{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"provenance":[],"gpuType":"T4"},"kernelspec":{"name":"python3","display_name":"Python 3"},"language_info":{"name":"python"},"accelerator":"GPU","widgets":{"application/vnd.jupyter.widget-state+json":{"73eb22912e4449298bd418b3b7e823da":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HBoxModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HBoxView","box_style":"","children":["IPY_MODEL_73ecc175f4b04408aea3786b5c9d0b2f","IPY_MODEL_c7988b785d3346b9a03133d441080b87","IPY_MODEL_dade3047b1944ef0a22e031eee7ef07e"],"layout":"IPY_MODEL_3384cac3e2e8406d9ea8bbfd65c87136"}},"73ecc175f4b04408aea3786b5c9d0b2f":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_7f770f22ae0e4c14926f8bf81049d4c2","placeholder":"​","style":"IPY_MODEL_617cb6ee91f14a11b8f024413280460d","value":"spiece.model: 100%"}},"c7988b785d3346b9a03133d441080b87":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"FloatProgressModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"ProgressView","bar_style":"success","description":"","description_tooltip":null,"layout":"IPY_MODEL_2d3fb1e8c53347c9966f6669cbe8f93f","max":791656,"min":0,"orientation":"horizontal","style":"IPY_MODEL_9e5ce0b8408b494bb583513bf0c1e4f8","value":791656}},"dade3047b1944ef0a22e031eee7ef07e":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_6cda654b41574bfcb9bf4a8ef8c0aa78","placeholder":"​","style":"IPY_MODEL_8470bd11052b455e8d10bf7b6eb5cb0d","value":" 792k/792k [00:00&lt;00:00, 3.05MB/s]"}},"3384cac3e2e8406d9ea8bbfd65c87136":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"7f770f22ae0e4c14926f8bf81049d4c2":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"617cb6ee91f14a11b8f024413280460d":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"2d3fb1e8c53347c9966f6669cbe8f93f":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"9e5ce0b8408b494bb583513bf0c1e4f8":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"ProgressStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","bar_color":null,"description_width":""}},"6cda654b41574bfcb9bf4a8ef8c0aa78":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"8470bd11052b455e8d10bf7b6eb5cb0d":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"7691709fd81c4a34927860dc8c91751c":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HBoxModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HBoxView","box_style":"","children":["IPY_MODEL_88bc50ab935b49a99292aa907990453d","IPY_MODEL_d8266e99c1fd4dc2a3e62bd0ab4524e0","IPY_MODEL_44b8fcdac57440fc8b72495697e6eae4"],"layout":"IPY_MODEL_97d5902c8d5b434dbd149d9dc98884e4"}},"88bc50ab935b49a99292aa907990453d":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_a4e20156bf5842b39315fe025791c534","placeholder":"​","style":"IPY_MODEL_80628e51d1e245e59a8057db248d35a4","value":"tokenizer.json: 100%"}},"d8266e99c1fd4dc2a3e62bd0ab4524e0":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"FloatProgressModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"ProgressView","bar_style":"success","description":"","description_tooltip":null,"layout":"IPY_MODEL_70ce768bdce441daab87f41ff24d6eb7","max":1389353,"min":0,"orientation":"horizontal","style":"IPY_MODEL_d893d3d5f92443f7bbead9749667f4b3","value":1389353}},"44b8fcdac57440fc8b72495697e6eae4":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_98119439c548494ca454c4750c79c65d","placeholder":"​","style":"IPY_MODEL_d128daf8ef9844bb885dc74223c81ef2","value":" 1.39M/1.39M [00:00&lt;00:00, 5.33MB/s]"}},"97d5902c8d5b434dbd149d9dc98884e4":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"a4e20156bf5842b39315fe025791c534":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"80628e51d1e245e59a8057db248d35a4":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"70ce768bdce441daab87f41ff24d6eb7":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"d893d3d5f92443f7bbead9749667f4b3":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"ProgressStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","bar_color":null,"description_width":""}},"98119439c548494ca454c4750c79c65d":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"d128daf8ef9844bb885dc74223c81ef2":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"4ce328f50ec846a3a33a577bb3af4774":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HBoxModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HBoxView","box_style":"","children":["IPY_MODEL_ca9186809d5b44718f3b34b60c4f7015","IPY_MODEL_d1872dac52fe42729838fc218cf48a07","IPY_MODEL_9a0732edb9ce423f87290430fa158f63"],"layout":"IPY_MODEL_de28447cbce44dc2a51e54fd5204815d"}},"ca9186809d5b44718f3b34b60c4f7015":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_48ec762c8ea34121853ab2ad6bc3e4e7","placeholder":"​","style":"IPY_MODEL_da835df94c1342e1a11c4309be6d7e10","value":"config.json: 100%"}},"d1872dac52fe42729838fc218cf48a07":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"FloatProgressModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"ProgressView","bar_style":"success","description":"","description_tooltip":null,"layout":"IPY_MODEL_08c901f3359044daaedce3384c412865","max":1208,"min":0,"orientation":"horizontal","style":"IPY_MODEL_f34d969eefa249dabf1db793e37822cf","value":1208}},"9a0732edb9ce423f87290430fa158f63":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_0794da8ad58c48679ada18842604fe7e","placeholder":"​","style":"IPY_MODEL_f3214b0068894e0eaad6de46da4ddc70","value":" 1.21k/1.21k [00:00&lt;00:00, 52.9kB/s]"}},"de28447cbce44dc2a51e54fd5204815d":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"48ec762c8ea34121853ab2ad6bc3e4e7":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"da835df94c1342e1a11c4309be6d7e10":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"08c901f3359044daaedce3384c412865":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"f34d969eefa249dabf1db793e37822cf":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"ProgressStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","bar_color":null,"description_width":""}},"0794da8ad58c48679ada18842604fe7e":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"f3214b0068894e0eaad6de46da4ddc70":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"12d2a3d58c8a4a618dbdc330c906def3":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HBoxModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HBoxView","box_style":"","children":["IPY_MODEL_3bd08447fff44b3aa14bd9cf5a9de3e1","IPY_MODEL_862b994d613d458193a2f1122620d9a3","IPY_MODEL_811ad5d09a7e4702a2d2c42e248649d3"],"layout":"IPY_MODEL_10455389e55b42259b5015fb6931c7e0"}},"3bd08447fff44b3aa14bd9cf5a9de3e1":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_c5bd1b7764004ef9b7e93c7118b97c1f","placeholder":"​","style":"IPY_MODEL_d0b4db582654429e92780b96b5c439ce","value":"model.safetensors: 100%"}},"862b994d613d458193a2f1122620d9a3":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"FloatProgressModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"ProgressView","bar_style":"success","description":"","description_tooltip":null,"layout":"IPY_MODEL_a5ba4c4f8d36425cb42f095570ee913f","max":891646390,"min":0,"orientation":"horizontal","style":"IPY_MODEL_837d7790b24d49dabea13905d5350c50","value":891646390}},"811ad5d09a7e4702a2d2c42e248649d3":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_4ef460b335b7444590c430a4b8bc82fd","placeholder":"​","style":"IPY_MODEL_8f8903b08bb543a99d0ac0c8c1916295","value":" 892M/892M [00:05&lt;00:00, 162MB/s]"}},"10455389e55b42259b5015fb6931c7e0":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"c5bd1b7764004ef9b7e93c7118b97c1f":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"d0b4db582654429e92780b96b5c439ce":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"a5ba4c4f8d36425cb42f095570ee913f":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"837d7790b24d49dabea13905d5350c50":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"ProgressStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","bar_color":null,"description_width":""}},"4ef460b335b7444590c430a4b8bc82fd":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"8f8903b08bb543a99d0ac0c8c1916295":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"ec1828f9d3bb4d40a1cb02c7a17bac33":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HBoxModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HBoxView","box_style":"","children":["IPY_MODEL_dcb5d24e65c74fe89aecb5b14139346d","IPY_MODEL_5d4680477e004459879f20cf530d32f4","IPY_MODEL_aa60ba0a44bd4b5eb96cd90a11898201"],"layout":"IPY_MODEL_f7bddc60aa0240ee82491739df915edb"}},"dcb5d24e65c74fe89aecb5b14139346d":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_675c5f258a8d408484ce97f0806a2ad3","placeholder":"​","style":"IPY_MODEL_a6dc881ed8fd47c68dacc2d8163b25c7","value":"generation_config.json: 100%"}},"5d4680477e004459879f20cf530d32f4":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"FloatProgressModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"ProgressView","bar_style":"success","description":"","description_tooltip":null,"layout":"IPY_MODEL_fd008b9ee15d49d3a75a3d34469161a0","max":147,"min":0,"orientation":"horizontal","style":"IPY_MODEL_f85341a4a9364047808d49e1e3fbdb67","value":147}},"aa60ba0a44bd4b5eb96cd90a11898201":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","model_module_version":"1.5.0","state":{"_dom_classes":[],"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"HTMLModel","_view_count":null,"_view_module":"@jupyter-widgets/controls","_view_module_version":"1.5.0","_view_name":"HTMLView","description":"","description_tooltip":null,"layout":"IPY_MODEL_a0429ab624b44019b67c383cfcd79cc7","placeholder":"​","style":"IPY_MODEL_5fc67793c7734fe282d2cd47b13358f7","value":" 147/147 [00:00&lt;00:00, 11.2kB/s]"}},"f7bddc60aa0240ee82491739df915edb":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"675c5f258a8d408484ce97f0806a2ad3":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"a6dc881ed8fd47c68dacc2d8163b25c7":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}},"fd008b9ee15d49d3a75a3d34469161a0":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"f85341a4a9364047808d49e1e3fbdb67":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"ProgressStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","bar_color":null,"description_width":""}},"a0429ab624b44019b67c383cfcd79cc7":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","model_module_version":"1.2.0","state":{"_model_module":"@jupyter-widgets/base","_model_module_version":"1.2.0","_model_name":"LayoutModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"LayoutView","align_content":null,"align_items":null,"align_self":null,"border":null,"bottom":null,"display":null,"flex":null,"flex_flow":null,"grid_area":null,"grid_auto_columns":null,"grid_auto_flow":null,"grid_auto_rows":null,"grid_column":null,"grid_gap":null,"grid_row":null,"grid_template_areas":null,"grid_template_columns":null,"grid_template_rows":null,"height":null,"justify_content":null,"justify_items":null,"left":null,"margin":null,"max_height":null,"max_width":null,"min_height":null,"min_width":null,"object_fit":null,"object_position":null,"order":null,"overflow":null,"overflow_x":null,"overflow_y":null,"padding":null,"right":null,"top":null,"visibility":null,"width":null}},"5fc67793c7734fe282d2cd47b13358f7":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","model_module_version":"1.5.0","state":{"_model_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_model_name":"DescriptionStyleModel","_view_count":null,"_view_module":"@jupyter-widgets/base","_view_module_version":"1.2.0","_view_name":"StyleView","description_width":""}}}}},"cells":[{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"vJUCiAQ9fI8T","executionInfo":{"status":"ok","timestamp":1711408783094,"user_tz":420,"elapsed":17845,"user":{"displayName":"Rishikesh Sivakumar","userId":"10957495338496577769"}},"outputId":"ecef027b-f15e-49c1-8669-336a717317e3"},"outputs":[{"output_type":"stream","name":"stdout","text":["Requirement already satisfied: transformers in /usr/local/lib/python3.10/dist-packages (4.38.2)\n","Requirement already satisfied: filelock in /usr/local/lib/python3.10/dist-packages (from transformers) (3.13.1)\n","Requirement already satisfied: huggingface-hub<1.0,>=0.19.3 in /usr/local/lib/python3.10/dist-packages (from transformers) (0.20.3)\n","Requirement already satisfied: numpy>=1.17 in /usr/local/lib/python3.10/dist-packages (from transformers) (1.25.2)\n","Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.10/dist-packages (from transformers) (24.0)\n","Requirement already satisfied: pyyaml>=5.1 in /usr/local/lib/python3.10/dist-packages (from transformers) (6.0.1)\n","Requirement already satisfied: regex!=2019.12.17 in /usr/local/lib/python3.10/dist-packages (from transformers) (2023.12.25)\n","Requirement already satisfied: requests in /usr/local/lib/python3.10/dist-packages (from transformers) (2.31.0)\n","Requirement already satisfied: tokenizers<0.19,>=0.14 in /usr/local/lib/python3.10/dist-packages (from transformers) (0.15.2)\n","Requirement already satisfied: safetensors>=0.4.1 in /usr/local/lib/python3.10/dist-packages (from transformers) (0.4.2)\n","Requirement already satisfied: tqdm>=4.27 in /usr/local/lib/python3.10/dist-packages (from transformers) (4.66.2)\n","Requirement already satisfied: fsspec>=2023.5.0 in /usr/local/lib/python3.10/dist-packages (from huggingface-hub<1.0,>=0.19.3->transformers) (2023.6.0)\n","Requirement already satisfied: typing-extensions>=3.7.4.3 in /usr/local/lib/python3.10/dist-packages (from huggingface-hub<1.0,>=0.19.3->transformers) (4.10.0)\n","Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests->transformers) (3.3.2)\n","Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests->transformers) (3.6)\n","Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests->transformers) (2.0.7)\n","Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests->transformers) (2024.2.2)\n"]}],"source":["!pip install transformers\n"]},{"cell_type":"code","source":["import pandas as pd\n","import torch\n","from transformers import T5Tokenizer, T5ForConditionalGeneration,Adafactor"],"metadata":{"id":"x5xCOksYrUUF"},"execution_count":null,"outputs":[]},{"cell_type":"code","source":["from google.colab import drive\n","drive.mount('/content/drive')\n"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"i9v01m_drYfe","executionInfo":{"status":"ok","timestamp":1711408814587,"user_tz":420,"elapsed":19471,"user":{"displayName":"Rishikesh Sivakumar","userId":"10957495338496577769"}},"outputId":"c1ba167b-f352-4d81-ca37-615765fc63df"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["Mounted at /content/drive\n"]}]},{"cell_type":"code","source":["csv_path = '/content/drive/MyDrive/data/text_patentability_data.csv'\n","\n","train_df=pd.read_csv(csv_path)\n","train_df=train_df.iloc[ :5,:]\n","#train_df=train_df.sample(frac = 1)\n","batch_size=1\n","num_of_batches=len(train_df)/batch_size"],"metadata":{"id":"pJ3C9O0es8Dy"},"execution_count":null,"outputs":[]},{"cell_type":"code","source":["train_df.head()"],"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":289},"id":"8mIUESIu1Lc4","executionInfo":{"status":"ok","timestamp":1711408820476,"user_tz":420,"elapsed":462,"user":{"displayName":"Rishikesh Sivakumar","userId":"10957495338496577769"}},"outputId":"afb6b5f7-b3e7-4222-a0db-88026c883b2f"},"execution_count":null,"outputs":[{"output_type":"execute_result","data":{"text/plain":["                                             summary  \\\n","0  BACKGROUND OF THE INVENTION 1. Field of Invent...   \n","1  FIELD OF THE INVENTION The invention relates t...   \n","2  FIELD OF THE INVENTION The invention relates t...   \n","3  BACKGROUND OF THE INVENTION 1. Field of the In...   \n","4  TECHNICAL FIELD OF THE INVENTION This inventio...   \n","\n","                                              detail  \\\n","0  DESCRIPTION OF EXAMPLE EMBODIMENTS An architec...   \n","1  DETAILED DESCRIPTION OF THE INVENTION The pres...   \n","2  DETAILED DESCRIPTION OF THE INVENTION The pres...   \n","3  DETAILED DESCRIPTION In the following descript...   \n","4  DETAILED DESCRIPTION OF THE INVENTION Referrin...   \n","\n","                                              claims status  \n","0  1. A distributed data processing system for co...    ISS  \n","1  1-41. (Cancelled) 42. An isolated recombinant ...    ISS  \n","2  1-41. (Cancelled) 42. An isolated recombinant ...    ISS  \n","3  1. A method comprising: providing a first comm...    ISS  \n","4  1. A method for transferring information betwe...    ISS  "],"text/html":["\n","  <div id=\"df-4985d267-a7f4-4826-8247-d2106d1683c7\" class=\"colab-df-container\">\n","    <div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>summary</th>\n","      <th>detail</th>\n","      <th>claims</th>\n","      <th>status</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>0</th>\n","      <td>BACKGROUND OF THE INVENTION 1. Field of Invent...</td>\n","      <td>DESCRIPTION OF EXAMPLE EMBODIMENTS An architec...</td>\n","      <td>1. A distributed data processing system for co...</td>\n","      <td>ISS</td>\n","    </tr>\n","    <tr>\n","      <th>1</th>\n","      <td>FIELD OF THE INVENTION The invention relates t...</td>\n","      <td>DETAILED DESCRIPTION OF THE INVENTION The pres...</td>\n","      <td>1-41. (Cancelled) 42. An isolated recombinant ...</td>\n","      <td>ISS</td>\n","    </tr>\n","    <tr>\n","      <th>2</th>\n","      <td>FIELD OF THE INVENTION The invention relates t...</td>\n","      <td>DETAILED DESCRIPTION OF THE INVENTION The pres...</td>\n","      <td>1-41. (Cancelled) 42. An isolated recombinant ...</td>\n","      <td>ISS</td>\n","    </tr>\n","    <tr>\n","      <th>3</th>\n","      <td>BACKGROUND OF THE INVENTION 1. Field of the In...</td>\n","      <td>DETAILED DESCRIPTION In the following descript...</td>\n","      <td>1. A method comprising: providing a first comm...</td>\n","      <td>ISS</td>\n","    </tr>\n","    <tr>\n","      <th>4</th>\n","      <td>TECHNICAL FIELD OF THE INVENTION This inventio...</td>\n","      <td>DETAILED DESCRIPTION OF THE INVENTION Referrin...</td>\n","      <td>1. A method for transferring information betwe...</td>\n","      <td>ISS</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>\n","    <div class=\"colab-df-buttons\">\n","\n","  <div class=\"colab-df-container\">\n","    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-4985d267-a7f4-4826-8247-d2106d1683c7')\"\n","            title=\"Convert this dataframe to an interactive table.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n","    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n","  </svg>\n","    </button>\n","\n","  <style>\n","    .colab-df-container {\n","      display:flex;\n","      gap: 12px;\n","    }\n","\n","    .colab-df-convert {\n","      background-color: #E8F0FE;\n","      border: none;\n","      border-radius: 50%;\n","      cursor: pointer;\n","      display: none;\n","      fill: #1967D2;\n","      height: 32px;\n","      padding: 0 0 0 0;\n","      width: 32px;\n","    }\n","\n","    .colab-df-convert:hover {\n","      background-color: #E2EBFA;\n","      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","      fill: #174EA6;\n","    }\n","\n","    .colab-df-buttons div {\n","      margin-bottom: 4px;\n","    }\n","\n","    [theme=dark] .colab-df-convert {\n","      background-color: #3B4455;\n","      fill: #D2E3FC;\n","    }\n","\n","    [theme=dark] .colab-df-convert:hover {\n","      background-color: #434B5C;\n","      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","      fill: #FFFFFF;\n","    }\n","  </style>\n","\n","    <script>\n","      const buttonEl =\n","        document.querySelector('#df-4985d267-a7f4-4826-8247-d2106d1683c7 button.colab-df-convert');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      async function convertToInteractive(key) {\n","        const element = document.querySelector('#df-4985d267-a7f4-4826-8247-d2106d1683c7');\n","        const dataTable =\n","          await google.colab.kernel.invokeFunction('convertToInteractive',\n","                                                    [key], {});\n","        if (!dataTable) return;\n","\n","        const docLinkHtml = 'Like what you see? Visit the ' +\n","          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n","          + ' to learn more about interactive tables.';\n","        element.innerHTML = '';\n","        dataTable['output_type'] = 'display_data';\n","        await google.colab.output.renderOutput(dataTable, element);\n","        const docLink = document.createElement('div');\n","        docLink.innerHTML = docLinkHtml;\n","        element.appendChild(docLink);\n","      }\n","    </script>\n","  </div>\n","\n","\n","<div id=\"df-4ee02cca-66ed-4687-a04c-71e9bbebecea\">\n","  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-4ee02cca-66ed-4687-a04c-71e9bbebecea')\"\n","            title=\"Suggest charts\"\n","            style=\"display:none;\">\n","\n","<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","     width=\"24px\">\n","    <g>\n","        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n","    </g>\n","</svg>\n","  </button>\n","\n","<style>\n","  .colab-df-quickchart {\n","      --bg-color: #E8F0FE;\n","      --fill-color: #1967D2;\n","      --hover-bg-color: #E2EBFA;\n","      --hover-fill-color: #174EA6;\n","      --disabled-fill-color: #AAA;\n","      --disabled-bg-color: #DDD;\n","  }\n","\n","  [theme=dark] .colab-df-quickchart {\n","      --bg-color: #3B4455;\n","      --fill-color: #D2E3FC;\n","      --hover-bg-color: #434B5C;\n","      --hover-fill-color: #FFFFFF;\n","      --disabled-bg-color: #3B4455;\n","      --disabled-fill-color: #666;\n","  }\n","\n","  .colab-df-quickchart {\n","    background-color: var(--bg-color);\n","    border: none;\n","    border-radius: 50%;\n","    cursor: pointer;\n","    display: none;\n","    fill: var(--fill-color);\n","    height: 32px;\n","    padding: 0;\n","    width: 32px;\n","  }\n","\n","  .colab-df-quickchart:hover {\n","    background-color: var(--hover-bg-color);\n","    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n","    fill: var(--button-hover-fill-color);\n","  }\n","\n","  .colab-df-quickchart-complete:disabled,\n","  .colab-df-quickchart-complete:disabled:hover {\n","    background-color: var(--disabled-bg-color);\n","    fill: var(--disabled-fill-color);\n","    box-shadow: none;\n","  }\n","\n","  .colab-df-spinner {\n","    border: 2px solid var(--fill-color);\n","    border-color: transparent;\n","    border-bottom-color: var(--fill-color);\n","    animation:\n","      spin 1s steps(1) infinite;\n","  }\n","\n","  @keyframes spin {\n","    0% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","      border-left-color: var(--fill-color);\n","    }\n","    20% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    30% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","      border-right-color: var(--fill-color);\n","    }\n","    40% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    60% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","    }\n","    80% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-bottom-color: var(--fill-color);\n","    }\n","    90% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","    }\n","  }\n","</style>\n","\n","  <script>\n","    async function quickchart(key) {\n","      const quickchartButtonEl =\n","        document.querySelector('#' + key + ' button');\n","      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n","      quickchartButtonEl.classList.add('colab-df-spinner');\n","      try {\n","        const charts = await google.colab.kernel.invokeFunction(\n","            'suggestCharts', [key], {});\n","      } catch (error) {\n","        console.error('Error during call to suggestCharts:', error);\n","      }\n","      quickchartButtonEl.classList.remove('colab-df-spinner');\n","      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n","    }\n","    (() => {\n","      let quickchartButtonEl =\n","        document.querySelector('#df-4ee02cca-66ed-4687-a04c-71e9bbebecea button');\n","      quickchartButtonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","    })();\n","  </script>\n","</div>\n","\n","    </div>\n","  </div>\n"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"dataframe","variable_name":"train_df","summary":"{\n  \"name\": \"train_df\",\n  \"rows\": 5,\n  \"fields\": [\n    {\n      \"column\": \"summary\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 5,\n        \"samples\": [\n          \"FIELD OF THE INVENTION The invention relates to the adenine nucleotide translocator (ANT) protein that is found in mitochondria of eukaryotic cells. More particularly, the invention relates to the production of ANT polypeptides and ANT fusion proteins using recombinant DNA technology; to novel labeled ligands for ANT proteins; and to assays useful for identifying and isolating ANT proteins and for screening compounds that interact with ANT, including high throughput screening. BACKGROUND OF THE INVENTION Mitochondria are the main energy source in cells of higher organisms, and these organelles provide direct and indirect biochemical regulation of a wide array of cellular respiratory, oxidative and metabolic processes. These include electron transport chain (ETC) activity, which drives oxidative phosphorylation to produce metabolic energy in the form of adenosine triphosphate (ATP), and which also underlies a central mitochondrial role in intracellular calcium homeostasis. Mitochondrial ultrastructural characterization reveals the presence of an outer mitochondrial membrane that serves as an interface between the organelle and the cytosol, a highly folded inner mitochondrial membrane that appears to form attachments to the outer membrane at multiple sites, and an intermembrane space between the two mitochondrial membranes. The subcompartment within the inner mitochondrial membrane is commonly referred to as the mitochondrial matrix. (For a review, see, e.g., Ernster et al., 1981 J. Cell Biol. 91:227s.) The cristae, originally postulated to occur as infoldings of the inner mitochondrial membrane, have recently been characterized using three-dimensional electron tomography as also including tube-like conduits that may form networks, and that can be connected to the inner membrane by open, circular (30 nm diameter) junctions (Perkins et al., 1997, Journal of Structural Biology 119:260). While the outer membrane is freely permeable to ionic and non-ionic solutes having molecular weights less than about ten kilodaltons, the inner mitochondrial membrane exhibits selective and regulated permeability for many small molecules, including certain cations, and is impermeable to large (&#x3e;&#x2dc;10 kDa) molecules. Altered or defective mitochondrial activity, including but not limited to failure at any step of the ETC, may result in catastrophic mitochondrial collapse that has been termed &#x201c;permeability transition&#x201d; (PT) or &#x201c;mitochondrial permeability transition&#x201d; (MPT). According to generally accepted theories of mitochondrial function, proper ETC respiratory activity requires maintenance of an electrochemical potential (&#x394;&#x3a8;m) in the inner mitochondrial membrane by a coupled chemiosmotic mechanism. Altered or defective mitochondrial activity may dissipate this membrane potential, thereby preventing ATP biosynthesis and halting the production of a vital biochemical energy source. In addition, mitochondrial proteins such as cytochrome c may leak out of the mitochondria after permeability transition and may induce the genetically programmed cell suicide sequence known as apoptosis or programmed cell death (PCD). MPT may result from direct or indirect effects of mitochondrial genes, gene products or related downstream mediator molecules and/or extramitochondrial genes, gene products or related downstream mediators, or from other known or unknown causes. Loss of mitochondrial potential therefore may be a critical event in the progression of diseases associated with altered mitochondrial function, including degenerative diseases. Mitochondrial defects, which may include defects related to the discrete mitochondrial genome that resides in mitochondrial DNA and/or to the extramitochondrial genome, which includes nuclear chromosomal DNA and other extramitochondrial DNA, may contribute significantly to the pathogenesis of diseases associated with altered mitochondrial function. For example, alterations in the structural and/or functional properties of mitochondrial components comprised of subunits encoded directly or indirectly by mitochondrial and/or extramitochondrial DNA, including alterations deriving from genetic and/or environmental factors or alterations derived from cellular compensatory mechanisms, may play a role in the pathogenesis of any disease associated with altered mitochondrial function. A number of degenerative diseases are thought to be caused by, or to be associated with, alterations in mitochondrial function. These include Alzheimer's Disease (AD); diabetes mellitus; Parkinson's Disease; Huntington's disease; dystonia; Leber's hereditary optic neuropathy; schizophrenia; mitochondrial encephalopathy, lactic acidosis, and stroke (MELAS); cancer; psoriasis; hyperproliferative disorders; mitochondrial diabetes and deafness (MIDD) and myoclonic epilepsy ragged red fiber syndrome. The extensive list of additional diseases associated with altered mitochondrial function continues to expand as aberrant mitochondrial or mitonuclear activities are implicated in particular disease processes. A hallmark pathology of AD and potentially other diseases associated with altered mitochondrial function is the death of selected cellular populations in particular affected tissues, which results from apoptosis (also referred to as &#x201c;programmed cell death&#x201d; or PCD) according to a growing body of evidence. Mitochondrial dysfunction is thought to be critical in the cascade of events leading to apoptosis in various cell types (Kroemer et al., FASEB J. 9:1277-87, 1995), and may be a cause of apoptotic cell death in neurons of the AD brain. Altered mitochondrial physiology may be among the earliest events in PCD (Zamzami et al., J. Exp. Med. 182:367-77, 1995; Zamzami et al., J. Exp. Med. 181:1661-72, 1995) and elevated reactive oxygen species (ROS) levels that result from such altered mitochondrial function may initiate the apoptotic cascade (Ausserer et al., Mol. Cell. Biol. 14:5032-42, 1994). Thus, in addition to their role in energy production in growing cells, mitochondria (or, at least, mitochondrial components) participate in apoptosis (Newmeyer et al., 1994, Cell 79:353-364; Liu et al., 1996, Cell 86:147-157). Apoptosis is apparently also required for, inter alia, normal development of the nervous system and proper functioning of the immune system. Moreover, some disease states are thought to be associated with either insufficient (e.g., cancer, autoimmune diseases) or excessive (e.g., stroke damage, AD-associated neurodegeneration) levels of apoptosis. For general reviews of apoptosis, and the role of mitochondria therein, see Green and Reed (1998, Science 281:1309-1312), Green (1998, Cell 94:695-698) and Kromer (1997, Nature Medicine 3:614-620). Hence, agents that effect apoptotic events, including those associated with mitochondrial components, might have a variety of palliative, prophylactic and therapeutic uses. The adenine nucleotide translocator (ANT), a nuclear encoded polypeptide that is a major component of the inner mitochondrial membrane, is responsible for mediating transport of ADP and ATP across the mitochondrial inner membrane. For example, ANT is believed to mediate stoichiometric ATP/proton exchange across the inner mitochondrial membrane, and ANT inhibitors (such as atractyloside or bongkrekic acid) induce MPT under certain conditions. Three human ANT isoforms have been described that differ in their tissue expression patterns and other mammalian ANT homologues have been described. (See, e.g., Wallace et al., 1998 in Mitochondria &#x26; Free Radicals in Neurodegenerative Diseases, Beal, Howell and Bodis-Woliner, Eds., Wiley-Liss, New York, pp. 283-307, and references cited therein.) ANT has also been implicated as an important molecular component of the mitochondrial permeability transition pore, a Ca2+-regulated inner membrane channel that, as described above, plays an important modulating role in apoptotic processes. As inner mitochondrial membrane proteins are believed to possess multiple hydrophobic membrane spanning domains, ANT polypeptides may exhibit, inter alia, poor intracellular solubility, toxic accumulations and/or the formation of inclusion bodies and other deleterious effects on respiratory homeostasis within a host cell due to ANT biological activity. Consequently, those having ordinary skill in the art have heretofore been unable to produce ANT, reliably or in sufficient quantities for a variety of uses, such as those provided herein. Because of the significance of mitochondria to respiratory, metabolic and apoptotic processes, and in view of the prominent role played by ANT in these and other mitochondrial acitivities, there is clearly a need for compositions and methods that permit the production of ANT proteins, including ANT fusion proteins; for novel ANT ligands; for methods to identify and isolate ANT proteins; and for methods of identifying and isolating agents that interact with ANT. The present invention fulfills these needs and provides other related advantages. These and other aspects of the present invention will become evident upon reference to the following detailed description and attached drawings. In addition, various references are set forth below which describe in more detail certain procedures or compositions (e.g., plasmids, vectors, etc.), and are therefore incorporated by reference in their entireties. SUMMARY OF THE INVENTION In its various aspects and embodiments the invention is directed to: A recombinant expression construct comprising at least one regulated promoter operably linked to a first nucleic acid encoding an adenine nucleotide translocator polypeptide; further comprising at least one additional nucleic acid sequence that regulates transcription; wherein the additional nucleic acid sequence that regulates transcription encodes a repressor of said regulated promoter; wherein the adenine nucleotide translocator polypeptide comprises a human adenine nucleotide translocator polypeptide; wherein the human adenine nucleotide translocator polypeptide is ANT1; wherein the human adenine nucleotide translocator polypeptide is ANT2; wherein the human adenine nucleotide translocator polypeptide is ANT3; wherein the adenine nucleotide translocator polypeptide is expressed as a fusion protein with a polypeptide product of a second nucleic acid sequence; wherein the polypeptide product of said second nucleic acid sequence is an enzyme; wherein said fusion protein localizes to membranes; wherein said membranes are mitochondrial membranes; wherein the adenine nucleotide translocator polypeptide is expressed as a fusion protein with at least one product of a second nucleic acid sequence encoding a polypeptide cleavable by a protease, said adenine nucleotide translocator polypeptide being separable from the fusion protein by cleavage with the protease; A host cell comprising a recombinant expression construct as provided; wherein the host cell is a prokaryotic cell; wherein the host cell Is a eukaryotic cell; wherein the eukaryotic cell is selected from the group consisting of a yeast cell, an insect cell and a mammalian cell; wherein the insect cell is an Sf9 cell or a Trichoplusia ni cell; at lacks at least one isoform of an endogenous adenine nucleotide translocator; in which expression of at least one gene encoding an endogenous adenine nucleotide translocator isoform is substantially impaired. A recombinant expression construct comprising at least one promoter operably linked to a nucleic acid molecule comprising a first nucleic acid sequence and a second nucleic acid sequence, said first nucleic acid sequence encoding an animal adenine nucleotide translocator polypeptide wherein the adenine nucleotide translocator polypeptide is expressed as a fusion protein with a polypeptide product of said second nucleic acid sequence; wherein the polypeptide product of said second nucleic acid sequence is an enzyme; wherein said fusion protein localizes to membranes; wherein said membranes are mitochondrial membranes; further comprising at least one additional nucleic acid sequence that regulates transcription; wherein the additional nucleic acid sequence that regulates transcription encodes a repressor of said promoter; wherein the adenine nucleotide translocator polypeptide comprises a human adenine nucleotide translocator polypeptide; wherein the human adenine nucleotide translocator polypeptide is ANT1; wherein the human adenine nucleotide translocator polypeptide is ANT2; wherein the human adenine nucleotide translocator polypeptide is ANT3; wherein the adenine nucleotide translocator polypeptide is expressed as a fusion protein with at least one product of a second nucleic acid sequence encoding a polypeptide cleavable by a protease, said adenine nucleotide translocator polypeptide being separable from the fusion protein by cleavage with the protease; a host cell comprising a recombinant expression construct as just described; wherein the host cell is a prokaryotic cell; wherein the host cell is a eukaryotic cell; wherein the eukaryotic cell is selected from the group consisting of a yeast cell, an insect cell and a mammalian cell; wherein the insect cell is an Sf9 cell or a Trichoplusia ni cell; that lacks at least one isoform of an endogenous adenine nucleotide translocator; in which expression of at least one gene encoding an endogenous adenine nucleotide translocator isoform is substantially impaired; wherein the expression construct is a recombinant viral expression construct; A method of producing a recombinant adenine nucleotide translocator polypeptide, comprising; culturing a host cell comprising a recombinant expression construct comprising at least one regulated promoter operably linked to a first nucleic acid encoding an adenine nucleotide translocator polypeptide; A method of producing a recombinant adenine nucleotide translocator polypeptide, comprising culturing a host cell comprising a recombinant expression construct comprising at least one promoter operably linked to a nucleic acid molecule comprising a first nucleic acid sequence and a second nucleic acid sequence, said first nucleic acid sequence encoding an animal adenine nucleotide translocator polypeptide wherein the adenine nucleotide translocator polypeptide is expressed as a fusion protein with a polypeptide product of said second nucleic acid sequence; A method of producing a recombinant adenine nucleotide translocator polypeptide, comprising culturing a host cell infected with the recombinant viral expression construct as provided above. An ANT polypeptide produced by the methods just described. An isolated human adenine nucleotide translocator polypeptide; wherein the human adenine nucleotide translocator polypeptide is recombinant ANT1 or a variant or fragment thereof, wherein the human adenine nucleotide translocator polypeptide is recombinant ANT2 or a variant or fragment thereof; wherein the human adenine nucleotide translocator polypeptide is recombinant ANT3 or a variant or fragment thereof; An isolated human adenine nucleotide translocator fusion protein comprising an adenine translocator polypeptide fused to at least one additional polypeptide sequence; wherein said one additional polypeptide sequence is an enzyme sequence or a variant or fragment thereof, wherein said fusion protein localizes to membranes; wherein said membranes are mitochondrial membranes; An isolated human adenine nucleotide translocator fusion protein comprising an adenine translocator polypeptide fused to at least one additional polypeptide sequence cleavable by a protease, said adenine nucleotide translocator polypeptide being separable from the fusion protein by cleavage with the protease. An isolated adenine nucleotide translocator fusion protein comprising a first polypeptide that is an animal adenine translocator polypeptide fused to at least one additional polypeptide sequence; wherein said one additional polypeptide sequence is an enzyme sequence or a variant or fragment thereof, that localizes to membranes; wherein said membranes are mitochondrial membranes. An isolated recombinant animal adenine nucleotide translocator fusion protein comprising an adenine translocator polypeptide fused to at least one additional polypeptide sequence cleavable by a protease, said adenine nucleotide translocator polypeptide being separable from the fusion protein by cleavage with the protease; wherein the additional polypeptide sequence is a polypeptide having affinity for a ligand. A method for determining the presence of an ANT polypeptide in a biological sample comprising contacting a biological sample suspected of containing an ANT polypeptide with an ANT ligand under conditions and for a time sufficient to allow binding of the ANT ligand to an ANT polypeptide; and detecting the binding of the ANT ligand to an ANT polypeptide, and therefrom determining the presence of an ANT polypeptide in said biological sample; wherein the adenine nucleotide translocator polypeptide comprises a human adenine nucleotide translocator polypeptide; wherein the human adenine nucleotide translocator polypeptide is ANT1; wherein the human adenine nucleotide translocator polypeptide is ANT2; wherein the human adenine nucleotide translocator polypeptide is ANT3; wherein the ANT ligand comprises atractyloside substituted at 6&#x2032; hydroxyl to form an atractyloside derivative; wherein the atractyloside is detectably substituted at the 6&#x2032; hydroxyl to form a detectable atractyloside derivative; wherein the detectable atractyloside derivative comprises a radioloabeled substituent; wherein the radiolabeled substituent is selected from the group consisting of 125I, 131I, 3H, 14C and 35S; wherein the detectable atractyloside derivative comprises a fluorescent substituent; wherein the ANT ligand further comprises a Eu3+ atom complexed to the atractyloside derivative; wherein the detectable atractyloside derivative comprises covalently bound biotin; wherein the atractyloside molecule is substituted at 6&#x2032; hydroxyl with an amine or an amine containing functionality to form an amine modified atractyloside derivative; wherein the atractyloside molecule is a carboxyatractyloside molecule that is substituted at 6&#x2032; hydroxyl to form an atractyloside derivative that is a carboxyatractyloside derivative. A method for isolating ANT from a biological sample, comprising contacting a biological sample suspected of containing an ANT polypeptide with an ANT ligand under conditions and for a time sufficient to allow binding of the ANT ligand to an ANT polypeptide; and recovering the ANT polypeptide, and thereby isolating ANT from a biological sample; wherein the ANT ligand is covalently bound to a solid phase; wherein the ANT ligand is non-covalently bound to a solid phase. A method for identifying an agent that binds to an ANT polypeptide, comprising contacting a candidate agent with a host cell expressing at least one recombinant ANT polypeptide under conditions and for a time sufficient to permit binding of the agent to said recombinant ANT polypeptide; and detecting binding of said agent to the recombinant ANT; wherein the host cell is a prokaryotic cell; wherein the prokaryotic cell is an E. coli cell; wherein the host cell is a eukaryotic cell; wherein the eukaryotic cell is selected from the group consisting of a yeast cell, an insect cell and a mammalian cell; wherein the insect cell is an Sf9 cell or a Trichoplusia ni cell; wherein the host cell lacks at least one isoform of an endogenous adenine nucleotide translocator; wherein host cell expression of at least one gene encoding an endogenous adenine nucleotide translocator isoform is substantially impaired. A method for identifying an agent that binds to an ANT polypeptide, comprising contacting a candidate agent with a biological sample containing at least one recombinant ANT polypeptide under conditions and for a time sufficient to permit binding of the agent to said ANT polypeptide; and detecting binding of said agent to the recombinant ANT polypeptide. A method for identifying an agent that interacts with an ANT polypeptide comprising contacting a biological sample containing recombinant ANT with a detectable ANT ligand in the presence of a candidate agent; and comparing binding of the detectable ANT ligand to recombinant ANT in the absence of said agent to binding of the detectable ANT ligand to recombinant ANT in the presence of said agent, and therefrom identifying an agent that interacts with an ANT polypeptide. An ANT ligand comprising atractyloside substituted at the 6&#x2032; hydroxyl to form an atractyloside derivative; wherein the atractyloside is detectably substituted at the 6&#x2032; hydroxyl to form a detectable atractyloside derivative; wherein the detectable atractyloside derivative comprises a radioloabeled substituent; wherein the radiolabeled substituent is selected from the group consisting of 125I, 131I, 3H, 14C and 35S; wherein the detectable atractyloside derivative comprises a fluorescent substituent; further comprising a Eu3+ atom complexed to the atractyloside derivative; wherein the detectable atractyloside derivative comprises covalently bound biotin; wherein the atractyloside molecule is substituted at 6&#x2032; hydroxyl with an amine or an amine containing functionality to form an amine modified atractyloside derivative; wherein the atractyloside molecule is a carboxyatractyloside molecule that is substituted at 6&#x2032; hydroxyl to form an atractyloside derivative that is a carboxyatractyloside derivative. An ANT ligand having the following structure(I): including stereoisomers and pharmaceutically acceptable salts thereof, wherein R1, R2 and R3 are as identified below. An assay plate for high throughput screening of candidate agents that bind to at least one ANT polypeptide, comprising an assay plate having a plurality of wells, each of said wells further comprising at least one immobilized recombinant ANT polypeptide or a variant or fragment thereof. A method of targeting a polypeptide of interest to a mitochondrial membrane, comprising expressing a recombinant expression construct encoding a fusion protein in a host cell, said construct comprising at least one regulated promoter operably linked to a nucleic acid molecule comprising a first nucleic acid sequence and a second nucleic acid sequence, said first nucleic acid sequence encoding an adenine nucleotide translocator polypeptide that is expressed as a fusion protein with a polypeptide product of said second nucleic acid sequence, wherein said second nucleic acid sequence encodes the polypeptide of interest. A method of targeting a polypeptide of interest to a mitochondrial membrane, comprising expressing a recombinant expression construct encoding a fusion protein in a host cell, said construct comprising at least one promoter operably linked to a nucleic acid molecule comprising a first nucleic acid sequence and a second nucleic acid sequence, said first nucleic acid sequence encoding an animal adenine nucleotide translocator polypeptide that is expressed as a fusion protein with a polypeptide product of said second nucleic acid sequence, wherein said second nucleic acid sequence encodes the polypeptide of interest; a pharmaceutical composition comprising an ANT ligand as just described. A pharmaceutical composition comprising an agent that binds to an ANT polypeptide identified as just described. A pharmaceutical composition comprising an agent that binds to an ANT polypeptide identified as described above. A pharmaceutical composition comprising an agent that interacts with an ANT polypeptide identified above. A method of treatment comprising administering to a subject any one of the just described the pharmaceutical compositions.\",\n          \"TECHNICAL FIELD OF THE INVENTION This invention is related to data processing systems and their architecture. In one aspect, it relates to a network component for retransmitting data packets in accordance with ID codes embedded therein in a distributed manner. BACKGROUND OF THE INVENTION The classification and management of data is one of the most difficult tasks faced by corporations, government entities, and other large users of information. Companies must classify their data in such a way to make it easy and simple for buyers to find and purchase their products. Data exchanges face a bigger challenge in that they must work with multiple companies and develop a comprehensive classification system for their buyers. One common way to create a search/classification system for specific products is to access and use government and/or industry specific classification systems (i.e., classification databases). However, no existing classification database is comprehensive enough to address all the issues associated with building a classification system. These issues include: uniform numbers for products that cross multiple industries, restricting products from inclusion in classification, and non-usage of slang or industry standard language to access or classify products. The classification databases frequently do not address all the products, thus resulting in inconsistencies even when companies use the same classification system. Additionally, many of the various classification systems conflict with each other. For example, a product might have several classification numbers if it crosses multiple industries. Still other companies might use third party classification systems approved by a governmental entity. This program requires companies to pay multiple fees and go through a lengthy administrative process. Even then it may not cover all products in an industry. Companies must make a conscious decision to initiate, implement and maintain these programs. These efforts can be costly, and for this reason, compliance is generally not high. A need therefore, exists, for a data processing system which automatically generates identification codes for specific products. Preferably, companies could use the automatically-generated identification codes in place of their existing identification codes. More preferably, the use of the automatically-generated identification codes can be phased-in gradually as the of user base expands. Under current practices, companies create search engines by developing hierarchies and families of products. They may create a thesaurus to encompass slang words. Companies often use drop down menus, key words and product description capabilities to enhance their systems. It is desired to classify the data in such a way as to minimize the responses generated by a search, and therefore more effectively guide the buyer through the system. However, under current practices, most exchanges offer barely adequate search capabilities for their buyers. Buyers must click through numerous drop down menus and then sort through multiple entries to accomplish their objectives. In many instances the buyer will fail to find the product that they seek. These existing processes could therefore be characterized as cumbersome, time consuming, frustrating and ineffective. A need therefore exists, for a product classification system which can facilitate simple, rapid and effective searching by prospective buyers. Another challenging data management task is the transmission of data between dissimilar systems. Even within the same corporate organization it is very common to find different system types, applications and/or information structures being used. Transmitting data between such systems can be a time-consuming and expensive task. Under current practices, data transfer between dissimilar systems is often facilitated by the use of customized software applications known as &#x201c;adapters&#x201d;. Some adapters &#x201c;pull&#x201d; data, i.e., extract it from the source system in the data format of the host system or host application, convert the data into another data format (e.g., EDI) and then sometimes convert it again into yet another data format (e.g., XML) for transmission to the destination system. Other adapters &#x201c;push&#x201d; data, i.e., convert the data from the transmission data format (e.g., XML) to an intermediate data format (e.g., EDI) if necessary, then convert it to the data format of the host system or application at the destination system, and finally loading the data into the destination system. All of these adapter steps are performed on the host systems using the host systems' CPU. Thus, in adapter-based systems, CPU load considerations may affect when and how often data pulls can be scheduled. For example, data pulls may be scheduled for late nights so as slow down the CPU during daytime ONTP (on line transaction processing). A need therefore exists for a system architecture which can allow the transmission of data between dissimilar systems while minimizing the associated load imposed on the host system CPU. Network routers are known which direct data packages on a network in accordance with ID codes embedded in the data packets. However, these routers typically direct data packets between similar nodes on a single network. It is now becoming increasingly common to transmit data across multiple networks, and even across different types of networks. A need therefore exists for a router which can direct data over networks of different types in accordance with associated ID codes. A need further exists for a router which can automatically transform a data packet having a first data format into a second data format. It is well known that when large amounts of data are being transmitted between systems, a system error (i.e., stoppage) and/or data loss (i.e., dropout) may occur. With conventional adapter-based system architectures, debugging a system stoppage can be very challenging because of the large number of conversion processes involved, and because most systems do not have an integrated way to indicate the point at which processing stopped, relying instead upon error logs. A need therefore exists for a system architecture in which processing status information is an integral part of the data packets transmitted over the networks. Further, with adapter-based systems, even after the processes have been debugged, it is often necessary to wait (e.g., until the time of day when host system CPU demand is low) to replace lost data in order to avoid adverse impact on the company's business. For example, if the host system is used for OLTP (on line transaction processing) during the day, pulling bulk data from the host system in order to replace data lost in a previous data transfer may be delayed until the late night hours. Of course, the delay in processing the data can have an adverse impact of its own. A need therefore exists for a system architecture which allows for the replacement of lost data while minimizing the impact on the source host system. SUMMARY OF THE INVENTION The present invention disclosed and claimed herein, in one aspect thereof, comprises a method for transferring information between first and second systems with dissimilar first and second database structures, respectively. The method includes the step of first extracting data from the first system and then routing the extracted data from the first system to a first conversion server. At the first conversion server, the data is converted from a format compatible with the first database structure to an intermediate format. The data in the intermediate format is then routed to a second conversion server. At the second conversion server, the data is converted from the intermediate format to a format compatible with the second database structure. The data converted at the second conversion server is then stored in the second system.\",\n          \"FIELD OF THE INVENTION The invention relates to the adenine nucleotide translocator (ANT) protein that is found in mitochondria of eukaryotic cells. More particularly, the invention relates to the production of ANT polypeptides and ANT fusion proteins using recombinant DNA technology; to novel labeled ligands for ANT proteins; and to assays useful for identifying and isolating ANT proteins and for screening compounds that interact with ANT, including high throughput screening. BACKGROUND OF THE INVENTION Mitochondria are the main energy source in cells of higher organisms, and these organelles provide direct and indirect biochemical regulation of a wide array of cellular respiratory, oxidative and metabolic processes. These include electron transport chain (ETC) activity, which drives oxidative phosphorylation to produce metabolic energy in the form of adenosine triphosphate (ATP), and which also underlies a central mitochondrial role in intracellular calcium homeostasis. Mitochondrial ultrastructural characterization reveals the presence of an outer mitochondrial membrane that serves as an interface between the organelle and the cytosol, a highly folded inner mitochondrial membrane that appears to form attachments to the outer membrane at multiple sites, and an intermembrane space between the two mitochondrial membranes. The subcompartment within the inner mitochondrial membrane is commonly referred to as the mitochondrial matrix. (For a review, see, e.g., Ernster et al., 1981 J. Cell Biol. 91:227s.) The cristae, originally postulated to occur as infoldings of the inner mitochondrial membrane, have recently been characterized using three-dimensional electron tomography as also including tube-like conduits that may form networks, and that can be connected to the inner membrane by open, circular (30 nm diameter) junctions (Perkins et al., 1997, Journal of Structural Biology 119:260). While the outer membrane is freely permeable to ionic and non-ionic solutes having molecular weights less than about ten kilodaltons, the inner mitochondrial membrane exhibits selective and regulated permeability for many small molecules, including certain cations, and is impermeable to large (&#x3e;&#x2dc;10 kDa) molecules. Altered or defective mitochondrial activity, including but not limited to failure at any step of the ETC, may result in catastrophic mitochondrial collapse that has been termed &#x201c;permeability transition&#x201d; (PT) or &#x201c;mitochondrial permeability transition&#x201d; (MPT). According to generally accepted theories of mitochondrial function, proper ETC respiratory activity requires maintenance of an electrochemical potential (&#x394;&#x3a8;m) in the inner mitochondrial membrane by a coupled chemiosmotic mechanism. Altered or defective mitochondrial activity may dissipate this membrane potential, thereby preventing ATP biosynthesis and halting the production of a vital biochemical energy source. In addition, mitochondrial proteins such as cytochrome c may leak out of the mitochondria after permeability transition and may induce the genetically programmed cell suicide sequence known as apoptosis or programmed cell death (PCD). MPT may result from direct or indirect effects of mitochondrial genes, gene products or related downstream mediator molecules and/or extramitochondrial genes, gene products or related downstream mediators, or from other known or unknown causes. Loss of mitochondrial potential therefore may be a critical event in the progression of diseases associated with altered mitochondrial function, including degenerative diseases. Mitochondrial defects, which may include defects related to the discrete mitochondrial genome that resides in mitochondrial DNA and/or to the extramitochondrial genome, which includes nuclear chromosomal DNA and other extramitochondrial DNA, may contribute significantly to the pathogenesis of diseases associated with altered mitochondrial function. For example, alterations in the structural and/or functional properties of mitochondrial components comprised of subunits encoded directly or indirectly by mitochondrial and/or extramitochondrial DNA, including alterations deriving from genetic and/or environmental factors or alterations derived from cellular compensatory mechanisms, may play a role in the pathogenesis of any disease associated with altered mitochondrial function. A number of degenerative diseases are thought to be caused by, or to be associated with, alterations in mitochondrial function. These include Alzheimer's Disease (AD); diabetes mellitus; Parkinson's Disease; Huntington's disease; dystonia; Leber's hereditary optic neuropathy; schizophrenia; mitochondrial encephalopathy, lactic acidosis, and stroke (MELAS); cancer; psoriasis; hyperproliferative disorders; mitochondrial diabetes and deafness ADD) and myoclonic epilepsy ragged red fiber syndrome. The extensive list of additional diseases associated with altered mitochondrial function continues to expand as aberrant mitochondrial or mitonuclear activities are implicated in particular disease processes. A hallmark pathology of AD and potentially other diseases associated with altered mitochondrial function is the death of selected cellular populations in particular affected tissues, which results from apoptosis (also referred to as &#x201c;programmed cell death&#x201d; or PCD) according to a growing body of evidence. Mitochondrial dysfunction is thought to be critical in the cascade of events leading to apoptosis in various cell types (Kroemer et al., FASEB J. 9:1277-87, 1995), and may be a cause of apoptotic cell death in neurons of the AD brain. Altered mitochondrial physiology may be among the earliest events in PCD (Zamzami et al., J. Exp. Med. 182:367-77, 1995; Zamzami et al., J. Exp. Med. 181:1661-72, 1995) and elevated reactive oxygen species (ROS) levels that result from such altered mitochondrial function may initiate the apoptotic cascade (Ausserer et al., Mol. Cell. Biol. 14:5032-42, 1994). Thus, in addition to their role in energy production in growing cells, mitochondria (or, at least, mitochondrial components) participate in apoptosis (Newmeyer et al., 1994, Cell 79:353-364; Liu et al., 1996, Cell 86:147-157). Apoptosis is apparently also required for, inter alia, normal development of the nervous system and proper functioning of the immune system. Moreover, some disease states are thought to be associated with either insufficient (e.g., cancer, autoimmune diseases) or excessive (e.g., stroke damage, AD-associated neurodegeneration) levels of apoptosis. For general reviews of apoptosis, and the role of mitochondria therein, see Green and Reed (1998, Science 281:1309-1312), Green (1998, Cell 94:695-698) and Kromer (1997, Nature Medicine 3:614-620). Hence, agents that effect apoptotic events, including those associated with mitochondrial components, might have a variety of palliative, prophylactic and therapeutic uses. The adenine nucleotide translocator (ANT), a nuclear encoded polypeptide that is a major component of the inner mitochondrial membrane, is responsible for mediating transport of ADP and ATP across the mitochondrial inner membrane. For example, ANT is believed to mediate stoichiometric ATP/proton exchange across the inner mitochondrial membrane, and ANT inhibitors (such as atractyloside or bongkrekic acid) induce MPT under certain conditions. Three human ANT isoforms have been described that differ in their tissue expression patterns and other mammalian ANT homologues have been described. (See, e.g., Wallace et al., 1998 in Mitochondria &#x26; Free Radicals in Neurodegenerative Diseases, Beal, Howell and Bodis-Wollner, Eds., Wiley-Liss, New York, pp. 283-307, and references cited therein.) ANT has also been implicated as an important molecular component of the mitochondrial permeability transition pore, a Ca2+-regulated inner membrane channel that, as described above, plays an important modulating role in apoptotic processes. As inner mitochondrial membrane proteins are believed to possess multiple hydrophobic membrane spanning domains, ANT polypeptides may exhibit, inter alia, poor intracellular solubility, toxic accumulations and/or the formation of inclusion bodies and other deleterious effects on respiratory homeostasis within a host cell due to ANT biological activity. Consequently, those having ordinary skill in the art have heretofore been unable to produce ANT, reliably or in sufficient quantities for a variety of uses, such as those provided herein. Because of the significance of mitochondria to respiratory, metabolic and apoptotic processes, and in view of the prominent role played by ANT in these and other mitochondrial acitivities, there is clearly a need for compositions and methods that permit the production of ANT proteins, including ANT fusion proteins; for novel ANT ligands; for methods to identify and isolate ANT proteins; and for methods of identifying and isolating agents that interact with ANT. The present invention fulfills these needs and provides other related advantages. These and other aspects of the present invention will become evident upon reference to the following detailed description and attached drawings. In addition, various references are set forth below which describe in more detail certain procedures or compositions (e.g., plasmids, vectors, etc.), and are therefore incorporated by reference in their entireties. SUMMARY OF THE INVENTION In its various aspects and embodiments the invention is directed to: A recombinant expression construct comprising at least one regulated promoter operably linked to a first nucleic acid encoding an adenine nucleotide translocator polypeptide; further comprising at least one additional nucleic acid sequence that regulates transcription; wherein the additional nucleic acid sequence that regulates transcription encodes a repressor of said regulated promoter; wherein the adenine nucleotide translocator polypeptide comprises a human adenine nucleotide translocator polypeptide; wherein the human adenine nucleotide translocator polypeptide is ANT1; wherein the human adenine nucleotide translocator polypeptide is ANT2; wherein the human adenine nucleotide translocator polypeptide is ANT3; wherein the adenine nucleotide translocator polypeptide is expressed as a fusion protein with a polypeptide product of a second nucleic acid sequence; wherein the polypeptide product of said second nucleic acid sequence is an enzyme; wherein said fusion protein localizes to membranes; wherein said membranes are mitochondrial membranes; wherein the adenine nucleotide translocator polypeptide is expressed as a fusion protein with at least one product of a second nucleic acid sequence encoding a polypeptide cleavable by a protease, said adenine nucleotide translocator polypeptide being separable from the fusion protein by cleavage with the protease; A host cell comprising a recombinant expression construct as provided; wherein the host cell is a prokaryotic cell; wherein the host cell is a eukaryotic cell; wherein the eukaryotic cell is selected from the group consisting of a yeast cell, an insect cell and a mammalian cell; wherein the insect cell is an Sf9 cell or a Trichoplusia ni cell; at lacks at least one isoform of an endogenous adenine nucleotide translocator; in which expression of at least one gene encoding an endogenous adenine nucleotide translocator isoform is substantially impaired. A recombinant expression construct comprising at least one promoter operably linked to a nucleic acid molecule comprising a first nucleic acid sequence and a second nucleic acid sequence, said first nucleic acid sequence encoding an animal adenine nucleotide translocator polypeptide wherein the adenine nucleotide translocator polypeptide is expressed as a fusion protein with a polypeptide product of said second nucleic acid sequence; wherein the polypeptide product of said second nucleic acid sequence is an enzyme, wherein said fusion protein localizes to membranes; wherein said membranes are mitochondrial membranes; further comprising at least one additional nucleic acid sequence that regulates transcription; wherein the additional nucleic acid sequence that regulates transcription encodes a repressor of said promoter; wherein the adenine nucleotide translocator polypeptide comprises a human adenine nucleotide translocator polypeptide; wherein the human adenine nucleotide translocator polypeptide is ANT1; wherein the human adenine nucleotide translocator polypeptide is ANT2; wherein the human adenine nucleotide translocator polypeptide is ANT3; wherein the adenine nucleotide translocator polypeptide is expressed as a fusion protein with at least one product of a second nucleic acid sequence encoding a polypeptide cleavable by a protease, said adenine nucleotide translocator polypeptide being separable from the fusion protein by cleavage with the protease; a host cell comprising a recombinant expression construct as just described; wherein the host cell is a prokaryotic cell; wherein the host cell is a eukaryotic cell; wherein the eukaryotic cell is selected from the group consisting of a yeast cell, an insect cell and a mammalian cell; wherein the insect cell is an Sf9 cell or a Trichoplusia ni cell; that lacks at least one isoform of an endogenous adenine nucleotide translocator; in which expression of at least one gene encoding an endogenous adenine nucleotide translocator isoform is substantially impaired; wherein the expression construct is a recombinant viral expression construct; A method of producing a recombinant adenine nucleotide translocator polypeptide, comprising; culturing a host cell comprising a recombinant expression construct comprising at least one regulated promoter operably linked to a first nucleic acid encoding an adenine nucleotide translocator polypeptide; A method of producing a recombinant adenine nucleotide translocator polypeptide, comprising culturing a host cell comprising a recombinant expression construct comprising at least one promoter operably linked to a nucleic acid molecule comprising a first nucleic acid sequence and a second nucleic acid sequence, said first nucleic acid sequence encoding an animal adenine nucleotide translocator polypeptide wherein the adenine nucleotide translocator polypeptide is expressed as a fusion protein with a polypeptide product of said second nucleic acid sequence; A method of producing a recombinant adenine nucleotide translocator polypeptide, comprising culturing a host cell infected with the recombinant viral expression construct as provided above. An ANT polypeptide produced by the methods just described. An isolated human adenine nucleotide translocator polypeptide; wherein the human adenine nucleotide translocator polypeptide is recombinant ANT1 or a variant or fragment thereof, wherein the human adenine nucleotide translocator polypeptide is recombinant ANT2 or a variant or fragment thereof, wherein the human adenine nucleotide translocator polypeptide is recombinant ANT3 or a variant or fragment thereof; An isolated human adenine nucleotide translocator fusion protein comprising an adenine translocator polypeptide fused to at least one additional polypeptide sequence; wherein said one additional polypeptide sequence is an enzyme sequence or a variant or fragment thereof, wherein said fusion protein localizes to membranes; wherein said membranes are mitochondrial membranes; An isolated human adenine nucleotide translocator fusion protein comprising an adenine translocator polypeptide fused to at least one additional polypeptide sequence cleavable by a protease, said adenine nucleotide translocator polypeptide being separable from the fusion protein by cleavage with the protease. An isolated adenine nucleotide translocator fusion protein comprising a first polypeptide that is an animal adenine translocator polypeptide fused to at least one additional polypeptide sequence; wherein said one additional polypeptide sequence is an enzyme sequence or a variant or fragment thereof, that localizes to membranes; wherein said membranes are mitochondrial membranes. An isolated recombinant animal adenine nucleotide translocator fusion protein comprising an adenine translocator polypeptide fused to at least one additional polypeptide sequence cleavable by a protease, said adenine nucleotide translocator polypeptide being separable from the fusion protein by cleavage with the protease; wherein the additional polypeptide sequence is a polypeptide having affinity for a ligand. A method for determining the presence of an ANT polypeptide in a biological sample comprising contacting a biological sample suspected of containing an ANT polypeptide with an ANT ligand under conditions and for a time sufficient to allow binding of the ANT ligand to an ANT polypeptide; and detecting the binding of the ANT ligand to an ANT polypeptide, and therefrom determining the presence of an ANT polypeptide in said biological sample; wherein the adenine nucleotide translocator polypeptide comprises a human adenine nucleotide translocator polypeptide; wherein the human adenine nucleotide translocator polypeptide is ANT1; wherein the human adenine nucleotide translocator polypeptide is ANT2; wherein the human adenine nucleotide translocator polypeptide is ANT3; wherein the ANT ligand comprises atractyloside substituted at 6&#x2032; hydroxyl to form an atractyloside derivative; wherein the atractyloside is detectably substituted at the 6&#x2032; hydroxyl to form a detectable atractyloside derivative; wherein the detectable atractyloside derivative comprises a radioloabeled substituent; wherein the radiolabeled substituent is selected from the group consisting of 125I, 131I, 3H, 14C and 35S; wherein the detectable atractyloside derivative comprises a fluorescent substituent; wherein the ANT ligand further comprises a Eu3+ atom complexed to the atractyloside derivative; wherein the detectable atractyloside derivative comprises covalently bound biotin; wherein the atractyloside molecule is substituted at 6&#x2032; hydroxyl with an amine or an amine containing functionality to form an amine modified atractyloside derivative; wherein the atractyloside molecule is a carboxyatractyloside molecule that is substituted at 6&#x2032; hydroxyl to form an atractyloside derivative that is a carboxyatractyloside derivative. A method for isolating ANT from a biological sample, comprising contacting a biological sample suspected of containing an ANT polypeptide with an ANT ligand under conditions and for a time sufficient to allow binding of the ANT ligand to an ANT polypeptide; and recovering the ANT polypeptide, and thereby isolating ANT from a biological sample; wherein the ANT ligand is covalently bound to a solid phase; wherein the ANT ligand is non-covalently bound to a solid phase. A method for identifying an agent that binds to an ANT polypeptide, comprising contacting a candidate agent with a host cell expressing at least one recombinant ANT polypeptide under conditions and for a time sufficient to permit binding of the agent to said recombinant ANT polypeptide; and detecting binding of said agent to the recombinant ANT; wherein the host cell is a prokaryotic cell; wherein the prokaryotic cell is an E. coli cell; wherein the host cell is a eukaryotic cell; wherein the eukaryotic cell is selected from the group consisting of a yeast cell, an insect cell and a mammalian cell; wherein the insect cell is an Sf9 cell or a Trichoplusia ni cell; wherein the host cell lacks at least one isoform of an endogenous adenine nucleotide translocator; wherein host cell expression of at least one gene encoding an endogenous adenine nucleotide translocator isoform is substantially impaired. A method for identifying an agent that binds to an ANT polypeptide, comprising contacting a candidate agent with a biological sample containing at least one recombinant ANT polypeptide under conditions and for a time sufficient to permit binding of the agent to said ANT polypeptide; and detecting binding of said agent to the recombinant ANT polypeptide. A method for identifying an agent that interacts with an ANT polypeptide comprising contacting a biological sample containing recombinant ANT with a detectable ANT ligand in the presence of a candidate agent; and comparing binding of the detectable ANT ligand to recombinant ANT in the absence of said agent to binding of the detectable ANT ligand to recombinant ANT in the presence of said agent, and therefrom identifying an agent that interacts with an ANT polypeptide. An ANT ligand comprising atractyloside substituted at the 6&#x2032; hydroxyl to form an atractyloside derivative; wherein the atractyloside is detectably substituted at the 6&#x2032; hydroxyl to form a detectable atractyloside derivative; wherein the detectable atractyloside derivative comprises a radioloabeled substituent; wherein the radiolabeled substituent is selected from the group consisting of 125I, 131I, 3H, 14C and 35S; wherein the detectable atractyloside derivative comprises a fluorescent substituent; further comprising a Eu3+ atom complexed to the atractyloside derivative; wherein the detectable atractyloside derivative comprises covalently bound biotin; wherein the atractyloside molecule is substituted at 6&#x2032; hydroxyl with an amine or an amine containing functionality to form an amine modified atractyloside derivative; wherein the atractyloside molecule is a carboxyatractyloside molecule that is substituted at 6&#x2032; hydroxyl to form an atractyloside derivative that is a carboxyatractyloside derivative. An ANT ligand having the following structure(I): including stereoisomers and pharmaceutically acceptable salts thereof, wherein R1, R2 and R3 are as identified below. An assay plate for high throughput screening of candidate agents that bind to at least one ANT polypeptide, comprising an assay plate having a plurality of wells, each of said wells further comprising at least one immobilized recombinant ANT polypeptide or a variant or fragment thereof. A method of targeting a polypeptide of interest to a mitochondrial membrane, comprising expressing a recombinant expression construct encoding a fusion protein in a host cell, said construct comprising at least one regulated promoter operably linked to a nucleic acid molecule comprising a first nucleic acid sequence and a second nucleic acid sequence, said first nucleic acid sequence encoding an adenine nucleotide translocator polypeptide that is expressed as a fusion protein with a polypeptide product of said second nucleic acid sequence, wherein said second nucleic acid sequence encodes the polypeptide of interest. A method of targeting a polypeptide of interest to a mitochondrial membrane, comprising expressing a recombinant expression construct encoding a fusion protein in a host cell, said construct comprising at least one promoter operably linked to a nucleic acid molecule comprising a first nucleic acid sequence and a second nucleic acid sequence, said first nucleic acid sequence encoding an animal adenine nucleotide translocator polypeptide that is expressed as a fusion protein with a polypeptide product of said second nucleic acid sequence, wherein said second nucleic acid sequence encodes the polypeptide of interest; a pharmaceutical composition comprising an ANT ligand as just described. A pharmaceutical composition comprising an agent that binds to an ANT polypeptide identified as just described. A pharmaceutical composition comprising an agent that binds to an ANT polypeptide identified as described above. A pharmaceutical composition comprising an agent that interacts with an ANT polypeptide identified above. A method of treatment comprising administering to a subject any one of the just described the pharmaceutical compositions.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"detail\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 5,\n        \"samples\": [\n          \"DETAILED DESCRIPTION OF THE INVENTION The present invention is directed generally toward adenine nucleotide translocator (ANT) polypeptides, which as provided herein may refer to any ANT isoform; to expression constructs containing nucleic acids encoding ANT and to natural and synthetic small molecules that interact with ANT, including ANT binding ligands. The present invention relates in part to the unexpected findings that bacterial, insect, yeast or mammalian expression systems can be designed for reliable production of recombinant human ANT polypeptides in significant quantities. In certain aspects the invention provides compositions and methods for producing recombinant ANT polypeptides that employ regulated promoters, and in certain of these and other aspects the invention provides compositions and methods for producing recombinant ANT polypeptides that are ANT fusion proteins. In certain preferred embodiments, the design of such expression systems includes the use of a host cell that lacks endogenous ANT or in which endogenous ANT gene expression is substantially impaired, as provided herein. The present invention thus also pertains in part to methods for producing and isolating recombinant ANT polypeptides, including human ANT polypeptides and in preferred embodiments human ANT3 polypeptides, that may then be used in various binding assays and screening assays and the like. In view of the surprising observation that expression of recombinant human ANT polypeptides can be achieved at levels enabling such uses of these ANT polypeptide products, the present invention provides assays (including high throughput assays) for identifying agents that bind to recombinant human ANT. Accordingly, the present invention further relates in part to novel human ANT ligands, the synthesis, selection and characterization of which would heretofore have not been possible given the need for expressed recombinant ANT polypeptides to use in binding assays. The invention also pertains to agents that interact with ANT, including agents that enhance or impair any ANT functions known to the art, including but not limited to those described herein, and to incorporation of such agents into pharmaceutical compositions and their use in therapeutic methods. As dicussed above, the present invention relates in part to the unexpected finding that recombinant adenine nucleotide translocator (ANT) polypeptides, which includes full length ANT proteins and polypeptides, fragments and variants thereof, and further includes ANT fusion proteins as provided herein, can be produced in useful amounts by using a recombinant expression vector having a regulatory nucleic acid operably linked to a nucleic acid encoding ANT. In particular, the invention provides compositions and methods for producing recombinant ANT polypeptides through the use of a regulated promoter; the invention also provides recombinant ANT polypeptides that are ANT fusion proteins. The invention also pertains to compositions and methods to identify the presence of ANT polypeptides and to isolate recombinant ANT, and in addition to screening assays for compounds that interact with ANT. Accordingly, the invention provides certain advantages with regard to regulation of mitochondrial function, and in particular regulation of the mitochondrial permeability &#x201c;pore&#x201d;. By way of background, four of the five multisubunit protein complexes (Complexes I, III, IV and V) that mediate ETC activity are localized to the inner mitochondrial membrane, which is the most protein rich of biological membranes in cells (75% by weight); the remaining ETC complex (Complex II) is situated in the matrix. ANT represents the most abundant of the inner mitochondrial membrane proteins. In at least three distinct chemical reactions known to take place within the ETC, positively-charged protons are moved from the mitochondrial matrix, across the inner membrane, to the intermembrane space. This disequilibrium of charged species creates an electrochemical potential of approximately 220 mV referred to as the &#x201c;protonmotive force&#x201d; (PMF), which is often represented by the notation &#x394;&#x3c8; or &#x394;&#x3c8;m and represents the sum of the electric potential and the pH differential across the inner mitochondrial membrane (see, e.g., Ernster et al., 1981 J. Cell Biol. 91:227s and references cited therein). This membrane potential drives ANT-mediated stoichiometric exchange of adenosine triphosphate (ATP) and adenosine diphosphate (ADP) across the inner mitochondrial membrane, and provides the energy contributed to the phosphate bond created when ADP is phosphorylated to yield ATP by ETC Complex V, a process that is &#x201c;coupled&#x201d; stoichiometrically with transport of a proton into the matrix. Mitochondrial membrane potential, &#x394;&#x3c8;m, is also the driving force for the influx of cytosolic Ca2+ into the mitochondrion. Under normal metabolic conditions, the inner membrane is impermeable to proton movement from the intermembrane space into the matrix, leaving ETC Complex V as the sole means whereby protons can return to the matrix. When, however, the integrity of the inner mitochondrial membrane is compromised, as occurs during MPT that may accompany a disease associated with altered mitochondrial function, protons are able to bypass the conduit of Complex V without generating ATP, thereby &#x201c;uncoupling&#x201d; respiration because electron transfer and associated proton pumping yields no ATP. Thus, mitochondrial permeability transition involves the opening of a mitochondrial membrane &#x201c;pore&#x201d;, a process by which, inter alia, the ETC and ATP synthesis are uncoupled, &#x394;&#x3c8;m collapses and mitochondrial membranes lose the ability to selectively regulate permeability to solutes both small (e.g., ionic Ca2+, Na+, K+, H+) and large (e.g., proteins). Without wishing to be bound by theory, it is unresolved whether this pore is a physically discrete conduit that is formed in mitochondrial membranes, for example by assembly or aggregation of particular mitochondrial and/or cytosolic proteins and possibly other molecular species, or whether the opening of the &#x201c;pore&#x201d; may simply represent a general increase in the porosity of the mitochondrial membrane. MPT may also be induced by compounds that bind one or more mitochondrial molecular components. Such compounds include, but are not limited to, atractyloside and bongkrekic acid, which are known to bind to ANT. Methods of determining appropriate amounts of such compounds to induce MPT are known in the art (see, e.g., Beutner et al., 1998 Biochim. Biophys. Acta 1368:7; Obatomi and Bach, 1996 Toxicol. Lett. 89:155; Green and Reed, 1998 Science 281:1309; Kroemer et al., 1998 Annu. Rev. Physiol. 60:619; and references cited therein). Thus certain mitochondrial molecular components, such as ANT, may contribute to the MPT mechanism. As noted above, it is believed that adenine nucleotide translocator (ANT) mediates ATP/proton exchange across the inner mitochondrial membrane, and that ANT inhibitors such as atractyloside or bongkrekic acid induce MPT under certain conditions. Hence, it is desirable to obtain ANT in sufficient quantities for structural and functional assays that provide, for example, ANT ligands and other agents that interact with ANT, which will be useful for therapeutic management of mitochondrial pore activity. See also U.S. Ser. No. 09/161,172, entitled &#x201c;Compositions and Methods for Identifying Agents that Alter Mitochondrial Permeability Transition Pores&#x201d;, which is hereby incorporated by reference. The compositions and methods of the present invention can be adapted to any prokaryotic or eukaryotic ANT, including plant and animal ANTs, which may further include, for example, yeast, vertebrate, mammalian, rodent, primate and human ANTs, for which amino acid sequences and/or encoding nucleic acids will be known to those familiar with the art. Three human ANT isoforms have been described that differ in their tissue expression patterns. (Stepien et al., 1992 J. Biol. Chem. 267:14592; see also Wallace et al., 1998 in Mitochondria &#x26; Free Radicals in Neurodegenerative Diseases, Beal, Howell and Bodis-Wollner, Eds., Wiley-Liss, New York, pp. 283-307, and references cited therein.) Nucleic acid sequences for cDNAs encoding these three human ANT isoforms have been reported (FIG. 1; See Neckelmann et al., Proc. Nat'l. Acad. Sci. U.S.A. 84:7580-7584 (1987) for huANT1 cDNA [SEQ ID NO:1]; Battini et al., J. Biol. Chem. 262:4355-4359 (1987) for huANT2 cDNA [SEQ ID NO:2], and Cozens et al., J. Mol. Biol. 206:261-280 (1989) for huANT3 cDNA [SEQ ID NO:3]; see FIG. 2 for amino acid sequences of huANT1 [SEQ ID NO:31] huANT2 [SEQ ID NO:32] and huANT3 [SEQ ID NO:33].), and ANT gene sequences have been determined for a number of species (See, e.g., Li et al., 1989 J. Biol. Chem. 264:13998 for huANT1 genomic DNA; Liew et al. GenBank Acc. #N86710 for huANT2; Shinohara et al., 1993 Biochim. Biophys. Acta 1152:192 for rat ANT gene; for others see also, e.g., Ku et al., 1990 J. Biol. Chem. 265:16060; Adams et al., 1991 Science 252:1651; and WO 98/19714.). ANT sequences among mammalian species are highly conserved; for example, at the amino acid level murine ANT1 and ANT2 exhibit 98% sequence identity with human ANT2. Full length amino acid sequences of at least 29 ANT proteins have been reported to date from a variety of animal and plant species, with most of these deduced from nucleic acid sequences. (Fiore et al., 1998 Biochimie 80:137-150) The present invention further relates to nucleic acids which hybridize to ANT encoding polynucleotide sequences as provided herein, as incorporated by reference or as will be readily apparent to those familiar with the art, if there is at least 70%, preferably at least 90%, and more preferably at least 95% identity between the sequences. The present invention particularly relates to nucleic acids which hybridize under stringent conditions to the ANT encoding nucleic acids referred to herein. As used herein, the term &#x201c;stringent conditions&#x201d; means hybridization will occur only if there is at least 95% and preferably at least 97% identity between the sequences. The nucleic acids which hybridize to ANT encoding nucleic acids referred to herein, in preferred embodiments, encode polypeptides which either retain substantially the same biological function or activity as the ANT polypeptides encoded by the cDNAs of FIG. 1 [SEQ ID NOS:1, 2 and 3], or the deposited expression constructs. As used herein, to &#x201c;hybridize&#x201d; under conditions of a specified stringency is used to describe the stability of hybrids formed between two single-stranded nucleic acid molecules. Stringency of hybridization is typically expressed in conditions of ionic strength and temperature at which such hybrids are annealed and washed. Typically &#x201c;high&#x201d;, &#x201c;medium&#x201d; and &#x201c;low&#x201d; stringency encompass the following conditions or equivalent conditions thereto: high stringency: 0.1&#xd7;SSPE or SSC, 0.1% SDS, 65&#xb0; C.; medium stringency: 0.2&#xd7;SSPE or SSC, 0.1% SDS, 50&#xb0; C.; and low stringency: 1.0&#xd7;SSPE or SSC, 0.1% SDS, 50&#xb0; C. The deposits referred to herein will be maintained under the terms of the Budapest Treaty on the International Recognition of the Deposit of Micro-organisms for purposes of Patent Procedure. These deposits are provided merely as convenience to those of skill in the art and are not an admission that a deposit is required under 35 U.S.C. &#xa7; 112. The sequences of the nucleic acids contained in the deposited materials, as well as the amino acid sequences of the polypeptides encoded thereby, are incorporated herein by reference and are controlling in the event of any conflict with any description of sequences herein. A licensee may be required to make, use or sell the deposited materials, and no such license is hereby granted. Nucleic Acids The nucleic acids of the present invention may be in the form of RNA or in the form of DNA, which DNA includes cDNA, genomic DNA, and synthetic DNA. The DNA may be double-stranded or single-stranded, and if single stranded may be the coding strand or non-coding (anti-sense) strand. A coding sequence which encodes an ANT polypeptide for use according to the invention may be identical to the coding sequence known in the art for any given ANT, as described above and, for example, as shown for human ANT1 [SEQ ID NO:1], human ANT2 [SEQ ID NO.2] and human ANT3 [SEQ ID NO:3] in FIG. 1, or to that of any deposited clone, or may be a different coding sequence, which, as a result of the redundancy or degeneracy of the genetic code, encodes the same ANT polypeptide as, for example, the cDNAs of FIG. 1 or the deposited expression constructs. The nucleic acids which encode ANT polypeptides, for example the human ANT polypeptides having the amino acid sequences of FIG. 2 [SEQ ID NOS:31-33] or any other ANT polypeptides for use according to the invention, or for the ANT polypeptides encoded by the deposited constructs may include, but are not limited to: only the coding sequence for the ANT polypeptide; the coding sequence for the ANT polypeptide and additional coding sequence; the coding sequence for the ANT polypeptide (and optionally additional coding sequence) and non-coding sequence, such as introns or non-coding sequences 5&#x2032; and/or 3&#x2032; of the coding sequence for the ANT polypeptide, which for example may further include but need not be limited to one or more regulatory nucleic acid sequences that may be a regulated or regulatable promoter, enhancer, other transcription regulatory sequence, repressor binding sequence, translation regulatory sequence or any other regulatory nucleic acid sequence. Thus, the term &#x201c;nucleic acid encoding an ANT polypeptide&#x201d; encompasses a nucleic acid which includes only coding sequence for the polypeptide as well as a nucleic acid which includes additional coding and/or non-coding sequence(s). The present invention further relates to variants of the herein described nucleic acids which encode for fragments, analogs and derivatives of an ANT polypeptide, for example the human ANT1, ANT2 and ANT3 polypeptides having the deduced amino acid sequences of FIG. 2 [SEQ ID NOS:31-33] or any ANT polypeptide, including ANT polypeptides encoded by the cDNAs of the deposited expression constructs. The variants of the nucleic acids encoding ANTs may be naturally occurring allelic variants of the nucleic acids or non-naturally occurring variants. As is known in the art, an allelic variant is an alternate form of a nucleic acid sequence which may have at least one of a substitution, a deletion or an addition of one or more nucleotides, any of which does not substantially alter the function of the encoded ANT polypeptide. Thus, for example, the present invention includes nucleic acids encoding the same ANT polypeptides as shown in FIG. 2 [SEQ ID NOS:31-33], or the same ANT polypeptides encoded by the cDNAs of the deposited expression constructs, as well as variants of such nucleic acids, which variants encode a fragment, derivative or analog of any of the polypeptides of FIG. 2 (SEQ ID NO:2) or the polypeptides encoded by the cDNAs of the deposited expression constructs. Variants and derivatives of ANT may be obtained by mutations of nucleotide sequences encoding ANT polypeptides. Alterations of the native amino acid sequence may be accomplished by any of a number of conventional methods. Mutations can be introduced at particular loci by synthesizing oligonucleotides containing a mutant sequence, flanked by restriction sites enabling ligation to fragments of the native sequence. Following ligation, the resulting reconstructed sequence encodes an analog having the desired amino acid insertion, substitution, or deletion. Alternatively, oligonucleotide-directed site-specific mutagenesis procedures can be employed to provide an altered gene wherein predetermined codons can be altered by substitution, deletion or insertion. Exemplary methods of making such alterations are disclosed by Walder et al. (Gene 42:133, 1986); Bauer et al. (Gene 37:73, 1985); Craik (BioTechniques, January 1985, 12-19); Smith et al. (Genetic Engineering: Principles and Methods, Plenum Press, 1981); Kunkel (Proc. Natl. Acad Sci. USA 82:488, 1985); Kunkel et al. (Methods in Enzymol. 154:367, 1987); and U.S. Pat. Nos. 4,518,584 and 4,737,462. Equivalent DNA constructs that encode various additions or substitutions of amino acid residues or sequences, or deletions of terminal or internal residues or sequences not needed for biological activity are also encompassed by the invention. For example, sequences encoding Cys residues that are not essential for biological activity can be altered to cause the Cys residues to be deleted or replaced with other amino acids, preventing formation of incorrect intramolecular disulfide bridges upon renaturation. Other equivalents can be prepared by modification of adjacent dibasic amino acid residues to enhance expression in yeast systems in which KEX2 protease activity is present. EP 212,914 discloses the use of site-specific mutagenesis to inactivate KEX2 protease processing sites in a protein. KEX2 protease processing sites are inactivated by deleting, adding or substituting residues to alter Arg-Arg, Arg-Lys, and Lys-Arg pairs to eliminate the occurrence of these adjacent basic residues. Lys-Lys pairings are considerably less susceptible to KEX2 cleavage, and conversion of Arg-Lys or Lys-Arg to Lys-Lys represents a conservative and preferred approach to inactivating KEX2 sites. Polypeptides and Fusion Proteins The present invention further relates to ANT polypeptides, and in particular to methods for producing recombinant ANT polypeptides by culturing host cells containing ANT expression constructs, and to isolated recombinant human ANT polypeptides, including, for example, the human ANT1, ANT2 and ANT3 polypeptides which have the deduced amino acid sequence of FIG. 2 [SEQ ID NOS:31-33] or which have the amino acid sequence encoded by the deposited recombinant expression constructs, as well as fragments, analogs and derivatives of such polypeptides. The polypeptides and nucleic acids of the present invention are preferably provided in an isolated form, and in certain preferred embodiments are purified to homogeneity. The terms &#x201c;fragment,&#x201d; &#x201c;derivative&#x201d; and &#x201c;analog&#x201d; when referring to ANT polypeptides or fusion proteins, or to ANT polypeptides or fusion proteins encoded by the deposited recombinant expression constructs, refers to any ANT polypeptide or fusion protein that retains essentially the same biological function or activity as such polypeptide. Thus, an analog includes a proprotein which can be activated by cleavage of the proprotein portion to produce an active ANT polypeptide. The polypeptide of the present invention may be a recombinant polypeptide or a synthetic polypeptide, and is preferably a recombinant polypeptide. A fragment, derivative or analog of an ANT polypeptide or fusion protein, including ANT polypeptides or fusion proteins encoded by the cDNAs of the deposited constructs, may be (i) one in which one or more of the amino acid residues are substituted with a conserved or non-conserved amino acid residue (preferably a conserved amino acid residue) and such substituted amino acid residue may or may not be one encoded by the genetic code, or (ii) one in which one or more of the amino acid residues includes a substituent group, or (iii) one in which the ANT polypeptide is fused with another compound, such as a compound to increase the half-life of the polypeptide (for example, polyethylene glycol), or (iv) one in which additional amino acids are fused to the ANT polypeptide, including amino acids that are employed for purification of the ANT polypeptide or a proprotein sequence. Such fragments, derivatives and analogs are deemed to be within the scope of those skilled in the art from the teachings herein. The polypeptides of the present invention include ANT polypeptides and fusion proteins having amino acid sequences that are identical or similar to sequences known in the art. For example by way of illustration and not limitation, the human ANT (&#x201c;huANT&#x201d;) polypeptides of FIG. 2 [SEQ ID NOS:31-33] are contemplated for use according to the instant invention, as are polypeptides having at least 70% similarity (preferably a 70% identity) to the polypeptides of FIG. 2 [SEQ ID NOS:31-33] and more preferably 90% similarity (more preferably a 90% identity) to the polypeptides of FIG. 2 [SEQ ID NOS: 31-33] and still more preferably a 95% similarity (still more preferably a 95% identity) to the polypeptides of FIG. 2 [SEQ ID NOS:31-33] and to portions of such polypeptides, wherein such portions of an ANT polypeptide generally contain at least 30 amino acids and more preferably at least 50 amino acids. As known in the art &#x201c;similarity&#x201d; between two polypeptides is determined by comparing the amino acid sequence and conserved amino acid substitutes thereto of the polypeptide to the sequence of a second polypeptide. Fragments or portions of the polypeptides of the present invention may be employed for producing the corresponding full-length polypeptide by peptide synthesis; therefore, the fragments may be employed as intermediates for producing the full-length polypeptides. Fragments or portions of the nucleic acids of the present invention may be used to synthesize full-length nucleic acids of the present invention. The term &#x201c;isolated&#x201d; means that the material is removed from its original environment (e.g., the natural environment if it is naturally occurring). For example, a naturally occurring nucleic acid or polypeptide present in a living animal is not isolated, but the same nucleic acid or polypeptide, separated from some or all of the co-existing materials in the natural system, is isolated. Such nucleic acids could be part of a vector and/or such nucleic acids or polypeptides could be part of a composition, and still be isolated in that such vector or composition is not part of its natural environment. The term &#x201c;gene&#x201d; means the segment of DNA involved in producing a polypeptide chain; it includes regions preceding and following the coding region &#x201c;leader and trailer&#x201d; as well as intervening sequences (introns) between individual coding segments (exons). As described herein, the invention provides ANT fusion proteins encoded by nucleic acids that have the ANT coding sequence fused in frame to an additional coding sequence to provide for expression of an ANT polypeptide sequence fused to an additional functional or non-functional polypeptide sequence that permits, for example by way of illustration and not limitation, detection, isolation and/or purification of the ANT fusion protein. Such ANT fusion proteins may permit detection, isolation and/or purification of the ANT fusion protein by protein-protein affinity, metal affinity or charge affinity-based polypeptide purification, or by specific protease cleavage of a fusion protein containing a fusion sequence that is cleavable by a protease such that the ANT polypeptide is separable from the fusion protein. Thus ANT fusion proteins may comprise polypeptide sequences added to ANT to facilitate detection and isolation of ANT. Such peptides include, for example, poly-His or the antigenic identification peptides described in U.S. Pat. No. 5,011,912 and in Hopp et al., (1988 Bio/Technology 6:1204), or the XPRESS&#x2122; epitope tag (Invitrogen, Carlsbad, Calif.). The affinity sequence may be a hexa-histidine tag as supplied, for example, by a pBAD/His (Invitrogen) or a pQE-9 vector to provide for purification of the mature polypeptide fused to the marker in the case of a bacterial host, or, for example, the affinity sequence may be a hemagglutinin (HA) tag when a mammalian host, e.g., COS-7 cells, is used. The HA tag corresponds to an antibody defined epitope derived from the influenza hemagglutinin protein (Wilson et al., Cell 37:767 (1984)). ANT fusion proteins may further comprise immunoglobulin constant region polypeptides added to ANT to facilitate detection, isolation and/or localization of ANT. The immunoglobulin constant region polypeptide preferably is fused to the C-terminus of an ANT polypeptide. General preparation of fusion proteins comprising heterologous polypeptides fused to various portions of antibody-derived polypeptides (including the Fc domain) has been described, e.g., by Ashkenazi et al. (PNAS USA 88:10535, 1991) and Byrn et al. (Nature 344:677, 1990). A gene fusion encoding the ANT:Fc fusion protein is inserted into an appropriate expression vector. In certain embodiments of the invention, ANT:Fc fusion proteins may be allowed to assemble much like antibody molecules, whereupon interchain disulfide bonds form between Fc polypeptides, yielding dimeric ANT fusion proteins. ANT fusion proteins having specific binding affinities for pre-selected antigens by virtue of fusion polypeptides comprising immunoglobulin V-region domains encoded by DNA sequences linked in-frame to sequences encoding ANT are also within the scope of the invention, including variants and fragments thereof as provided herein. General strategies for the construction of fusion proteins having immunoglobulin V-region fusion polypeptides are disclosed, for example, in EP 0318554; U.S. Pat. No. 5,132,405; U.S. Pat. No. 5,091,513; and U.S. Pat. No. 5,476,786. The nucleic acid of the present invention may also encode a fusion protein comprising an ANT polypeptide fused to other polypeptides having desirable affinity properties, for example an enzyme such as glutathione-S-transferase. As another example, ANT fusion proteins may also comprise an ANT polypeptide fused to a Staphylococcus aureus protein A polypeptide; protein A encoding nucleic acids and their use in constructing fusion proteins having affinity for immunoglobulin constant regions are disclosed generally, for example, in U.S. Pat. No. 5,100,788. Other useful affinity polypetides for construction of ANT fusion proteins may include streptavidin fusion proteins, as disclosed, for example, in WO 89103422; U.S. Pat. No. 5,489,528; U.S. Pat. No. 5,672,691; WO 93/24631; U.S. Pat. No. 5,168,049; U.S. Pat. No. 5,272,254 and elsewhere, and avidin fusion proteins (see, e.g., EP 511,747). As provided herein and in the cited references, ANT polypeptide seqences may be fused to fusion polypeptide sequences that may be full length fusion polypeptides and that may alternatively be variants or fragments thereof. The present invention also provides a method of targeting a polypeptide of interest to a membrane, and in particular embodiments to a cellular membrane, and in further embodiments to a mitochondrial membrane. This aspect of the invention is based on the unexpected observation that certain recombinant expression constructs as provided herein, which constructs include a nucleic acid encoding a first polypeptide that is an ANT polypeptide, and that is expressed as a fusion protein with a second polypeptide sequence, provide recombinant ANT fusion proteins capable of preferentially localizing to cell membranes. In certain embodiments the cell membrane is a prokaryotic cell membrane such as a bacterial cell membrane, for example an E. coli membrane. In other embodiments the cell membrane is a eukaryotic cell membrane such as a yeast or a mammalian cell membrane, for example a membrane of any eukaryotic cell contemplated herein. A cell membrane as used herein may be any cellular membrane, and typically refers to membranes that are in contact with cytosolic components, including intracellular membrane bounded compartments such as mitochondrial inner and outer membranes as described above, and also intracellular vesicles, ER-Golgi constituents, other organelles and the like, as well as the plasma membrane. In preferred embodiments, an ANT fusion protein may be targeted to a mitochondrial membrane. In other preferred embodiments, for example, recombinant expression constructs according to the invention may encode ANT fusion proteins that contain polypeptide sequences that direct the fusion protein to be retained in the cytosol, to reside in the lumen of the endoplasmic reticulum (ER), to be secreted from a cell via the classical ER-Golgi secretory pathway, to be incorporated into the plasma membrane, to associate with a specific cytoplasmic component including the cytoplasmic domain of a transmembrane cell surface receptor or to be directed to a particular subcellular location by any of a variety of known intracellular protein sorting mechanisms with which those skilled in the art will be familiar. Accordingly, these and related embodiments are encompassed by the instant compositions and methods directed to targeting a polypeptide of interest to a predefined intracellular, membrane or extracellular localization. Vectors The present invention also relates to vectors and to constructs that include nucleic acids of the present invention, and in particular to &#x201c;recombinant expression constructs&#x201d; that include any nucleic acids encoding ANT polypeptides according to the invention as provided above; to host cells which are genetically engineered with vectors and/or constructs of the invention and to the production of ANT polypeptides and fusion proteins of the invention, or fragments or variants thereof, by recombinant techniques. ANT proteins can be expressed in mammalian cells, yeast, bacteria, or other cells under the control of appropriate promoters. Cell-free translation systems can also be employed to produce such proteins using RNAs derived from the DNA constructs of the present invention. Appropriate cloning and expression vectors for use with prokaryotic and eukaryotic hosts are described by Sambrook, et al., Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbor, N.Y., (1989). Generally, recombinant expression vectors will include origins of replication and selectable markers permitting transformation of the host cell, e.g., the ampicillin resistance gene of E. coli and S. cerevisiae TRP1 gene, and a promoter derived from a highly-expressed gene to direct transcription of a downstream structural sequence. Such promoters can be derived from operons encoding glycolytic enzymes such as 3-phosphoglycerate kinase (PGK), &#x3b1;-factor, acid phosphatase, or heat shock proteins, among others. The heterologous structural sequence is assembled in appropriate phase with translation initiation and termination sequences. Optionally, the heterologous sequence can encode a fusion protein including an N-terminal identification peptide imparting desired characteristics, e.g., stabilization or simplified purification of expressed recombinant product. Useful expression constructs for bacterial use are constructed by inserting into an expression vector a structural DNA sequence encoding a desired protein together with suitable translation initiation and termination signals in operable reading phase with a functional promoter. The construct may comprise one or more phenotypic selectable marker and an origin of replication to ensure maintenance of the vector construct and, if desirable, to provide amplification within the host. Suitable prokaryotic hosts for transformation include E. coli, Bacillus subtilis, Salmonella typhimurium and various species within the genera Pseudomonas, Streptomyces, and Staphylococcus, although others may also be employed as a matter of choice. Any other plasmid or vector may be used as long as they are replicable and viable in the host. As a representative but nonlimiting example, useful expression vectors for bacterial use can comprise a selectable marker and bacterial origin of replication derived from commercially available plasmids comprising genetic elements of the well known cloning vector pBR322 (ATCC 37017). Such commercial vectors include, for example, pKK223-3 (Pharmacia Fine Chemicals, Uppsala, Sweden) and GEM1 (Promega Biotec, Madison, Wis., USA). These pBR322 &#x201c;backbone&#x201d; sections are combined with an appropriate promoter and the structural sequence to be expressed. Following transformation of a suitable host strain and growth of the host strain to an appropriate cell density, the selected promoter, if it is a regulated promoter as provided herein, is induced by appropriate means (e.g., temperature shift or chemical induction) and cells are cultured for an additional period. Cells are typically harvested by centrifugation, disrupted by physical or chemical means, and the resulting crude extract retained for further purification. Microbial cells employed in expression of proteins can be disrupted by any convenient method, including freeze-thaw cycling, sonication, mechanical disruption, or use of cell lysing agents; such methods are well know to those skilled in the art. Thus, for example, the nucleic acids of the invention as provided herein may be included in any one of a variety of expression vector constructs as a recombinant expression construct for expressing an ANT polypeptide. Such vectors and constructs include chromosomal, nonchromosomal and synthetic DNA sequences, e.g., derivatives of SV40; bacterial plasmids; phage DNA; baculovirus; yeast plasmids; vectors derived from combinations of plasmids and phage DNA, viral DNA, such as vaccinia, adenovirus, fowl pox virus, and pseudorabies. However, any other vector may be used for preparation of a recombinant expression construct as long as it is replicable and viable in the host. The appropriate DNA sequence(s) may be inserted into the vector by a variety of procedures. In general, the DNA sequence is inserted into an appropriate restriction endonuclease site(s) by procedures known in the art. Standard techniques for cloning, DNA isolation, amplification and purification, for enzymatic reactions involving DNA ligase, DNA polymerase, restriction endonucleases and the like, and various separation techniques are those known and commonly employed by those skilled in the art. A number of standard techniques are described, for example, in Ausubel et al. (1993 Current Protocols in Molecular Biology, Greene Publ. Assoc. Inc. &#x26; John Wiley &#x26; Sons, Inc., Boston, Mass.); Sambrook et al. (1989 Molecular Cloning, Second Ed., Cold Spring Harbor Laboratory, Plainview, N.Y.); Maniatis et al. (1982 Molecular Cloning, Cold Spring Harbor Laboratory, Plainview, N.Y.); and elsewhere. The DNA sequence in the expression vector is operatively linked to at least one appropriate expression control sequences (e.g., a promoter or a regulated promoter) to direct mRNA synthesis. Representative examples of such expression control sequences include LTR or SV40 promoter, the E. coli lac or trp, the phage lambda PL promoter and other promoters known to control expression of genes in prokaryotic or eukaryotic cells or their viruses. Promoter regions can be selected from any desired gene using CAT (chloramphenicol transferase) vectors or other vectors with selectable markers. Two appropriate vectors are pKK232-8 and pCM7. Particular named bacterial promoters include lacI, lacZ, T3, T7, gpt, lambda PR, PL and top. Eukaryotic promoters include CMV immediate early, HSV thymidine kinase, early and late SV40, LTRs from retrovirus, and mouse metallothionein-I. Selection of the appropriate vector and promoter is well within the level of ordinary skill in the art, and preparation of certain particularly preferred recombinant expression constructs comprising at least one promoter or regulated promoter operably linked to a nucleic acid encoding an ANT polypeptide is described herein. In certain preferred embodiments the expression control sequence is a &#x201c;regulated promoter&#x201d;, which may be a promoter as provided herein and may also be a repressor binding site, an activator binding site or any other regulatory sequence that controls expression of a nucleic acid sequence as provided herein. In certain particularly preferred embodiments the regulated promoter is a tightly regulated promoter that is specifically inducible and that permits little or no transcription of nucleic acid sequences under its control in the absence of an induction signal, as is known to those familiar with the art and described, for example, in Guzman et al. (1995 J. Bacteriol. 177:4121), Carra et al. (1993 EMBO J. 12:35), Mayer (1995 Gene 163:41), Haldimann et al. (1998 J. Bacteriol. 180:1277), Lutz et al. (1997 Nuc. Ac. Res. 25:1203), Allgood et al. (1997 Curr. Opin. Biotechnol. 8:474) and Makrides (1996 Microbiol. Rev. 60:512), all of which are hereby incorporated by reference. In other preferred embodiments of the invention a regulated promoter is present that is inducible but that may not be tightly regulated. In certain other preferred embodiments a promoter is present in the recombinant expression construct of the invention that is not a regulated promoter; such a promoter may include, for example, a constitutive promoter such as an insect polyhedrin promoter as described in the Examples or a yeast phosphoglycerate kinase promoter (see, e.g., Giraud et al., 1998 J. Mol. Biol. 281:409). The expression construct also contains a ribosome binding site for translation initiation and a transcription terminator. The vector may also include appropriate sequences for amplifying expression. Transcription of the DNA encoding the polypeptides of the present invention by higher eukaryotes may be increased by inserting an enhancer sequence into the vector. Enhancers are cis-acting elements of DNA, usually about from 10 to 300 bp that act on a promoter to increase its transcription. Examples including the SV40 enhancer on the late side of the replication origin bp 100 to 270, a cytomegalovirus early promoter enhancer, the polyoma enhancer on the late side of the replication origin, and adenovirus enhancers. As noted above, in certain embodiments the vector may be a viral vector such as a retroviral vector. For example, retroviruses from which the retroviral plasmid vectors may be derived include, but are not limited to, Moloney Murine Leukemia Virus, spleen necrosis virus, retroviruses such as Rous Sarcoma Virus, Harvey Sarcoma virus, avian leukosis virus, gibbon ape leukemia virus, human immunodeficiency virus, adenovirus, Myeloproliferative Sarcoma Virus, and mammary tumor virus. The viral vector includes one or more promoters. Suitable promoters which may be employed include, but are not limited to, the retroviral LTR; the SV40 promoter; and the human cytomegalovirus (CMV) promoter described in Miller, et al., Biotechniques 7:980-990 (1989), or any other promoter (e.g., cellular promoters such as eukaryotic cellular promoters including, but not limited to, the histone, pol III, and &#x3b2;-actin promoters). Other viral promoters which may be employed include, but are not limited to, adenovirus promoters, thymidine kinase (TK) promoters, and B19 parvovirus promoters. The selection of a suitable promoter will be apparent to those skilled in the art from the teachings contained herein, and may be from among either regulated promoters or promoters as described above. The retroviral plasmid vector is employed to transduce packaging cell lines to form producer cell lines. Examples of packaging cells which may be transfected include, but are not limited to, the PE501, PA317, &#x3c8;-2, &#x3c8;-AM, PA12, T19-14X, VT-19-17-H2, &#x3c8;CRE, &#x3c8;CRIP, GP+E-86, GP+envAm12, and DAN cell lines as described in Miller, Human Gene Therapy, 1:5-14 (1990), which is incorporated herein by reference in its entirety. The vector may transduce the packaging cells through any means known in the art. Such means include, but are not limited to, electroporation, the use of liposomes, and CaPO4 precipitation. In one alternative, the retroviral plasmid vector may be encapsulated into a liposome, or coupled to a lipid, and then administered to a host. The producer cell line generates infectious retroviral vector particles which include the nucleic acid sequence(s) encoding the ANT polypeptides or fusion proteins. Such retroviral vector particles then may be employed, to transduce eukaryotic cells, either in vitro or in vivo. The transduced eukaryotic cells will express the nucleic acid sequence(s) encoding the ANT polypeptide or fusion protein. Eukaryotic cells which may be transduced include, but are not limited to, embryonic stem cells, embryonic carcinoma cells, as well as hematopoietic stem cells, hepatocytes, fibroblasts, myoblasts, keratinocytes, endothelial cells, and bronchial epithelial cells. As another example of an embodiment of the invention in which a viral vector is used to prepare the recombinant ANT expression construct, in one preferred embodiment, host cells transduced by a recombinant viral construct directing the expression of ANT polypeptides or fusion proteins may produce viral particles containing expressed ANT polypeptides or fusion proteins that are derived from portions of a host cell membrane incorporated by the viral particles during viral budding. In another preferred embodiment, ANT encoding nucleic acid sequences are cloned into a baculovirus shuttle vector, which is then recombined with a baculovirus to generate a recombinant baculovirus expression construct that is used to infect, for example, Sf9 or Trichoplusia ni (PharMingen, Inc., San Diego, Calif.) host cells, as described in Baculovirus Expression Protocols, Methods in Molecular Biology Vol. 39, Christopher D. Richardson, Editor, Human Press, Totowa, N.J., 1995; Piwnica-Worms, &#x201c;Expression of Proteins in Insect Cells Using Baculoviral Vectors,&#x201d; Section II in Chapter 16 in: Short Protocols in Molecular Biology, 2nd Ed., Ausubel et al., eds., John Wiley &#x26; Sons, New York, N.Y., 1992, pages 16-32 to 16-48. Host Cells In another aspect, the present invention relates to host cells containing the above described recombinant ANT expression constructs. Host cells are genetically engineered (transduced, transformed or transfected) with the vectors and/or expression constructs of this invention which may be, for example, a cloning vector, a shuttle vector or an expression construct. The vector or construct may be, for example, in the form of a plasmid, a viral particle, a phage, etc. The engineered host cells can be cultured in conventional nutrient media modified as appropriate for activating promoters, selecting transformants or amplifying particular genes such as genes encoding ANT polypeptides or ANT fusion proteins. The culture conditions for particular host cells selected for expression, such as temperature, pH and the like, will be readily apparent to the ordinarily skilled artisan. The host cell can be a higher eukaryotic cell, such as a mammalian cell, or a lower eukaryotic cell, such as a yeast cell, or the host cell can be a prokaryotic cell, such as a bacterial cell. Representative examples of appropriate host cells according to the present invention include, but need not be limited to, bacterial cells, such as E. coli, Streptomyces, Salmonella tvphimurillm; fungal cells, such as yeast; insect cells, such as Drosophila S2, Trichoplusia ni (PharMingen, San Diego, Calif.) and Spodoptera Sf9; animal cells, such as CHO, COS or 293 cells; adenoviruses; plant cells, or any suitable cell already adapted to in vitro propagation or so established de novo. The selection of an appropriate host is deemed to be within the scope of those skilled in the art from the teachings herein. Various mammalian cell culture systems can also be employed to express recombinant protein. Examples of mammalian expression systems include the COS-7 lines of monkey kidney fibroblasts, described by Gluzman, Cell 23:175 (1981), and other cell lines capable of expressing a compatible vector, for example, the C127, 3T3, CHO, HeLa and BHK cell lines. Mammalian expression vectors will comprise an origin of replication, a suitable promoter and enhancer, and also any necessary ribosome binding sites, polyadenylation site, splice donor and acceptor sites, transcriptional termination sequences, and 5&#x2032; flanking nontranscribed sequences, for example as described herein regarding the preparation of ANT expression constructs. DNA sequences derived from the SV40 splice, and polyadenylation sites may be used to provide the required nontranscribed genetic elements. Introduction of the construct into the host cell can be effected by a variety of methods with which those skilled in the art will be familiar, including but not limited to, for example, calcium phosphate transfection, DEAE-Dextran mediated transfection, or electroporation (Davis et al., 1986 Basic Methods in Molecular Biology). As will be aprreciated by those of ordinary skill in the art, in certain situations it may be desirable to prepare the compositions of the invention and to practice the methods of the invention under conditions where endogenous ANT expression by a host cell is compromised, in order to provide advantages associated with the expression of a desired ANT encoding construct. For example, detection of particular ANT encoding nucleic acid sequences or ANT polypeptides that are highly similar to those encoded by the host cell genome may be facilitated by inhibiting host cell ANT gene expression. As another example, where functional activity of an exogenously introduced recombinant ANT polypeptide is to be determined in a host cell or in a biological sample derived therefrom, it may also be advantageous to inhibit endogenous host cell ANT gene expression. Thus, in certain preferred embodiments of the invention, host cells may lack at least one isoform of an endogenous ANT, and in certain preferred embodiments the host cells may lack all endogenous ANT isoforms. For example, in the yeast system described by Giraud et al. (1998 J. Mol. Biol. 281:409) a S. cerevisiae triple null mutant is described that lacks all three yeast ANT isoforms and is unable to grow under anaerobic conditions. In other preferred embodiments, expression in host cells of at least one gene encoding an endogenous ANT isoform is substantially impaired. Substantial impairment of endogenous ANT isoform expression may be achieved by any of a variety of methods that are well known in the art for blocking specific gene expression, including site-specific or site-directed mutagenesis as described above, antisense inhibition of gene expression, ribozyme mediated inhibition of gene expression and generation of mitochondrial DNA depleted (&#x3c1;0) cells. Identification of oligonucleotides and ribozymes for use as antisense agents and DNA encoding genes for targeted delivery for genetic therapy involve methods well known in the art. For example, the desirable properties, lengths and other characteristics of such oligonucleotides are well known. Antisense oligonucleotides are typically designed to resist degradation by endogenous nucleolytic enzymes by using such linkages as: phosphorothioate, methylphosphonate, sulfone, sulfate, ketyl, phosphorodithioate, phosphoraridate, phosphate esters, and other such linkages (see, e.g., Agrwal et al., Tetrehedron Lett. 28:3539-3542 (1987); Miller et al., J. Am. Chem. Soc. 93:6657-6665 (1971); Stec et al., Tetrehedron Lett. 26:2191-2194 (1985); Moody et al., Nucl. Acids Res. 12:4769-4782 (1989); Uznanski et al., Nucl. Acids Res. (1989); Letsinger et al., Tetrahedron 40:137-143 (1984); Eckstein, Annu. Rev. Biochem. 54:367-402 (1985); Eckstein, Trends Biol. Sci. 14:97-100 (1989); Stein In: Oligodeoxynucleotides. Antisense Inhibitors of Gene Expression, Cohen, Ed, Macmillan Press, London, pp. 97-117 (1989); Jager et al., Biochemistry 27:7237-7246 (1988)). Antisense nucleotides are oligonucleotides that bind in a sequence-specific manner to nucleic acids, such as mRNA or DNA. When bound to mRNA that has complementary sequences, antisense prevents translation of the mRNA (see, e.g., U.S. Pat. No. 5,168,053 to Altman et al.; U.S. Pat. No. 5,190,931 to Inouye, U.S. Pat. No. 5,135,917 to Burch; U.S. Pat. No. 5,087,617 to Smith and Clusel et al. (1993) Nucl. Acids Res. 21:3405-3411, which describes dumbbell antisense oligonucleotides). Triplex molecules refer to single DNA strands that bind duplex DNA forming a colinear triplex molecule, thereby preventing transcription (see, e.g., U.S. Pat. No. 5,176,996 to Hogan et al., which describes methods for making synthetic oligonucleotides that bind to target sites on duplex DNA). According to this embodiment of the invention, particularly useful antisense nucleotides and triplex molecules are molecules that are complementary to or bind the sense strand of DNA or mRNA that encodes an ANT polypeptide or a protein mediating any other process related to expression of endogenous ANT genes, such that inhibition of translation of mRNA encoding the ANT polypeptide is effected. A ribozyme is an RNA molecule that specifically cleaves RNA substrates, such as mRNA, resulting in specific inhibition or interference with cellular gene expression. There are at least five known classes of ribozymes involved in the cleavage and/or ligation of RNA chains. Ribozymes can be targeted to any RNA transcript and can catalytically cleave such transcripts (see, e.g., U.S. Pat. No. 5,272,262; U.S. Pat. No. 5,144,019; and U.S. Pat. Nos. 5,168,053, 5,180,818, 5,116,742 and 5,093,246 to Cech et al.). According to certain embodiments of the invention, any such ANT mRNA-specific ribozyme, or a nucleic acid encoding such a ribozyme, may be delivered to a host cell to effect inhibition of ANT gene expression. Ribozymes, and the like may therefore be delivered to the host cells by DNA encoding the ribozyme linked to a eukaryotic promoter, such as a eukaryotic viral promoter, such that upon introduction into the nucleus, the ribozyme will be directly transcribed. As used herein, expression of a gene encoding an endogenous adenine nucleotide translocator isoform is substantially impaired by any of the above methods for inhibiting when cells are substantially but not necessarily completely depleted of functional DNA or functional mRNA encoding the endogenous ANT isoform, or of the relevant ANT polypeptide. ANT isoform expression is substantially impaired when cells are preferably at least 50% depleted of DNA or mRNA encoding the endogenous ANT (as measured using high stringency hybridization as described above) or depleted of ANT polypeptide (as measured by Western immunoblot as described herein, see also, e.g., Giraud et al. 1998 J. Mol Biol. 281:409); and more preferably at least 75% depleted of endogenous ANT DNA, mRNA or polypeptide. Most preferably, ANT isoform expression is substantially impaired when cells are depleted of &#x3e;90% of their endogenous ANT DNA, mRNA, or polypeptide. Alternatively, expression of a gene encoding an endogenous adenine nucleotide translocator isoform may be substantially impaired through the use of mitochondrial DNA depleted &#x3c1;0 cells, which are incapable of mitochondrial replication and so may not continue to express functional ANT polypeptides. Methods for producing &#x3c1;0 cells are known and can be found, for example in PCT/US95/04063, which is hereby incorporated by reference. Protein Production The expressed recombinant ANT polypeptides or fusion proteins may be useful in intact host cells; in intact organelles such as mitochondria, cell membranes, intracellular vesicles other cellular organelles; or in disrupted cell preparations including but not limited to cell homogenates or lysates, submitochondrial particles, uni- and multilamellar membrane vesicles or other preparations. Alternatively, expressed recombinant ANT polypeptides or fusion proteins can be recovered and purified from recombinant cell cultures by methods including ammonium sulfate or ethanol precipitation, acid extraction, anion or cation exchange chromatography, phosphocellulose chromatography, hydrophobic interaction chromatography, affinity chromatography, hydroxylapatite chromatography and lectin chromatography. Protein refolding steps can be used, as necessary, in completing configuration of the mature protein. Finally, high performance liquid chromatography (HPLC) can be employed for final purification steps. The polypeptides of the present invention may be a naturally purified product, or a product of chemical synthetic procedures, or produced by recombinant techniques from a prokaryotic or eukaryotic host (for example, by bacterial, yeast, higher plant, insect and mammalian cells in culture). Depending upon the host employed in a recombinant production procedure, the polypeptides of the present invention may be glycosylated or may be non-glycosylated. Polypeptides of the invention may also include an initial methionine amino acid residue. Samples A &#x201c;biological sample containing mitochondria&#x201d; may comprise any tissue or cell preparation in which intact mitochondria capable of maintaining a membrane potential when supplied with one or more oxidizable substrates such as glucose, malate or galactose are or are thought to be present. Mitochondrial membrane potential may be determined according to methods with which those skilled in the art will be readily familiar, including but not limited to detection and/or measurement of detectable compounds such as fluorescent indicators, optical probes and/or sensitive pH and ion-selective electrodes (See, e.g., Ernster et al., 1981 J. Cell Biol. 91:227s and references cited therein; see also Haugland, 1996 Handbook of Fluorescent Probes and Research Chemicals&#x2014;Sixth Ed., Molecular Probes, Eugene, Oreg., pp. 266-274 and 589-594.). By &#x201c;capable of maintaining a potential&#x201d; it is meant that such mitochondria have a membrane potential that is sufficient to permit the accumulation of a detectable compound (e.g., DASPMI[2-,4-dimethylaminostyryl-N-methylpyridinium], TMRM [tetramethylrhodamine methyl ester], etc.) used in the particular instance. A biological sample containing mitochondria may, for example, be derived from a normal (i.e., healthy) individual or from an individual having a disease associated with altered mitochondrial function. Biological samples containing mitochondria may be provided by obtaining a blood sample, biopsy specimen, tissue explant, organ culture or any other tissue or cell preparation from a subject or a biological source. The subject or biological source may be a human or non-human animal, a primary cell culture or culture adapted cell line including but not limited to genetically engineered cell lines that may contain chromosomally integrated or episomal recombinant nucleic acid sequences, immortalized or immortalizable cell lines, somatic cell hybrid or cytoplasmic hybrid &#x201c;cybrid&#x201d; cell lines, differentiated or differentiatable cell lines, transformed cell lines and the like. A &#x201c;biological sample&#x201d; may comprise any tissue or cell preparation as just described for a biological sample containing mitochondria, but does not require the presence of intact mitochondria. Thus a &#x201c;biological sample&#x201d; may comprise any tissue or cell preparation and a &#x201c;biological sample containing at least one recombinant ANT polypeptide&#x201d; comprises any tissue or cell preparation in which an expressed recombinant ANT polypeptide or fusion protein as provided herein is thought to be present. A biological sample may, for example, be derived from a recombinant cell line or from a transgenic animal. Biological samples containing recombinant ANT may be provided by obtaining a blood sample, biopsy specimen, tissue explant, organ culture or any other tissue or cell preparation from a subject or a biological source. The subject or biological source may be a human or non-human animal, a primary cell culture or culture adapted cell line including but not limited to genetically engineered cell lines that may contain chromosomally integrated or episomal recombinant nucleic acid sequences, immortalized or immortalizable cell lines, somatic cell hybrid or cytoplasmic hybrid &#x201c;cybrid&#x201d; cell lines, differentiated or differentiatable cell lines, transformed cell lines and the like. Proteins As described herein, isolation of a mitochondrial pore component or a mitochondrial molecular species with which an agent identified according to the methods of the invention interacts refers to physical separation of such a complex from its biological source, and may be accomplished by any of a number of well known techniques including but not limited to those described herein, and in the cited references. Without wishing to be bound by theory, a compound that &#x201c;binds a mitochondrial component&#x201d; can be any discrete molecule, agent compound, composition of matter or the like that may, but need not, directly bind to a rnitochondrial molecular component, and may in the alternative bind indirectly to a mitochondrial molecular component by interacting with one or more additional components that bind to a mitochondrial molecular component. These or other mechanisms by which a compound may bind to and/or associate with a mitochondrial molecular component are within the scope of the claimed methods, so long as isolating a mitochondrial pore component also results in isolation of the mitochondrial molecular species that directly or indirectly binds to the identified agent. Thus, for example, as provided herein, any ANT polypeptide including recombinant ANT polypeptides and fusion proteins may be a mitochondrial molecular component and/or a mitochondrial pore component, and any ANT ligand or agent that binds to an ANT polypeptide may be a compound that binds a mitochondrial component and/or an agent that affects mitochondrial pore activity. As described herein, the mitochondrial permeability transition &#x201c;pore&#x201d; may not be a discrete assembly or multisubunit complex, and the term thus refers instead to any mitochondrial molecular component (including, e.g. a mitochondrial membrane per se) that regulates the inner membrane selective permeability where such regulated function is impaired during MPT. As used herein, mitochondria are comprised of &#x201c;mitochondrial molecular components&#x201d;, which may be any protein, polypeptide, peptide, amino acid, or derivative thereof; any lipid, fatty acid or the like, or derivative thereof; any carbohydrate, saccharide or the like or derivative thereof, any nucleic acid, nucleotide, nucleoside, purine, pyrimidine or related molecule, or derivative thereof, or the like; or any other biological molecule that is a constituent of a mitochondrion. &#x201c;Mitochondrial molecular components&#x201d; includes but is not limited to &#x201c;mitochondrial pore components&#x201d;. A &#x201c;mitochondrial pore component&#x201d; is any mitochondrial molecular component that regulates the selective permeability characteristic of mitochondrial membranes as described above, including those responsible for establishing &#x394;&#x3a8;m and those that are functionally altered during MPT. Isolation and, optionally, identification and/or characterization of the mitochondrial pore component or components with which an agent that affects mitochondrial pore activity interacts may also be desirable and are within the scope of the invention. Once an agent is shown to alter MPT according to the methods provided herein and in U.S. Ser. No. 09/161,172, those having ordinary skill in the art will be familiar with a variety of approaches that may be routinely employed to isolate the molecular species specifically recognized by such an agent and involved in regulation of MPT, where to &#x201c;isolate&#x201d; as used herein refers to separation of such molecular species from the natural biological environment. Thus, for example, once an ANT ligand is prepared according to the methods provided herein, such approaches may be routinely employed to isolate the ANT polypeptide. Techniques for isolating a mitochondrial pore component such as an ANT polypeptide or fusion protein may include any biological and/or biochemical methods useful for separating the complex from its biological source, and subsequent characterization may be performed according to standard biochemical and molecular biology procedures. Those familiar with the art will be able to select an appropriate method depending on the biological starting material and other factors. Such methods may include, but need not be limited to, radiolabeling or otherwise detectably labeling cellular and mitochondrial components in a biological sample, cell fractionation, density sedimentation, differential extraction, salt precipitation, ultrafiltration, gel filtration, ion-exchange chromatography, partition chromatography, hydrophobic chromatography, electrophoresis, affinity techniques or any other suitable separation method that can be adapted for use with the agent with which the mitochondrial pore component interacts. Antibodies to partially purified components may be developed according to methods known in the art and may be used to detect and/or to isolate such components. Affinity techniques may be particularly useful in the context of the present invention, and may include any method that exploits a specific binding interaction between a mitochondrial pore component and an agent identified according to the invention that interacts with the pore component. For example, because ANT ligands as provided herein and other agents that influence MPT can be immobilized on solid phase matrices, an affinity binding technique for isolation of the pore component may be particularly useful. Alternatively, affinity labeling methods for biological molecules, in which a known MPT-active agent or a novel ANT ligand as provided herein may be modified with a reactive moiety, are well known and can be readily adapted to the interaction between the agent and a pore component, for purposes of introducing into the pore component a detectable and/or recoverable labeling moiety. (See, e.g., Pierce Catalog and Handbook, 1994 Pierce Chemical Company, Rockford, Ill.; Scopes, R. K., Protein Purification: Principles and Practice, 1987, Springer-Verlag, New York; and Hermanson, G. T. et al., Immobilized Affinity Ligand Techniques, 1992, Academic Press, Inc., California; for details regarding techniques for isolating and characterizing biological molecules, including affinity techniques. Characterization of mitochondrial pore component molecular species, isolated by MPT-active agent affinity techniques described above or by other biochemical methods, may be accomplished using physicochemical properties of the pore component such as spectrometric absorbance, molecular size and/or charge, solubility, peptide mapping, sequence analysis and the like. (See, e.g., Scopes, supra.) Additional separation steps for biomolecules may be optionally employed to further separate and identify molecular species that co-purify with mitochondrial pore components. These are well known in the art and may include any separation methodology for the isolation of proteins, lipids, nucleic acids or carbohydrates, typically based on physicochemical properties of the newly identified components of the complex. Examples of such methods include RP-HPLC, ion exchange chromatography, hydrophobic interaction chromatography, hydroxyapatite chromatography, native and/or denaturing one- and two-dimensional electrophoresis, ultrafiltration, capillary electrophoresis, substrate affinity chromatography, immunoaffinity chromatography, partition chromatography or any other useful separation method. Preferably extracts of cultured cells, and in particularly preferred embodiments extracts of biological tissues or organs may be sources of mitochondrial molecular components, including ANT polypeptides. Preferred sources may include blood, brain, fibroblasts, myoblasts, liver cells or other cell types. ANT Ligands As noted above, the binding of the adenine nucleotide translocator (ANT) is responsible for mediating transport of ADP and ATP across the mitochondrial inner membrane. ANT has also been implicated as the critical component of the mitochondrial permeability transition pore, a Ca2+ regulated inner membrane channel that plays an important modulating role in apoptotic processes. Additionally, ANT activity appears to be related to changes in ANT polypeptide conformation within the mitochondrial membrane, as evidenced by studies using agents that are capable of binding to ANT. (Block et al., 1986 Meths. Enzymol. 125:658) Accordingly, it is another aspect of the present invention to provide compositions and methods for producing and identifying agents that bind to ANT, which agents are also referred to herein as ANT ligands. Binding interactions between ANT and a variety of small molecules are known to those familiar with the art. For example, these interactions include binding to ANT by atractyloside, carboxyatractyloside, palmitoyl-CoA, bongkrekic acid, thyroxin, eosin Y and erythrosin B. (See, e.g., Stubbs, 1979 Pharm. Ther. 7:329; Klingenberg et al., 1978 Biochim. Biophys. Acta 503:193; Sterling, 1986 Endocrinol. 119:292; Majima et al., 1998 Biochem. 37:424; Block et al. 1986 Meths. Enzymol. 125:658; for erythrosin B and additional ANT inhibitors, see Beavis et al. 1993 J. Biol. Chem. 268:997; Powers et al. 1994 J. Biol. Chem. 269:10614.) The ANT ligands of the present invention represent novel atractyloside derivatives. Atractyloside (ATR) and its known derivatives, including carboxyatractyloside (CATR), naphthoyl-ATR, MANT-ATR and other AIR derivatives (see, e.g., Boulay et al., Analytical Biochemistry 128:323-330,1983; Roux et al., Analytical Biochemistry 234:31-37,1996; Lauquin et al., FEBS Letters 67:306-311,1976; and Gottikh et al., Tetrahedron 26:4419-4433, 1970; for other known ATR derivatives see, e.g., Block et al., 1986 Meths. Enzymol. 125:658) have proven invaluable in the elucidation of the structure and the mechanism of action of the adenine nucleotide translocator. According to the ANT ligands of the invention, the binding mode of ATR to ANT allows for modifications of the ATR 6&#x2032;-hydroxyl functionality without significantly altering ATR binding affinity for ANT. Thus, ANT ligands as provided herein may be ATR derivatives modified by chemical substitution at the 6&#x2032; hydroxyl position. In particular, the novel ANT ligands as provided herein further include long linker moieties at the 6&#x2032; position, which linkers may include a 6&#x2032;-amine linker, thereby permitting additional chemical modification to the ANT ligand as will be appreciated by those skilled in the art and as illustrated in the non-limiting Examples. Also, as shown in Examples 6-11, such linkers as provided herein may have carbon chain backbones of 2-20 carbon atoms, and in preferred embodiments 2-6 carbon atoms. The invention therefore provides ANT ligands that may be intermediates for conjugation to a variety of additional chemical moieties to yield further ATR derivatives that are ANT ligands within the scope of the invention. These include ANT ligands to which 125I may be covalently attached under mild reaction conditions; the invention also includes ANT ligands to which reactive amine groups may be covalently linked. ANT ligands which are such amine-containing ATR derivatives may then be reacted with a variety of fluorophores and haptens bearing, for example, reactive isothiocyanate, N-hydroxysuccinimide ester, anhydride and other useful functionalities to yield stable ATR derivatives including, for example, derivatives that have thiourea, amide or other linkages. Thus, ANT ligands as provided herein also include ATR derivatives that are detectable by virtue of substituents introduced at the 6&#x2032; position. Accordingly, detectable ATR derivatives as herein provided include ATR derivatives having a 6&#x2032; hydroxyl substitution that includes a radiolabeled substituent, for example 125I, 131I, 3H, 14C or 35S. Other ANT ligands that are detectable ATR derivatives may comprise fluorescent substituents, including those appropriately tagged with reporter molecules such as fluorophores and haptens having utility in high throughput screening assays for identifying agents that bind to ANT. More specifically, in preferred embodiments, an ANT ligand according to the present invention that includes fluorescent substituents has an extinction coefficient &#x3e;10,000 M&#x2212;1 (see Table 1); further, this property provides an advantage for using such ANT ligands according to the methods provided herein, and in particular for use in high throughput screening assays. Additionally, the ANT ligands of the invention exhibit high affinities for ANT, and in preferred embodiments have binding constants in the nanomolar range. In certain embodiments of the invention, ANT ligands may be ATR derivatives such as ATR-lanthanide chelating agents, which have utility in time-resolved fluorescence detection, for example detection of these compounds complexed to a lanthanide ion such as Eu3+, Th3+, Sm3+ and Dy3+. In addition, ANT ligands may comprise ATR conjugated to readily detectable substituents such as highly fluorescent moieties, for example by way of illustration and not limitation, cyanine and coumarin derivatives. These and other highly fluorescent substituents permit the synthesis, according to the methods of the invention, of ANT ligands that are detectable with extremely high sensitivities. Those familiar with the art are aware of additional fluorescent substituents that may be used, for example, those disclosed in Haugland, 1996 Handbook of Fluorescent Probes and Research Chemicals&#x2014;Sixth Ed., Molecular Probes, Eugene, Oreg. In other embodiments, the invention provides detectable ANT ligands produced by coupling of biotin-NHS ester with the ATR derivatives as disclosed herein; these and other ANT ligands similarly generated according to the instant methods can be detected with commercially available enzyme-avidin conjugates using, for example, calorimetric, fluorescent or chemiluminescent techniques. In one embodiment, the ANT ligands of this invention have the following structure(I): including stereoisomers and pharmaceutically acceptable salts thereof, wherein R1 is hydroxyl, halogen, &#x2014;OC(&#x2014;O)R4 or &#x2014;NHR4; R2 is hydrogen or &#x2014;C(&#x2550;O)R5; R3 is &#x2014;CH3 or &#x2550;CH2; R4 is &#x2014;X-aryl, &#x2014;X-substituted aryl, &#x2014;X-arylalkyl, &#x2014;X-substituted arylalkyl, X-heteroaryl, or &#x2014;X-heteroarylalkyl, wherein X is an optional arido or alylamido linker moiety; and R5 is alkyl. As used herein, the above terms have the meanings set forth below. &#x201c;Amido&#x201d; means &#x2014;NHC(&#x2550;O)&#x2014; or&#x2014;C(&#x2550;O)NH&#x2014;. &#x201c;Alkylamido&#x201d; means -(alkyl)-NHC(&#x2014;O)&#x2014; or -(alkyl)-C(&#x2550;O)NH&#x2014;, such as &#x2014;CH2NHC(&#x2550;O)&#x2014;, &#x2014;CH2CH2NHC(&#x2550;O)&#x2014;, &#x2014;CH2C(&#x2550;O)NH&#x2014;, &#x2014;CH2CH2C(&#x2550;O)NH&#x2014;, and the like. &#x201c;Alkyl&#x201d; means a straight chain or branched, noncyclic or cyclic, unsaturated or saturated aliphatic hydrocarbon containing from 1 to 8 carbon atoms. Representative saturated straight chain alkyls include methyl, ethyl, n-propyl, n-butyl, n-pentyl, n-hexyl, and the like; while saturated branched alkyls include isopropyl, sec-butyl, isobutyl, tert-butyl, isopentyl, and the like. Representative saturated cyclic alkyls include cyclopropyl, cyclobutyl, cyclopentyl, cyclohexyl, and the like; while unsaturated cyclic alkyls include cyclopentenyl and cyclohexenyl, and the like. Unsaturated alkyls contain at least one double or triple bond between adjacent carbon atoms (referred to as an &#x201c;alkenyl&#x201d; or &#x201c;alkynyl&#x201d;, respectively). Representative straight chain and branched alkenyls include ethylenyl, propylenyl, 1-butenyl, 2-butenyl, isobutylenyl, 1-pentenyl, 2-pentenyl, 3-methyl-1-butenyl, 2-methyl-2-butenyl, 2,3-dimethyl-2-butenyl, and the like; while representative straight chain and branched alkynyls include acetylenyl, propynyl, 1-butynyl, 2-butynyl, 1-pentynyl, 2-pentynyl, 3-methyl-1 butynyl, and the like. &#x201c;Aryl&#x201d; means an aromatic carbocyclic moiety such as phenyl or naphthyl (i.e., 1- or 2-naphthyl). &#x201c;Arylalkyl&#x201d; means an alkyl having at least one alkyl hydrogen atoms replaced with an aryl moiety, such as benzyl, &#x2014;(CH2)2phenyl, &#x2014;(CH2)3phenyl, and the like. &#x201c;Heteroaryl&#x201d; means an aromatic heterocycle ring of 5- to 10 members and having at least one heteroatom selected from nitrogen, oxygen and sulfur, and containing at least 1 carbon atom, including both mono- and bicyclic ring systems. Representative heteroaryls are pyridyl, furyl, benzofuranyl, thiophenyl, benzothiophenyl, quinolinyl, pyrrolyl, indolyl, oxazolyl, benzoxazolyl, imidazolyl, benzimidazolyl, thiazolyl, benzothiazolyl, isoxazolyl, pyrazolyl, isothiazolyl, pyridazinyl, pyrimidinyl, pyrazinyl, triazinyl, cinnolinyl, phthalazinyl, and quinazolinyl. &#x201c;Heteroarylalkyl&#x201d; means an alkyl having at least one alkyl hydrogen atom replaced with a heteroaryl moiety, such as &#x2014;CH2pyridinyl, &#x2014;CH2pyrimidinyl, and the like. &#x201c;Heterocycle&#x201d; means a 5- to 7-membered monocyclic, or 7- to 10-membered bicyclic, heterocyclic ring which is either saturated, unsaturated, or aromatic, and which contains from 1 to 4 heteroatoms independently selected from nitrogen, oxygen and sulfur, and wherein the nitrogen and sulfur heteroatoms may be optionally oxidized, and the nitrogen heteroatom may be optionally quarternized, including bicyclic rings in which any of the above heterocycles are fused to a benzene ring. The heterocycle may be attached via any heteroatom or carbon atom. Heterocycles include heteroaryls as defined above. Thus, in addition to the heteroaryls listed above, heterocycles also include morpholinyl, pyrrolidinonyl, pyrrolidinyl, piperidinyl, hydantoinyl, valerolactamyl, oxiranyl, oxetanyl, tetrahydrofuranyl, tetrahydropyranyl, tetrahydropyridinyl, tetrahydroprimidinyl, tetrahydrothiophenyl, tetrahydrothiopyranyl, tetrahydropyrimidinyl, tetrahydrothiophenyl, tetrahydrothiopyranyl, and the like. &#x201c;Heterocyclealkyl&#x201d; means an alkyl having at least one alkyl hydrogen atom replaced with a heterocycle, such as &#x2014;CH2morpholinyl, and the like. The term &#x201c;substituted&#x201d; as used herein means any of the above groups (i.e., alkyl, aryl, arylalkyl, heteroaryl, heteroarylalkyl, heterocycle and heterocyclealkyl) wherein at least one hydrogen atom is replaced with a substituent. In the case of a keto substituent (&#x201c;C(&#x2550;O)&#x201d;) two hydrogen atoms are replaced. Substituents include halogen, hydroxy, alkyl, haloalkyl, aryl, substituted aryl, arylalkyl, substituted arylalkyl, heterocycle, substitued heterocycle, heterocyclealkyl or substituted heterocyclealkyl. &#x201c;Halogen&#x201d; means fluoro, chloro, bromo and iodo. &#x201c;Haloalkyl&#x201d; means an alkyl having at least one hydrogen atom replaced with halogen, such as trifluoromethyl and the like. &#x201c;Alkoxy&#x201d; means an alkyl moiety attached through an oxygen bridge (i.e., &#x2014;O-alkyl) such as methoxy, ethoxy, and the like. In one embodiment, R2 is &#x2014;C(&#x2550;O)CH2CH(CH3)2, R3 is &#x2550;CH2, and the ANT ligand is an atractyloside derivative having the following structure (II): wherein R1 is as defined above. In another embodiment, R2 is &#x2014;C(&#x2550;O)CH2CH(CH3)2, R3 is &#x2014;CH3, and the ANT ligand is a dihydro-atractyloside derivative having the following structure (III): wherein R1 is as defined above. In still a further embodiment, R2 is &#x2014;OH, R3 is &#x2550;CH2, and the ANT ligand is an apoatractyloside derivative having the following structure (IV): wherein R1 is as defined above. In more specific embodiments of structures (II), (III) and (IV), R1 is &#x2014;OC(&#x2550;O)(aryl), &#x2014;OC(&#x2550;O)(substituted aryl), &#x2014;OC(&#x2550;O)(arylalkyl), &#x2014;OC(&#x2550;O)(substituted arylalkyl), &#x2014;NH(CH2)2NHC(&#x2550;O)(arylalkyl), &#x2014;NH(CH2)2NHC(&#x2550;O)(substituted arylalkyl). Representative R1 moieties in this regard include &#x2014;OC(&#x2550;O)(phenyl), &#x2014;OC(&#x2550;O)(1-naphthyl), &#x2014;OC(&#x2550;O)(substituted phenyl), &#x2014;OC(&#x2550;O)(substituted 1-naphthyl), &#x2014;OC(&#x2550;O)(CH2)1-3(phenyl), &#x2014;OC(&#x2550;O)(CH2)1-3(substituted phenyl), &#x2014;NH(CH2)2NHC(&#x2550;O)(CH2)1-3(phenyl), &#x2014;NH(CH2)2NHC(&#x2550;O)(CH2)1-3(substituted phenyl). In this context, representative substituted phenyl moieties include (but are not limited to) 4-hydroxyphenyl, 3-iodo-4-hydroxyphenyl, 3,5-iodo-4-hydroxyphenyl, 4-(4-hydoxyphenyl)phenyl, 4-(3-iodo-4-hyroxyphenyl)phenyl, 3-methyl-4-hyroxyphenyl, and 3-methyl-4-hydroxy-5-iodophenyl. The ANT ligands of structure (I) may readily be made by one skilled in the art of organic chemisty and, more particularly, by the techniques disclosed in Examples 6-11. The compounds of the present invention may generally be utilized as the free base. Alternatively, the compounds of this invention may be used in the form of acid addition salts. Acid addition salts of the free amino compounds of the present invention may be prepared by methods well known in the art, and may be formed from organic and inorganic acids. Suitable organic acids include maleic, fumaric, benzoic, ascorbic, succinic, methanesulfonic, acetic, oxalic, propionic, tartaric, salicylic, citric, gluconic, lactic, mandelic, cinnamic, aspartic, stearic, palmitic, glycolic, glutamic, and benzenesulfonic acids. Suitable inorganic acids include hydrochloric, hydrobromic, sulfuric, phosphoric, and nitric acids. Thus, the term &#x201c;pharmaceutically acceptable salt&#x201d; of structure (I) is intended to encompass any and all acceptable salt forms. With regard to stereoisomers, the compounds of structure (I) may have chiral centers and may occur as recemates, reacemic mixtures and as individual enantiomers or diastereomers. All such isomeric forms are included within the present invention, including mixtures thereof. Furthermore, some of the crystalline forms of the compounds of structure (I) may exist as polymorphs, which are included in the present invention. In addition, some of the compounds of structure (I) may also form solvates with water or other organic solvents. Such solvates are similarly included within the scope of this invention. Activities of ANT ligands are typically calculated from the IC50 as the concentration of a compound necessary to displace 50% of the detectable (i.e., detectably labeled, for example, radiolabeled) ligand from ANT molecules, which may be present as isolated or purified polypeptides or as components of preparations containing isolated mitochondria or submitochondrial particles (SMP) using established ligand binding assays or modifications thereof. For example, ANT ligands may be tested for their ability to compete with radiolabeled ATR, or with a radiolabeled ATR derivative such as compound 24 as provided herein, for binding to isolated ANT polypeptides or to ANT present in isolated mitochondria or SMP. As another example, the relative affinities for ANT of various ANT ligands as provided herein may be determined by a fluorescence assay that exploits the flourescent properties of compound 22 (Example 11), a naphthoyl-ATR derivative that is an ANT ligand having a fluorescence excitation peak at 312 nm and an emission peak at 400 nm. When compound 22 is bound to ANT, the fluorescence is quenched. When, however, compound 22 is displaced from ANT by a known concentration of ATR or an ATR derivative that is an ANT ligand, fluorescence dequenching that results from displacement of the fluorophore can be measured in real time. Briefly, a mitochondrial preparation (see, e.g., Example 13) is washed and resuspended in a suitable buffer in the presence of compound 22 (e.g., 10 mM Tris-120 mM KCl containing 3.6 nmoles of compound 22 per mg mitochondrial protein, 10 min at room temperature), washed to remove unbound fluorophore and placed in a fluorometer equipped with a light source and filter set appropriate for the fluorophore. Fluorescence intensity is monitored as a function of time, and a candidate ANT ligand is then added to determine its ability to compete with compound 22 for binding to ANT, as evidenced by a change in detectable relative fluorescence intensity units. After the fluorescence signal has stabilized, any additional compound 22 that remains bound to ANT is displaced by adding an excess (e.g., &#x3bc;M quantities) of ATR as a competitive inhibitor, to determine maximal signal intensity and therefrom calculate the proportion of compound 22 displaced by the candidate ANT ligand. Those having familiarity with the art will appreciate that variations and modifications may be made to ANT-binding assays such as those illustrated above and described in the Examples for determing IC50 values of candidate ANT ligands, and which are not intended to be limiting. Activity of each ANT ligand is reported as a &#x201c;Ki&#x201d; value calculated by the following equation: K i = IC 50 1 + L / K D where L=radioligand and KD=affinity of radioligand for receptor (Cheng and Prusoff, Biochem. Pharmacol. 22:3099, 1973). ANT ligands of this invention have a Ki of 100 &#x3bc;M or less. In a preferred embodiment of this invention, the ANT ligands have a Ki of less than 10 &#x3bc;M, and more preferably less than 1 &#x3bc;M. To this end, ANT ligands of this invention having a Ki of less than 100 &#x3bc;M include compound 5 (Example 7), compound 6 (Example 8), and compounds 22, 23, 24, 26, 29, 33, 35, 37, and 38 (Example 11). Preferred ANT ligands having a Ki of less than 10 &#x3bc;M include compounds 6, 22, 23, 24, 29, 33, 35, and 38, and more preferred ANT ligands having a Ki of less than 1 &#x3bc;M include compounds 6, 24, 33, and 38, as well as ATR. Assays It is another aspect of the invention to provide compositions and methods for the determination of the presence of ANT polypeptides and for the identification of agents that bind to, or that interact with, ANT polypeptides. Such compositions and methods will be useful for diagnostic and prognostic purposes, for example in the determination of the existence of altered mitochondrial function which, as described above, may accompany both normal and disease states. These compositions and methods will also be useful for the identification of agents that alter or regulate mitochondrial function based on ANT roles in mitochondrial activities, for example by way of illustration and not limitation, maintenance of mitochondrial membrane potential, ATP biosynthesis, induction of apoptosis, MPT and other mitochondrial function. In certain preferred embodiments these compositions and methods are useful as high throughput screening assays. In certain aspects the invention provides a method for determining the presence of an ANT polypeptide in a biological sample, comprising contacting a sample suspected of containing an ANT polypeptide with an ANT ligand under conditions and for a time sufficient to allow binding of the ANT ligand to an ANT polypeptide, and detecting such binding. &#x201c;ANT ligands&#x201d; according to these aspects of the invention may include any novel ANT ligands as provided herein. The use of human ANT1, ANT2 and ANT3 according to these methods represent particularly preferred embodiments. Other preferred embodiments include the use of any ANT polypeptide or ANT fusion protein as provided herein. Accordingly, the instant method for determining the presence of ANT polypeptide in a sample will be useful for monitoring expression of ANT encoding constructs provided herein. In some preferred embodiments an ANT fusion protein is used that is a GST fusion protein, and in other preferred embodiments the ANT fusion protein is a His-tagged fusion protein. As provided herein, the biological sample may be a cell, a mitochondrion, submitochondrial particles, a cell membrane (including any cellular membrane as described herein), a cell extract, cell conditioned medium, a tissue homogenate or an isolated ANT. In other aspects, the invention provides a method for identifying an agent that binds to an ANT polypeptide, comprising contacting a candidate agent with a host cell expressing at least one recombinant ANT polypeptide under conditions and for a time sufficient to permit binding of the agent to the ANT polypeptide and detecting such binding. In various preferred embodiments the host cell may be a prokaryotic cell or a eukaryotic cell. In certain other preferred embodiments the host cell may lack at least one isoform of an endogenous ANT, for example, due to a mutation in one or more endogenous ANT encoding genes. In certain other embodiments host cell expression of at least one gene encoding an endogenous ANT isoform is substantially impaired, for example, through the use of ANT nucleic acid-specific ribozyme or antisense constructs as provided herein, or through the use of &#x3c1;0 cells, as also provided herein. According to other embodiments of this aspect of the invention, it may be preferred to use intact cells or, alternatively, to use permeabilized cells. Those having ordinary skill in the art are familiar with methods for permeabilizing cells, for example by way of illustration and not limitation, through the use of surfactants, detergents, phospholipids, phospholipid binding proteins, enzymes, viral membrane fusion proteins and the like; through the use of osmotically active agents; by using chemical crosslinking agents; by physicochernical methods including electroporation and the like, or by other permeabilizing methodologies. In other aspects, the invention provides a method for identifying an agent that binds to an ANT polypeptide comprising contacting a candidate agent with a biological sample containing at least one recombinant ANT polypeptide under conditions and for a time sufficient to permit binding of the agent to the ANT polypeptide, and detecting such binding. The use of human ANT1, ANT2 and ANT3 according to these methods represent particularly preferred embodiments. Other preferred embodiments include the use of any ANT polypeptide or ANT fusion protein as provided herein. In some preferred embodiments an ANT fission protein is used that is a GST fusion protein, and in other preferred embodiments the ANT fusion protein is a His-tagged fusion protein. As provided herein, the biological sample may be a cell, a mitochondrion, submitochondrial particles, a cell membrane (including any cellular membrane as described herein), a cell extract, cell conditioned medium, a recombinant viral particle, a tissue homogenate or an isolated ANT. Detection of binding may be by any of a variety of methods and will depend on the nature of the candidate agent being screened. For example, certain candidate agents are inherently detectable as a consequence of their physicochemical properties, such as will be apparent to those skilled in the art and including spectrophotometric, colorimetric, fluorimetric, solubility, hydrophobic, hydrophilic, electrostatic charge, molecular mass or other physicochemical properties. As another example, certain candidate agents may be radioactively labeled with a readily detectable radionuclide, as is well known in the art. Certain candidate agents may also be directly or indirectly detectable by ANT protein affinity methodologies, for example by their ability to interfere with binding of an ANT-specific antibody to an ANT; or by their being removable from an assay solution using a protein affinity reagent that binds to a fusion polypeptide present as a portion of an ANT fusion protein. A candidate agent bound to an ANT polypeptide may be detected by any method known for the detection, identification or characterization of relevant molecules, including spectrophotometric, mass spectrometric, chromatographic, electrophoretic, calorimetric or any other suitable analytical technique. In another aspect the invention provides a method for identifying an agent that interacts with an ANT polypeptide comprising contacting a biological sample containing recombinant ANT with a detectable ANT ligand (or a known detectable molecule capable of binding to ANT) in the presence of a candidate agent, and comparing binding of the detectable ANT ligand (or known detectable ANT binding molecule) to recombinant ANT in the absence of the agent to binding of the detectable ANT ligand (or known detectable ANT binding molecule) to recombinant ANT in the presence of the agent, and therefrom identifying an agent that interacts with an ANT polypeptide. It will be appreciated that in certain preferred embodiments this aspect provides competitive binding assays wherein novel ANT ligands as provided hereinabove are useful. However, this aspect of the invention need not be so limited and may be modified to employ known detectable ANT binding molecules, in which case it should be pointed out that the selection of biological sample and/or of recombinant ANT as provided by the present invention offer unexpected advantages heretofore unknown in the art. Examples of known detectable ANT-binding molecules include suitably labeled ATP, ADP, ATR, CATR, palmitoyl-CoA, bongkrekic acid, thyroxin, eosin Y and erythrosin B or other ANT-binding molecules known in the art. (See, e.g., Block et al., 1986 Meths. Enzymol 125:658.) The use of human ANT1, ANT2 and ANT3 according to these methods represent particularly preferred embodiments. Other preferred embodiments include the use of any ANT polypeptide or ANT fusion protein as provided herein. In some preferred embodiments an ANT fusion protein is used that is a GST fusion protein, and in other preferred embodiments the ANT fusion protein is a His-tagged fusion protein. As provided herein, the biological sample may be a cell, a mitochondrion, submitochondrial particles, a cell membrane (including any cellular membrane as described herein), a cell extract, cell conditioned medium, a recombinant viral particle, a tissue homogenate or an isolated ANT. The ANT ligands compounds are preferably part of a pharmaceutical composition when used in the methods of the present invention. The pharmaceutical composition will include at least one of a pharmaceutically acceptable carrier, diluent or excipient, in addition to one or more ANT ligands and, optionally, other components. &#x201c;Pharmaceutically acceptable carriers&#x201d; for therapeutic use are well known in the pharmaceutical art, and are described, for example, in Remingtons Pharmaceutical Sciences, Mack Publishing Co. (A. R. Gennaro edit. 1985). For example, sterile saline and phosphate-buffered saline at physiological pH may be used. Preservatives, stabilizers, dyes and even flavoring agents may be provided in the pharmaceutical composition. For example, sodium benzoate, sorbic acid and esters of p-hydroxybenzoic acid may be added as preservatives. Id. at 1449. In addition, antioxidants and suspending agents may be used. Id. &#x201c;Pharmaceutically acceptable salt&#x201d; refers to salts of the compounds of the present invention derived from the combination of such compounds and an organic or inorganic acid (acid addition salts) or an organic or inorganic base (base addition salts). The compounds of the present invention may be used in either the free base or salt forms, with both forms being considered as being within the scope of the present invention. The pharmaceutical compositions that contain one or more ANT substrates/ligands compounds may be in any form which allows for the composition to be administered to a patient. For example, the composition may be in the form of a solid, liquid or gas (aerosol). Typical routes of administration include, without limitation, oral, topical, parenteral (e.g., sublingually or buccally), sublingual, rectal, vaginal, and intranasal. The term parenteral as used herein includes subcutaneous injections, intravenous, intramuscular, intrasternal, intracavernous, intrameatal, intraurethral injection or infusion techniques. The pharmaceutical composition is formulated so as to allow the active ingredients contained therein to be bioavailable upon administration of the composition to a patient. Compositions that will be administered to a patient take the form of one or more dosage units, where for example, a tablet may be a single dosage unit, and a container of one or more compounds of the invention in aerosol form may hold a plurality of dosage units. For oral administration, an excipient and/or binder may be present. Examples are sucrose, kaolin, glycerin, starch dextrins, sodium alginate, carboxymethylcellulose and ethyl cellulose. Coloring and/or flavoring agents may be present. A coating shell may be employed. The composition may be in the form of a liquid, e.g., an elixir, syrup, solution, emulsion or suspension. The liquid may be for oral administration or for delivery by injection, as two examples. When intended for oral administration, preferred composition contain, in addition to one or more ANT substrates/ligands compounds, one or more of a sweetening agent, preservatives, dye/colorant and flavor enhancer. In a composition intended to be administered by injection, one or more of a surfactant, preservative, wetting agent, dispersing agent, suspending agent, buffer, stabilizer and isotonic agent may be included. A liquid pharmaceutical composition as used herein, whether in the form of a solution, suspension or other like form, may include one or more of the following adjuvants: sterile diluents such as water for injection, saline solution, preferably physiological saline, Ringer's solution, isotonic sodium chloride, fixed oils such as synthetic mono or digylcerides which may serve as the solvent or suspending medium, polyethylene glycols, glycerin, propylene glycol or other solvents; antibacterial agents such as benzyl alcohol or methyl paraben; antioxidants such as ascorbic acid or sodium bisulfite; chelating agents such as ethylenediarinetetraacetic acid; buffers such as acetates, citrates or phosphates and agents for the adjustment of tonicity such as sodium chloride or dextrose. The parenteral preparation can be enclosed in ampoules, disposable syringes or multiple dose vials made of glass or plastic. Physiological saline is a preferred adjuvant. An injectable pharmaceutical composition is preferably sterile. A liquid composition intended for either parenteral or oral administration should contain an amount of ANT substrates/ligands compound such that a suitable dosage will be obtained. Typically, this amount is at least 0.01 wt % of an ANT substrates/ligands compound in the composition. When intended for oral administration, this amount may be varied to be between 0.1 and about 70% of the weight of the composition. Preferred oral compositions contain between about 4% and about 50% of ANT substrates/ligands compound(s). Preferred compositions and preparations are prepared so that a parenteral dosage unit contains between 0.01 to 1% by weight of active compound. The pharmaceutical composition may be intended for topical administration, in which case the carrier may suitably comprise a solution, emulsion, ointment or gel base. The base, for example, may comprise one or more of the following: petrolatum, lanolin, polyethylene glycols, beeswax, mineral oil, diluents such as water and alcohol, and emulsifiers and stabilizers. Thickening agents may be present in a pharmaceutical composition for topical administration. If intended for transdermal administration, the composition may include a transdermal patch or iontophoresis device. Topical formulations may contain a concentration of the ANT substrates/ligands compound of from about 0.1 to about 10% w/v (weight per unit volume). The composition may be intended for rectal administration, in the form, e.g., of a suppository which will melt in the rectum and release the drug. The composition for rectal administration may contain an oleaginous base as a suitable nonirritating excipient. Such bases include, without limitation, lanolin, cocoa butter and polyethylene glycol. In the methods of the invention, the ANT substrates/ligands compound(s) may be administered through use of insert(s), bead(s), timed-release formulation(s), patch(es) or fast-release formulation(s). It will be evident to those of ordinary skill in the art that the optimal dosage of the ANT substrates/ligands compound(s) may depend on the weight and physical condition of the patient; on the severity and longevity of the physical condition being treated; on the particular form of the active ingredient, the manner of administration and the composition employed. It is to be understood that use of an ANT substrates/ligands compound in a chemotherapy can involve such a compound being bound to an agent, for example, a monoclonal or polyclonal antibody, a protein or a liposome, which assist the delivery of said compound. EXAMPLES The following Examples are offered by way of illustration and not by way of limitation. Example 1 Cloning and Expression of His-Tagged Human ANT Proteins in Bacteria A. PCR Amplification of ANT cDNAs Total cellular RNA prepared from whole human brain was obtained from a commercial source (Clontech, Palo Alto, Calif.). The RNA was purified by treatment with RNase-free DNase I (Roche Molecular Biochemicals, formerly Boehringer Mannheim Biochemicals, Indianapolis, Ind.) using 1 ul of DNase I (10 u/ul) in a buffer containing 40 mM Trsi-HCI, pH 7.0, 6 mM magnesium chloride and 2 mM calcium chloride for 30 minutes at 37&#xb0; C. This treatment was followed by two phenoUchloroform extractions, one chloroform extraction and an ethanol precipitation in the presence of sodium acetate. The RNA pellet was collected by centrifugation, washed with 70% ethanol, air dried, and resuspended in RNase-free sterile water. The RNA was reverse transcribed to generate cDNA using RNase H-deficient Reverse Transcriptase (SUPERSCRIPT&#x2122;; Life Technologies, Rockville, Md.). ANT cDNAs were amplified by polymerase chain reactions (PCR) in a thermal cycler using the following primers, AMPLITAQ&#x2122; DNA Polymerase (Perkin-Elmer), and reagents and buffers supplied in a GENEAMP&#x2122; PCR Reagent Kit (Perkin-Elmer), according to the manufacturer's instructions. In the following representations of the PCR primers, underlined nucleotides indicate sequences complementary to the 5&#x2032;-ends and 3&#x2032;-ends of the ANT cDNAs and double-underlined nucleotides indicate recognition sequences for the restriction enzymes XhoI (recognition sequence: 5&#x2032;-CTCGAG) and Asp718 (recognition sequence: 5&#x2032;-GGTACC). For human ANT1 (huANT1; SEQ ID NO:1), the following primers were used: Forward (sense): 5&#x2032;-TTATATCTCGAGTATGGGTGATCACGCTTGGAGCTT SEQ ID NO:4 CCTAAAG and Reverse (antisense): 5&#x2032;-TATATAGGTACCTTAGACATATTTTTTGATCTCATC SEQ ID NO:5 ATACAAC. For human ANT2 (huANT2; SEQ ID NO:2), the following primers were used: Forward (sense): 5&#x2032;-TTATATCTCGAGTATGACAGATGCCGCTGTGTCCTT SEQ ID NO:6 CGCCAAG and Reverse (antisense): 5&#x2032;-TATATAGGTACCTTATGTGTACTTCTTGATTTCATC SEQ ID NO:7 ATACAAG. For human ANT3 (huANT3; SEQ ID NO:3), the following primers were used: Forward (sense): 5&#x2032;-TTATATCTCGAGTATGACGGAACAGGCCATCTCCTT SEQ ID NO:8 CGCCAAA and Reverse (antisense): 5&#x2032;-TATATAGGTACCTTAGATCACCTTCTTGAGCTCGTC SEQ ID NO:9 GTACAGG. B. Generation of ANT Expression Constructs PCR products were digested with the restriction endonucleases XhoI and Asp718 (both enzymes from Roche Molecular Biochemicals) according to the manufacturer's recommendations using manufacturer-supplied reaction buffers. Restricted DNAs were purified by horizontal agarose gel electrophoresis and band extraction using the UltraClean GelSpin kit (Mo Bio Laboratories, Inc., Solana Beach, Calif.). The expression vector pBAD/His (&#x201c;B&#x201d; derivative; Invitrogen, Carlsbad, Calif.) was used. This vector contains the following elements operably linked in a 5&#x2032; to 3&#x2032; orientation: the inducible, but tightly regulatable, araBAD promoter; optimized E. coli translation initiation signals; an amino terminal polyhistidine(6&#x394;His)-encoding sequence (also referred to as a &#x201c;His-Tag&#x201d;); an XPRESS&#x2122; epitope-encoding sequence; an enterokinase cleavage site which can be used to remove the preceding N-terminal amino acids following protein purification, if so desired; a multiple cloning site; and an in-frame termination codon. Plasmid pBAD/His DNA was prepared by digestion with the restriction endonucleases XhoI and Asp718 according to the manufacturer's instructions and subjected to horizontal agarose gel electrophoresis and band extraction using the UltraClean GelSpin kit (Mo Bio Laboratories). Restricted ANT cDNAs were ligated into the linearized plasmid with restricted expression vector DNA using T4 DNA ligase (New England Biolabs, Beverly, Mass.) using the manufacturer's reaction buffer and following the manufacturer's instructions. Competent recA1 hsdR endA1 E. coli cells (strain TOP10F&#x2032;; Invitrogen, Catalog #C3030-03) were transformed with ligation mixtures containing the prokaryotic vector construct according to the manufacturer's instructions. Single colonies were selected and grown in 3-5 ml of LB broth (Sambrook, J., Fritsch, E. F., and Maniatis, T., Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1989) containing 50 &#x3bc;g/ml ampicillin (Roche Molecular Biochemicals). Plasmid DNA was isolated from the bacterial cultures using the WIZARD&#x2122; Plus Series 9600 Miniprep Reagents System (Promega, Madison, Wis.). The recombinant huANT nucleotide sequences present in the expression constructs were determined and their authenticity confirmed relative to the published ANT sequences (FIG. 1; See Neckelmann et al., Proc. Nat'l. Acad Sci. U.S.A. 84:7580-7584 (1987) for huANT1; Battini et al., J. Biol. Chem. 262:4355-4359 (1987) for huANT2, and Cozens et al., J. Mol. Biol. 206:261-280 (1989) for huANT3.) by DNA sequencing using the PRISM&#x2122; Ready BIG DYE&#x2122; Terminator Cycle Sequencing Kit (The Perkin-Elmer Corp., Norwalk, Conn.) and the following sequencing primers 5&#x2032;-TATGCCATAGCATTTTTATCC (SEQ ID NO:10) and 5&#x2032;-CGCCAAAACAGCCAAGCT (SEQ ID NO:11). For each human ANT sequence, both primers are located inside the vector sequence adjacent to the DNA insertion. Sequence data was analyzed using the SEQUENCE NAVIGATOR&#x2122; analysis software package (Perkin-Elmer). This huANT3 expression construct was named pMK3A-huANT3. The expression plasmids encoding His-tagged human ANT1, ANT2 and ANT3 are referred to herein as follows: For human ANT1, &#x201c;pMK1 (His-tagged huANT1)&#x201d; or &#x201c;pMK1&#x201d;; for human ANT2, &#x201c;pMK2 (His-tagged huANT2)&#x201d; or &#x201c;pMK2&#x201d;; for human ANT3 &#x201c;pMK3A (His-tagged hu ANT3&#x201d; or &#x201c;pMK3A&#x201d;; for human ANT3 from which extraneous linker N-terminal amino acids are deleted as detailed below, &#x201c;pMK3B (His-tagged hu ANT3, shortened epiotpe linker)&#x201d; or &#x201c;pMK3B&#x201d;. Plasmids pMKI, pMK2 and pMK3A have been deposited at the American Type Culture Collection (ATCC; Manassas, Va.) on Nov. 3, 1998, and given the accession numbers ATCC 98969, ATCC 98970 and ATCC 98971, respectively. The expression constructs comprising nucleotide sequences encoding human ANT1 (pMK1-huANT1) and human ANT2 (pMK2-huANT2) were restriction mapped to confirm their structures. The nucleotide sequences of plasmids pMK1-huANT1 and pMK2-huANT2 are determined using the methods and primers (SEQ ID NOS:10 and 11) described above. Treatment of the recombinant huANT3 protein expressed from pMK3A-huANT3 with enterokinase liberates the His-Tag/XPRESS&#x2122; epitope polypeptide from the huANT3 protein; however, the resultant huANT3 protein comprises several extraneous N-terminal amino acids (i.e., Pro-Ser-Ser-Ser-Met, where &#x201c;Met&#x201d; indicates the amino acid encoded by the translation initiation codon of huANT3). Although the extraneous amino acids probably have little or no effect on the recombinant huANT3 protein, a derivative expression construct in which the nucleotide sequence encoding the extraneous amino acids are deleted was prepared in the following manner. The QUIK-CHANGE&#x2122; Site-Directed Mutagenesis Kit (Stratagene, La Jolla, Calif.) essentially according to the manufacturer's instructions. Briefly, a reaction mixture comprising purified pMK3A-huANT3 DNA, the mutagenic oligonucleotide primers 5&#x2032;-GGAGATGGCCTGTTCCGTCATCTTATCGTCATCGTCGTACAGATC (SEQ ID NO:12; the underlined sequence is the reverse complement of the 5&#x2032; end of the huANT3 reading frame), and 5&#x2032;-GATCTGTACGACGATGACGATAAGATGACGGAACAGGCCATCTCC (SEQ ID NO: 13; the underlined sequence corresponds to the 5&#x2032; end of the huANT3 reading frame), Pfu DNA polymerase and dNTPs in manufacturer-supplied reaction buffer was prepared. The mutagenic oligonucleotide primers were present in excess and cycles of DNA synthesis was carried out in a thermal cycler according to the manufacturer's protocol. The reaction products were treated with the restriction enzyme DpnI, which cleaves methylated and hemi-methylated DNAs but leaves unmethylated DNA (i.e., annealed products of the reaction) intact, and used to transform EPICUREAN COLI&#x2122; XL-1-Blue E. coli cells (Stratagene). Plasmid DNA was prepared from twelve randomly selected transformants and the nucleotide sequence of the region containing the multiple cloning site cassette was determined according to the methods described above. Of the twelve plasmids, only one retained the original sequence found in pMK-huANT3, and three contained undesired point mutations. One of the eight &#x201c;correct&#x201d; plasmids was chosen and named pMK3B-huANT3. C. Expression of His-Tagged huANT3 Cultures of E. coli cells containing pMK3A-huANT3 were grown in LB media containing 50 ug/ml ampicillin to mid-log phase (OD600&#x2dc;0.5) and induced for 3-4 hours with increasing doses of arabinose (i.e., 0.00002%, 0.0002%, 0.002%, 0.02%, and 0.2 %). One ml of each culture was centrifuged at 5,000&#xd7;g for 10 minutes at 4&#xb0; C. to pellet the cells. Cell pellets were resuspended, and the cells were lysed, by adding 100 ul of Phosphate Buffered Saline (PBS; pH 7.4) containing 1% cholate, 1% n-dodecyl maltoside, and 0.1% 2-mercaptoethanol (in the preceding text, and throughout the specification, unless specified otherwise, all chemicals are from Sigma, St. Louis, Mo.). Total protein content in the lysates was determined using the BCA (bicinchoninic acid; Smith et al., 1985, Anal. Biochem. 150:76-85) Protein Assay kit (Pierce Chemical Co., Rockford, Ill.). Ten &#x3bc;g of total protein were loaded per lane onto an SDS polyacrylamide gel, electrophoresed and transferred to a nitrocellulose membrane (HYBOND&#x2122; ECL Nitrocellulose Membrane, Amersham Pharmacia Biotech, formerely Amersham Life Sciences, Piscataway, N.J.). Human ANT3 fusion proteins were detected in a western blot using ANT1-XPRESS&#x2122; Antibody (Invitrogen) and horseradish peroxidase-conjugated anti-mouse secondary antibody (Amersham Pharmacia Biotech) according to the manufacturers' instructions. The results are shown in FIG. 3. From left to right in the figure, the following samples are shown: lanes &#x201c;M&#x201d;, molecular weight markers, lane &#x201c;0&#x201d;, untransformed E. coli cells; lane &#x201c;o/n&#x201d;, E. coli comprising pMK3A-huANT3 grown overnight without induction; lane &#x201c;1&#x201d;-&#x201c;5&#x201d;, E. coli comprising pMK3A-huANT3 grown induced with increasing doses of arabinose (0.00002%, 0.0002%, 0.002%, 0.02% and 0.2 %, respectively). As expected, untransformed (lane 0) and uninduced (lane o/n) E. coli showed no XPRESS&#x2122;-huANT3 material. However, expression of recombinant ANT3 fusion protein with a molecular weight of 36.6 kD was observed in lanes 3 and 4 (0.002% and 0.02% arabinose, respectively). No XPRESS&#x2122;-huANT3 material was detected in lanes 1 and 2 (0.00002% and 0.0002% arabinose, respectively) indicating that the degree of induction was insufficient under these conditions. Cells that were grown in the presence of the highest concentration of arabinose (0.2%, lane 5) began to lyse and died before the time of harvest; consequently, no recombinant protein was detected. This indicated that very high expression of recombinant huANT in E. coli caused cell death, as is sometimes the case during overexpression of heterologous proteins in bacteria. D. Recombinant huANT3 Localizes to the Bacterial Membrane In order to locate the expressed human ANT 3 within E. coli cells, cells were grown in culture and induced with arabinose as described above, and then fractionated into different compartments (e.g., membranes, inclusion bodies and cytosol). Bacteria were pelleted by centrifugation at 5,000&#xd7;g for 10 minutes at 4&#xb0; C. The cell pellets were resuspended in {fraction (1/10)} volume of cell buffer A (50 mM Tris-HCl, pH 8.0, 2 mM EDTA, 100 ug/ml lysozyme, and 0.1% Triton X-100) and incubated for 15 minutes at 30&#xb0; C. in an orbital shaker. The cell mixture was sonicated for 2 minutes and membranes were pelleted by centrifugation at 12,000&#xd7;g for 15 minutes at 4&#xb0; C. The supernatant, representing the cytosol, was removed for analysis (FIG. 4, lane 4), as was a portion of the pellet containing membranes and inclusion bodies (FIG. 4, lane 3). The remaining portion of the pellet was washed twice with cell buffer B (10 mM Tris-HCl, pH 7.0, 0.1 mM EDTA, and 1 mM DTT) and centrifuged at 12,000&#xd7;g for 15 minutes at 4&#xb0; C. The pellet was resuspended in cell buffer C (20 mM Tris-HCl, pH 8.0, 100 mM sodium chloride, and 6 M guanidinium hyrochloride) and incubated for 1 hour at room temperature. The solution was then centrifuged at 12,000&#xd7;g for 15 minutes at 4&#xb0; C. The supernatant (containing solubilized inclusion bodies; lane 1, FIG. 4) and the pellet (containing insoluble inclusion bodies; lane 2, FIG. 4) were analyzed by Western blotting as described above. The results are shown in FIG. 4. Recombinant huANT 3 (molecular weight 36.6 kD) was detected in lanes 2, 3, and 4, as well as the positive control lane (+) (total cell lysate previously tested for presence of ANT3 protein by Western immunoblot analysis, as described above). The greatest amount of recombinant huATN3 was detected in lane 3, which represents the membrane fraction. This indicates that the majority of the huANT3 fusion protein integrated into the E. coli cellular membrane. Smaller protein signals were visible in lanes 2 and 4, representing the insoluble inclusion body fraction which might have contained some membranes with integrated ANT 3, and the cytosolic fraction where protein synthesis takes place. No protein was detectable in the soluble inclusion body fraction in lane 1, indicating that controlled expression of ANT3 in the bacteria did not result in the formation of inclusion bodies, which is an undesirable consequence of over-expression of some heterologous proteins in bacteria. E. Purification of ANT Proteins ANT proteins, and ANT fusion proteins, produced by the expression systems described herein have been purified using a variety of methods. The purification of ANT proteins, particularly human ANT proteins, is described in this Example. Regardless of which of following protein purification methods is used, or others that can be derived from the present disclosure, it is important to add sufficient amounts of DNase and RNase to eliminate the viscosity associated with some bacterial lysates (typically 10 &#x3bc;g/mL of each enzyme; both from Roche Biochemicals) when the bacterial cells are lysed (or immediately thereafter). An alternative or additional means by which viscosity has been minimized and ANT solubility has been optimized is vigorous sonication, as opposed to standard sonication, of the lysates. The term &#x201c;vigorous sonication&#x201d; refers to, for example, sonication with a Branson Sonifier (Model 450) 2&#xd7;(30 seconds each time) at 50% duty cycle and 80% output using a tapered, flat-tipped probe (as opposed to sonication with a cup and horn apparatus). Although either type of sonication will suffice, better yields have typically been observed when vigorous sonication has been used. Furthermore, in various ANT purification methods that have been used, it was often desirable to make the lysate at least 1% Triton-X, in order to solubilize the maximum possible amount of ANT protein, after which insoluble material is removed by a high-speed (i.e., about 100,000 g) spin. Typically, protease inhibitors such as, for example, pepstatin, leupeptin, phenylmethylsulfonyl fluoride (PMSF) and/or aprotinin (all from Sigma) have been present at effective levels (typically 10 &#x3bc;g/mL) during the preparation. Depending on the particular ANT protein or ANT fusion protein being isolated, all four protease inhibitors or any effective combination thereof are used. For example, in preparations of GST-huANT3 fusion proteins, best results were seen when all four protease inhibitors were used, although acceptable results have been obtained when only leupeptin and pepstatin were used. One method incorporates novel methods with several techniques previously used only for purifying ANT proteins from non-human mammals, i.e., bovine cardiac tissue and rats (Aquila et al., 1982, Hoppe-Seyler's Z. Physiol. Chem. 363:345-349; and Sterling, 1986, Endocrinology 119:292-295). In brief, bacterial cells expressing a GST-ANT3 fusion protein were lysed by lysozyme treatment, and 14C-palmityl&#x2014;CoA (Sigma) was added at a concentration of 50 nmol per gram of E. coli. Because it associates with ANT proteins, 14C-palmityl-CoA acts as a radiolabeled tracer that can be used to follow the ANT protein in subsequent purification steps. The lysates were then sonicated and made 6% Triton X-100 (Sigma) and incubated at 4&#xb0; C. for 1 hr to solubilize material. A high-speed spin was used to remove insoluble material, and the resulting solute was applied either (1) for small scale preparations, to hydroxyapatite beads (Bio-Rad Laboratories, Hercules, Calif.), or (2) in the case of larger preparations (i.e., &#x2267;1 liter of bacterial culture), to a hydroxyapatite column (Bio-Rad) essentially according to the manufacturer's instructions. Unlike other intramembrane mitochondrial proteins, ANT has a low affinity for hydroxyapatite (Klingenberg et al., 1978, Biochim. et Biophys. Acta 503:193-210). The hydroxyapatite column was eluted with Column Buffer A (10 mM MOPS, pH 7.2, 100 Mn NaCl, 9.5% Triton X100) and washed with Column Buffer B (10 mM MOPS, pH 7.2, 100 mM NaCl, 400 mM sodium phosphate). Non-recombinant ANT proteins from non-human species are eluted in the void volume with Column Buffer A, and the GST-huANT3 fusion protein was expected to be present in the void volume as well; Column Buffer B was used to wash the column in the event that GST-huANT3 fusion protein behaves differently. Samples were collected in such a manner as to have a final concentration of 30 of mM octyl glucoside (Calbiochem), a nonionic detergent that helps solubulize ANT proteins with minimal effect on activity (Sterling, 1986, Endrocrinol. 119: 292-295). The bead-extracted supernatant or column eluent was collected, and Triton X-100 was removed therefrom using the EXTRACTI-GEL&#x2122; affinity matrix (Pierce) essentially according to the manufacturer's instructions (see also Berman et al., 1985, Biochemistry 24:7140-7147). Varying amounts of GST-huANT3 prepared in the above manner were subject to PAGE and the gel was stained using a colloidal blue protein stain (Novex, San Diego, Calif.). The stained gel displayed a single band having a molecular weight corresponding to that predicted for the fusion protein. Based on the intensity of bands from samples of varying volumes, and the known volume of the preparation and minimal sensitivity of the stain, the yield from 100 mL of bacterial culture was estimated to be about 50 ug. In one of the lanes of the gel, approximately 500 ng of protein was loaded, and no contaminating bands were detected; this indicates that the GST-huANT3 protein was from at least about 90% pure to at least about 95% pure. GST-huANT3 fission proteins (see preceding Examples) have been purified by this method, and other ANT fusion proteins, including His-tagged huANT3 and other His-tagged ANT proteins, are purified in like fashion. Purified huANT fusion proteins are used to produce purified human ANT proteins as follows. GST-huANT fusion proteins are further purified via glutathione-agarose beads (Sigma) essentially according to the manufacturer's instructions. In brief, a solution comprising GST-huANT fusion proteins is contacted with glutathione-agarose beads, and the beads are washed to release undesirable contaminants. Next, the [bead:GST-huANT] complexes are treated with an appropriate enzyme, i.e., one that separates the huANT polypeptide from the remainder of the fusion protein. In the case of the GST-huANT3 fusion protein described herein (i.e., that encoded by pMK3C), thrombin (Sigma) cleaves the fusion protein in such a manner so as to produce two polypeptides: a first polypeptide corresponding to the GST moiety, and a second polypeptide which corresponds to human ANT3 with an additional six amino acids (i.e., Gly-Ser-Pro-Gly-Ile-Leu) present at its N-terminus. His-tagged huANT fusion proteins are further purified via Nickel-coated resins (such as, e.g., PROBOND&#x2122; Ni2+ charged agarose resin; Invitrogen) essentially according to the manufacturer's instructions. In brief, a solution comprising His-tagged huANT fusion proteins is contacted with the Nickel-coated resin, and the resin is washed to release undesirable contaminants. Next, the [resin:His-tagged huANT] complexes are treated with an appropriate enzyme, i.e., one that separates the huANT polypeptide from the remainder of the fusion protein. In the case of the His-tagged huANT3 fusion proteins described herein, enterokinase (Sigma, or EKMAX&#x2122; from Invitrogen may be used) cleaves the fusion protein in such a manner so as to produce two polypeptides: a first polypeptide comprising the His-tag and XPRESS&#x2122; epitope moieties, and a second polypeptide which corresponds to human ANT3. If the expression construct used is pMK3A, the resultant purified human ANT3 protein has an additional four amino acids (i.e., Pro-Ser-Ser-Ser) at its N-terminus. If pMK3B is the expression construct present in the cells from which His-tagged huANT3 is isolated, the resultant purified human ANT3 protein has the sequence of native huANT3, i.e., SEQ ID NO:3. In both of the preceding purification steps, an ANT fusion protein bound to a solid support is treated with an enzyme (i.e., thrombin or enterokinase) that liberates an ANT protein from the remainder of the fusion protein, which remains bound to the solid support. ANT protein is released into the liquid phase which is then collected to generate a solution comprising the ANT protein and some amount of the liberating enzyme. The amount of liberating enzyme needed is minimal because the treatment is catalytic in nature; nevertheless, some enzyme remains in the preparation. If desired, enzyme molecules may be removed from the preparation using any of a variety of means known in the art. For example, an enzyme may be removed from a solution by contacting the solution with a resin conjugated to a ligand having a high affinity for the enzyme. In the case of enterokinases, one such resin is the EK-AWAY&#x2122; resin (Invitrogen) which comprises the soybean trypsin inhibitor having a high affinity for enterokinases. Methods of treating GST fusion proteins with thrombin and purifying the desired recombinant protein have been described previously (see, for example, Smith and Corcoran, Unit 16.7 in Chapter 16 in Short Protocols in Molecular Biology 2nd Ed., Ausubel et al., eds, John Wiley &#x26; Sons, New York, N.Y., 1992, pages 16-28 to 16-31. In general, however, any suitable means for separating the liberating enzyme from any given ANT protein may be used. F. Growth Inhibition As noted in the above discussion of the results presented in FIG. 3, very high expression of recombinant huANT3 in E. coli caused cell death. Such a result is sometimes observed during over-expression of heterologous proteins in bacteria. Although not wishing to be bound by any particular theory, because the recombinant huANT3 protein localized to the bacterial membrane, and because ANT3 functions as an ATP/ADP exchanger in the inner mitochondrial membrane and under appropriate conditions may exhibit pore properties suggestive of a role in membrane permeability, one possible explanation for the observed cell death would be an inappropriate enhancement of the permeability of the bacterial membrane. If this in fact the case, inhibitors of mitochondrial ANT might prevent the death of E. coli overexpressing huANT3. As noted above, under certain conditions atractyloside or bongkrekic acid may exhibit inhibition of ANT activity, such that either of these inhibitors, other known ANT-active agents and potentially other ANT ligands as provided herein may be employed in the instant Example described using bongkrekic acid. In order to test this hypothesis, the following experiments are carried out. E. coli harboring pMK3A-huANT3 are grown with no arabinose or with 0.2% or more arabinose, the latter concentration having been previously shown to induce toxic levels of huANT3, and various concentrations (0, 5, 20, 50 and 200 &#x3bc;M) of bongkrekic acid (Biomol Research Laboratories, Inc., Plymouth Meeting, Mass.), an inhibitor of ANT (Henderson and Lardy, 1970, J. Biol. Chem. 245:1319-1326) that binds to ANT (see, e.g., Vignais et al., 1976, Biochim. Biophys. 440:688-696). The ability of bongkrekic acid to prevent the lysis of E. coli overexpressing huANT3, or any other ANT protein for that matter, indicates that the toxic effect of such overexpression is due to an activity associated with normally functioning ANT. ANT proteins produced by this expression system, and others described herein, are also purified using known methods for purifying ANT proteins from humans and other mammals. See for example, Klingenberg et al., 1978 Biochim. Biophys. Acta 503:193-210; Aquila et al., 1982 Hoppe-Seyler's Z. Physiol. Chem. 363:345-349; and Sterling, 1986 Endocrinol. 119:292-295. The bacterial toxicity of extreme overexpression of ANT in this system can be used to screen and identify novel inhibitors of ANT, as such compounds will be expected to also prevent lysis of E. coli overexpressing ANT proteins. In order to achieve a greater degree of specificity for the ANT protein produced from an expression vector, the yeast expression system for ANT proteins (see Example 4, infra) is used in a mutant yeast strain that is resistant to bongkrekic acid (Lauquin et al., 1975 FEBS Letters 35:198-200). Example 2 Expression of GST-huANT3 Fusion Proteins A. Generation of GST-huANT3 Expression Constructs Human ANT3 cDNA was amplified from pMK3A-huANT3 by PCR as in Example 1 but using the following primers. In the following representations of PCR primers, underlined nucleotides indicate sequences complementary to the 5&#x2032;-ends and 3&#x2032;-ends of the ANT cDNAs and double-underlined nucleotides indicate recognition sequences for the restriction enzymes XhoI (recognition sequence: 5&#x2032;&#x2014;CTCGAG) or EcoRi (recognition sequence: 5&#x2032;-GAATTC). The primers used for PCR amplification were: Forward (sense): 5&#x2032;-CCCGGGGAATTCTGATGACGGAACAGGCCATCTC SEQ ID NO:14 C and Reverse (antisense): 5&#x2032;-CCCGGGCTCGAGTTAGAGTCACCTTCTTGAGCTC SEQ ID NO:15 The expression vector pGEX-4T-2 (Amersham Pharmacia Biotech) was used to generate huANT3 fusion proteins comprising an enzymatic polypeptide and an ANT polypeptide. This vector comprises a lacIq (repressor) gene a tac promoter operably linked to a glutathione S-transferase (GST) gene from Schistosoma japonicum. (Smith et al., 1988, Gene 67:31-40), the coding sequence of which has been modified to comprise a thrombin cleavage site-encoding nucleotide sequence immediately 5&#x2032; from a multiple cloning site. GST fusion proteins can be detected by Western blots with anti-GST or by using a colorimetric assay; the latter assay utilizes glutathione and 1-chloro-2-4-dinitrobenzene (CDNB) as substrates for GST and yields a yellow product detectable at 340 nm (Habig et al., 1974, J. Biol. Chem. 249:7130-7139). GST fusion proteins produced from expression constructs derived from this expression vector can be purified by, e.g., glutathione affinity chromatography, and the desired polypeptide released from the fusion product by thrombin. Thus, this expression vector provides for the rapid purification of fusion proteins, and release of proteins with relatively few extraneous N-terminal amino acids, although the resulting recombinantly produced protein contains two additional amino acids at the amino terminus (Gly-Ser). The tac promoter may be induced by the addition to cultured cells of, e.g., 1-5 mM isopropyl-beta-D-thiogalactopyranoside (IPTG; Fluka, Milwaukee, Wis.) and provides for high-level expression. Plasmid pGEX-4T-2 was prepared by digestion with the restriction endonucleases EcoRI and Ap718 according to the manufacturer's instructions and subjected to horizontal agarose gel electrophoresis and band extraction using the UltraClean GelSpin kit (Mo Bio Laboratories) Restricted ANT cDNAs were ligated with the restricted expression vector DNA as described in the preceding Example. Single colonies were selected for grown in 3-5 ml of LB broth containing 50 ug/ml ampicillin (Roche Molecular Biochemicals), and plasmid DNA was isolated from the bacterial cultures using the WIZARD&#x2122; Plus Series 9600 Miniprep Reagents System (Promega). To confirm their authenticity, the recombinant huANT nucleotide sequences present in the pGEX deriavtive plasmid were determined as described in the preceding Example using the previously described oligonucleotide primers and 5&#x2032; and 3&#x2032; PGEX Sequencing Primers (Amersham Pharmacia Biotech). The resultant GST-huANT3 expression construct was named pMK3C-GST-huANT3 (also referred to herein as pMK3C). Plasmid pMK3C has been deposited at the American Type Culture Collection (ATCC; Manassas, Va.) on Nov. 3, 1998, and given the accession number ATCC 98973. Thrombin treated recombinant huANT3 protein produced from the pMK3C-GST-huANT3 expression construct includes several extraneous N-terminal amino acids, i.e., Gly-Ser-Pro-Gly-Ile-Leu-Met, where &#x201c;Met&#x201d; indicates the amino acid encoded by the translation initiation codon of huANT3. There is, however, no evidence that the extraneous six amino terminal amino acids have any effect on the resultant recombinant huANT3 protein. In order to confirm expression of the GST-huANT3 fusion protein, the following experiments were carried out. Eight independently isolated pMK3C-GST-huANT3 transformants and one control (vector-transformed) isolate were grown overnight in LB-ampicillin and then diluted 1:20 in 2 ml of fresh media. After 3 hours of growth at 37&#xb0; C., IPTG was added to a final concentration of 0.1 mM. Cell growth was continued for 2 hours, after which 1.5 of cells were tranferred to microfuge tubes, pelleted, resuspended in 300 uL of cold PBS containing 1% Triton X-100, and sonicated twice for 8 seconds. The sonicates were spun for 5 min. at 4&#xb0; C., the supernatant was transferred to fresh microfuge tubes and 50 uL of glutathione-agarose beads (Sigma) were added to produce a 50% slurry. After a 5 min. incubation at ambient temperature, the beads were spun and washed with 1 ml of PBS three times. The washed pellet was resuspended in SDS sp1 buffer (62.5 mM Tris, pH 6.8, 2% SDS, 10% glycerol, 5% beta-mercaptoethanol and sufficient bromophenol blue to provide visible coloration), and 30 uL of each preparation (equivalent to 15 uL of culture) was subjected to SDS-PAGE. The gel was stained using a Colloidal Coomassie (G-250) Staining Kit (Novex, San Diego, Calif.). A band of the predicted molecular weight of the GST-huANT3 fusion protein was readily apparent, with the same intensity, in each of the 8 preparations from pMK3C-GST-huANT3 transformants; this band was absent in the control preparation. B. Western Blot Analysis of Expression of huANT3 Fusion Proteins E. coli transformed with either (1) pMK3A-huANT3 (the pBAD/His-huANT3 expression construct) or (2) pMK3C-GST-huANT3 (the pGEX/GST-huANT3 expression construct) were lysed by the addition of lysozyme (100 &#x3bc;g/&#x3bc;l; Sigma) for 20 min at room temperature, followed by one freeze/thaw cycle. The negative control for the former transformant was a parallel culture of the transformed cells that had not undergone arabinose induction. The control for the latter transformant was a parallel culture of E. coli that had been transformed with the pGEX-4T-2 vector only. Total protein concentrations of each lysate were determined using the BCA Protein Assay kit (Pierce Chemical Co.), and equivalent amounts of total protein from each lysate preparation were mixed with equivalent volumes of 2&#xd7; Laemmli electrophoresis buffer and subjected to SDS-PAGE. The proteins were electrophoretically transferred to nitrocellulose, which was then contacted with antibodies against the appropriate epitope included in each vector (i.e., ANT1-XPRESS&#x2122; from Invitrogen for pMK3A-huANT3 and polyclonal goat anti-GST from Amersham Pharmacia Biotech, formerly Nycomed Amersham plc and Pharmacia &#x26; UpJohn Inc. for pMK3C-GST-huANT3). In a separate experiment, the bacterial lysate from the pMK3C-GST-huANT3 transformants was incubated with agarose-glutathione beads (Sigma) according to the manufacturer's instructions (see the preceding section and Smith et al., Expression and Purification of Glutathione S-Transferase Fusion Proteins, Unit 16.7 of Chapter 16 in: Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, N.Y., 1992, pages 16-28 to 16-31). The beads were suspended in Laemmli sample buffer and subjected to SDS-PAGE and Western analysis as described above. Although the yield of GST-huANT3 was low, perhaps because the fusion protein is inserted into the bacterial membrane, a sufficient amount of material was recovered for the experiment. The results (FIG. 5) show that a specific band of the predicted molecular weight (His-Tag+enterokinase site+antigenic site+huANT3=38 kDa) was observed in the arabinose induced E.coli that were transformed with the pBAD/his-huANT3 vector, but was absent in the non-induced control culture. Similarly, a band corresponding to GST-huANT3 was observed in the pMK3C-GST-huANT3-transformed E.coli, while only the unaltered GST band was observed in control E.coli transformed with the expression vector. Purification of the GST-huANT3 fusion protein using agarose-GSH beads produced a band of equivalent size to that observed in the crude lysate of pMK3C-GST-huANT-transformed bacteria. Example 3 Expression of ANT3 in Insect Cells A. Generation of Baculovirus Expression Constructs DNA comprising nucleotide sequences encoding huANT3 was amplified by PCR from a whole human brain cDNA library (Clontech) using the following primers. In the following representations of PCR primers, underlined nucleotides indicate sequences complementary to the 5&#x2032;-ends and 3&#x2032;-ends of the ANT cDNAs and double-underlined nucleotides indicate recognition sequences for the restriction enzymes BamHI (recognition sequence: 5&#x2032;-GGATCC) or EcoRI (recognition sequence: 5&#x2032;-GAATTC). The PCR primers used were: Forward (sense): 5&#x2032;-TTATAGGATCCATGACGGAACAGGCCATCTCCTT SEQ ID NO:16 CGCCAAA and Reverse (antisense): 5&#x2032;-TTAAAGAATTCTTAGATCACCTTCTTGAGCTCGT SEQ ID NO:17 CGTACAG. PCR products were digested with the restriction endonucleases BamHI (New England Biolabs) and EcoRI (New England Biolabs) according to the manufacturer's recommendations. Subsequent purification was carried out by horizontal agarose gel electrophoresis and band extraction using the UltraClean GelSpin kit (Mo Bio Laboratories, Inc.). The Baculovirus transfer vector pBlueBacHis2 (B version, Invitrogen) comprises, in 5&#x2032; to 3&#x2032; orientation, a constitutive polyhedrin promotor operably linked to nucleotide sequences encoding (1) a translation initiation sequence, (2) an N-terminal polyhistidine sequence, (3) an XPRESS&#x2122; epitope tag for detection and purification of the recombinant protein and (4) an enterokinas cleavage site, followed by a multiple cloning site wherein cDNAs can be inserted. The transfer vector pBlueBacHis2 was prepared by digestion with the restriction endonucleases BamHI and EcoRI according to the manufacturer's recommendation, and the restricted DNA was subject to horizontal agarose gel electrophoresis and band extraction using the UltraClean GelSpin kit (Mo Bio Laboratories, Inc.). The restricted PCR products were ligated with the restricted expression vector DNA as in the preceding Examples. Competent E. coli TOP10F&#x2032; cells (Invitrogen) were transformed with the ligation recation following the manufacturer's instructions. Single colonies were selected for growth in 3-5 ml of LB broth containing 50 ug/ml ampicillin. Plasmid DNA was isolated from the bacterial cultures using the WIZARD&#x2122; Plus Series 9600 Miniprep Reagents System (Promega). The recombinant ANT gene sequences were determined and their authenticities confirmed (SEQ ID NOS: 1, 2 and 3 correspond to human ANTs 1, 2 and 3, respectively) by DNA sequencing using the Prism Ready Dye Terminator Cycle Sequencing Kit (Perkin-Elmer, Catalog #402080) and the following primers: Polyhedrin Forward Sequencing Priming Site, 5&#x2032;-AAATGATAACCATCTCGC (SEQ ID NO:18); Baculovirus Reverse Sequencing Priming Site, 5&#x2032;-ACTTCAAGGAGAATTTCC(SEQ ID NO:19); primers internal to the ANT 3 coding sequence (sense strand), 5&#x2032;-ACTTCGCCTTCACGGATA (SEQ ID NO:20); and 5&#x2032;-TACGGCCAAGGGCATTCT (SEQ ID NO:21); primers internal to the ANT 3 coding sequence (antisense strand), 5&#x2032;-TGAAGCGGAAGTTCCTAT (SEQ ID NO:22); and 5&#x2032;-ATGCCGGTTCCCGTACGA (SEQ ID NO:23). Sequence data were analysed using the SEQUENCE NAVIGATOR&#x2122; analysis software package (Perkin-Elmer). An isolated plasmid having the correct sequence was named pMK4A-huANT3. Although pMK4A-huANT3 contains authentic huANT3-encoding sequences, the ANT3 reading frame is not synchronous with the reading frame of the His-Tag/XPRESS&#x2122; epitope of the expression vector. Accordingly, pMK4A-huANT3 is not expected to produce recombinant ANT protein, although cells harboring it may be used as controls. In order to generate an in-frame derivative of pMK4A-huANT3, the plasmid was mutagenized using the QUIK&#x2014;CHANGE&#x2122; Site-Directed Mutagenesis Kit (Stratagene) as in Example 1, except that the mutagenic oligonucleotide primers used were 5&#x2032;-GGCCTGTTCCGTCATCTTATCGTCATCGTCG (SEQ ID NO:24; the underlined sequence is the reverse complement of the 5&#x2032; end of the huANT3 reading frame), and 5&#x2032;-CGACGATGACGATAAGATGACGGAACAGGCC (SEQ ID NO:25; the underlined sequence corresponds to the 5&#x2032; end of the huANT3 reading frame). Several transformants were isolated, and plasmid DNA purified therefrom. The nucleotide sequences of the plasmid DNAs were determined and one having the &#x201c;correct&#x201d; sequence was identified and named pMK4B-huANT3. The baculovirus expression plasmids encoding human ANT3 are referred to as &#x201c;pMK4A (baculovirus shuttle, out-of-frame hu ANT3) or &#x201c;pMK4A&#x201d;; and &#x201c;pMK4B (baculovirus shuttle, in-frame hu ANT3)&#x201d; or &#x201c;pMK4B&#x201d;. Plasmid pMK4B has been deposited at the American Type Culture Collection (ATCC; Manassas, Va.) on Nov. 3, 1998, and given the accession number ATCC 98972. In order to insert sequences encoding the huANT3 protein (and assoicated regulatory sequences) into the baculovirus genome, insect cells (MAXBAC&#x2122; Spodoptera frugiperda Sf9 cells, Invitrogen, Carlsbad, Calif.; or Trichoplusia ni cells, PharMingen, San Diego, Calif.) were co-transfected with the baculoviral transfer construct pMK4B-huANT3 and linear baculoviral (Autographa californica nuclear polyhedrosis virus, AcMNPV) DNA engineered to contain a promoterless 3&#x2032; fragment of the lacZ gene (BAC-N-BLUE&#x2122;, Invitrogen) using the BAC-N-BLUE&#x2122; Transfection Kit (Invitrogen) following the manufacturer's instructions. Recombinant baculovirus plaques express functional beta-galactosidase and were identified as blue plaques in the presence of X-gal (5-bromo4-chloro-3-indoyl-beta-D-glactosidase). These recombinant viruses are expression constructs that express human ANT3 polypeptide in insect cells, as shown by the following experiments. B. Western Blot Analysis of Baculovirus Expression Systems High titer viral stock was produced, and recombinant protein was expressed in infected Sf9 (Invitrogen, Carlsbad, Calif.) or T. ni (PharMingen, San Diego, Calif.) cells according to the manufacturer's instructions (see also Piwnica-Worms, Expression of Proteins in Insect Cells Using Baculovirus Vectors, Section II of Chapter 16 in: Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, New York, 1992, pages 16-32 to 16-48; Kitts, Chapter 7 in: Baculovirus Expression Protocols, Methods in Molecular Biology, Vol. 39, C. R. Richardson, Ed., Humana Press, Totawa, N.J., 1995, pages 129-142). Transfected Sf9 cells were pelleted by centrifugation and lysed by adding 100 &#x3bc;l of MSB buffer (210 mM mannitol (Sigma), 70 mM sucrose (Fluka), 50 mM Tris-HCl, pH 7.4, 10 mM EDTA) and performing three freeze-thaw cycles. A total cellular fraction, a cytosolic fraction, a submitochondrial partical fraction, a mitochondrial fraction and a plasma membrane fraction were prepared as follows. The cell lysate was centrifuged at 600 g for 10 minutes at 4&#xb0; C. to prepare a plasma membrane pellet. The supernatant was removed and set aside. The plasma membrane pellet was washed with 100 ul of MSB, centrifuged at 600 g for 10 minutes at 4&#xb0; C., and used for the analysis. The supernatant was removed, combined with the first supernatant and mixed. Half of the supernatant was used to prepare a mitochondrial fraction and a cytosolic fraction by centrifugation at 14,000 g for 15 minutes at 4&#xb0; C.; the pellet represents the mitochondrial fraction and the supernatant represents the cytosol. The other half of the supernatant was centrifuged at 14,000 g for 15 minutes at 4&#xb0; C. to produce a mitochondria-containing pellet that was resuspended in MSB, incubated with 0.25 mg/ml digitonin (Roche Molecular Biochemicals, formerly Boehringer Mannheim, Indianapolis, Ind.) for 2 min and sonicated for 3 min at 50% duty cycle in a cup-horn sonicator to produce submitochondrial particles (SMPs). (See Example 13 for details regarding mitochondrial preparation from transfected T. ni cells.) The protein content for each fraction was determined using the BCA Protein Assay kit (Pierce Chemical Co.), and 8 ug of total protein were loaded per lane onto an SDS polyacrylamide gel, electrophoresed and transferred to a HYBOND&#x2122; ECL Nitrocellulose Membrane (Amersham Life Science). Fusion proteins were detected in a western blot using ANT1-XPRESS&#x2122; Antibody (Invitrogen, Catalog #R910-25) and horseradish peroxidase-conjugated anti-mouse secondary antibody (Amersham Life Science) following the manufacturers' instructions. The results of the Western analysis are shown in FIG. 6. Recombinant GST-huANT3 fusion protein (molecular weight 36.6 kD) was detected in total cells, mitochondria, submitochondrial particles and the plasma membrane. The signal was most intense in mitochondria and submitochondrial particles, whereas no band was detectable in the cytosolic fraction. These data suggest that the human recombinant huANT3 fusion protein integrated into the mitochondrial membranes much more efficiently than into the plasma membranes. Furthermore, all of the recombinant protein integrated into membranes since no signal was detected in the cytosolic fraction. The final lane of the autoradiogram shows His-tagged huANT3 isolated from cell lysates using magnetic agarose beads coupled to Ni according to the manufacturers instructions (Qiagen; Hilden, Germany). Thus, as in E. coli, huANT3 is expressed in the baculovirus/Sf9 system. Furthermore, recombinantly produced 6&#xd7;His- and epitope-tagged huANT3 fusion protein is appropriately localized to the mitochondria in Sf9 cells despite the presence of over 35 extraneous N-terminal amino acids, and can be isolated from cellular fractions by means that take advantage of the His-Tag moiety's affinity for metals such as, e.g., nickel. Example 4 Expression of ANT3 in Yeast A. Expression Constructs and Host Cells Human ANT3 cDNA was amplified by PCR as in Example 1 but using the following primers. In the following representations of PCR primers, underlined nucleotides indicate sequences complementary to the 5&#x2032;-ends and 3&#x2032;-ends of the ANT cDNAs and double-underlined nucleotides indicate recognition sequences for the restriction enzymes XhoI (recognition sequence: 5&#x2032;-CTCGAG) or Asp718 (recognition sequence: 5&#x2032;-GGTACC). The primers used for PCR amplification were: Forward (sense; SEQ ID NO:28): 5&#x2032;-TTAATGGGTACCATGACGGAACAGGCCATCTCCTTCGCCAAA, and Reverse (antisense; SEQ ID NO:29): 5&#x2032;-TTATACTCGAGTTAGATCACCTTCTTGAGCTCGTCGTACAGG. PCR products, and expression vector DNAs, were digested with the restriction endonucleases XhoI and Asp718 (both enzymes from Roche Molecular Biochemicals) according to the manufacturer's recommendations using manufacturer-supplied reaction buffers. The expression vector pYES2 (Invitrogen) was used. This vector contains a multiple cloning site located immediately downstream from an inducible GAL1 promoter, as well as the 2u origin of replication and the S. cerevisiae URA3 gene for high-copy maintenance and selection in ura3 yeast cells, respectively. The restricted DNAs were purified by horizontal agarose gel electrophoresis and band extraction using the UltraClean GelSpin kit (Mo Bio Laboratories), ligated to each other, and used to transform E. coli cells, as in the preceding Examples. Plasmid DNA was isolated from several transformants, and the nucleotide sequence of the insert DNA was determined and confirmed to be that of huANT3. One confirmed plasmid was chosen to be used for further study and was designated pMK5A (huANT3). A second yeast huANT3 expression vector, pMK5B, was constructed as follows. Plasmids pMK5A and pYESTrp2 (Invitrogen) were digested with restriction enzymes BglI and PvulI (both from New England Biolabs) and gel purified, ligated and used to transform E. coli as above. The expression vector pYES2Trp is similar to pYES2 but comprises a TRP1 selectable marker. Plasmid DNA was isolated from several transformants and restriction mapped to confirm the structure of the expected expression construct. One confirmed plasmid was chosen to be used for further study and was designated pMK5B (huANT3). A third yeast huANT3 expression vector, pMK5C, was constructed using the expression vector pYPGE2, which comprises a TRP1 selectable marker and the strong PGK promoter upstream from a multiple cloning site (Brunelli and Pall, 1993 Yeast 9:1299-1308). Plasmid pYPGE2 DNA was digested with XhoI and Asp718, gel-purified and ligated with the XhoI- and Asp718-restricted huANT3 PCR product of Example 1. The ligation mixture was used to transform E. coli, and plasmid DNA was isolated from several transformants and restriction mapped to confirm the structure of the expected expression construct. One confirmed plasmid was chosen to be used for further study and was designated pMK5C (huANT3). In order to generate yeast expression systems, the S. cerevisiae strain INVScl (MAT&#x3b1;, his3&#x394;1, leu2, trp1-289, ura3-52) was transformed with purified pMK5A, pMK5B and pMK5C DNAs using the S.c. EASYCOMP&#x2122; Transformation Kit (Invitrogen). A second S. cerevisiae strain, J&#x394;I&#x394;3 (MAT&#x3b1;, ade2-1, leu2-3, leu2-112, his3-11, his3-15, trp1-1, ura3-1, can1-100, AAC1::LEU2, AAC2::HIS3, AAC3::URA3) was also transformed with the expression constructs. The AAC genes encode the three isoforms of the mitochondrial ADP/ATP translocator in S. cerevisiae and are interrupted in strain J&#x394;1&#x394;3 (Giraud et al., J. Mol. Biol. 281:409-418 (1998)). It is thus expected that transformants of J&#x394;1&#x394;3, which are incapable of expressing endogenous ANT (AAC) proteins, will only express the human ANT protein encoded by the expression construct with which they have been transformed. B. Northern Blot Analyses of Yeast Expression Systems In order to examine levels of huANT3 mRNA production in strain J&#x394;1&#x394;3, Northern analyses of cells transformed with pMK5B and pMK5C were performed according to methods known in the art. In brief, transformed cells and control (untransformed) cells grown to mid-log phase, harvested and lysed. RNA was extracted from the lysates, electrophoresed and transferred to a nitrocellulose filter (see Treco, Preparation of Yeast RNA, Unit 13.12 of Chapter 13 in Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, N.Y. (1992), 13:44-46 and Seldon, Analysis of RNA by Northern Hybridization, Unit 4.9 of Chapter 4, Id., 4:23-25). The XhoI- and Asp718-restricted huANT3 PCR product of Example 1 was radiolabelled and used as a probe, and an RNA preparation from human spleen tissue was used as a positive control. The results (FIG. 10) demonstrate the appropriately-sized ANT3-specific RNA is produced in human spleen and in yeast cells transformed with either expression vector, but not in untransformed yeast cells. The pYPGE2-derived expression construct pMK5C, which directs ANT3 expression from the PGK promoter, clearly results in more ANT3 RNA than the pYES2Trp-derived construct pMK5B, in which ANT3 expression is driven by the GAL1 promoter. In either case, however, significant levels of huANT3-specific RNA were produced in a yeast background that lacks any endogenous adenosine nucleotide translocator proteins. C. Western Analyses of Yeast Expression Systems 1. Production of antibody to huANT3 As the huANT3 produced from the yeast expression constructs lacks an epitope tag, it was necessary to produce an antibody to huANT3 in order to evaluate recombinant production of the protein. A monspecific (antipeptide) antibody specific to huANT3 was prepared as follows. A synthetic polypeptide corresponding to a portion of huANT3 located near the carboxy terminus and predicted to have high antigenicity according to the Jameson-Wolf Index (Wolf et al., Comput. Appl Biosci. 4:187-191 (1988)) was synthesized using known means by Alpha Diagnostic International (San Antonio, Tex.) and determined to be at least about 70% pure, preferably at least about 90% pure, by HPLC and MS analyses. The sequence of the synthetic polypeptide (SEQ ID NO:30) is: Cys-Trp-Arg-Lys-Ile-Phe-Arg-Asp-Glu-Gly-Gly-Lys- Ala-Phe-Phe The synthetic polypeptide was conjugated to a carrier molecule, keyhole limpet hemocyanin (KLH), using MSB (m-maleimidobenzoyl-N-hydroxysuccinimide ester; Pierce Chemical Co., Rockford, Ill.), and the conjugated material was used to immunize several rabbits, according to known means (Collawn and Paterson, Units 11.14 and 11.15 in Chapter 11 in: Short Protocols in Molecular Biolog, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, N.Y. (1992) 11:37-41. The rabbits were or are bled at 0 (preimmune, 2 mL), 7, 9, 11, 13 (15 mL for each bleed) or 15 weeks (50 mL) post-inoculation. Sodium azide (0.1%) was or is added to the bleeds as preservative. 2. Western analyses Western analyses of yeast expression systems are performed essentially as described in the preceding Examples, except that different methods are used to prepare protein preparations from yeast cells as opposed to bacterial or insect cells. Such methods of isolating proteins from yeast are known in the art (see, for example, Dunn and Wobbe, Preparation of Protein Extracts from Yeast, Unit 13.13 of Chapter 13 in Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds. John Wiley &#x26; Sons, New York, N.Y.(1992), 13:46-50). The intracellular distribution of huANT3 in, e.g., membrane or mitochondrial fractions, is determined as in the preceding Examples. Example 5 Expression of ANT3 In Mammalian Cells The preceding Examples describe a variety of means by which ANT and ANT fusion proteins can be recombinantly produced in various systems. Although such ANT proteins can be used in a variety of assays (see infra), it may be desirable to isolate large amounts of the native ANT protein from mammalian cells. In particular, as described in this Example, it may be desirable to produce recombinant viral particles in which ANT proteins are displayed in the viral envelope. Such ANT-displaying viral particles are expected to be very stable and useful in a variety of assays including, for example, those in which compounds binding to ANT proteins are screened and identified. Another useful outcome of mammalian expression systems is the generation and isolation of human mitochondria in which a particular ANT isoform is over-represented in order to determine the specific biological role(s) of such isoforms. For example, ANT3 is apparently ubiquitously expressed in human tissues, whereas ANT1 is primarily expressed in heart and skeletal muscle (Stepien et al., 1992, J. Biol. Chem. 267:14592-14597). Directed overexpression of huANT1 in cultured heart or muscle cells is expected to result in mitochondria that contain mostly the ANT1 isoform. Such &#x201c;ANT isoform-enriched&#x201d; mitochondria can be isolated and tested for various mitochondrial functions. Constructs for expressing ANT proteins in mammalian cells are prepared in a stepwise process. First, expression cassettes that comprise a promoter (and associated regulatory sequences) operably linked to nucleotide sequences encoding an ANT protein are constructed in bacterial plasmid-based systems; these expression cassette-comprising constructs are evaluated and optimized for their ANT-producing ability in mammalian cells that are transiently transfected therewith. Second, the ANT expression cassettes are transferred to viral systems that produce recombinant proteins during lytic growth of the virus (e.g., SV40, BPV, EBV, adenovirus; see below) or from a virus that can stably integrate into and transduce a mammalian cellular genome (e.g., a retroviral expression construct). A. Transient Expression With regards to the first step, commercially available &#x201c;shuttle&#x201d; (i.e., capable of replicaton in both E. coli and mammalian cells) vectors that comprise promoters that function in mammalian cells and can be operably linked to an ANT-encoding sequence include, but are not limited to, SV40 late promoter expression vectors (e.g., pSVL, Pharmacia), glucocorticoid-inducible promoter expression vectors (e.g., pMSG, Pharmacia), Rous sarcoma enhancer-promoter expression vectors (e.g., pRc/RSV, Invitrogen) and CMV early promoter expression vectors, including deriavtives thereof having selectable markers to agents such as Neomycin, Hygromycin or ZEOCIN&#x2122; (e.g., pRc/CMV2, pCDM8, pcDNA1.1, pcDNA1.1/Amp, pcDNA3.1, pcDNA3.1/Zeo and pcDNA3.1/Hygro, Invitrogen) In general, preferred shuttle vectors for ANT genes are those having selectable markers (for ease of isolation and maintenance of transformed cells) and inducible, and thus regulatable, promoters (as overexpression of ANT genes may have toxic effects) Methods for transfecting mamallian cells are known in the art (see, Kingston et al., &#x201c;Transfection of DNA into Eukaryotic Cells,&#x201d; Section I of Chapter 9 in: Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, N.Y., 1992, pages 9-3 to 9-16). A control plasmid, such as pCH110 (Pharmacia), may be cotransfected with the ANT expression construct being examined so that levels of ANT can be normalized to a gene product expressed from the control plasmid. Western analyses of mammalian expression systems are performed essentially as described in the preceding Examples, except that different methods are used to prepare protein preparations from mamallian cells as opposed to bacterial, insect or yeast cells. Such methods of isolating proteins from yeast are known in the art (see, for example, Kingston and Sheen, Unit 9.6A and Brasier, Unit 9.6B of Chapter 9 in: Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, N.Y., 1992, pages 9-17 to 9-23). Preferred expression cassettes, consisting essentially of a promoter and associated regulatory sequences operably linked to an ANT gene of interest, are identified by the ability of cells transiently transformed with a vector comprising a given ANT expression cassette to express high levels of ANT protein when induced to do so; these expression cassettes are incorporated into viral expression vectors. B. Viral Expression Nucleic acids, preferably DNA, comprising preferred expression cassettes are isolated from the transient expression constructs in which they were prepared, characterized and optimized (see preceding section). A preferred method of isolating such expression cassettes is by amplification by PCR, although other methods (e.g., digestion with appropriate restriction enzymes) can be used. Preferred expression cassettes are introduced into viral expression vectors, preferably retroviral expression vectors, in the following manner. A DNA molecule comprising a preferred expression cassette is introduced into a retroviral transfer vector by ligation (see preceding Examples). Two types of retroviral transfer vectors are known in the art: replication-incompetent and replication-competent. Replication-incompetent vectors lack viral genes necessary to produce infectious particles but retain cis-acting viral sequences necessary for viral transmission. Such cis-acting sequences include the &#x3a8; packaging sequence, signals for reverse transcription and integration, and viral promoter, enhancer, polyadenylation and other regulatory sequences. Replication-competent vectors retain all these elements as well as genes encoding virion structural proteins (typically, those encoded by genes designated gag, pol and env) and can thus form infectious particles in a variety of cell lines. In contrast, these functions are supplied in trans to replication-incompetent vectors in a packaging cell line, i.e, a cell line that produces mRNAs encoding gag, pol and env genes but lacking the &#x3a8; packaging sequence. See, generally, Cepko, Unit 9.10 of Chapter 9 in: Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, N.Y., 1992, pages 9-30 to 9-35. A retroviral construct comprising an ANT expression cassette produces RNA molecules comprising the cassette sequences and the &#x3a8; packaging sequence. These RNA molecules correspond to viral genomes that are encapsidated by viral structural proteins in an appropriate cell line (by &#x201c;appropriate&#x201d; it is meant that, for example, a packaging cell line must be used for constructs based on replication-incompetent retroviral vectors). Infectious viral particles are then produced, and released into the culture supernatant, by budding from the cellular membrane. The infectious particles, which comprise a viral RNA genome that includes the ANT expression cassette, are prepared and concentrated according to known methods. It may be desirable to monitor undesirable helper virus, i.e., viral particles which do not comprise an ANT expression cassette. See, generally, Cepko, Units 9.11, 9.12 and 9.13 of Chapter 9 in: Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, N.Y., 1992, pages 9-36 to 9-45. Viral particles comprising an ANT expression cassette are used to infect in vitro (e.g., cultured cells) or in vivo (e.g., cells of a rodent, or of an avian species, which are part of a whole animal). Tissue explants or cultured embryos may also be infected according to methods known in the art. See, generally, Cepko, Unit 9.14 of Chapter 9 in: Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, N.Y., 1992, pages 9-45 to 9-48. Regardless of the type of cell used, production of ANT protein is directed by the recombinant viral genome. In a preferred embodiment, recombinantly produced ANT proteins are inserted into the cell membrane of cultured cells. Because the retroviral expression construct produces viral particles by budding of the cell membrane, the resultant viral particles delivered to the culture supernatant have ANT protein incorporated into their capsules, preferably on the surface of the particles. Such ANT-displaying viral particles are expected to provide a stable format for ANT proteins and to thus be useful in assays using ANT proteins, either directly or as a source material from which ANT can be further purified. If it is desired to minimize the amount of ANT protein inserted into mitochondrial membranes, &#x3c1;0 cells, which have been treated in such a manner as to be nearly or completely devoid of mitochondria, are used as host cells. C. ANT Antisense Constructs Antisense versions of the preceding transient and viral ANT expression constructs are prepared by exchanging the antisense (non-encoding) strand for a sense (ANT protein encoding) strand in a construct. Such ANT antisense constructs are useful as research reagents, i.e., to reduce levels of expression of one or more isoforms in a cell transformed or infected with such a construct in order to determine the effects of such treatment on cellular physiology. ANT antisense constructs are also useful as gene therapy agents that interfere with the translation of one or more isoforms of ANT. Example 6 Synthesis and Properties of Representative ATR Derivatives A number of atractyloside (ATR) derivatives were prepared for use as ligands for adenine nucleotide translocators (ANTs) in the context of high-throughput screening assays. These compounds bind with high affinity (i.e., in the nM range) to ANT and are thus useful for screening libraries of chemical compounds for molecules having high specificity for ANT (regardless of isoform) The structure of ATR is set forth below as compound (1). Compounds (3) and (4) represent novel fluorescent derivatives of ATR, while compound (2) is an ATR derivative which permits introduction of the 125I under mild conditions. Purification Compounds 2, 3 and 4 were purified by silica gel chromatography using CH2Cl2/MeOH/AcOH (75:25:1) as the eluting solution. Detection was achieved by staining with a 0.5% solution of vanillin in H3PO4H2O (1/1). Further purification was accomplished by reversed-phase HPLC using a Microsorb C8 column (250&#xd7;10 mm). The column was eluted at a flow rate of 2.0 mL/min with a linear gradient of methanol/acetic acid/1 M ammonium acetate 98:1:1 (&#x201c;Solvent B&#x201d;) and H2O acetic acid/1 M ammonium acetate aqueous solution 98:1:1 (&#x201c;Solvent A&#x201d;). The effluent was monitored for absorbance at 254 nm. Compound-containing fractions were pooled, evaporated, and repeatedly co-evaporated with added methanol (3&#xd7;5 mL). Synthesis of Compound 2 Atractyloside 1 (0.10 mmol) was dried by repeated evaporation of added pyridine (3&#xd7;5 mL) and the resulting gummy residue dissolved in pyridine (5 mL). To the resulting solution, 0.20 mmol of toluenesulfonyl chloride was added. The reaction mixture was stirred at ambient temperature for 1.5 h. Then, another portion of toluenesulfonyl chloride (0.20 mmol) was added and the reaction left stirring an additional 1.5 h. 1 mL of methanol was added to the reaction mixture which was then stirred for 0.5 h, after which solvents were removed by evaporation. Residual pyridine was removed by evaporation of additional methanol (5&#xd7;10 mL). Silica gel chromatography followed by reversed-phase HPLC using a linear gradient of 50-80% of solvent B in solvent A for 30 min. resulted in the compound 2 eluting at 68% solvent B. Yield: 4.3 mg, 4.9%. ESI-MS (M-H) found:879, calc.:879. Synthesis of Compounds 3 and 4 7-Diethylamino-2-oxo-2H-chromene-3-carboxylic acid or 0.20 mmol of 1-pyrenebutyric acid and 0.60 mmol of 1,1&#x2032;-carbonyldiimidazole in 1 mL of dimethylformamide were allowed to react for 15 min. To the activated carboxylic acid was added a solution of atractyloside 1 in H2O (4 mL) and the resulting reaction mixture was stirred at ambient temperature for 16 h. Evaporation left a gummy residue which was purified by silica gel chromatography followed by reversed-phase HPLC. Using a linear gradient of 10-80% of solvent B in solvent A for 50 min (for compound 3) or 50-100% of solvent B in solvent A for 50 min (for compound 4) resulted in compound 3 eluting at 75% B and compound 4 eluting at 82% B. Yields: compound 3, 3.1 mg, 8.0%; compound 4, 1.3 mg, 3.6%. ESI-MS (M-H) compound 3 found:968, calc.: 968; compound 4 found:995, calc.:995. Properties of Representative ATR Derivatives As summarized in Table 1 below, compounds 3 and 4 were found to be more advantageous in terms of fluorescence characteristics and sensitivity compared to the existing ATR derivatives Naphthoyl-ATR and MANT-ATR as reported by Boulay et al., Analytical Biochemistry 128:323-330,1983; Roux et al., Analytical Biochemistry 234:31-37,1996; and Lauquin et al., FEBS Letters 67:306-311,1976. TABLE 1 Excitation Emission Extinction Coefficient (M&#x2212;1) ATR Derivative (nm) (nm) (Predicted) Naphthoyl-ATR 300 405 6,200 MANT-ATR 350 460 5,800 Compound 4 341 391 17,420 Compound 3 417 470 46,400 Example 7 Synthesis of Representative ATR Derivative The further representative ATR derivative, compound 5, was prepared by the procedure set forth below. Synthesis of Compound 5 Dipotassium atractylate (0.10 mmol) was dissolved in 50% aq. ethanol (5 mL) and palladium on charcoal (10%, 17 mg) was added to the reaction mixture. After flushing the system with hydrogen, the reaction mixture was stirred under an atmosphere of hydrogen gas for 3 h. Removal of catalyst by filtration through Celite, washing with 50% aq. ethanol (10 mL), and evaporation of solvents afforded a white solid. Yield after thorough drying under high vacuum; 78.3 mg (97.3%). ESI-MS (M-2H+K) found:765, calc.:765. 1H-NMR analysis confirmed the absence of alkenic protons: DMSO-d6) &#x3b4;0.88(d, 3H), 0.89(d, 3H), 1.02(d, 3H). Example 8 Synthesis of Representative Iodinated ATR Derivative Compound 2 of Example 6 may be used as intermediate for conjugation of variety of chemical moieties to yield further ATR derivatives. In this example, compound 2 is employed to introduce 125I under mild conditions to yield the following compound 6. Synthesis of Compound 6 Five &#x3bc;l of 0.2 M sodium phosphate (pH 5) was combined with 21 ul of Na125I (9.25 mCi) in its shipping container (specific activity, 2100 Ci/mmol; Amersham, Piscataway, N.J.). Ten ul (200 ug, 212 nmol) of compound 2 of Example 1 was added to the mixture. The pH was checked with litmus paper to confirm that it did not rise above pH 5. The mixture was allowed to stand at ambient temperature overnight (17.5 hours) to yield radiolabelled compound 6. (Non-radioactive iodinated ATR derivative, for use as a &#x201c;cold&#x201d; competitor in binding studies, may be prepared in the same manner using unlabeled iodine). The iodinated derivative was purified over a C18 analytical column (4&#xd7;6&#xd7;250 mm) (Phenomenex, Torrance, Calif.) using a 25%-55% acetonitrile gradient in running buffer (1% triethylammonium acetate, pH 4.5). A flow rate of 1 ml/min was used to run the gradient over 30 min. The desired product eluted at 25 min. ESI-MS. 835 (M-H), 707 (m-2H-I). Example 9 Synthesis of Representative ATR Derivatives Activation of carboxylic acids with carbonyl diimidazole and their reaction with ATR has been the method of choice for synthesis of various 6&#x2032;-O-acyl derivatives The relatively low reactivity of the 6&#x2032;-hydroxyl of ATR and the presence of an allylic secondary hydroxyl in the aglycon as well as the sulfated glucose moiety, are all factors that have a negative impact on the efficiencies of these acylation reactions. Hence, yields are generally poor and the approach requires a large excess of acylating reagents. Two strategies for introduction of an amine functionality in the ATR system are described below that permit synthesis of a broader range of ATR derivatives. In the first strategy, as depicted by Scheme 1, displacement of the primary tosylate from compound 2 (Example 1) with azide followed by reduction yields the corresponding 6&#x2032;-amine (compound 7). Alternatively, the amine group can be introduced as part of a spacer, which permits introduction of more sterically demanding functional moieties. Thus, reacting the 6&#x2032;-O-succinoyl derivative (compound 8; see Brandolin et al., 1974 FEBS Lett. 46:149.) with a monoprotected diamine followed by deprotection affords compound 9 as illustrated by Reaction Scheme 2. The amine-containing ATR derivatives 7 and 9 may then be reacted with a variety of fluorophors and haptens bearing reactive isothiocyanate, N-hydroxysuccinimide ester and anhydride functionalities to yield stable ATR-derivatives having thiourea and amide linkages. Representative ATR derivatives that were prepared include ATR-lanthanide chelating agents (compounds 10, 11, 12, 13, 20 and 21) that have utility for time-resolved fluorescence detection of these compounds complexed to Eu3+. In addition, ATR was conjugated to cyanine (compounds 14 and 15) and fluorescein analogues (compounds 16 and 17) that are detectable by fluorescence with extremely high sensitivities. Coupling of biotin-NHS ester with the ATR derivatives of compounds 7 and 9 provided ATR-biotin conjugates (compounds 18 and 19) that can be detected with commercially available enzyme-avidin conjugates using colorimetric, fluorescent or chemiluminescent techniques. More specifically, a solution of compound 2 in DMF was treated with azide ion for 8 hours at 80&#xb0; C. to give the 6&#x2032;-azido-ATR, that was purified by silica gel chromatography using a CH2Cl2/CH3OH solvent system supplemented with 1% acetic acid. Staudinger-reduction using 1.5 equivalents of triphenylphosphine in a THF/H2O mixture for 4 hours at RT afforded the amine of compound 7, that was isolated after silica gel chromatographic purification. To accommodate more sterically demanding functional moieties, 6&#x2032;-O-succinoyl-ATR may be condensed with commercially available monoprotected diamines (Calbiochem-Novabiochem Corp, San Diego, Calif.) to produce ATR-mono-protected amine derivatives. Thus, EDC-mediated coupling of 6&#x2032;-O-succinoyl-ATR in DMF with 1.1 equivalents of mono-protected FMOC diamines yield the amide that was deprotected using piperidine or 1,8-diazabicyclo[5.4.0]undec-7-ene (DBU) in acetonitrile to furnish the ATR derivative of compound 9. The amines were purified by silca gel chromatography as described above. The ATR-amine derivatives of compounds 7 and 9 were coupled to a variety of fluorophors, chelates and haptens that contained amine-reactive functionalities, such as isothiocyanates, anhydrides and NHS esters in aqueous DMF to generate the ATR derivatives of compounds 10 through 19: These compounds were purified by a combination of silica gel chromatography and preparative reverse phase chromatography on a C-8 column using CH3OH/H2O gradient containing 0.1-1% acetic acid. Example 10 Synthesis of Representative ATR Derivatives and Intermediate Therefor General Procedure for Coupling Atractyloside or Dihydroatractyloside to Organic Acids Carboxylic acid (200 &#x3bc;mol) and 1,1&#x2032;-carbonyldiimidazole (700 &#x3bc;mol) were dissolved in DMF (2 mL) and stirred for 15 min. To the activated acid, a solution of atractyloside (ATR) (100 &#x3bc;mol, 85 mg) or dihydroatractyloside (100 &#x3bc;mol) in DMF:H2O (1:2, 6 mL) was added (in 1 mL portions over ca 30 sec). The reaction mixture was stirred at room temperature for 60 min, after which solvents were removed by rotary evaporation on a water bath in which the temperature was kept below 40&#xb0; C. The residue was stripped of traces of DMF by repeated evaporation of added EtOH:H2O (1:1, 3&#xd7;20 mL). The residue was then taken up in MeOH: H2O (1:1, 10 mL), sonicated if necessary, and filtered through a 0.45 &#x3bc;m filter. Evaporation, re-dissolution in &#x2dc;1.3 mL buffer, and purification by reverse phase BPLC furnished the desired 6&#x2032;-O-acyl-ATR in yields ranging from 5-15%. Reverse Phase HPLC Conditions for Purification of Atractyloside Derivatives Purification by reverse phase HPLC (RP-BPLC) was performed in a 10&#xd7;250 mm C-8 column, using a gradient of MeOH:AcOH:1 M NH4OAc (buffer B, 98:1:1) in H2O:AcOH:1 M NH4OAc (buffer A, 98:1:1). Typically, a gradient of B in A from 50-80% over 30 min was employed. For more lipophilic derivatives, the gradient was from 60-90% or 70-100% B over the same time period. Synthesis of Dihydroatractyloside Atractyloside, dipotassium salt (254 mg, 0.3 mmole) in 15 ml of EtOH/2O (1:1) was hydrogenated under atmospheric pressure for 3 hours using 51 mg of activated palladium/carbon (10%) as catalyst. The catalyst was removed by filtration through celite and the celite bed was washed with 15 ml of EtOH/H2O. The filtrates were concentrated by rotary evaporation, followed by drying under high vacuum overnight to provide the product as white solid (236 mg), that was pure by NMR. 6&#x2032;-Tosyl-Atractyloside Atractyloside.3H2O (255 mg. 0.3 mmole)was dried by co-evaporation with dry pyridine (3&#xd7;5 ml) and then was kept under high vacuum for 16 hrs. The dried atractyloside was dissolved in 15 ml of dry pyridine and 114 mg of tosyl chloride (0.6 mmole) was added. The reaction was stirred for 2 hrs at 23&#xb0; C. and then an additional 114 mg of tosyl chloride was added and the reaction was allowed to continue for an additional 1.5 hrs. Methanol (1 ml) was added to the reaction mixture to scavenge excess tosyl chloride and the mixture was stirred for several minutes. The mixture was evaporated to dryness, and residual pyridine was removed by evaporation of added methanol (2&#xd7;30 ml). The crude product was partially purified by silica gel flash chromatography (CH2Cl2/MeOH, 3:1 with 1% AcOH). The product containing fractions were dissolved in 3 ml of H2O:MeOH:1M NH4OAc:HOAc (49:49:1:1) and purified by RP-HPLC using a 50%-80% gradient of MeOH:AcOH:1M NH4OAc (buffer B, 98:1:1) in H2O:AcOH: 1 M NH4OAc (buffer A, 98:1:1). Product containing fractions were pooled, evaporated, and subjected to additional co-evaporation of 3&#xd7;10 ml of MeOH. Tosyl-atractyloside was obtained as a glassy material weighing 60 mg. General Procedure for Lactoperoxidase&#x2014;Catalyzed Iodination of 4-Hydroxyphenyl Derivatives To a solution of 4&#x2032;-hydroxy-biphenyl-4-carboxylic acid atractylosid-6&#x2032;-yl ester (1.0 mg, 1.0 mmol) in H2O (120 &#x3bc;L) were added aqueous solutions of lactoperoxidase (50 &#x3bc;L, 200 IU/mL), NaI (10 &#x3bc;L, 100 mM) and H2O2 (20 &#x3bc;L, 100 mM). The reaction was left at room temperature for 1 h after which it was frozen at &#x2212;20 &#xb0; C. The next day (&#x2dc;20 hours later) the reaction mixture was thawed and subjected to RP-HPLC. Three major peaks were eluted and electrospray ionization-mass spectrometic (ESI-MS) analysis confirmed their identity as unreacted starting material, and monoiodinated, and diiodinated atractyloside derivatives. These conditions can be modified to drive the reaction completely to the mono and di-iodinated forms with additional aliquots of NaI and/or enzyme. General for Synthesis of Iodophenols The iodination procedure described in Acta Chem, Scand. 12, 188 (1958) was used for mono-iodination of 3-(4-hydroxyphenyl)propionic acid, 4&#x2032;-hydroxy-4-biphenylcarboxylic acid and 4-hydroxybenzoic acid. This method is also applicable for the mono-iodination of 3-(4-methoxyphenyl)-propionic acid and di-iodination of 3-(3-iodo-4-hydroxyphenyl)propionic acid. Thus a solution of KI (1.99 gm, 12 mmole) and iodine (1.22 gm, 4.8 mmole) in 20 ml of H2O was added in a dropwise fashion to a solution of 3-(4-hydroxyphenyl)propionic acid (0.83 gm, 5 mmole) in 100 ml of concentrated aqueous ammonia solution over 20 min. The reaction mixture was stirred for an additional 40 min and then subjected to vacuum to remove the ammonia. The mixture was dried further by rotary evaporation to afford an oily residue. The crude material was partitioned between 2M HCl (50 ml) and ether (2&#xd7;50 ml) and the ether layers were combined and concentrated to give a yellowish solid residue. Flash silica gel chromatography using 95:5 CH2Cl2/MeOH as eluant, concentration of product containing fractions and recrystallization in 1:1 benzene hexane afforded 790 mg of 3-iodo-4-hydroxyphenylpropionic acid. 3-(3,5-Diiodo4-hydroxyphenyl)propionic acid was prepared in similar fashion using 5.2 equivalents of KI and 2.1 equivalents of 12. Following crystallization from toluene, the di-iodo derivative was obtained in 77% yield. 5-iodo-6-hydroxy-2-naphthoic Acid Mono-iodination of 6-hydroxy-2-naphthoic acid and 4-hydroxybenzoic acid was carried out following the procedure of Edgar and Falling (J. Org. Chem. 55, 5287, 1990). Thus, 0.75 gm of (4.34 mmol) was dissolved in 19 ml of MeOH and 1.04 gm of NaI (and 0.27 gm of NaOH was added. The solution was cooled to 0&#x25a1;C and aqueous NaOCl (4% solution, 12.9 ml) was added dropwise over 75 min. The resulting mixture was stirred for 1 hr at 0&#xb0; C. and then treated with 7 ml of 10% aqueous sodium metabisulfite. The mixture was adjusted to pH 7 using 5% HCl and extracted with 40 ml of ether. The organic layer was washed with brine and dried over MgSO4. The solution was concentrated to an off-white solid, that was recrystallized from toluene/CH3OH to provide 0.42 gm of 5-iodo-6-hydroxy-2-naphthoic acid. Reaction of 6&#x2032;-Tostylatractyloside with 1,2-ethylenediamine 6&#x2032;-Tosyl-Atractyloside (25 mg) was dissolved in 2 ml of 1,2-ethylenediamine and the mixture was stirred at 23&#xb0; C. overnight. The 1,2-ethylenediamine was removed in vacuo, the residue was dissolved in MeOH/H2O (2:1) and 10.6 mg of the product was isolated in by RP-HPLC using the conditions described above. Proton nmr and mass spectra indicate the loss of the isovaleryl group. Reaction of N-(6-Deoxy-apo-atractylosyl)-ethanediamine with Bolton Hunter Reagent N-(6-Deoxy-apo-atractylosyl)-ethanediamine (10.6 mg) in 2.75 ml of DMF/DMSO (8:3) was reacted with 60 mg of 4-hydroxyphenylpropionyl-N-hydroxysuccinimidyl ester (Bolton Hunter reagent) at 23&#xb0; C. for 16 hrs. The solution was diluted with water and purified by RP-HPLC to afford 9.1 mg of the desired compound. Lactoperoxidase catalyzed iodination of the compound leads to a quantitative conversion to the di-iodo-apo-atractyloside derivative. Reaction of Atractyloside with Succinic Anhydride Atractyloside.3H2O (255 mg. 0.3 mmole)was dried by co-evaporation with dry pyridine (3&#xd7;5 ml) and then was kept under high vacuum for 16 hrs. The dried atractyloside was dissolved in 6 ml of dry pyridine and 60 mg of succinic anhydride (0.6 mmole) was added. The mixture was kept at 80&#xb0; C. for 30 min, another 60 mg (0.6 mmole) of succinic anhydride was added and the reaction mixture was stirred for an additional 3 hrs. The pyridine was removed in vacuo, and the residue was triturated with 10 ml of MeOH. The 6&#x2032;-O-succinyl-ATR derivative was collected by filtration as a white solid, washed with MeOH and dried overnight over P2O5. Reaction of 6&#x2032;-O-succinylatractyloside With Tyramine and Iodination 6&#x2032;-O-Succinylatractyloside (85 mg) was dissolved in 2 ml of DMF and 28 mg of tyramine and 100 mg of PyBOP was added. The mixture was stirred at 23&#xb0; C. for 16 hrs The crude mixture was subjected to RP-HPLC and the desired amide was isolated in 19.7 mg. Lactoperoxidase catalyzed iodination of the product using the standard conditions described above provides the mono and the di-iodinated products. Synthesis of 4-(4-Hydroxy-3-Methyl)-Butyric Acid To 4 gm (30 mmol) of AlCl3 in 50 ml of 1,2-dichloroethane at 0&#xb0; C. was added 2.7 gm (27 mmol) of succinic anhydride and the mixture was stirred for 20 min. 2-Methylanisole (3.1 ml, 25 mmol) was added, and the reaction mixture was warmed to 23&#xb0; C. and stirred for 12 hrs. The mixture was poured into 300 ml of ice-cold water and the precipitate was filtered off. The precipitate was washed with 2&#xd7;300 ml of water to afford a white solid. The solid material was dried under vacuum to afford 3.51 gm of product that was used in the next reaction. 3-(4-Methoxy-3-methylbenzoyl)propionic acid (4.4 gm, 20 mmol) and 4.49 gm of KOH pellets (80 mmol) were dissolved in 30 ml of ethylene glycol and 3.88 ml of hydrazine hydrate (80 mmol) was added to the stirred solution in four portions. The resulting reaction mixture was heated at 155&#xb0; C. for 24 hrs in an oil bath. After cooling, the reaction mixture was taken up in 100 ml of benzene and washed with 10% aqueous citric acid. The organic layer was washed with another portion of citric acid, dried over anhydrous sodium sulfate, and concentrated in vacuo to afford an oily residue that crystallized upon standing. The material was triturated with hot hexane and the solvent was evaporated off to afford 3.55 gm of a crystalline solid that was homogeneous by silica gel tlc using hexane/ethyl acetate (8:2) as eluting solvent. 4-(4-Methoxy-3-methylphenyl)butyric acid (3.12 gm) was heated in 120 ml of a 1:1 mixture of 48% aqueous HBr/acetic acid at 155&#xb0; C. for 24 hrs. The reaction mixture was cooled to room temperature and was extracted with 200 ml of benzenelether (1:1). The organic layer was dried over anhydrous magnesium sulfate and concentrated to afford a light brown solid residue. The reaction products were separated by silica gel flash chromatography using hexane/ethyl acetate (3:1) as eluting solvent to provide 0.86 mg of 4-(4-hydroxy-3-methyl)-butyric acid as a light yellow solid. Representative Synthesis of Atractyloside Derivatives To a solution of 3-(4-hydroxyphenyl)propionic acid (HPP) (0.498 g, 3.0 mmole) in 10 ml of anhydrous DMF was added carbonyldiimidazole (0.486 g, 3.0 mmole). The mixture was stirred at room temperature for 30 minutes and added in portions (2 ml/hour) to a solution of atractyloside (ATR) (0.086 g, 0.1 mmole) in 1 ml of anhydrous DMF. The reaction mixture was stirred at room temperature overnight and quenched with 1 ml of water. The solvent was evaporated under vacuum and the residue was dissolved in ethyl acetate (75 ml) and water (50 ml). The aqueous layer is separated, extracted with ethyl acetate (3&#xd7;75 ml) and concentrated under vacuum. The residue was dissolved in 1.5 ml of methanol/water(1/1), filtered through a 0.2 mm filter and purified using HPLC with a preparative C-8 column (microsorb, 10&#xd7;250 mm) using a linear gradient elution of 30%-60% solvent B with a flow rate of 2.0 ml/min (solvent A: H2O/HOAc/NH40ac (1.0 M, aq.): {1000/111}; solvent B: CH3OH/HOAc/NH4OAc(1.0 M, aq.): {1000/1/1}). The title compound (compound 36 of Example 11 below) was obtained as a white film (6.2 mg). Example 11 Further Representative ATR Derivatives Following the procedures set forth in Example 10, the following ATR derivatives were prepared. Atractyloside Derivatives Cpd R MS NMR 22 &#x2002;879.3 (M &#x2212; H)&#x2212;&#x2002;799.3 (M &#x2212; SO3&#x2014;H)&#x2212; 1H NMR (CD3OD) &#x3b4;0.63(s,3H),0.95(d,3H), 0.96(d,3H),4.94(s,1H), 5.09(s,1H),7.53(m,2H), 7.60(m,1H),7.93(d,1H), 8.07(d,1H),8.26(d,1H), 8.88(d,1H) 23 1083 (M &#x2212; H)&#x2212;1003 (M &#x2212; SO3&#x2014;H)&#x2212; 1H NMR (CD3OD) &#x3b4;0.65(s,3H,)0.95(d,3H), 0.96(d,3H),5.02(s,1H), 5.13(s,1H),6.61(ddd,2H), 6.69(d,2H),6.86(dd,2H), 7.34(d,1H),8.33(dd,1H), 8.67(s,1H) 24 &#x2002;999 (M &#x2212; H)&#x2212; 1H NMR (CD3OD) &#x3b4;0.94(s,3H),0.95(d,3H), 0.96(d,3H),5.05(s,1H), 5.15(s,1H),6.75(d,1H), 7.04(dd,1H),7.52(d,1H) 25 &#x2002;921 (M &#x2212; H)&#x2212;&#x2002;841 (M &#x2212; SO3&#x2014;H)&#x2212; 1H NMR (CD3OD) &#x3b4;0.75(s,3H),0.95(d,3H), 0.96(d,3H),4.97(s,1H), 5.08(s,1H),6.89(d,2H), 7.52(d,2H),7.64(d,2H), 8.07(d,2H) 26 1047 (M &#x2212; H)&#x2212;&#x2002;967 (M &#x2212; SO3&#x2014;H)&#x2212; 1H NMR (CD3OD) &#x3b4;0.73(s,3H),0.95(d,3H), 0.96(d,3H),4.98(s,1H), 5.09(s,1H),6.93(d,1H), 7.52(dd,1H),7.61(d,2H), 7.98(d,1H),8.08(d,2H) 27 &#x2014; 1H NMR (CD3OD) &#x3b4;0.81(s,3H),0.95(d,3H), 0.96(d,3H),5.04(s,1H), 5.15(s,1H),6.80(d,2H), 7.90(d,2H),7.64(d,2H) 28 1097.1 (M &#x2212; H)&#x2212;1017.0 (M &#x2212; SO3&#x2014;H)&#x2212; 1H NMR (CD3OD) &#x3b4;0.83(s,3H),0.95(d,3H), 0.96(d,3H),5.03(s,1H), 5.15(s,1H),6.84(d,1H), 7.88(dd,1H),8.33(d,1H) 29 1043.2 (M &#x2212; 2H + Na)&#x2212;1021.2 (M &#x2212; H)&#x2212; 1H NMR (CD3OD) &#x3b4;0.66(s,3H),0.95(d,3H), 0.96(d,3H),4.93(s,1H), 5.05(s,1H),7.22(d,1H), 7.87(d,1H),8.04(dd,1H), 8.08(d,1H),8.50(d,1H) 30 &#x2002;944 (M &#x2212; H)&#x2212;&#x2002;864 (M &#x2212; SO3&#x2014;H)&#x2212; 1H NMR (CD3OD) &#x3b4;0.97(s,3H),0.98(d,6H), 5.06(s,1H),5.17(s,1H), 5.70(d,2H),7.03(d,2H). 31 1070 (M &#x2212; H)&#x2212;&#x2002;990 (M &#x2212; SO3&#x2014;H)&#x2212; &#x2014; 32 1116 (M &#x2212; SO3&#x2014;H)&#x2212; &#x2014; 33 1125 (M &#x2212; H)&#x2212;1147 (M &#x2212; 2H + Na)&#x2212; 1H NMR (CD3OD) &#x3b4;0.95(s,3H),0.96(s,6H), 5.05(s,1H),5.15(s,1H), 7.58(s,2H) 34 &#x2002;923.2 (M &#x2212; 2H + Na)&#x2212;&#x2002;901.3 (M &#x2212; H)&#x2212; &#x2014; 35 1049.3 (M &#x2212; 2H + Na)&#x2212; &#x2014; 36 &#x2002;873 (M &#x2212; H)&#x2212;&#x2002;895 (M &#x2212; 2H + Na)&#x2212; 1H NMR (CD3OD) &#x3b4;0.94(s,3H),0.96(d,6H), 5.05(s,1H),5.16(s,1H), Dihydroactractyloside Derivatives Cpd R MS NMR 37 &#x2002;897 (M &#x2212; 2H + Na)&#x2212;&#x2002;875 (M &#x2212; H)&#x2212; 1H NMR (CD3OD) &#x3b4;0.93(s,3H),0.95(d,3H), 0.96(d,3H),1.08(d,3H), 6.69(d,2H),7.02(d,2H), resonances from alkenic protons absent 38 1105 (M &#x2212; 2H + Na)&#x2212;1083 (M &#x2212; H)&#x2212; 1H NMR (CD3OD) &#x3b4;0.93(s,3H),0.95(d,3H), 0.96(d,3H),1.08(d,3H), 6.74(d,1H),7.05(dd,1H), 7.53(d,1H), resonances from alkenic protons absent Apoactractyloside Derivatives Cpd R MS NMR 39 &#x2002;831 (M &#x2212; H)&#x2212; 1H NMR (CD3OD) &#x3b4;1.00(s,3H),5.07(s,1H), 5.17(s,1H),6.69(d,2H), 7.03(d,2H) 40 1083 (M &#x2212; H)&#x2212;1105 (M &#x2212; 2H + Na)&#x2212; &#x2014; Example 12 Binding Assays Using Recombinant huANT3 A. GST-huANT3 Recombinantly Produced in E. coli: Following arabinose induction, transformed or sham transformed (vector only) E. coli were collected by centrifugation at 2000 g for 10 min. The bacterial pellets were resuspended in MSB, to which lysozyme (100 gg/ml) was added. After 20 min at room temperature, the lysates were subjected to one freeze-thaw cycle followed by sonication as described above. The resultant membrane preparation was used for binding assays. To estimate maximal binding and the extent of overexpression of the huANT3, 25 &#x3bc;g of membrane protein was incubated with varying amounts of [32P]ATP (1-500 &#x3bc;M) in binding buffer (120 mM KCl, 10 mM Tris, 1 mM EDTA, pH 7.4) for 2 hr at room temperature. The membranes with bound ATP were sedimented by centrifugation at 5000 g for 5 min, and washed once with binding buffer. Membrane pellets were then mixed with 5 ml scintillation cocktail and counted. The results are presented in Table 2. TABLE 2 Saturation Binding of [32P]ATP to E. coli Membranes cpm bound by: Sham huANT3- [ATP], uM Transformed Cells Producing Cells 0.1 109 191 0.5 95 49 1.0 147 325 5.0 123 N.D. 10 214 263 50 549 2,727 100 718 5,772 500 2,140 9,715 N.D., not determined The data presented in Table 2 indicate that the affinity of the ATP binding was &#x2dc;6 &#x3bc;M. ATP binding was completed abolished by the addition of atractyloside (10 &#x3bc;M) to the assay. These results support the contention that the measured ATP binding was predominantly to recombinantly produced ANT3 protein. Agarose-glutathione beads were incubated with solubilized (using Dnase, Rnase and 0.1% Triton X-100; see Example 1, section D) bacterial lysate (see Example 2), and substituted for the E. coli membranes in binding assays. Best results (i.e., more specific binding) were seen when the beads were preincubated with bovine serum albumin (BSA, 0.1%) Specific ATP binding (1,070 cpm) was also observed in this experiment (compare to nonspecific binding of 279 cpm in the presence of 10 mM non-radiolabeled ATP). B. huANT3 from a Sf9/Baculovirus Expression System Sf9 cells expressing huANT3 were grown in spinner flasks. The cells were harvested by centrifugation at 2,000 g for 5 min. The cell pellet was resuspended in MSB and subjected to 3 freeze-thaw cycles. Cell membranes and debris were removed by centrifugation at 600 g for 5 min; mitochondria were collected by centrifuging the supernatant at 20,000 g for 15 min. The mitochondrial pellets were suspended in MSB, and used for binding assays as described above. Homologous competition of [32P]ATP binding was performed using 25 &#x3bc;g mitochondrial protein per assay. As illustrated in FIG. 7, ATP bound to the mitochondria with Kd=13 &#x3bc;M, a value consistent with ATP binding to ANT. Furthermore, the ATP binding was displaced by low concentrations of atractyloside (FIG. 8). Homologous competition binding assays using [125I]atractyloside revealed specific binding with Kd=12 nM (FIG. 9). These findings are consistent with the presence of functional huANT3 in the mitochondrial preparations. His-tagged huANT3 protein was purified from baculovirus-infected Sf9 solubilized cell lysates using Ni-agarose magnetic beads; Sf9 cells that had not been infected were used as negative controls. The beads were incubated with [32P]ATP (1 or 100 &#x3bc;M) for 2 hr. The beads were washed and then counted to determine the amount of bound ATP. As shown in Table 3, the [32P]ATP binding was significantly higher in material recovered from the infected cells than in the controls. Binding saturation had essentially been achieved with 1 &#x3bc;M ATP. TABLE 3 Binding of [32P]ATP to Purified His-Tagged huANT3 cpm bound by: Control huANT3- [ATP], uM (Uninfected) Cells Producing Cells 1.0 43 149 100 30 160 Example 13 Competitive Binding Assays Atractyloside analogs (Example 6; Table 1; see also Examples 7-9) were used in pseudo-homologous competition binding assays using Sf9/huANT3 mitochondria. Mitochondria (see Example 12, 25 &#x3bc;g/tube) were incubated with 0.5 nM [125I]atractyloside and varying concentrations of non-radiolabeled atractyloside or fluorescent atractyloside derivatives as described above (FIGS. 7-9). The results (Table 4) show that three of the atractyloside derivatives (MANT-, Pyrene- and Coumarin-atractyloside) had relative binding affinities similar to that of authentic atractyloside (IC50&#x3c;500 mM relative to atractyloside). Each of these derivatives is fluorescent, and may therefore be useful as detectable ligands for binding assays. TABLE 4 Competitive Binding Assays Using [125I]Atractyloside cpm [125I]Atractyloside bound in the presence of: [ATR Derivative], nM ATR COU-ATR PYR-ATR MANT-ATR 0.0 227 437 437 437 1.0 224 391 350 &#x2014; 50 146 &#x2014; &#x2014; &#x2014; 10 &#x2002;42 371 349 229 50 &#x2002;26 &#x2014; &#x2014; &#x2014; 100 &#x2014; 277 362 195 1,000 &#x2002;36 174 238 &#x2014; 5,000 &#x2002;45 &#x2014; &#x2014; &#x2014; 10,000 &#x2003;0 100 &#x2003;0 &#x2014; Abbreviations and symbols: ATR, atractlyoside. COU-ATR, Coumarin-atractyloside, (Table 1, compound 3). PYR-ATR, Pyrene-atractyloside, (Table 1, compound 4). MANT-ATR, MANT-atractyloside, (Table 1, Roux et al. 1996 Anal. Bioch. 234:31) - - - , not determined. Atractyloside analogs (Example 6; Table 1; see also Examples 7-9) were also used in pseudo-homologous competition binding assays using T. ni/huANT3 mitochondria or bovine mitochondria. Mitochondria from noninfected T. ni cells, or T. ni cells infected with a baculovirus expressing huANT3 (see Example 3) were prepared as follows: T. ni cells were prepared by a subcontractor (PharMingen, San Diego, Calif.) as portions of about 250 mg of cells per tube. Each portion was suspended in 1 ml of MSB with protease inhibitors (leupeptin, final concentration 10 ug/ml; pepstatin, final concentration 10 ug/ml; aprotinin, final concentration, 2 ug/ml; phenylmethylsulfonyl fluoride, [PMSF], final concentration, 100 &#x3bc;M; all from Sigma Chemical Co., St. Louis, Mo.). The resuspended cell suspensions were frozen and thawed twice, then homogenized using a rotating teflon-coated probe and a close-fitting glass container (10 passes). The cellular homogenate was centrifuged (3,700 rpm, approximately 1,500&#xd7;g) at 4&#xb0; C. for 5 minutes; this supernatant from the first spin was saved. The pellet was washed with about 500 &#x3bc;l of MSB with protease inhibitors, centrifuged (3,800 rpm, approximately 1,600&#xd7;g) at 4&#xb0; C. for 5 minutes, and supernatant from this spin was combined with the supernatant from the first spin. The combined supernatant was centrifuged (14,000 rpm, approximately 20,800&#xd7;g) at 4&#xb0; C. for 15 minutes, and the pellet was resuspended in 300 &#x3bc;l of a 1:1 solution of (a) 20 mM MOPS and (b) MSB, wherein both (a) and (b) contain the previously described protease inhibitors. The resultant suspension was frozen and thawed three times. Bovine mitochondria were prepared as follows: Essentially all of the fat and cholesterol in clogged arteries was removed from two bovine hearts which were then cut into 1-inch cubes. The cubes were ground in a meat grinder using the fine setting. Three hundred (300) gm portions of the ground heart were weighed out and, to each was added 400 ml of Isolation Buffer (IB; 250 mM sucrose, 1 mM sodium succinate, 0.2 mM K+ EDTA, 10 mM Tris-base, pH 7.8). (All buffers were filter sterilized, and column buffers were degassed, and, unless otherwise noted, all steps were carried out at 0 to 4&#xb0; C. on ice or in pre-cooled rotors and centrifuges.) The preparations were mixed in a blender two times for 15 seconds on high setting and, in between and after blends, the pH was adjusted to 7.8 with 2M Tris-base. The homogenate was centrifuged for 20 minutes at 1,200&#xd7;g, and the supernatant was poured through two layers of cheese cloth and adjusted to pH 7.8 with 2M Tris-base. The supernatant was then centrifuged for 30 minutes at 11,000&#xd7;g. The supernatant was decanted, and the buff-colored outer pellet was dislodged with about 10 ml of IB and discarded. The brown inner pellet (heavy mitochondria) was resuspended in IB (about 10 ml per pellet). The pellets were homogenized in a glass-teflon homogenizer (2 passes at high drill speed). Samples were combined and centrifuged for 30 minutes at 11,000&#xd7;g. The supernatant was decanted, and the pellets were resuspended in 60 ml of IB per 900 gm of ground heart. This centrifugation step was repeated and the pellets were finally resuspended in IB (60 ml per 900 gm of ground heart). One kilogram of beef heart typically yields about one (1) gram of mitochondria. The mitochondrial preparations were divided into aliquots (typically, 50 &#x3bc;l for T. ni mitochondria or 20 &#x3bc;l for bovine mitochondria) and then either used directly in assays or flash frozen and stored at &#x2212;80&#xb0; C. The total protein content in the mitochondrial preparations was determined using the enhanced protocol (30 minutes at 60&#xb0; C.; see http://www.ruf.rice.edu/&#x2dc;bioslabs/methods/protein/BCA.html) of the bicinchoninic acid (BCA) assay (available in kit form from Pierce, Rockford, Ill.). In the &#x201c;Tube Assay,&#x201d; mitochondria (from about 1 to 10 &#x3bc;g of total protein) were resuspended in 100 &#x3bc;l of Tris-KCl buffer with 0.1% BSA, pH 7.4. 125I-labeled compound 24 (Example 11) was added to a final concentration of 0.5 nM. When used, competitors were added at these concentration ranges: unlabeled atractyloside or compound 24, final concentration from 5 nM to 10 &#x3bc;M; unlabeled ADP (a lower affinity competitor) was added at a final concentration of 500 nM to 1 mM. The reaction mixes were incubated on ice for 60 minutes and then pelleted by centrifugation (approximately 16,000&#xd7;g) for 11 minutes at 4&#xb0; C. Unbound 125I-compound 24 was removed by aspiration. The pellets were contact-washed with Tris-KCl buffer, pH 7.4, and recentrifuged. The resultant pellets were aspirated and the radioactivity (dpm) in each was determined by gamma counting. Representative results are shown in FIGS. 11-18. The data presented in FIGS. 11 and 12 show that mitochondria (5 &#x3bc;g of protein/tube) from both beef heart (FIG. 11) and T. ni cells expressing huANT3 (FIG. 12) specifically bind 125I-compound 24 in a manner that is inhibited by increasing concentrations of unlabeled compound 24, but, as expected, little or no binding is seen when mitochondria are excluded from the reaction mixes. FIGS. 13 and 14 show competitive inhibition of 125I-compound 24 binding to mitochondria (1 &#x3bc;g of protein/tube) from beef heart (FIG. 13) and T. ni cells expressing huANT3 (FIG. 14) by compound 24 that is not detectably labeled, unlabeled atractyloside (ATR), and unlabeled adenosine diphosphate (ADP). In both instances, ATR and compound 24 yield comparable competition curves, although ATR appears to have a slightly higher affinity than compound 24. However, both ATR and compound 24 bind with much higher (about 1,000 fold) affinity than the low affinity ANT ligand ADP. FIG. 15 shows the competitive inhibition, by unlabeled ATR, of binding of 125I-compound 24 to mitochondria (1 &#x3bc;g of protein/tube) from T. ni cells expressing huANT3 and control T. ni cells (i.e., non-infected T. ni cells). As shown in FIG. 15, there was only slight inhibition of 125I-compound 24 binding to control (nontransformed) mitochondria by higher concentrations of ATR. In contrast, binding of 125I-compound 24 to mitochondria from T. ni cells expressing huANT3 was increasingly inhibited by higher concentrations of ATR. FIG. 16 shows competitive inhibition, by unlabeled compound 24 and by bongkrekic acid (BKA), of 125I-compound 24 binding to mitochondria (1 &#x3bc;g of protein/tube) from beef heart. BKA effectively displaced labeled compound 24, albeit with a slightly lower affinity than unlabeled compound 24. FIG. 17 shows competitive inhibition of 125I-compound 24 binding to beef heart mitochondria by either of the ATR derivatives, compound 23 (see Example 11) and compound 28 (see Example 11). As shown in FIG. 17, ATR exhibited an IC50 of approximately 44 nM, compound 23 an IC50 of approximately 105 nM, and compound 28 an IC50 of approximately 695 nM. In FIG. 18, data are presented depicting competitive inhibition of 125I-compound 24 binding to beef heart mitochondria by the ATR derivative compound 5 (see Example 7). As shown in FIG. 18, the IC50 for compound 5 was approximately 3.3 &#x3bc;M. Competitive binding assays were also performed using recombinant His-tagged huANT3 (see Example 3) immobilized on Ni beads (FIG. 19) instead of mitochondria. To prepare the bead-immobilzed huANT3, mitochondria from T. ni cells infected with a baculovirus expressing huANT3 (see Example 3) were solubilized with 0.5% O-glucopyranoside in the presence of 0.5 nM 125I-compound 24, Ni-agarose beads (Qiagen, Hilden, Germany), and various concentrations of ATR or BKA as unlabeled competitors. After 1 hour at 4&#xb0; C., the beads were washed and radioactivity that remained associated with the beads was counted. (Background binding of [125I] compound 24 to Ni-agarose beads (Qiagen) in the absence of His-tagged huANT3 was approximately 700-800 cpm and was not subtracted from the radioactivity shown in FIG. 19.) The results (FIG. 19) show that both ATR and BKA effectively compete with compound 24 in a manner similar to that observed in assays using intact mitochondria (cf. FIGS. 14 and 15). Example 14 High Throughput Screening Assay for Compounds Targeted to ANT Proteins and Polypeptides The recombinantly produced ANT proteins, ANT fusion proteins and detectably labeled ANT ligands described herein are incorporated into automated assay systems. Such automated systems are useful for high throughput screening (HTS) of candidate ANT-binding compounds or chemical libraries comprising such compounds. Such compounds may be further characterized and developed as drug candidates and drugs useful for preventing, treating or curing diseases or disorders resulting from the overexpression or dysfunction of one or more ANT proteins or from the overexpression or dysfunction of a factor that positively regulates or stimulates ANT proteins. A preferred element of many automated assay systems is the incorporation of a target molecule (in the present instance, an ANT protein) into a 96-well plate. This format is readily adaptable for use in a variety of automated label detection systems. For HTS assays, robotic label detection systems are preferred. As one example of an HTS comprising the elements describes herein, the GST-huANT3 fusion protein of Example 2 is contacted with REACTI-BIND&#x2122; glutathione-coated 96-well plates (Pierce). Glutathione coated strip-well plates are preferably used for assays comprising radiolabeled ANT ligands (e.g., iodinated atractyloside derivates; see Example 7), whereas black opaque glutathione coated 96-well plates are preferred for assays comprising fluorescent ANT ligands (such as are described in, e.g., Examples 6-9); both types of glutathione coated plates are commercially available (Pierce). In a typical assay, 1 to 50 ug of GST-huANT3 protein (i.e., total solubilized protein prepared as in Example 2) is added per gluthathione-coated well to each well of a 96-well plate. Iodinated atractyloside derivate (125I-ATR) is added to the wells (0.5 nmol/well). In a control experiment, unlabeled atractyloside (ATR; Sigma) is used as a &#x2018;mock&#x2019; drug at a concentration of from about 1 to about 10,000 nM. That is, unlabeled ATR is used to displace a labeled atractyloside derivative (e.g., 125I-ATR). Unlabeled ATR thus acts as a positive control for an HTS in which various compounds are screened for their ability to displace a labeled ANT ligand. As an example of the automated label detection systems used in the HTS assays of the Example, when the detectably labeled ANT ligand of the assay is 125I-ATR, an automatic gamma counter is used. Alternatively, 125I-ATR can be used in scintillation proximity assays (SPA). For example, a GST-huANT fusion protein is contacted with ScintiStrip 96-well plates coated with glutathione (EG&#x26;G Wallac). The polystyrene of these plates contains a scintillating agent that emits beta radiation when excited by a gamma emitter in close proximity thereto. The beta radiation is then detected by any appropriate automatic beta counter. When fluorescent ANT ligands are used in the HTS assay, an automatic fluorescence counter is used and may be, for example, a FLUOROCOUNT&#x2122; Counter (Packard Instrument Company, Meriden, Conn.).\",\n          \"DETAILED DESCRIPTION OF THE INVENTION Referring now to FIG. 1, there is illustrated a system diagram for the presently disclosed system. There are illustrated three transactional systems, 102, 104 and 106. Transaction system 102 is comprised of a router 108 that is interfaced with a network mesh 110, which network mesh 110 is local to the system 102. The network mesh 110 allows the router 108 to interface with various system nodes. There is provided a host system node 114 that is the node at which a transaction arises. Also attached to the network mesh 110 is an archival server 116 and a conversion server 118, the function of which will be described hereinbelow. Since the host system 114, the servers 116 and 118, and the router 108 are all in the same network mesh 110, they communicate in a common protocol to that of the network mesh 110, and also may have the ability to communicate over the network mesh 110 with other network protocols that presently exist and any future protocols that would be developed at a later time. This allows data packets to be transferred between the various nodes on the network mesh 110. The router 108 is also provided with various media interfaces 120 and 122. Media interface 120 allows the router 108 to interface with a private network 124 which could be any type of private network such as a local area network (LAN) or a wide area network (WAN). This private network 124 can have other systems attached thereto such that the router 108 can forward data through this network 124. The media interface 122 is interfaced with a global public network (GPN) 126, which is typically referred to as the &#x201c;Internet.&#x201d; This allows the router 108 to interface with the GPN 126 and the multitude of resources associated therewith, as are well known in the art. The system 106 is similar to the system 102 in that it has associated therewith a central router 128. The router 128 is interfaced with a network mesh 130, which network mesh 130 is also interfaced with a universal ID server 132 and a universal web server 134. The router 128 is also interfaced with the GPN 126 with a media interface 136. As such, the router 108 could effectively interface with the router 128 and the network resources in the form of the universal ID server 132 and the universal web server 134, the operation of which will be described hereinbelow. The third system, the system 104, is comprised also of a central router 136, similar to the routers 108 and 128. The router 136 is interfaced on the local side thereof to a local network mesh 138. Local network mesh 138 has associated therewith three host or transaction nodes, a transaction node 140 associated with a system A, a transaction node 142 associated with a system B and a transaction node 144 associated with a system C, the transaction nodes 140-144 all interfacing with the network mesh 138. In addition, the system 104 has associated with its local network mesh 138 a core ID server 146, an account ID server 148, a conversion server 150 and an archival server 152. Router 136 is operable to interface with the private network 124 via a media interface 154, interfaced with the GPN 126 via a media interface 156 and also to a transmission medium 158 through a media interface 160. The transmission medium 158 is an application specific medium that allows information to be transmitted to an end user output device 162 through a media interface device 164 or to be received therefrom. As will be described hereinbelow, this end user output device might be a fax machine, and the transmission medium 158 a telephone system or the such that allows data in the form of facsimile information to be transmitted from the router 136 through the transmission medium 158 to the end user output device 162 for handling thereof. The transmission medium 158 may be merely a public telephone network (PTN) that allows the number of the end user output device 162 to be dialed, i.e., addressed, over the network or transmission medium 158, the call answered, a hand shake negotiated, and then the information transferred thereto in accordance with the transaction that originated in the access to the transmission medium 158. The transmission medium could include a satellite transmission system, a paging transmission system, or any type of other medium that interfaces between one of the routers and a destination/source device. This will be described in more detail hereinbelow. In addition to allowing the router 136 to directly interface with an end user device 162 via the interface 160, there is also provided a fifth transaction node 166 that is disposed on the GPN 126 and has access thereto via a media interface 168. The transaction node 166 is operable to interface with any of the routers 108, 128 or 136. In operation, as will be described in more detail hereinbelow, each of the transaction nodes 114, 140, 142, 144, is able, through the use of the disclosed system, to complete a transaction on the system and utilize the system to send information to or retrieve information from another transaction node on the system. In the private network 124, there is illustrated a phantom line connection between the router 108 and the router 136. In order to facilitate a connection between, for example, transaction node 140 for system A and, for example, transaction node 114 for system D, it is necessary to create a unique data packet of ID's that can be transmitted via the router 136 through the network 124 and to, transaction node 114 for system D. This unique proprietary transaction packet that is transmitted utilizes various distributed resources in order to allow this transaction packet to be processed within the system and transmitted over a defined route that is defined in an initial transaction profile that is stored in the system at various places in a distributed manner. This will be described in more detail hereinbelow. Additionally, the router 136 could also allow one of the transaction nodes 140-144 to interface with the router 108 through the GPN 126 such that a transaction can be completed with the transaction node 114 for system D. This would also be the case with respect to interfacing with the universal ID server 132 or the universal web server 134, the transaction node 166 for system E or with the end user output device 162. Each of the routers 108-128 and 136 have associated therewith a data cache 170, 172 and 180, respectively. Whenever a particular router in one of the systems 102-106 has data routed thereto, data may be cached, then processed either outside the system or internal to the system, or the data is maintained in the cache for later transmittal. The general operation of a transaction would require one of the transaction nodes to determine what type of transaction was being made and the destination of that transaction. If it were determined that the transaction would be between transaction node 140 and transaction node 114 on system 102, a unique transaction packet would be generated that would have unique transaction IDs associated therewith that defined the routing path in the system and the transaction associated therewith while processing what needed to be done between the two transaction nodes. As will be described hereinbelow, this transaction is distributed over the entire system, with only a portion thereof disposed at the transaction node itself. It is the unique transaction codes or IDs that are embedded in the information that is sent to the system that allows the transaction to be carried out in a distributed manner at all of the various elements along the path of the transaction. As a further example, consider that transaction node 114 for system D utilizes a different database than transaction node 140, i.e., the two nodes are in general incompatible and require some type of conversion or calculation to interface data and transactional information. With the transaction determined at the transaction node originating the transaction, and a unique transaction packet created with the unique transaction information contained therein, all the necessary information to complete the transaction and the routing of data follows the transaction packet through the system. This, in association with information disposed in other elements or nodes of the system, allows the transaction to be completed in a distributed manner. In particular, the transaction packet is transmitted to various nodes which perform those discrete functions associated with the transaction packet for the purpose of converting, routing, etc. to ensure that the transaction packet arrives at the correct destination and achieves the correct transaction. Referring now to FIG. 2, there is illustrated a diagrammatic view of the system 104 and a transaction between transaction nodes on the network mesh 138, which network mesh 138 is illustrated as a general network. It is noted that network mesh 138 could be any type of network, such as an Ethernet, a satellite, a Wide Area Network or a Local Area Network. The transaction is illustrated as occurring between transaction node 140 for system A and transaction node 142 for system B. Although the details of a transaction will be described in more detail hereinbelow, this transaction is illustrated in a fairly simple form for exemplary purposes. The transaction is initiated at transaction node 140 to generate information that will be transmitted to transaction node 142 for system B. When the transaction is generated, the type of transaction is determined, the manner in which the information is to be transmitted to transaction node 142 is determined and the route that it will take is determined, and all of the information is embedded in a transaction packet. This is a predetermined transaction that is completed with the use of IDs that are utilized by various systems on the network to appropriately route information and to possibly perform intermediate processes on the packet and the data associated therewith. Further, transaction node 140 has associated therewith information to allow the data that needs to be transferred to be transferred in a predetermined manner in accordance with a known profile of how transaction node 142 wants the transaction to be completed and in what form the data is to be run. For example, it may be that transaction node 140 desires to order a particular product in a particular quantity from transaction node 142. The data associated with the transaction or transactions would be assembled, in accordance with a predetermined transaction profile as determined by the system beforehand and in accordance with a business relationship between the transacting parties, and forwarded to the appropriate location in the appropriate format to be received and processed by transaction node 142. These are transactions that transaction node 140 typically receives and handles in their day-to-day business. As such, all transaction node 142 desires to do is to receive the transaction in a manner that is compatible with its operational environment. By using various conversion algorithms, routing algorithms and the such, the transaction can be effected between the two systems in a distributed manner. Although not illustrated in FIG. 2, and as will be described hereinbelow, there is an initial setup that defines a profile for a transaction and a profile for a transaction node in the system. Whenever it is desirable for transaction node 140 for system A, for example, to create a business relationship with transaction node 142, this business relationship will be set up on the system as a transaction profile. Once the transaction node is set up, the various information that is necessary for the two transaction nodes to converse will be set up on the system and &#x201c;propagated&#x201d; over the system such that the transaction profile is &#x201c;distributed&#x201d; about the system. This will be described in more detail hereinbelow. In the transaction illustrated, the first step is to create the transaction packet and route it to the router 136. This is facilitated over a path &#x201c;A&#x201d; through the network 138. The router 136 is then operable to examine the contents of the transaction packet and the IDs associated therewith with a look-up table (LUT) 202. In the LUT 202, the router 136 determines that this transaction packet is associated with a particular transaction and that the transaction requires that any information for this type of transaction being received from transaction node 140 be transferred to the conversion server 150. The router 136 then reassembles the packet and transfers this transaction packet over the network 138 to the conversion server on a path &#x201c;B&#x201d; and also stores the information in its associated data cache. Router 136 has, as such, &#x201c;handed off&#x201d; the transaction to the conversion server 150 and then created a record in its local cache 180. (This could be stored in non local cache also, such as at the archive server 152.) It is noted that the transaction packet may be converted at each node along the path, depending upon the transaction and the action to be taken at each node. At the conversion server 150, the received packet from the path &#x201c;B&#x201d; is examined to determine information associated therewith. The conversion server 150 also has an LUT associated therewith, an LUT 204. The conversion server 150 recognizes that the information came from the router 136 and has a predetermined transaction associated therewith merely from examining the IDs, processing them through the LUT 204 and then determining what type of process is to be performed on the data packet and the contents thereof and where to forward them to. For example, the operation of the conversion server could be as simple as converting the data from an SML language to an XML language, it could be utilized to translate between languages, or any other type of conversion. Primarily, the contents of the transaction packet and associated data that was retrieved from the database associated with transaction node 140, associated with the transaction therein, may require conversion in order to be compatible with the destination transaction node 142. The conversion server 150 places the data in the appropriate format such that it will be recognized and handled by the transaction node 142. The specific manner by which this conversion is achieved is that setup in the initial setup when the business relationship between the two transaction nodes 140 and 142 was defined. The reason that this particular conversion was performed is that the agreed upon transaction set these parameters in the system for this portion of the transaction which is stored in the LUT 204 at the conversion server 150. After the conversion server 150 has processed data in accordance with the transaction IDs within the data packet, the transaction data packet is then reassembled with the destination address of the router 136 and transferred back to the router 136 via a path &#x201c;C,&#x201d; which may also modify the transaction packet to some extent, as will be described in more detail hereinbelow. Router 136 recognizes this data packet as having come from the conversion server 150 and performs a look-up in the LUT 202 to determine that this particular transaction, determined from the transaction IDs associated therewith, requires data received from conversion server 150 to be transferred to the transaction node 142. The data is then assembled in a transaction packet and transmitted to transaction node 142 along the path &#x201c;D.&#x201d; Additionally, the previous cached data in cache 180 is replaced by the new data that is forwarded to transaction node 142. In some instances, it is desirable to archive the data associated with this transaction. This is facilitated by the archive server 152, wherein the data transmitted to the transaction node 142 along the path &#x201c;D&#x201d; is also transferred to the archive server 152 along a path &#x201c;D&#x2032;.&#x201d; As will be described hereinbelow, the entire transaction is determined by a unique transaction packet that has embedded therein routing information in the form of ID packets, data, etc. The ID packets are unique numbers that are recognized by each of the nodes in the network to which it is routed. By recognizing an ID packet, the particular router 136, conversion server 150, etc., has information associated therewith that allows it to perform the portion of the transaction associated with that particular node, i.e., the conversion server 150 performs a conversion and then routes it back to the router 136. In this manner, the originating transaction node need not embed all the transaction information therein and actually effect a direct connection, through a network or otherwise, to the destination transaction node in order to complete the transaction, nor does the originating transaction node require all the transaction information to be associated therewith. As such, the transaction itself is distributed throughout the network in a predetermined manner. Referring now to FIG. 3, there is illustrated a diagrammatic view of the manner in which the packet is modified through the transaction. An originating transaction packet 302 is generated at the originating transaction node 140. This is then transferred to the router 136, wherein the router 136 evaluates the transaction packet, determines where it is to be sent and then converts the packet to a &#x201c;conversion transaction packet&#x201d; 304, which is basically the transaction packet that is designated by the router 136 for transmittal to the conversion server 150 via the path &#x201c;C&#x201d; with the necessary information in the form of ID packets, data, etc., that will be required by the conversion server 150 to perform its portion of the transaction, it being noted that the transaction packet may undergo many conversions as it traverses through the system. The conversion server 150 then processes the data contained in the conversion transaction packet and then, after processing, converts it to a router transaction packet 306 for transmission back to the router 136. The router 136 then converts this to a destination transaction packet 308 for transmission to the destination. It is noted that the conversion server 150, after receiving the router transaction packet, has no knowledge of where the destination of the transaction packet will be eventually, as it has only a small portion of the transaction associated therewith. All it is required to know is that the transaction packet requires a certain action to be taken, i.e., the conversion process, and then this transaction packet must be transmitted back to the router 136. Since this transaction packet always has associated therewith the necessary ID information as to the transaction, each node that the transaction packet is transferred to or through will recognize where the transaction packet came from, what to do with the transaction packet and then where to transfer it to. Each node then will transfer the transaction packet to the destination eventually in a &#x201c;daisy chain&#x201d; manner. Referring now to FIGS. 4A-4D, there are illustrated diagrammatic views of the packet transmission which facilitates transmission of a transaction packet between transaction nodes or even nodes in a network. Prior to describing the formation of and transmission of the transaction packet, the distinction must be made between a &#x201c;data&#x201d; packet and a &#x201c;transaction&#x201d; packet. In general, data is transmitted over a network in a packetized manner; that is, any block of data, be it large or small, is sent out in small &#x201c;chunks&#x201d; that define the packet. However, the packet is a sequence of fields that represent such things as headers, footers, error correction codes, routing addresses and the data which is sent as an intact &#x201c;unit.&#x201d; Sometimes, the data contained in the packet is actually a small portion of the actual overall data that is to be transmitted during a data transfer operation of some predetermined block of data. These packets typically have finite length fields that are associated therewith and some even have longer variable length fields for the data. However, for large blocks of data, the data will be divided up into smaller sub-blocks that can be sent in each packet. Therefore, for example, a large block of data would be sent to the network controller for transmission over a compatible network to a network controller on a receiving device for assembly thereat. The block of data, if it were large enough not to be compatible with a single data packet, would be divided up into sub-blocks. Each of these sub-blocks is disposed within a data packet and transmitted to the receiving device which, once receiving it, will ensure that this data is then sequenced into a combined block of data. If, for example, one of the data packets had an error in it, this would be communicated back to the transmitting device and that particular data packet possibly retransmitted or the entire block of data retransmitted. Since each data packet has a sequence number when sending a group of data packets that represent one block of data, the individual packets that each contain a sub-block of data can be reassembled to provide at the receiving device the entire packet. This packetising of data is conventional. With specific reference to FIG. 4A, there is illustrated the manner by which the data is actually transmitted. Typically, network controllers are arranged in multiple &#x201c;layers&#x201d; that extend from an application layer down to a transport or network layer that inserts the data into a new format that associates a data field 402 with a header 406. The embodiment of FIG. 4A is referred to as an IPX data flow controller. As noted hereinabove, whenever a computer is attached to a network, it becomes a node on a network and is referred to as a workstation. When information is sent between the nodes, it is packaged according to the protocol rules set up in the network and associated with the network controller. The rules are processes that must be complied with to utilize the operating system protocol layers&#x2014;the application layer, the presentation layer, the session layer, the transport layer, the network layer, the data link and the physical layer&#x2014;in order to actually output a sequence of logical &#x201c;1's&#x201d; and &#x201c;0's&#x201d; for transmission on the network mesh. At the network layer, the data field 402, which was generated at the application layer, is associated with the header 406. This particular configuration is then sent down to the data link which is illustrated as a block 408 which basically associates the data field 402 with a UDP header 410 and then translates this data field 402, UDP header 410 and the IPX header 406 which is then translated into a new data field 412. This new data field 412 at the datalink is then associated with IPX header 414 which is then again converted to a data field 414 associated with a media access controller (MAC) header 416 which is then compatible with the physical layer. The physical layer is the network mesh. This data field 414 and header 416 are what is transferred to the network and what is received by the receiving device. The receiving device, upon receiving the MAC header 416, recognizes an address as being associated with that particular receiving device and then extracts the data field 414 therefrom, which is again utilized to extract the header 414 for examination purposes and, if it is compatible with the device, then the data field 412 is extracted and so on, until the data field 402 is extracted. Of course, data field 402 is only extracted if the data packet comprised of the MAC header 416 and data field 414 is that directed to the particular receiving device. It is noted that all devices on the network mesh will receive the data packet, i.e., they can all &#x201c;see&#x201d; the data packet traveling across the network mesh. However, the data will only be extracted by the addressed one of the devices on the system. In this manner, a unique Universal Resource Locator (URL) can be defined for each device on the system. Typically, in an Ethernet environment, each network controller will have a unique serial number associated therewith, there never being two identical serial numbers in any network controller or network card for the Ethernet environment. In the transaction packet, there are provided a plurality of smaller packets that are referred to as &#x201c;ID packets&#x201d; that are generated in the application level. This basically comprises the data field 402. The transaction packet is formulated with a plurality of these ID packets and data that are generated at the transaction node and modified at other nodes. This transaction packet, once formed, is transmitted to the network level in such a manner that the appropriate header will be placed thereon to send it to the appropriate location. Therefore, the software or process running at the particular transmitting node on the network will have some type of overhead associated therewith that defines the address of the source node on the network and also the address of the destination node. Therefore, when data is received by any one of the nodes, it can recognize the defined field for the destination address as being its address. Further, it can utilize the information in the source address field, which is at a particular location in the data packet, to determine where the data came from. Referring specifically to FIG. 4B, there is illustrated a diagrammatic view of an ID packet 430. The ID packet 430, in the present disclosure, is comprised of a plurality of IDs, a core ID 432, a device ID 434 and an item ID 436. The core ID 432 is that associated with a particular entity on the network such as a corporation. For example, if a corporation had a profile set up, it would be assigned this particular core ID when initiated. The device ID 434 is the unique ID of the device on the network. The core ID could be the corporation, and the device ID could be the computer or program that is assigning item IDs. For example, if company ABC had an assigning device, a computer EFG, the computer EFG would be the creator of the ID packet. If device EFG wanted to assign a vendor ID to a particular vendor&#x2014;the item, then vendor ID would be set to HIJ. The value for the data packet would then be ABC/EFG/HIJ. Note that the ID is actually not letters, but a combination of codes and time stamps. Each of the core ID 432, device ID 434 and item ID 436 are comprised of two blocks, a group block 438 and an individual block 440. The group block and the individual block 440 are comprised of a prefix 442, a time stamp 446 and a sequence number 448. The prefix is a sequence of predetermined prefixes that define various items associated with a particular group or individual. For example, it could be that the setup of the profile define this individual as a vendor that had an account which was a core ID and other various prefix values. As such, many different companies or organizations could have the same prefix. However, once the prefix is defined, then the time that it is created is set in the time stamp 446 and then a sequence number is associated therewith in the field 448. Therefore, since only one entity will ever assign the time stamp and sequence values, the entire block, 438 will comprise a unique value or number or ID for that particular group. For example, if company A set up a profile, it would be assigned a unique number that would always identify that particular company and this would never change. As far as the individual block 440, this is a block that further defines the core ID. For example, there may be five or six different divisions within a corporation such that this can be a subclassification. The notable aspect for this particular core ID 432 is that it comprises a unique ID in the system and will define certain aspects of the overall ID packet 430, as well as the device ID 432, 434 and item ID 436. When all three of these core ID 432, device ID 434 and item ID 436 are combined, this defines a unique ID packet 430 that is associated with various information such as transactions, messages, pointers, etc. These are set up originally in the universal ID server 132 in a profile origination step (not described) wherein a particular operation can be associated with the ID packet 430. This would essentially constitute a relational database somewhere in the system. Therefore, as will be described in more detail hereinbelow, when this ID packet 430 is assembled into the transaction packet, it is only necessary for any node to examine each of the ID packets and determine if any of the ID packets define operations to be performed by that particular ID packet. For example, if the ID packet represented a transaction such as a conversion, then the conversion server 150 in, for example, system 104, would recognize the particular ID packet indicating a conversion operation and also it might require information as to the destination node which is typically information contained in an ID packet, among other information, which defines exactly the process that must be carried out. For example, it may be that information is to be converted from one language to another which is indicated by an ID packet merely by the ID itself. With a combination of that ID packet indicating that transaction and the unique ID packet associated with the destination, the conversion server could make a decision that a particular process is to be carried out. This is facilitated since a relational database will be associated with the conversion server 150 that will run a particular process therein. It is not necessary to send any information to the conversion server 150 as to exactly what must be carried out; rather, only the particular ID is necessary which comprises a &#x201c;pointer&#x201d; to a process within the conversion server 150. Once the conversion is complete, then the process that is running can utilize another ID packet contained therein for the purpose of determining which device in the node is to receive the results of the process and exactly how those results should be packaged in a new transaction packet, it being noted that the transaction packet can be modified many times along with the transaction as it traverses through the system. Referring now to FIG. 4C, there is illustrated a diagrammatic view of a transaction packet 460. The transaction packet in FIG. 4C is illustrated as being a plurality of &#x201c;stacked&#x201d; packets referred to as IDP1, IDP2, IDP3 and IDP4, followed by a data field 462, followed by additional ID packets, IDP5 and IDP6 and so on. This transaction packet 460 can have any length, it being noted that the length is due to the number of ID packets, those being fixed length, and possibly variable length data field 462. By examining the ID packets as they arrive, which occurs in a sequential manner, then each of the ID packets can determine what follows and what action should be taken. For example, IDP4 may be an ID packet that defines exactly the length of the field 462 and what data is contained therein. Typically, these will be in finite length blocks. Referring now to FIG. 4D, there is illustrated a more detailed example of a transaction packet 464. In this transaction packet, there are provided a plurality of ID packets, IDP1, IDP2, IDP3-IDP6, and so on. IDP1 is associated with a transaction packet defining a predetermined transaction. As noted hereinabove, this is merely a pointer to a process that is defined in code on the recipient node, if that node is actually going to utilize the transaction. It is noted that this transaction packet IDP1 may be a transaction that is designated for another node. Following the IDP1 data packet is provided the IDP2 data packet which is associated with a message number. A message number comprises the real ID of a line of data in the database of the transmitting transaction node. Followed by this message number would be a block number in the IDP3 data packet followed by a block of data in a data packet 466. The message number and block number define the sequence of the data packet 466 for later assembly. This could then be followed by the data packet IDP4 for another message number and IDP5 for a block number followed by a data packet 468 associated with the message number and block number of IDP4 and IDP5, respectively. This is then followed by the IDP6 data packet for another transaction. Therefore, various message numbers, block numbers, data, transactions, process IDs, etc. can be transmitted in ID packets, it being noted that all that is sent is a unique value that, in and of itself, provides no information. It is only when there is some type of relational database that contains pointers that can be cross-referenced to the unique ID packets that allows the information in the ID packet to be utilized. If it is a transaction, as described hereinabove, then that transaction could be carried out by recognizing the pointer to that process disposed at the node that is processing the data. Referring now to FIG. 5, there is illustrated a detail of a database at the source transaction node H1. This is by way of example only. In this example, there are provided three separate tables that exist in the database. These are tables that can be formed as a result of the transaction or exist as a result of other transactions. It is noted that these particular tables are in the &#x201c;native&#x201d; database of the transaction node. Typically, the databases will always be arranged in rows and columns with a row identification address (RID) associated with each row. With the row address, one can identify where the data is for the purpose of extracting the data, updating the data, etc. When data is accessed from the database or is processed by the database with the system of the present disclosure, information is associated with each row of data in two separate proprietary columns, which columns are proprietary to the system of the present disclosure. They are a column 502 and a column 504. The column 502 is a date stamp on a given row, such that the particular row when accessed can be date stamped as to the time of access. A row ID that is proprietary is also associated with the accessed row. Therefore, whenever a row is accessed, it is date stamped and assigned a row ID. In this manner, even if the data is reorganized through a database packing operation or the such, the data can still be found. As such, a unique identification number for a given row can be generated with the proprietary row ID or the proprietary RID and the date stamp, such that a large number of proprietary numbers can be realized. When the databases are generated and put in the appropriate formats, it is desirable to transfer data that is stored for the purpose of a transaction to actually facilitate or execute the transaction. This utilizes a unique &#x201c;Extent&#x201d; for that transaction, which Extent is defined by an arrow 506 that converts the data in the appropriate manner to a proprietary transaction packet 508. The Extent 506, as will be described hereinbelow, is operable to determine how to process data, extract it from the various tables, even creating intermediate tables, and then assemble the correct ID packets with the appropriate data in a transaction packet and transfer this transaction packet to the network. Since the transaction is a profiled transaction for the whole network, the entire decision of how to route the data and ID packets to the destination and the manner in which the data is handled or delivered to the destination is not necessarily determined in the Extent at the H1 transaction node. Rather, only the information necessary to &#x201c;launch&#x201d; the transaction from the transaction node Hi is required and which ID packets are to be included. Once it is launched to network, this unique transaction packet travels through the network and is processed in accordance with the unique ID packets embedded in therein. Referring now to FIG. 6, there is illustrated a diagrammatic view of two transaction nodes 602, labeled H1, and 604, labeled H2, in a system that are both associated with individual routers 606, labeled R1, and router 608, labeled R2. Router 606(R1) is interfaced with the transaction node 602 through a local network 610, which also has associated therewith two conversion servers 612 and 616, labeled C1 and C2, respectively. The router 606(R1) is interfaced with router 608(R2) via a network 618. Router 608(R2) is interfaced with transaction node 604 through a local network 620, network 620 also interfaced with a conversion server 622 labeled C3. In operation, there will be a channel defined for any given transaction. This channel will define the path that is necessary to traverse an order to &#x201c;hit&#x201d; all the necessary processing nodes in order to effect the transaction in the appropriate manner and in an appropriate format that will be compatible with transaction node 604 when it arrives thereat. Similarly, if the transaction node 604 desires to utilize the same transaction back to node H1, it would merely use the same channel but in the reverse direction. Similarly, another transaction could be defined from the transaction node 604 to 604 directed toward transaction node 602, requiring an additional channel. Of course, each of these would also require a unique feed ID packet that would define the various software that generated the various channels, ID packets and the data packets, as described hereinabove. Referring now to FIGS. 7A and 7B, are illustrated graphical depictions of two channels. In FIG. 7A, there is illustrated a channel from H1 to H2 labeled &#x201c;0011.&#x201d; This channel requires the data to be generated at H1 and transferred to R1. At R1, a conversion operation is determined to be required and the data is merely sent to converter C1 (after possible caching at the router.) At conversion server C1, the conversion is performed and then it is reassembled and passed back to R1. At R1, it is determined that the data packet has arrived from C1, and the next step is to send it to converter C2. Converter C2 then performs the appropriate conversion operation, based upon the feed ID packet and the other unique ID packets in the transaction packet, and then transfers the transaction packet back to R1. At R1, it is determined that this transaction packet must be sent to another router, which is router R2. When sent to router R2, the routing information could be global or it could be network specific, i.e., the channels might be specific only to the systems associated with the appropriate router. In a situation like this, an intermediate &#x201c;joiner ID&#x201d; is generated that defines a particular relationship. This is an intermediate ID that is created for the purpose of this particular transaction. This joiner ID then is generated and the information sent to the router R2 which indicates that router R2 is to transmit the transaction packet to H2. It is known in this particular channel and transaction that the transaction packet is already appropriately conditioned for receipt by H2 and H2 will receive the transaction packet, and know what type of transaction is to be performed at H2, i.e., it is aware of the unique ID packets and their meaning, such as the feed ID, packet how to process information once received, etc. It therefore understands the parameters within which the transaction is to be effected. In FIG. 7B, there is illustrated another channel, channel &#x201c;0022&#x201d; for performing another transaction from H2 over to H1. This channel requires that the transaction packet be sent from H2 over to R2 and then from R2 over to C3 for conversion. After conversion, the transaction packet is sent from C3 over to R2 and then from R2 over to R1 with a joiner ID, similar to that of FIG. 7A. At R1, the data is transferred directly to H1. If the transaction for this particular channel is to be transmitted back to H2 along the same channel, the reverse path would be utilized. Referring now to FIG. 8, there is illustrated a flow chart for initiating a transaction. When the transaction is initiated, it is initiated at a block 802 and then a transaction table is created. This transaction table will have data associated therewith with rows of data therein in a predetermined format that is associated with the native database of the transaction node. This transaction table will then have each row therein stamped with a proprietary date and a proprietary RID, as indicated by the function block 804. Thereafter, the transaction flow will be analyzed, in a function block 806, to determine how the data is to be arranged and transferred. This transaction is then scheduled for transmission, in a function block 808. This is facilitated with a process wherein various calls are created for each block of data in the database, as indicated by a function block 810 and then a run ID is created in a function block 812. After the schedule has been generated and queued, the program then flows to an End block 814. Referring now to FIG. 9, there is illustrated a flow chart depicting the operation of analyzing the transaction flow in the block 806. The flow begins at a function block 902 to extract select data from the database and assign destination information and source information thereto, i.e., determine that the transaction comes from H1 and flows to H2. During this extraction operation, the type of extraction is determined, as indicated by block 901. It may be a partial extraction or a full extraction. The partial extraction is one in which less than all of the data for a given transaction is extracted, whereas the full extraction extracts all the desired data in a single continuous operation. The program in function block 902 operates in a filter mode and flows to a decision block 916 to determine if there is a restriction on the data which, if determined to be the case, will result in filtering by predetermined parameters, indicated by function block 918. This restriction operation is a filter operation that sets various parameters as to how the data is &#x201c;pulled&#x201d; or extracted. If not restricted, or, after restriction (filtering), the program will flow to a block 920 to a function block 904 to then assign a transaction ID to the data. Optionally, there could be assigned thereto a joiner ID in the event that it was determined the data should go across to systems and the joiner ID were appropriate. This joiner ID will be described hereinbelow. The program then flows to a function block 906 wherein a message number is assigned to each transaction. This message number is associated with a row of data. The program then flows to a function block 908 to determine block flow. Typically, in databases, the data is extracted in one large block of records. For example, a given transaction may require 10,000 records to be transferred over the network. However, it may be that the recipient transaction node desires only 500 records at a time as a function of the manner in which they conduct business. This, as noted hereinabove, is what is originally defined in the profile for the business relationship or the transactional relationship between the two transaction nodes. This, again, is predefined information. After determining the block flow, the program flows to a decision block 910 to determine if this is to be a divided block flow, i.e., the block is to be split up into sub blocks. If so, the program flows to a function block 912 to assign a block ID to each sub-block, such that the blocks can be assembled at a later time. The program then flows to a decision block 914. If it is not to be divided, the program will flow from the decision block 910 to the input of decision block 914. Decision block 914 determines if more data is to be extracted from the local database of the transaction node initiating the transaction and, if so, the program flows back to the input of function block 902 to pull more data. Once the data associated with the transaction has been extracted, the program will flow to a block 920 to return the operation. Referring now to FIG. 10, there is illustrated a diagrammatic view of a sample transaction table. The transaction table is basically comprised of the message number, the transaction ID, the joiner ID (if necessary), the row ID and date with proprietary identification system and the block ID. Also, a RUN ID can be assigned to each block as it is being processed. The row ID in a column 1002 and the date in a column 1004 is different from the database defining row ID in that they are always associated with the row. The row ID in the database is defined as function of the database and can actually change through various rearranging of the database at the transaction node. Referring now to FIG. 11, there is illustrated a flow chart depicting the operation of actually exporting the data from the transaction table. This is initiated at a block 1100 and then flows to a function block 1102 to pull the first block in accordance with the Extent that is running. It should be understood that all of the flow charts from the start of the transaction to the end of a transaction are associated with a predetermined transaction Extent. This Extent, as will be described hereinbelow, is a sequence of instructions or codes that are downloaded to the particular node to allow the node to conduct its portion of the transaction in the predetermined manner defined by the transaction profile that is distributed throughout the system. Not all of the necessary transaction information is contained here but, rather, only the information or the process steps necessary to create and transmit the transaction packet out of the system in the correct manner. Once the data is pulled in accordance with the Extent running on the transaction node, the program will flow from the function block 1102 to a decision block 1104 to determine if a caching operation is to be performed. If not, the program will flow to a function block 1106 to process the block as pulled. If caching is required, the program will flow to a decision block 1108 to determine if the caching is done, before transmitting the blocks and, when complete, the program will flow to a decision block 1108, along with the output of the function block 1106. The decision block 1108 determines whether an encryption operation is to be performed. If the data is to be encrypted prior to transmitting over the network, the program will flow to a function block 1110. If not, both function block 1110 and decision block 1108 will flow to the input of a function block 1112 to assemble the data packet. It is noted that the encryption operation is something that is optional and does require the overhead in each recipient node to decrypt the data. This function will not be described with respect to the remainder of the data flow. Once at the function block 1112, the transaction packet is assembled. The program then flows to function block 1114 to determine if the transaction packet is completely assembled and, once complete, the program will flow to a transmit block 1116. Referring now to FIG. 12, there is illustrated a flow chart for the transaction packet assembly operation, as initiated at a block 1202. The program flows to the function block 1204 to determine the size of the data packet, whether it is a small group of ID packets in the transaction packet or plural ID packets in the transaction packet. Once the size of the transaction packet has been determined, the program flows to a function block 1206 to determine the router to which information is to be transmitted. It is noted that more than one router could be on a network. The router is determined, of course, as a function of the particular Extent that is running, this being the path to which the packet will be routed. Once determined, the program will flow to a function block 1208 to insert the feed ID and the channel ID. It is noted that the feed ID and the channel ID are inherently a part of the Extent, this having been determined at the generation of the feed Extent which was generated during the profiling operation, as will be described hereinbelow. The program then flows to function block 1210 to attach the Run ID thereto and then to a Return Block 1212. Referring now to FIG. 13, there is illustrated a diagrammatic view of a transaction or process that is originated at an origin node 1302 for transmission to the destination node 1304 on a single outgoing channel. As noted hereinabove, the outgoing channel defines the route and the transaction. The origin node at 1302 utilizes a local Extent, indicated by box 1306, to generate the transaction. In this transaction, there are a number of IDs that are generated. One is a &#x201c;RUN ID,&#x201d; one is a &#x201c;FEED ID,&#x201d; and the third is a &#x201c;CHAN ID.&#x201d; Although there may also be other ID packets that are generated, these three packets can basically define an entire transaction or process. The origin node 1302, which can comprise the host node or the such, generates the transaction packet comprised of at least the RUN ID, the FEED ID and a CHANNEL ID and forwards it to a first process node 1306 which processes the received transaction packet in accordance with the above noted processes which then requires the transaction packet to be modified and transferred to a second process node 1308 for further processing, which then forwards this to a third processing node 1310 and then to a fourth processing node 1312 before final routing to the destination node 1304. The destination node 1304, as described hereinabove, can be the system router. Additionally, the router could be one of the processing nodes 1306-1312. This process will use a single outgoing channel for transferring a transaction packet from the origin node 1302 over to the destination node 1304. At the destination node 1304, the information could be transferred out of the channel to another channel, as will be described hereinbelow. Overall, this processing channel is defined graphically as a block 1314. This graphical representation indicates that a transaction packet is generated and the process through various nodes in accordance with distributed processing described hereinabove to route the transaction packet along various processing nodes to the destination node 1304 for handling thereat. Referring now to FIG. 14, there is illustrated a diagrammatic view of two channels adjacent to each other. In this embodiment, there is illustrated an origin node 1402 which is operable to generate a transaction packet, as described hereinabove, through two processing nodes 1404 and 1406 to a router 1408, labeled R1. This router R1 is substantially the same as the destination node 1304 in the single outgoing channel noted with respect to FIG. 13. This combination of the origin node 1402, the two processing nodes 1404 and 1406 and the router 1408 comprise an outgoing channel. A second channel is associated with a destination node 1410. The overall transaction or process is operable to generate the transaction at the origin node 1404 and route it finally to the destination node 1410 for completion of the transaction. However, once the router 1408 has received the transaction packet, it then passes it over to a router 1412 labeled R2, which constitutes an incoming channel for the destination node 1410. The router 1412 receives the packet from router 1408 and passes it through two processing nodes 1414 and 1416 to the destination node 1410. As noted hereinabove, the two systems, the one associated with router 1408 and the one associated with router 1412 could handle the transaction packet and the ID packets associated therewith in a similar manner, i.e., that is, they could utilize the same packet IDs. However, for security purposes, the origin node 1402 and the destination node 1410 utilize a different set of ID packets referred to as joiner ID packets to transfer information therebetween. As such, within the outgoing channel associated with router 1408 and origin node 1402, there would be a defined set of system assign IDs that would be proprietary to the origin node 1402. It may be that the actual identification of these IDs is something that the origin node 1402 would not want to share with the destination node 1410. Therefore, the origin node 1402 and the destination node 1410 negotiate a relational database that associates an arbitrary joiner ID with various IDs at the origin node 1402 such that the IDs have no meaning in any system other than for the business relationship between the outgoing channel and the incoming channel for the origin node 1402 and destination node 1410, respectively. These joiner IDs are illustrated in tables of FIG. 14A. You can see that router R1 has a table associated therewith wherein the joiner ID &#x201c;0128&#x201d; is associated with an ID packet &#x201c;XXXX.&#x201d; Whenever this joiner ID is received by router R2, a table for router R2 is examined to determine that this joiner ID &#x201c;0128&#x201d; is associated with an ID packet &#x201c;ZZZZ&#x201d; therein. For example, it may be that there is a unique ID associated with origin node 1402 that defines it in an overall system. However, it may be that destination node 1410 defines the origin node 1402 in a different manner, i.e., as &#x201c;ZZZZ.&#x201d; Rather than redefine the joiner ID as &#x201c;XXXX&#x201d; in its system, it merely needs to have a joiner ID that defines the relationship between the two systems. Therefore, whenever the joiner ID &#x201c;0128&#x201d; is received as an ID packet, the router R2 will convert this joiner ID to the ID packet &#x201c;ZZZZ&#x201d; such that it now recognizes that ID packet as the vendor number of the origin node 1402 within its system. Other than within the system associated with destination node 1410, this has no meaning. With respect to the joiner IDs, the joiner ID can be associated with the transaction packet in any position along the processing path. Typically, the joiner ID is assigned at the origin node 1404 when running the Extent associated therewith, i.e., it is initially defined when the feed and the channel are assigned. However, it could actually be assigned at the router 1408. Referring now to FIG. 15 there are illustrated three separate processing blocks 1502, 1504 and 1506, similar to the processing block 1314. Each of these processing blocks 1502, 1504 and 1506 represent a single channel and a processing system. For example, processing node 1502 could represent a single company and its associated router, conversion server, ID server, archival server and host node. When performing a transaction to transfer to another system, the transaction packet is generated within the processing node 1502, processed therethrough in accordance with the distributed processing system as described hereinabove and then output from the processing block 1502 over to a second channel 1508 for routing to the processing block 1504. The processing block 1504 represents a third channel and an independent and self-contained processing block. For example, the processing node 1504 may be an intermediate processing node that allows independent processing of a transaction or processing event for transfer to the processing block 1506. This could be, for example, a satellite system that constitutes an intermediate processing step. Once the transaction has been processed through the third channel, this is then transferred to a fourth channel 1510 for transfer to the block 1506, which comprises a fifth channel. Each of these channels and each of these processing blocks comprise separate distinct processing operations which all operate on the same transaction packet (although the transaction packet may be modified somewhat). Initially, the processing block 1502 originates at an originating node therein the transaction. This transaction has a channel and feed associated therewith, which channel comprised all of the channels from the origin to the destination at processing block 1506. Referring now to FIG. 16, there is illustrated a diagrammatic view of how the channel IDs and the feed IDs change as the transaction packet is processed through various processing nodes. As described hereinabove, a channel is defined as the route that a transaction path is to take through the various processing nodes. Since the processing is distributed, the transaction packet must be routed to each node in order that the appropriate processing be carried out on that transaction packet. Since the processing is predefined with respect to the channel ID, very little information needs to be disposed within the transaction packet in order to effect the processing. This transaction packet and the packet IDs associated therewith in the form of the feed ID, the channel ID, etc., define the portion of the processing that is to be carried out at each processing node, i.e., these constituting process pointers at each processing node. With respect to the channel ID, this basically remains the same in the transaction packet as the transaction packet traverses a group of processing nodes. However, the feed ID will change. The feed ID basically constitutes an instruction that is generated at one processing node for transfer to the second processing node that defines the processes that are to be carried out. In general, this feed ID is a &#x201c;tracer&#x201d; that follows the process to flow from node to node. As such, when one node receives a transaction ID from another processing node, it recognizes that the process is that associated with the channel ID, but it also recognizes where in the process the transaction packet is. For example, a router may handle a transaction packet a number of times in order to effect transfer to one or more conversion servers, effect transfer to an ID server, etc. With the use of the feed ID, the router now has knowledge of what process is to be carried out in the overall transaction process when it receives the transaction packet from a given processing node. Additionally, another aspect that the feed ID provides is the tracing function wherein a failure at any point along the process path can now be tracked to the previous process that was carried out. With specific respect to FIG. 16, there are provided a plurality of processing nodes 1602 labeled N1, N2, . . . , NK. Each of the processing nodes 1602, as described hereinabove, carry out a portion of the overall transaction process which was predistributed to the processing node. Each of the processing nodes 1602 carries out a plurality of processes, labeled P1, P2 and P3 for exemplary purposes. It should be understood that any number of processes could exist at a particular processing node 1602 that could be associated with a given channel ID or multiple channel IDs for many other transactions apart from the current transaction. It is noted that each processing node can handle many different processes and transactions. Once a transaction ID packet is configured, each processing node will receive that transaction packet, examine the transaction packet and determine exactly which process must be performed on that transaction packet, all of the effected with only a few ID packets of a fixed length. When the transaction is initiated, it is initiated at the origin node, illustrated as a node 1604 for generation of a feed ID and a channel ID, labeled FEED1 and CHID1. This indicates at the origin node 1604 that this transaction packet is to be transferred to processing node N1. When processing node N1 receives the transaction packet, it recognizes that the process to be carried out is defined by the feed ID and it has associated therewith a FEED1 block 1606 that defines the process that is to be carried out. This block 1606 then can select between the available processes P1-P3 for application to the transaction packet. Once a transaction packet has been processed in accordance with the selected one of the processes (it may possibly require more than one process for the processing), then the feed number is changed to the next feed ID, FEED2, and then the transaction packet is transferred with the same channel ID, CHID1, to the next processing node, node N2. At this node, the processing node recognizes that this is the FEED2 feed ID and processes the data in accordance with a block 1608 for this particular feed ID. Again, this selects between a plurality of processes for operation on the transaction packet. Once processed, then the feed ID is incremented and the transaction packet transferred until it reaches the last processing node in the processing chain, the processing node NK. At this node, this processing node will receive the feed ID, FEEDK, and the same channel ID, CHID1. This will be processed with processing block 1610 in accordance with the feed ID to select the process that is to be applied to the transaction packet and then this is transferred out to the destination. It can be seen that this &#x201c;hopping&#x201d; operation allows the transaction packet to be passed from one processing node to another. By incrementing the feed ID along the processing chain, each processing node can determine uniquely what process is to be carried out in the overall processing chain. However, it should also be understood that the feed ID provides this tracer operation, but could be eliminated. It could be that all that is required is the channel ID. Each processing node would receive the channel ID and the processing associated therewith could be indicative of the process to be carried out by recognizing where the channel ID came from. Therefore, an entire transaction could be carried out with a single ID packet. For example, suppose that a transaction involved a conventional normal transaction between two business entities that involve the transfer of 100 widgets to a particular warehouse. Once the business relationship is defined between two companies, then a single channel ID could be transferred to the destination company which, upon receipt, would recognize that a particular transaction was to be carried out in a particular way for this particular vendor. It may be that there are some conversions that are required during the process, which will require the ID packet to be transferred to a conversion server to possibly assign a joiner ID to the channel Id in order to provide some security to the system to prevent actual information at the origin in the form of its unique vendor ID, etc., to be transferred to the destination node. As such, it may be that some type of conversion operation would be required to assign a joiner ID during the process in the first company's system for transfer to the second company's system. It is noted that a company system is that defined by a router, a network mesh, an ID server and a host node. Typically, the ID server, the host node, the conversion server, and the network mesh are all typically associated and &#x201c;owned&#x201d; by a particular company. Referring now to FIG. 17, there is illustrated a diagrammatic view of how the feed is incremented. This is initiated at a start block 1702 and then proceeds to various feed blocks for the feeds FEED1, FEED2, . . . , FEEDK. The process must go through each of the feed blocks and, at each of the feed blocks, carry out the associated process. Therefore, the transaction packet in effect not only carries a channel ID that can be utilized at a particular processing node to determine what transaction is being processed but also receive intermediate instructions to indicate what processes in the transaction are to be carried out. As noted hereinabove, it may be that the router is involved in the actual transaction a number of times. Although a plurality of processes are predetermined as being associated with the given transaction, the processes that are applied to the transaction packet are determined as a function of where in the process the transaction is. The feed IDs indicate the position in the transaction for the purposes of determining which predetermined transaction processes are to be applied to the transaction packet when received at a particular processing node. Additionally, the feed IDs also provide for some failure analysis in the event that a failure occurs. For example, in FIG. 15, one could examine any transaction or process from the origin to the final destination at any place in the process and determine where in the process it was. Referring now to FIG. 18, there is illustrated a flow chart depicting the operation of running the process at a given process node. The program is initiated at a block 1802 and then proceeds to a function block 1804 to read the feed ID received in the transaction packet. The program then flows to a function block at 1806 to run the process or processes associated with that feed ID and then to a decision block 1808 to determine if all the processes have been run. If not, the program continues running processes in the block 1806 and, when complete, the program flows to a function block 1810 to increment to the next feed number and then transmit the transaction packet to the next processing node, as indicated by a return block 1812. Referring now to FIG. 19, there is illustrated a diagrammatic view of a plurality of channels which indicate processing from an origin to a destination in each channel and then handing off to a second channel or second system. These are defined as channels CH1, CH2 and CH3. In channel CH1, there is provided an origin node 1902 and a destination node 1904 with two processing nodes 1906 associated therewith. In the second channel, CH2, there is provided an origin node 1908 and a destination node 1910 with three intermediate processing nodes 1912. In the third channel, CH 3, there is provided an origin node 1914 and a destination node 1916 and three processing nodes 1918. The transaction is initiated at the origin node 1902 for final transmission to the destination node 1916. However, between the destination nodes 1904 and 1908, there is provided a line of demarcation 1920, with a similar line of demarcation 1922 disposed between destination node D2 and origin node 1914. The destination node 1904 could be a router and the origin node 1908 could be a router in channel CH2. The line of demarcation 1920 indicates that the first channel, CH1, basically &#x201c;hands off&#x201d; the transaction to the second channel CH2 which processes the transaction in accordance with a predetermined process set forth therein in a distributed manner across the various processing nodes for handing it off to the third channel, CH3. Each of the line of demarcations 1920 and 1922 define distinct boundaries such that the transaction packet can be considered independently handled for each of the channels. For example, it may be that in order to transfer from CH1 to CH2, a joiner ID is provided. When handing off from destination 1910 to origin 1914 across line of demarcation 1922, a second joiner ID&#x2032; may be required. Referring now to FIG. 20, there is illustrated a diagrammatic view of one of the systems of 102-108 wherein a non-system node 2002 is interfaced with the system 104 through a network 2006, which interfaces with the router 136. The non-system node 2002, since it is not part of the overall system 104, is not identified in the system per se without some processing in the system 104. In general, the non-system node 2002 first must be identified and the transaction associated with its access to the router 136 identified. Once this identification is made, then the necessary transaction packet is assembled and the transaction conducted in accordance with the process described hereinabove. For example, the non-system node 2002 will initiate a transaction merely by contacting the router 136. This could merely be the transmission of a request to a specified URL of the router 136 on the network 2006. The router 136, upon recognizing the URL of the non-system node 2002, i.e.,the source URL, would recognize that a transaction is being initiated. The router would then create a transaction packet and route it to the conversion server 150. The conversion server 150 would then convert information received from the non-system node 2002 over to a format compatible with a transaction to be conducted with, for example, transaction node 140 on the network mesh 138 in the system 104. As an example of a transaction, consider that the non-system node 2002 wanted to send an order via e-mail to transaction node 140. To facilitate this, non-system node 2002 would fill out a form in a predetermined order with information disposed in predetermined fields. This e-mail would then be routed to the router 136. The router 136 would recognize the source of the e-mail and the fact that it was an e-mail. By recognizing both the source of the e-mail and the fact that it is e-mail, the router 136 would now recognize a transaction. It would create a form ID for the non-system node 2002, which would define the type of form that is to be routed to the conversion server 150, and various other IDs that are associated with the transaction. This form and the form ID, in addition to other identification information in the form of ID packets, would be sent to the conversion server 150. The conversion server 150 would then extract the information from the form in accordance with the form ID pointer, and convert this to information associated with the transaction. This would then be transferred to transaction node 140. Referring now to FIG. 21, there is illustrated a flow chart depicting the operation of the router 136 when receiving information from within the system and from outside of the system. The operation of the router 136 is operable to receive data in the form of packetized data from the non-system node 2002. This is indicated at decision block 2102. The program then proceeds to decision block 2104 to determine whether this is a system packet. If so, then this indicates that this is a system node and the program will proceed to a function block 2106 to process the received transaction packet in a normal mode. If it is not a system packet or transaction packet, the program would flow to a function block 2108 to convert the packet to a system packet and then to the function block 2106. Referring now to FIG. 22, there is illustrated a block diagram of a simplified embodiment of FIG. 20. In this embodiment, there is illustrated a situation wherein the non-system transaction node 2002 can do nothing more than access the router 136 and transfer information thereto. As such, the router 136 must have some type of ID process, indicated by block 2202, by which to recognize the non-system node 2002 and associate the transaction packet therewith, which involves the use of a form ID, as described hereinabove. Once the transaction packet is created by the router 136, then the transaction packet is routed to the conversion server 150 and a conversion process, as indicated by block 2204, is run and the information received from the non-system node 2002 converted to the appropriate format to complete the transaction. Referring now to FIG. 23, there is illustrated an alternate embodiment of the embodiment of FIG. 22, wherein the non-system transaction node 2002 has software associated therewith that allows it to form the transaction packet. The non-system node 2002 has an ID process block 2302 associated therewith that allows the non-system node 2002 to create a transaction packet. The non-system node 2002 has a definite ID on the system which has been defined in the original setup wherein the ID process in block 2302 was created and &#x201c;pushed&#x201d; out to the non-system node 2002. Whenever a transaction is to be implemented, the ID process is run and a transaction packet assembled. This transaction packet is then forwarded to the router 136, in accordance with information in the transaction packet. This is due to the fact that the transaction packet created by the ID process 2302 has a channel ID and the such contained therein. Once the router 136 receives the transaction packet, it recognizes this transaction packet as one that exists on the system and routes it in accordance with a routing process in a process block 2304. Thereafter, this transaction packet is modified, if necessary, and routed to the conversion server 150 for processing thereby. The routing to the conversion server 150 is in accordance with the channel definition set forth in the ID process 2302. Thereafter, the information is processed as described hereinabove with respect to FIG. 22. ID Packet Referring now to FIG. 24, there is illustrated a more detailed diagrammatic view of the ID packet that constitutes the proprietary portion of a transaction packet that is transferred over the network, it being noted that this ID packet is typically embedded within a data transmission between the network with all of the commensurate overhead associated with such a transfer. As was described hereinabove, this ID packet represents the smallest fixed length portion of a transaction packet. The ID packet is divided into three sections, a core ID section 2402, a device ID section 2404 and an item ID section 2406. Each of the sections 2402-2406 are divided into two sections, a &#x201c;Group&#x201d; ID and a &#x201c;Individual&#x201d; ID section. A detail is illustrated of the core section 2402. Each of the Group and Individual sections are comprised of three sections, a preamble section 2408, a time stamp section 2410 and a sequence section 2412. As described hereinabove, the preamble section 2408 comprises a classification section that is comprised of a plurality of &#x201c;classifiers.&#x201d; The time stamp section 2410 and the sequence section 2412 provide a unique value that, when associated with a classifier section 2408, provides a unique group value for the core section 2402. The Individual section is also organized as such. In the preamble section 2408 of the Group section, it can be seen that there are a number of classifiers associated therewith. Of these, one classifier will always be the classifier &#x201c;G.&#x201d; There can be multiple other classifiers, it being understood that the number of classifiers is finite. As will be described hereinbelow, each of these classifiers is comprised of a single alpha character, there being twenty-six alpha characters, each of which can be represented by an ASCII value which is a finite length value. Of course, this limits the number of values to twenty-six for each classifier field. There could be any type of value system utilized, it only being necessary that the field be a fixed length. For example, if the field were defined as a digital word having a four bit length, this would provide 24 values. With respect to the preamble 2408 on the Individual section, this also has a finite number of classifier fields, one of which will be the classifier &#x201c;I&#x201d; designating this as an Individual ID. The core ID 2402, device ID 2404 and item ID 2406 are illustrated in Table 1 as follows: TABLE 1 CORE (WHO) DEVICE (WHERE) ITEM (WHAT) Corporation or Assignee of the Packet, Object, e.g., article, net Entity e.g., computer, phone, etc. address, real estate property, etc. The core ID 2402 is directed toward the basic owner of the ID packet. This, for example, could be a corporation, such as Corporation ABC. The device ID is associated with the device that assigned the values in the packet. For example, this could actually be the ID of the computer, the phone, etc. that actually was responsible for assigning the packet. The item ID is the subject of the data packet or the object, i.e., an article of commerce, a network address, a real estate property or the such. This is referred to as the &#x201c;Who, Where, What&#x201d; aspect of the ID packet. For example, Corporation ABC is originally defined as the owner of the ID packet. A unique core ID is initially associated with the ABC corporation wherein a defined classification preamble 2408 is associated therewith and then a unique time stamp and sequence number. This classifying preamble 2408 may actually be identical to the classification associated with other corporations in the system. However, once the time stamp and sequence number are associated with the preamble 2408, this core ID becomes unique as to that corporation or entity against others. When an object or item is being incorporated into an ID packet, i.e., an ID packet is being created to uniquely define that item in the system, there is some device on the system that actually creates this ID packet. For example, it might be that a catheter is being uniquely defined in a company. There will be possibly a computer terminal on which the information is entered. This computer terminal has an ID in the system and it is this ID that comprises the device ID. Therefore, once the ID packet is created, the entity (corporation) then owns the ID packet. The object, i.e., the catheter, is classified and is also known which device assigned the ID packet or created the ID packet. Referring now to FIG. 25, there is illustrated a more detailed diagram of the preamble 2408. The preamble 2408, as described hereinabove, is comprised of a plurality of fields. These are referred to in FIG. 25 as &#x201c;F1, F2, F3, F4, F5, . . . &#x201d; There are a fixed number of fields for the preamble 2408 which, in the present disclosure, are fixed for each Group ID and Individual ID for each of the core, device and item IDs. However, it could be that the fields differ between preambles, the only requirement being that they do not differ between ID packets. A typical five field preamble section of an ID is illustrated in Table 2 as it exists in the database, understanding that more fields may be incorporated. TABLE 2 F1 F2 F3 F4 F5 TS/SEQ CONTENT A B Z C W XXXX &#x2014; C T Q I C XXXX &#x2014; F L A K L XXXX &#x2014; G M B R S XXXX &#x2014; With reference to Table 2, it is described hereinabove that each field has an alpha character associated therewith. This alpha character has a predefined relationship for the classifier. For example, if a field were associated with the type of ID, there could be two values, one associated with a permanent ID and one associated with a joiner ID. This would therefore be a field having only two values. It could be that this utilized the alpha characters &#x201c;P&#x201d; and &#x201c;J.&#x201d; However, it could use any alpha character (number, character, symbol, etc.), it being recognized that the value or relationship (meaning) of the characters is unimportant; rather, it is the relationship of that packet disposed in other locations in the system that is important. In TABLE 2, it can be seen that the database associated with a particular ID has associated therewith the fields in the preamble, the time stamp/sequence field (TS/SEQ) section in addition to a content column. The content column defines what this preamble is associated with. For example, if this were the Group ID in the core ID 2402, then this could refer to, for example, a content of &#x201c;chemical corporations.&#x201d; If this were Corporation ABC, then the Individual ID would have a preamble field that might be common with other individual corporations but the TS/SEQ section would be unique only to that corporation and the content associated with that particular corporation would have the term &#x201c;Corporation ABC&#x201d; in the content column. It may be that there are ten corporations that have identical preambles but different TS/SEQ values and, therefore, the core ID 2402 would be unique to that corporation. Each of the Group ID and Individual IDs for the core, device and item IDs in the ID packet would be configured similarly. As will be described hereinbelow, although each of the fields in the preamble 2408 is defined as having only 26 values due to the choice of an alpha character as the classifier, one of the fields can be combined with the TS/SEQ value to provide a larger value associated therewith. Since the TS/SEQ value can comprise a unique and very large number, it does not constitute a classifier as such. By combining the twenty-six alpha numeric values each with the TS/SEQ value, the number of classifiers for that particular field becomes very large. For example, if one wanted to define a field in the preamble for the item ID 2406 as the field that defines the item, more than twenty-six item classifiers can now be provided. As a simple example, it could be that there are a plurality of catheter types in a company such as a pulmonary catheter, a cardiac catheter, etc. If there are more than twenty-six of these types of catheters, there would be required more than twenty-six classifier values. By combining an alpha character with the time stamp, the number of available classifiers can be increased in value. Referring now to FIG. 26, there is illustrated a diagrammatic view of the classification scheme. There are illustrated four fields that are being classified in a preamble, it being understood that more or less fields could be defined for the preamble structure, with only three values illustrated for each field. However, each of these values can be conditional upon the previous path, as will be described hereinbelow. In the field F1, there are illustrated three classifier values, A, B, C. The classifier of interest in field F1 is &#x201c;A.&#x201d; There are illustrated three paths from this classifier, since field F2 is only associated with three classifiers, these being again, A, B, and C. It should be understood that the classifications being associated with the classifier A is not necessarily the same classifier associated with the classifier A in field F1. Also, the classifier B in field 1 may also point to three separate classifiers A, B and C in field F2. However, it should be understood that the classifier A in field F2 that the classifier B in field F1 could point to may not be the same as classifier A in field F2 pointed to by the classifier A in field F1. The classifier in any one of the fields below field F1 has a value that may be conditioned upon the classifier in the previous field from which it derives. It can be seen that each of the classifiers in field F2 will point to one or more classifiers in the next field F3, there being illustrated three, A, B and C. Further, field F4 further expands this will three classifiers, A, B and C for each of the classifiers in field F3. Again, although there are illustrated as multiple classifiers A in field F3, they are not identical in value or classification function but, rather, they are unique to the associated path. With reference to FIG. 27, there is illustrated a single path through a given preamble of a field width of four. In the Group ID, for example, the preamble may be classified as &#x201c;A&#x201d; in field F1 and it may point to classifier &#x201c;B&#x201d; in field F2. Although the path could go to classifiers &#x201c;A&#x201d; or &#x201c;C&#x201d; only one path is selected. At field F2, classifier B points to classifier &#x201c;A&#x201d; in field F 3 and classifier &#x201c;A&#x201d; in field F 3 points to classifier &#x201c;B&#x201d; in field F4. Therefore, once it has been determined that field F1 has classifier A, then the next determination must be which of the classifiers in field F2 associated with classifier A in field F1 will be selected. It is this association of classifiers in a lower field with those in an upper field that defines the classification scheme. Again, it could be that classifier &#x201c;B&#x201d; in field F1 could point to a classifier &#x201c;B&#x201d; in field F2 that is different than that associated with classifier &#x201c;A&#x201d; in field F1. However, it could be that some fields have identical classifiers for each of the above fields. For example, in the Group ID, the last field will always be &#x201c;G&#x201d; defining the Group ID as such (not a conditional classifier.) The individual ID will always have a &#x201c;I&#x201d; in the last field thereof defining it as such. Therefore, there need not be any association between fields though there can be an association. With respect to the Individual ID, this follows the same path as the Group ID with the exception that it is defined as having values of &#x201c;D,&#x201d; &#x201c;E,&#x201d; and &#x201c;F.&#x201d; The ID that is generated will be stored in a table in the database of the ID server with alpha titles that can be searched, in association with the code associated therewith. A typical table in the database is illustrated in Table 3. In Table 3, the field F1 is associated with an ID that is either a permanent ID or joiner ID. This is referred to as P/J in one column, this is defined as a permanent or joiner field with the code associated with the permanent field being a &#x201c;P&#x201d; and the code associated with the joiner field with the joiner value being a &#x201c;J.&#x201d; The second field F2 is associated with different types of devices are Individual IDs or Group IDs, defined, in this embodiment as a profile type, a network type or a system type. Therefore, the one column will define the type as being profile, network or system and the code associated with the profile type will be &#x201c;F,&#x201d; the code associated with a profile type would be &#x201c;P,&#x201d; with a network type would be &#x201c;N&#x201d; and with a system type would be &#x201c;S.&#x201d; Field F3 is associated with an item which could be a type of computer such as an Apple computer, an item such as a catheter, a URL for a network address or the name of a system such as AVC or with a system referred to as a PPLL, this basically being an acronym for some type of system in the industry, as an arbitrary example. In this example, the code is the combination of an alpha character plus the time stamp for that row, to provide a large number of values therefor. In field F4, this is the category of the ID which, in this example can either be a core ID or a vendor ID. If it is a core ID, it will have a code of &#x201c;C&#x201d; and if it is a vendor ID, it will have a code of &#x201c;V.&#x201d; There will also be a time stamp associated with each row. It can be seen that there are two IDs having identical values in all of these fields with the exception that field F3 is associated with different catheters. As such, the code value would be distinguishable between the two because the code P+TS is associated with a different time stamp. This is what makes these two IDs distinct, even though they are associated with the same item, they are both vendor IDs, they are both permanent IDs and they are both profile IDs. By utilizing the time stamp in association with a alpha character, a much larger number of items can be defined for this particular field. Referring now to FIG. 28, there is illustrated a diagrammatic view of the method in which the data packet is created and the database populated with the data packet. Initially, a profile screen 2802 is provided which provides a plurality of user modifiable fields 2804 that allow the user to insert information. Each of these fields is utilized for the classification operation. Sometimes, this is an interactive system wherein inserting information into one field will result in another type of field being made available. For example, if somebody were classifying a data packet as being associated with a network, it might be that the URL of the network were provided as a possible input for another classifier, whereas that particular classifier, the URL, might not be appropriate for a previous classifier. Once the user has inserted all of the necessary information, then the flow would move to a block 2806 wherein the information that is input by the user would be classified into the preamble of the appropriate ID in the data packet. This, as described hereinabove, would be required in order to classify all of the IDs in the ID packet. For example, when filling the profile, a corporate name would be specified which automatically would pull up the core ID for that corporation. Of course, the device that is being utilized to fill in the profile would already be known and would constitute the device ID. The remaining portion of the profile 2802 would be utilized for the purpose of providing the item profile. The classifier would assemble all of this information and then flow to a block 2808 wherein the data packet is populated and the database is populated, as indicated by block 2810. This population of the database would provide information associated with the ID packet, as set forth in Table 3, such that all of the information necessary to identify a ID packet is contained therein. Table 3 is as follows: TABLE 3 F1 F2 F3 F4 P/J Code TYPE Code ITEM Code CATEG Code F5 Perm P Profile P Apple D+TS CORE C &#x2014; Perm P Profile P Cath P+TS VN V &#x2014; Perm P Profile P Cath P+TS VN V &#x2014; Perm P Network N URL P+TS VN V &#x2014; Perm P System S AVC A+TS VN V &#x2014; Join J Profile P Cath Z+TS CORE C &#x2014; Join J Profile P Cath F+TS CORE C &#x2014; Join J Network N URL L+TS VN V &#x2014; Join J System S PPLL N+TS VN V &#x2014; As such, the ID packet now provides a method to &#x201c;point&#x201d; to a specific row in the database, due to the fact that all of the preambles and the time stamps exist. Although Table 3 illustrated only a single ID in the ID packet, it should be understood that each ID packet is represented by all of the IDs, which comprise a single row in the database. This database is typically populated at the ID server and then the ID server, as described hereinabove, &#x201c;pushes&#x201d; all of the ID packets in the database to the respective account servers such as the conversion server, the router, etc. Also as noted hereinabove, some of these ID packets could identify processes. In this situation, it might be that all of the information in the database and an ID server need not be transferred to each and every one of the accounts such as the conversion server and the router. Only the information associated with data packets that would be processed or handled by that particular server would be required at the conversion server, router, for example. Referring now to FIG. 29 there is illustrated a flow chart depicting the operation of entering a profile. The program is initiated at a block 2902 and then proceeds to a block 2904 to enter the profile, this typically performed by a user. It could be that, additionally, a profile that is received in the form of a filled out &#x201c;form&#x201d; that is provided by some input device from a non-system user. That is, for example, ordering a product from a system node in a transaction. If the profile already exists, as determined by a decision block 2906, then the program will flow to a function block 2910 to use an existing ID. However, if the ID does not presently exist, the program will flow along a &#x201c;N&#x201d; path to a function block 2912 wherein a time stamp will be applied and then to function block 2914 where a sequence number will be assigned. Typically, if this particular device is creating new packets, a different sequence number will be attached to the various time stamp in a predetermined sequence. However, this could be a random sequence. The program then flows to a function block 2916 to store the ID and then to a decision block 2918 to determine if more profiles are to be entered. This is also the destination of the function block 2910. If more are required, the decision block 2918 will flow back to the input of function block 2904 and, if not, the program will flow to an End Block 2920. Referring now to FIG. 30, there is illustrated a diagrammatic view for defining a single ID in an ID packet. This ID is associated with the profile for a butterfly catheter. This typically will be the item ID. There are provided, for example, six fields, the first associated with whether it is a permanent or a joiner ID, defined by a &#x201c;P&#x201d; or a &#x201c;J,&#x201d; a second field associated with whether it is a profile, which is indicated by &#x201c;P,&#x201d; an item type defining what type the item is, indicated by a word as a user would input it, a fourth field associated with the actual item, i.e., that it is a butterfly catheter (the lowest classification), a fifth field for the overall type of ID packet, this being an &#x201c;ID&#x201d; packet, indicated by an &#x201c;I,&#x201d; indicated by &#x201c;C&#x201d; or a &#x201c;V,&#x201d; respectively, and a sixth field associated with the type of ID it is, an Individual ID, &#x201c;I&#x201d; or a Group ID &#x201c;G.&#x201d; In the first profile input, the user indicates it as being a permanent ID, a profile and types out the word &#x201c;catheter&#x201d; for the item type, and types and the word &#x201c;butterfly&#x201d; of the item that it is associated with an ID, &#x201c;J,&#x201d; and that it is an item ID indicated by an &#x201c;I.&#x201d; The term &#x201c;catheter&#x201d; is associated with an alpha letter &#x201c;C&#x201d; and the word butterfly is associated with the letter &#x201c;B.&#x201d; When this is first created, the ID that is generated is &#x201c;PPCBITS/S.&#x201d; The second item that is entered is identical to the first one in that the user indicated this as being a butterfly catheter. The system will recognize all of the first three and last two classifiers as being identical to others in the system and it will also recognize that the term &#x201c;butterfly&#x201d; as identical to a previous one that was entered. This type of search during the classification operation is performed by actually looking at the database in the non-coded column for the particular word in the field. This essentially looks at the spelling of the word. Since the spelling is the same as a previous one and the first three and last two fields are the same, then this will be identical to an ID packet that exists and a new ID packet need not be created. However, suppose a situation occurred where the user misspelled the term &#x201c;butterfly&#x201d; as &#x201c;butterfly.&#x201d; In this situation, the database search would not turn up this misspelling (this is assumed that the system does not have some type of spell check to allow adaptability to this type of situation) which basically determines this as a new item in the database. As such, a new alpha character will be associated with the item field, i.e., the fourth field, which is the alpha character &#x201c;L&#x201d; associated with the time stamp and this will comprise a new row in the database. For the last example, suppose that the item that is to be classified as a butterfly catheter with the correct spelling, but that the fifth field is a pulmonary description. In this event, this will be a different ID and may actually result in a different alpha character for the fourth field associated with the item. As illustrated, this can be assigned as an alpha character &#x201c;P,&#x201d; which may be different, but it uniquely identifies this as a different item associated with a pulmonary catheter. However, it is the time stamp that makes it unique even if the same character is used. Referring now to FIG. 31, there is illustrated a diagram of a system for layering data packets received from different systems that are potentially &#x201c;non-like&#x201d; systems. There are illustrated three systems, a system 3102, a system 3104 and a system 3106, labeled system &#x201c;A,&#x201d; &#x201c;B&#x201d; and &#x201c;C,&#x201d; respectively. Each of these systems operates in a different environment and may actually have a different database structure. For example, one might utilize an Oracle database with a specific and clearly defined database structure and another system might utilize a different database structure. Each of these database structures is an independent structure with possibly separate methods for identifying vendors and the such, i.e., there can actually be a different vendor number in each system for the same vendor or a different product number for a common product. However, in the overall system utilizing the ID packets, there can only be one common ID for a packet associated with any vendor or item. For example, if a field were present for an employee number associated with an employee, a field present for the days worked and a field present for the days out of the office, each of these particular types of data would be reflected in a different format in each database. Therefore, a specific employee number from one database would have to be converted into an ID packet format for the master system such that both systems employee number could be recognized, categorized and analyzed, or transferred from one system to the other. The manner for converting data and information in one database to the master system is provided by the extensions referred to hereinabove as &#x201c;Extents,&#x201d; that provide a software program for retrieving information from the non-master database and converting it to ID packets from the master system. System 3102 has associated therewith an Extent 3108, system 3104 has an Extent 3110 associated therewith and system 3106 has an Extent 3112 associated therewith. Each of the Extents 3108 is operable to retrieve the data and forward it to a conversion server 3114 as ID packets. The interface connection between the Extents 3108-3112 and the conversion server 3114 are illustrated as separate connections, but they are actually transferred through the network. Additionally, there could be multiple inputs to the conversion server from different networks. Each of the Extents is interfaced to an ID server 3116, which ID server 3116, which ID server 3116 is operable to &#x201c;push&#x201d; IDs for various items and the such to each of the associated Extents. For example, if system 3102 had associated therewith database information that was to be converted over to an ID packet out of the ID packets associated therewith would be stored in the Extent 3108. When initially set up, system 3102 would recognize for example, that each employee in its database required a separate ID packet to uniquely identify that employee. These would be set up by the ID server 3116 and pushed to the appropriate Extent 3108. Therefore, whenever system 3102 transferred an employee number as part of a data transfer to the conversion server 3114 or any other account server on the system, it would be processed through the Extent 3108 and the appropriate ID packet generated, i.e., extracted from the associated ID packet table of the Extent 3108, and then forwarded to the conversion server 3114. In the example of FIG. 31, the conversion server 3114 is illustrated as the destination of the information for the purpose of layering, as will be described hereinbelow. However, it should be understood that all of the data will first go to a router and then to the appropriate account server, if necessary. The illustration of FIG. 31 is simplified for this example. When data from system A is received for a particular conversion operation, it is stored in a database 3118 in a first location 3120. All the data from system 3104 is associated with a location 3122 and all the information from system 3106 is associated with a location 3124 in database 3118. This information is layered, such that common ID packet types, such as employee numbers, are arranged in a predetermined format. This is illustrated in a Table 3126, which is organized to illustrate four ID packets, IDP1, IDP2, IDP3 and IDP4. IDP1 may be employee numbers which are arranged in three locations, such that they all are in a common column. It should be understood that each of the IDPs can be different for employee numbers, i.e., each employee has a separate distinct ID packet. As such, if system 3102 and system 3106 both had the same employee in their database, they would have a common ID packet associated with the ID server 3116, this being set up initially. It can be seen, therefore, that the layering system allows a transaction or an analysis to pull data from non-like systems, convert it to like data in an organized structure and dispose it in a common table that will allow analysis thereof. An example of this will be described hereinbelow. Referring now to FIG. 32, there is illustrated a diagrammatic view of the transaction system for utilizing ID packets to converse between two systems through a master space. As described hereinabove, this master space includes the router, the network mesh, the core servers, the ID server, etc. that are required to process data packets. In FIG. 32, this system is illustrated with a block 3202 that defines the master data system. The master data system is essentially a system that receives, routes and operates on data packets to perform processes, etc. As described hereinabove, each of these ID packets constitutes a pointer to some process associated with traversal of information through the master data system 3202 from an origination point outside the system to a destination point outside the system through the master data system 3202 or to a point within the master data system for processing thereof. This processing system is referred to with a block 3204 which is operable that is also provided a master ID server 3206 that contains the ID packets that are operable with the system, these referred to as internal ID packets. These are differentiated from external ID packets for an external system, which is not disclosed herein. There is provided an external system 3208 that interfaces with the master data system 3202 via a conversion block 3310, system 3308 having a local database 3312 that is associated with its native database language or structure. Similarly, there is provided a second system 3314 that is interfaced with the master data system through a conversion block 3316 and has associated therewith a native database 3318. In order for system 3308 to interface with system 3314, it is necessary to extract data, convert it to an ID packet that is compatible with a master data system 3202, process it therein and then route it to system 3314 through the conversion block 3316, at which time it arrives at system 3314 in a structure similar to the native database 3318. This allows non-like systems to communicate with each other as long as they have a common space to go through. In order to operate in this manner, there must be some type of conversion to the master data space. This is not necessarily defined by the system itself, but, rather, the master data system 3202 through its ID server 3206 defines the manner by which each system will communicate therethrough. As such, this is a push operation with the definition. Not only are the parameters of the definition assigned, but the actual ID packet that is communicating therebetween. For example, there may actually be a common item, such as a catheter, that exists in both databases. By having this information determined by the master ID server 3206, an ID packet can be generated in the master ID server 3206 and associated with the same items in the two different databases 3312 and 3218. As such, it is important that the master ID server be able to identify the ID packet and associate it with the same item in two different databases such that, when pushing the ID packet to one of the systems, it also pushes the associated relationship to information in the database 3312 or 3318. For example, an employee number in database 3312 has a certain format and value that is set up in the master ID server 3206 as being related to a specific ID packet. When the ID packet is transferred to the conversion block 3310, it is associated with its value in the database 3312. Therefore, whenever the value in database 3312 is sent to the conversion block 3310, this value acts as a pointer and the appropriate ID packet can then be forwarded to the master data system 3202. Referring now to FIG. 33, there is illustrated an alternate embodiment of the embodiment of FIG. 32. In this system, there are provided two systems, a system 3302 and a system 3304. System 3302 has associated therewith a master data system 3306 and a master ID server 3308. System 3304 has associated therewith a master data system 3310 and a master ID server 3312. There is provided one external system, system 3314 associated with system 3302 in a conversion block 3316 disposed between system 3314 and master data system 3306. There is associated in a local database 3318 with system 3314. ID server 3308 is internal to the master data system 3306. Therefore, whenever system 3314, which is part of system 3302, communicates with master data system 3306, it will use internal ID packets associated with the ID server 3308, as described hereinabove. However, when conversing with master data system 3310, the ID packets are different, they are those associated with ID server 3312, these being external to system 3302. Therefore, master data system 3306 has stored in ID server 3308 external ID packets associated with the external side of the system, i.e., all other systems that are external thereto. System 3304 has associated therewith an external system node 3320, which communicates with master data system 3310 through a conversion block 3322 and also has associated therewith a local database 3324. When a transaction occurs which requires information to be transmitted from system 3314 over to system 3320, a data packet will be generated for information in the local database 3318. For example, if a simple transaction such as an employee number was required to be transferred to system 3320 for operations thereon as a portion of a process, the employee number would be extracted from D database 3318 with the conversion block 3316, as part of the overall transaction. This employee number would be converted to an internal ID packet associated with system 3302. At the master data system 3306, information in the ID server 3308 would be utilized to determine the external data packet to be transferred to master data system 3310. As described hereinabove, it could actually be the ID packet associated with the employee number that resides in ID server 3312. Alternatively, it could be a joiner ID packet which is a negotiated ID packet between the two systems, such that the actual ID packet associated with the employee number in either of the systems 3302 or 3304 is not known to the other. Once the ID packet, with a joiner ID packet, are transferred from master data system 3306 to master data system 3310, it is the processed in accordance with the transaction and transferred to the conversion block 3322 as the appropriate ID packet for that employee number. This is then converted to the format of database 3324 and processed by system 3320. Referring now to FIG. 34, there is illustrated a diagrammatic view of an example of a transaction. In this transaction, it is desirable to have information about employees as to the number of days they worked and the number of days they did not work. This information is analyzed in the master data system 3202. Therefore, the first thing that must be performed is a conversion from the employee number to a data packet, the days in information to a data packet and the days out information to a data packet. The employee number has previously been determined through a profiling operation to be defined as a unique ID packet. Therefore, a relational database can be utilized to pull the employee number from a database that is associated with the conversion block. The days in information can also be a unique data packet. For example, there could be a unique data packet for the days in information for values from 1-364, each different. Alternatively, there can be a single ID packet associated with the days in field and then a collateral or ancillary value data field that could be transmitted after the ID packet, as described hereinabove with respect to variable length data. This is the same situation with the days out field. The information is illustrated in a table 3402 in the native database. This is converted to a packetized value for a given row in a transaction packet. The first ID packet, IDPKT P, 3404 is generated to indicate the process that is being carried out, i.e., employee information regarding the days in and days out as being transferred to the master data system 3202 for the purpose of evaluating information in a particular process. This is followed by an ID packet 3406 labeled &#x201c;IDPKT EM&#x201d; for the employee number. Followed by that would be an ID packet 3408 for the days in. This is followed by an ID packet 3412 for the days out information. At the End of the information is provided a termination data packet 3418. This represents a single row of information being transferred, although it should be understood that the initiation of the process could constitute multiple rows and information in the form of an ID packet could be forwarded as a part of the transaction packet indicating the block size of the data that would be sent. This is then &#x201c;stacked&#x201d; in a stack 3420 such that it is stacked in a processing string as opposed to an organized data structure of columns and rows. Since the data is comprised of data packets, it is possible to place the data in such an organization. Referring now to FIG. 34A, there is illustrated a diagrammatic view of how the database is populated with ID packets. It can be seen that there are two columns, one for employee and one for an ID packet that represents data in and data out. It can be seen that unlike data is stored in the second column, i.e., that the information regarding days in is different than that regarding days out in that it would normally be contained within different columns of a database. This facilitates the processing operation. Therefore, by utilizing ID packets, the ID packets can be assembled in single columns representing different data. Further, they can be assembled in the column in the sequence in which information is to be processed in the later analysis routine. Core ID Generator Referring now to FIG. 35, there is illustrated a diagrammatic view of the operation of generating ID packets in the system. There is illustrated a network 3502 which has associated therewith a generic host node 3504 and a generic account 3506. These two nodes are merely nodes that are disposed on the systems that require knowledge of various ID packets in the system in order to process various portions of a transaction. The ID packets are created in an ID packet generator block 3508, this interfaced to the network 3502. The ID packet generator block 3508 is actually a program that can be implemented at any node on the system. It is, as such, a functional element. It interfaces with an ID packet database 3510, which can also be disposed locally with the ID packet generator 3508 or at another location on the network. It is only noted that the ID packet database 3510 is associated with the functionality in the ID packet generator 3508, regardless of where the system that it resides. Referring now to FIG. 36, there is illustrated a more detailed diagram of the operation of the ID packet generator. The ID packet generator 3508 is generally initiated with the input of a data input device 3602, this allowing an individual or corporation to input information to the system in the form of a profile in a predetermined format. As will be described hereinbelow, this profile is predetermined and sets various fields that are to be filled in by the individual inputting the data. This information is input to a profiler block 3604 which takes the information received from the data input device 3602 in the particular fields and associates it with a given profile format. There is typically a profile number associated with the profile, such that the fields can be input in a finite way. The profiler 3604, as will be described hereinbelow, performs the classification associated with the creation of an ID packet. At each node in the ID packet generator 3508, there is provided predetermined information about the ID packet. This typically is information about the corporation that owns the node associated with the ID packet generator 3508 and the actual identification of the node at which the ID packet generator 3508 resides. Therefore, there will be provided a core ID generator 3606 that has the owner of the system, i.e., the corporation that is doing the ID packet generation operation, selected from a core ID database 3608. This will provide the first portion of the ID packet, this being the core ID. There will also be a predetermined device ID generated by device ID generator 3610, selected from a device ID database 3609. Although not illustrated, this would actually be generated in another classification operation, which is not described herein, but which is similar to that associated to the item ID that will be described herein. In order to determine the item ID, this being the purpose for creating an ID packet and filling in the profile, an item ID generator 3612 is provided. As described hereinabove, the item ID generator 3612 is operable to generate the item ID, which is associated with a group ID and an individual ID. The group ID will typically be predetermined although it does not have to be, and then the individual ID must be determined as to its classification and as to its uniqueness. The uniqueness, as set forth hereinabove, is that associated with the time stamp, provided by block 3614 and a sequence, provided by sequence generator 3616. A classifier 3618 is provided that operates in conjunction with the profiler block 3604 to determine the classification of the item. This classification, in conjunction with the sequence and the time stamp, are combined together to provide an individual ID. The resulting item ID, as also described hereinabove, comprises a group ID and an individual ID. Once the item ID has been generated, then the ID packet is generated by combining the item ID, core ID and device ID together with a summing block 3618. This is then stored in the database 3510 in conjunction with the profile information. Although the classifier 3618 can utilize the information in the input information provided to the profile block 3604, all this information may not be part of the classification scheme. As such, all of the information utilized to classify the item ID and the additional information not necessarily utilized therefor will be stored in a database in association with the created ID packet. This is illustrated by a table 3624 which is comprised of a profile number, associated with the profile that created the overall profile, profile information in a column 3626 and the associated ID packet in a column 3628. This is the information that typically will be transferred with the ID packet, i.e., when another node receives an ID packet and associated information, it could actually utilize the profile information associated therewith in the column 3626 to create a new ID packet, since this constitutes the bulk of the information. Additionally, as will be described hereinbelow, information that was not classified would actually have links to other ID packets. For example, if the ID packet were utilized to classify a butterfly catheter, it may be that the classification system, at its lowest level, will only classify butterfly catheters. Additional information could be provided as to the color of the catheter. For example, if the butterfly catheter were red, thin, or the such, there would be provided a link to all ID packets having the word &#x201c;red&#x201d; disposed therein as any portion of the profile. All information in the profile is linked and not just the non-classified portion. In order to search the ID packet database, it would only be necessary to utilize the classification system to &#x201c;drill down&#x201d; to all ID packets associated with butterfly catheters to the classification preamble in the item ID (typically the individual ID in the item ID), and then filter this search with the links to the word &#x201c;red.&#x201d; This will be described in more detail hereinbelow. Once the ID packets have been generated, the second portion of the operation of the ID packet generator 3508 is the propagation operation. In this operation, various programs, referred to as &#x201c;Extents,&#x201d; are initiated by a propagation engine 3630 to extract the appropriate Extent propagation algorithm from a storage area 3632 which will define how information is propagated from the database 3510 to various nodes in the network, it being understood that the node on which the ID packet generator resides could actually be a node to which ID packets are transferred. This propagation operation is performed via a scheduling operation or a triggering operation, as noted by block 3634. Therefore, there could be some external trigger or internal trigger that results in the propagation of information or could just be a scheduling operation. Once the trigger/scheduler has indicated that a particular Extent should be performed, i.e., there is a predetermined process initiated or launched, then select ID packets are propagated to the appropriate node. For example, it may be that a particular transaction requires certain portions of the ID packet database to reside at a conversion server and at the host node. When these ID packets are created, a propagation Extent will indicate that all data associated with a particular profile, for example, to be transferred to select ones of the nodes. Further, as will be described hereinbelow, there are process ID packets that can be generated and propagated in a similar manner. It is noted that not all ID packets are required at each node nor are all Extents (noting that the Extents are actually ID packet or groups of ID packets) required at each node. Therefore, this propagating Extent at block 3632 will define where the ID packets are transferred, this being for the purpose of carrying out the transaction at each respective node in the process/transaction path. Referring now to FIG. 37, there is illustrated a flow chart for creating a profile. The program is initiated at a function block 3702 and then proceeds to a function block 3704 to pull up the select profile for interface with a user. Once the user has interfaced the profile, data is input to the profile, as indicated by a function block 3706, this information being input to select fields. Once the select fields have been filled in and the profile has been accepted, the program will flow to a function block 3708 wherein the device ID and the core ID will be fetched to provide the first two portions of the ID packet. The program will then flow to a function block 3710 to generate the classification portion of the ID packet. This may involve generating the classification portion for both the group ID and the individual ID. However, if only the item ID is to be classified, then only the classification portion of the individual ID of the item ID will be generated in the block 3710. The program then flows to a function block 3712 wherein the time stamp and sequence number are applied, rendering this ID packet as to the individual ID or the group ID or both. The program then flows to a function block 3714 to create the ID packet by assembling the device ID, core ID and item ID together. The program then flows to a function block 3716 to store the ID packet and the associated profile information in the database and initial copy in the block 3718. The resulting data packet is illustrated in FIG. 37a in that the generated ID packet in the first field 3720 is associated with two types of information&#x2014;standard information in a field 3722 and nonstandard information in a field 3724. Standard information is information that is generated for all items of the type profile being created. Of the standard information in field 3722, there are provided two regions, classification information which is required to form the preamble in the individual ID or group ID and nonclassification information which is information such as the color &#x201c;red&#x201d; associated with a butterfly catheter in the example described hereinabove that is not subject to classification, i.e., that is not required for the generation of the classification in the block 3710. There is also provided nonstandard information which can be stored in association with the ID packet in field 3720. This information constitutes items that only exist with respect to a creator system and may not be information that is defined or desired on a global basis. Effectively, this is similar to allowing a creator to add notes to a profile. Referring now to FIG. 38, there is illustrated a flow chart for the operation of creating the ID packet, which is initiated at a block 3802 and then proceeds to a block 3804 to generate the item ID. It combines the classification generated in the block 3710 with the time stamp and sequence number generated in the block 3712. Once the item ID is generated, then the program proceeds to a function block 3806 to link attributes of the item, these creating an input in the profiling operation in block 3706. These attributes are linked to an entry in an attribute table. This attribute table links such things as &#x201c;red&#x201d; to all ID packets in the system with that attribute. Even though an attribute is utilized in the classification operation, this attribute is still linked to an attribute table. For example, there might be an attribute that is entered into the profile that is associated with classification and some that are not associated with classification. For example, a butterfly catheter may have a color associated therewith, such as red, yellow or green. This is not considered important enough to constitute a classifier. Paint, on the other hand may utilize this term &#x201c;red&#x201d; as a classifier. Therefore, the term &#x201c;red&#x201d; for both the butterfly catheter and the paint would be linked to the same attribute table. One could then search all item IDs that have associated therewith the color red, regardless of what they were. Once the attribute has been linked, the program then flows to a function block 3808 to assemble the core, device and item ID in the block 3714. The program then flows to a Return block 3810. Referring now to FIG. 39, there illustrated a diagrammatic view of the screen that is presented to the user. There is provided a primary screen 3902 which has a plurality of fields associated therewith. The example in FIG. 39 is associated with inputting information regarding a new birth in a hospital. Each child is considered to be an item that has associated therewith a unique ID packet. Of course, the core ID would be that of the hospital, the device ID would be that of the actual device generated behind the packet, i.e., the unique device ID of the node generating the profile, and the item ID that is unique to the child. It should be understood that the group ID in the item ID would probably be the same for all children. The individual ID, on the other hand, would be unique to that particular child. Interestingly enough, there may be two children that have the same exact classifier, but that have a different time stamp and sequence number, i.e., they would therefore be unique. The difference is in the profile information that is associated with that particular individual. For example, there are provided a plurality of fields, one field 3904 for the name, a field 3906 for the gender, a field 3908 for the address, a field 3910 for the weight, a field 3912 for the date of birth, a field 3914 for the length, a field 3916 for parental data, a field 3918 for an internal reference number, this being an example of the nonstandard information that will be associated with a profile, a field 3920 for the doctor and a field 3922 for image links. Note that these image links would be non-standard information that would be links to images in a system and these links would not necessarily be desired by other systems. The primary profile 3902 has associated therewith a profile number 3924 that is associated with this profile in the system. When this profile is initially created, there is provided a very long standard information profile 3926 that defines the standard information that must be associated with the child. For example, there is provided a device ID field 3928, a core ID field 3930 and a classification field 3932. This predetermines what the device ID and the core ID will be and also predetermines all or a portion of the classifiers associated with the item ID and/or the group ID. There may also be a title field 3934 for the title of the profile. Therefore, this standard profile template 3926 is utilized to create substantially all of the information needed to create the ID packet. In fact, if the classification is the same for all children, then the information in the profile screen 3902 would be nonclassification information that would be considered standard information to retrieve (although some of this may be nonstandard information such as the internal reference number in the field 3918.) However, typically, there will be one or two classifiers that will not be standard for every child associated with the individual ID portion of the item ID. For example, it may be that there is a classifier for the ethnicity of the child or eye color. It can be seen that all of the fields in the profile 3902 are defined fields and the information therein will be linked to the attribute table. For example, although gender in field 3906 may not be a classifier, it will be linked to the attribute table. Therefore, all ID packets having a profile with the term &#x201c;female&#x201d; associated therewith can be searched through the attribute table. Referring now to FIG. 40, there is illustrated a flow chart depicting the operation of propagating the ID packets, once created, to select ones of the network nodes or other locations in the system. The program is initiated at a block 4002 and then proceeds to a decision block 4004 to determine if a trigger operation has been received. This trigger operation can be an external trigger or it could be a scheduling operation determined by the scheduler. The program, once determining a trigger is present, proceeds to a function block 4006 to run the propagate Extent for the triggered item. As noted hereinabove, each processor transaction on the system may require a certain group or groups of ID packets to be associated with that transaction. These ID packets must reside on the appropriate node in the system to which the transaction will be relayed during the process. It is therefore important that the appropriate ID packets associated with either item IDs or process IDs or even network address IDs to be resident at the node once the transaction packet, in its original form or modified form, is transferred thereto. As such, this propagation Extent will be run for particular processes or groups of processes or various transactions. Once the propagation Extent has been run, this propagation Extent will pull data from the database 3510, as indicated by a function block 4008. The program will then flow to a function block 4010 to create a transaction packet, which transaction packet is operable, in accordance with the operation of the Extent, to transfer ID packets to another location on the network. It is very similar to the situation described hereinabove wherein data is transmitted to a destination node. In this situation, the destination node is the node to which ID packets are to be transferred, this being data. It should be understood that, not only are ID packets transmitted, but the profile information associated with an ID packet is transmitted. As such, an entire row of the database 3510 will be transferred. And, therefore, it should be considered to be data. Once the transaction packet has been created, it is transmitted to the destination node, as indicated by function block 4012 and then the program proceeds to a function block 4014 to perform an acknowledgment operation and determine if the transaction packet has been received. This will be described hereinbelow. If so, the program will flow on the &#x201c;Y&#x201d; path to a function block 4016 to set a flag indicating that the data in the database, i.e., the updated ID packets or newly created ID packets, have been appropriately transmitted to the destination one of the nodes at which the ID packets must be stored. Once the flag is set, the program will flow to a Return block 4018. If acknowledgment has not been received, then the program will flow along the &#x201c;N&#x201d; path from decision block 4014 to a function block 4020 to run a propagate Extent indicating that there has been a failure of the propagation algorithm. This may result in a page or E-mail being sent to a technician or generation of a failure log or report. This will then be handed off to either an individual or another process to service. The program will then flow from function block 4020 to an End block 4022. Referring now to FIG. 41, there is illustrated a flow chart depicting the operation of the acknowledgment operation in decision block 4014. This is initiated at a block 4102 and then proceeds to a decision block 4104 to determine if an acknowledgment has been received. If so, the program will flow along the &#x201c;y&#x201d; path to the Return block 4018 through the function block 4016 to set the flag (not shown). If, however, the acknowledgment has not been received the program will flow to a decision block 4106 to determine if a time out operation has occurred. The program will loop back around to the input of decision block 4104 until the time out has occurred, at which time the program will flow to a function block 4108 to transmit a look-up ping to each of the destination nodes, i.e., this being a &#x201c;push&#x201d; operation wherein the ID server that generated the ID packets and propagated the ID packets will determine whether they have been received by the destination node. Each destination node has a table that is created at the ID server that represents the ID packets that are disposed therein and the associated profile information. There is also provided a column indicating a flag representing the successful transfer or lack thereof. Therefore, each entry must have a &#x201c;ping&#x201d; sent to the destination node. This ping basically defines the address at the destination location, this being known at the ID server, which will determine if information has been received thereat. The program will flow to a decision block 4110 to determine if a return acknowledgment has been received indicating that the ID packet in fact resides at the ping address which is achieved by sending the principal address back to the ID server. If an acknowledgment is received, the program flows to a Return block 4112, substantially that equal to Return block 4018 indicating that the flag is set in the table and, if not acknowledged, the program will flow to a decision block 4112 to 4114 to determine if there is a transmission time out. If so, the program will flow along a &#x201c;Y&#x201d; path to a failure block 4116 and, if not timed out, the program will flow along an &#x201c;N&#x201d; path to a retransmit 4118 to retransmit the ping. Once retransmitted, the program will flow back to the end of the function block 4108. When the ID packet is propagated, it is facilitated in two ways. First, the ID packet with profile information is sent. This would result in the ID packet constituting the primary &#x201c;ping key.&#x201d; All that is necessary to send is the ID packet in order to determine the address at the destination. This destination address is then returned with the ID packet and stored at the ID server. In the second case, only the profile information is propagated. This requires a field in the profile to be defined as the ping key. For example, when sending information to a host system, it may not be desirable for the ID packets to be disseminated. In this situation only profile information is sent. For a vendor profile, the vendor name (or number) is the ping key, as defined when the profile is set up. The ID server transmits the profile information to the extent running on the host, which then has knowledge of which native tables in the host the information must be routed/linked to. Once transferred, then the destination address of the ping key (vendor name in this example) is returned for storage at the ID server. Since the ID server has knowledge of all the destination addresses (there could be more than one for each ID Packet), this facilitates system clean up. For example, if a vendor needed to be changed ro deleted, then the ID server as a central repository could repropagate the changes to all of the linked to destination addresses. Referring now to FIG. 42, there is illustrated a flow chart for the look-up ping operation, initiated at a block 4202. The program will then flow to a function block 4204 to send a request to the router, it being noted that the router is the first place that the ping will be sent. In the preferred embodiment, the request to determine if an address has been sent is handled by the router. The request is sent to the router and then the router communicates to the destination node, as indicated by a function block 4206. The function block makes a determination as to whether the transmitted ID packet and its profile information resides at the destination node. This is indicated by a decision block 4208. If it has been sent there, the program will flow along a &#x201c;Y&#x201d; path to a block 4210 to send an acknowledgment signal back to the ID server and, if not, a nonacknowledgment signal will be sent back, as indicated by a function block 4212. The program will then return, as indicated by a block 4214. Referring now to FIG. 43, there is illustrated a flow chart depicting the operation of defining the profile. This is an operation wherein the overall templates for the profile are defined. This program is initiated at a block 4302 and then proceeds to a function block 4304 to set the core ID and then to a function block 4306 to set the device ID. The program then flows to a decision block 4308 to determine if the group ID and the item ID is fixed. If not, the program will flow to a function block 4310 to set the field parameters for the group. This is an operation wherein certain field parameters will be defined for the groups and, once filled in, they will set the classifiers for the group ID. If the group is fixed, the program will flow along a &#x201c;Y&#x201d; path to a function block 4312 wherein the group ID is set as a fixed item. Once the group has been set, either as a fixed group ID or as a substantially fixed group ID (the group can either be a set item as to both the time stamp and the sequence or, if a parameter is to be set, then a time stamp and sequence would be added after the group ID classifier has been defined), the program will flow to a decision block 4314 to determine if the individual ID is fixed. As noted hereinabove, there may be situations wherein the item ID is always a set ID in terms of the classifiers. If the individual ID is fixed, the program will flow along an &#x201c;N&#x201d; path to a function block 4316 where the field parameters for the individual ID are set. These, as described hereinabove, describe the classifiers for the individual ID. If the classifiers for the field are set, the program will flow along the &#x201c;y&#x201d; path to a function block 4318 to set the fields for the individual ID in the classifiers. The program will then flow to a function block 4320, it being noted that, during the set up of the profile, the time stamp and sequence will typically be added for at least the individual ID portion of item ID. At the function block 4320, additional attribute fields will be defined which are not a portion of the classification operation. These attributes will then be linked to the attribute table, as indicated by a function block 4322, the attributes linked being both the ones associated with the classification and the ones associated with the nonclassification operation. The program will then flow to a decision block 4324 wherein a decision will be made as to whether the content is limited, i.e., if a color were the type of field indicated by function blocks 4320 and 4322, i.e., the title of the field, then the content may be limited to a pull down menu of the multiple colors. If so, the program will flow to a function block 4326 wherein the available entry for that particular field will be noted. If not, the program will flow along an &#x201c;N&#x201d; path to a decision block 4328 to determine if the field has been validated. Decision block 4328 indicates the field as being an &#x201c;open&#x201d; field, wherein the program will flow along the &#x201c;N&#x201d; path to a function block 4330 or whether the field is required to be validated, as indicated by the flow along a &#x201c;Y&#x201d; path to a function block 4332 wherein a &#x201c;valid&#x201d; flag will be set. The validation operation is one that links the field to the attribute table, and will define the contents thereof as linkable when populated. This facilitates searching of the field, when the ID packet is created. For example, if an address field such as &#x201c;Street&#x201d; is defined, this would be linked to the street attribute in the attribute table. When this is filled in upon creating the ID Packet, then the actual street name will be linked to the dictionary. If it is open, then this field is not linked to the attribute table or the contents linked to the dictionary. Once the field is defined as an open field or a field that must be validated, the program will flow to a decision block 4334 to determine if additional fields are to be added. Once all fields have been added, the program will flow to a &#x201c;Done&#x201d; block 4336. In the operation of defining the attribute field type, i.e., the title of the field, this will link the field to the attribute table. This will be done before information is added thereto. As such, when information is added in a profile and the profile is accepted and the ID packets generated, the information defined in the profile that is associated with the ID packet will contain all the field names and the content of those fields. The links to the profile number are already preset, such that a new link need not be made. Therefore, when a new profile is generated, the unique address of that profile is the ID packet, since the ID packet is a unique value in and of itself. As soon as this ID packet is generated, it will immediately link to each of the field types in the attribute table. When content is added, a procedure must be followed wherein a dictionary is accessed to determine if the word is a correct spelling and then a decision made as to what the word is associated with. For example, it might be that a word is entered into a field having multiple meanings. This would be presented to the user once the content was entered such that the user could select the meaning of the term such that it will point to the correct meaning in the attribute table. This dictionary can also check for spelling mistakes, language translations, etc. Referring now to FIG. 44, there is illustrated a diagrammatic view of a system for propagating the ID packets from one ID server to a second system having an associated ID server. There is illustrated a first system 4402 having an ID server 4404 associated therewith, the ID server 4404 having an associated ID packet database 4406. The ID server 4404 interfaces with a local network 4408 having a router 4410 associated therewith. The system 4402 utilizes the router 4410 to interface with a gateway 4412. The gateway 4412 interfaces with a second system 4416 via an associated router 4418. The router 4418 interfaces with a local network 4420 for the system 4416. The system 4416 also has associated therewith its individual ID server 4422 interfaced with the local network 4420, the ID server 4422 having associated therewith its own ID packet database 4424. In operation, each of the ID servers 4404 can service their own systems to generate ID packets therefor. Each of the systems also has associated therewith other nodes, such as a host node 4426 for system 4402 and a host node 4428 for the system 4416. When each of the respective ID servers generates ID packets locally, they can each download them to their respective hosts 4426 or 4428 or even the associated routers 4410 and 4418. However, in some situations involved with transactions between two systems, it is necessary to provide ID packets from one system to the other. This will result in, for example, the flow of ID packets from server 4404 to system 4416 for storage in the various nodes associated therewith. Typically, the ID server 4422 will receive the ID packets and then propagate these ID packets to the various nodes associated therewith. Referring now to FIG. 45, there is illustrated a diagrammatic view of transfer of an ID packet and information from one system to another. There is illustrated a first system associated with a company &#x201c;A&#x201d; 4502, a second company, company &#x201c;B&#x201d; 4504. Company A desires to send data packets to company B. These ID packets and the associated information such as the profile, the profile numbers, etc. are stored in an internal database 4506 at Company A. The data is stored in a table format, as illustrated in table 4508. This table will be organized in rows and columns, each row comprising all the information necessary for transfer, this being an ID packet, and all of the profile information associated therewith. When information is transmitted from one company to another, it can be transmitted as the unique ID packet wherein the ID packet provides a &#x201c;pointer&#x201d; or address for the information. However, in certain situations, the particular ID packet associated with the company and for use internal to the company may not be of such a nature that the company would desire to transmit the information. For example, the core ID portion of the ID packet is unique to that company and this information may not be something that the company would want to be broadcast. Therefore, they create a new ID packet value as a joiner ID packet that is transmitted to the other company. Typically, this joiner ID packet will have a different value. It may in fact have the same preamble in the group ID and individual ID for any of the core ID, device ID or item IDs. However, the time stamp could be different. As such, this would be a different value. The reason for maintaining the preamble portion of the group ID and/or the individual ID for any of the three parts of the ID packet would be to maintain the classification system associated with all of the information in the ID packet. Although the classification information is identical or substantially identical, the time stamp and sequence number would be different, thus rendering the ID packet a different value. This function is facilitated by joiner conversion block 4510. When the joiner table has been created with the joiner conversion block 4510, this will provide a second cross-reference table which will be basically a &#x201c;pointer&#x201d; to the ID packet and the table 4508. The internal database 4506 will maintain this joiner ID. Basically, it is a cross-reference table such that information can be transmitted back and forth with different unique ID codes that will only be recognizable by the internal database 4506. One use of the joiner ID packet is to terminate the connection by merely erasing the joiner ID packet such that it will not be recognized, this termination not affecting the database 4508. When the information has been appropriately converted or not converted, it is transmitted to the system 4504 and input to an external database 4512 at the system 4504. This external database stores the external data received from external systems, this being one or more systems, in a table 4514, which table organizes the information in the form of the ID packet originally generated from the transmitting system (the External ID Packet) and the associated profile information, it being remembered that this External ID packet may be a joiner ID packet. Additionally, the system 4516 then interacts with the data to create an internal ID packet. This internal ID packet is associated with the profile information for the originally received ID packet, but actually constitutes a separate value. Since it has all of the information necessary to &#x201c;classify&#x201d; the data, it can go through the classification operation, as described hereinabove, to generate a new ID packet. This internal ID packet is then associated with the originally received ID packet and also the profile information. When it is necessary to utilize the information in the table 4514 for internal operations at the system 4504, only the internal ID packet and the associated profile information will be transmitted or propagated to various other systems. When the external data is utilized, it typically can be filtered with a filter 4516, which filter 4516 will only fetch a certain amount of the data for a particular system. This filter will filter off the ID packets originally received and then transmit it to a filtered internal database 4518. With the use of the filtered internal database, significantly less information is downloaded. For example, when an internal database in another system, the system 4502, transmits data, it may transmit all of a portion of its database, i.e., such as its entire data catalogue. However, the internal side of system 4504 may not desire to have all of the catalogue transferred down to the various nodes. Therefore, a particular Extent will run on the internal side of system 4504 to determine what data in the catalogue is selected for propagation to the various nodes. This is the information that is stored in the filtered internal database in the form of basically the internal ID packet and the profile information, as set forth in a table 4520. This filtered internal database information constitutes a database of ID packets which then must be propagated to the various systems through a propagate block 4522, which has been described hereinabove with respect to FIG. 26. This propagate block is operable to then propagate information to any of the nodes in the system 4504, as indicated by a block 4524, which will then associate the propagated ID packets for storage in a system node database 4526 in the form of a table 4528. As noted hereinabove, this will be organized in the form of the internal ID packets and the associated profile information, it being noted that this will probably not be as large as the table 4520 in the filtered internal database 4518. When the propagate block 4522 propagates, the table 4528 will also contain a link to the internal address of the System B Node 4524 at which the ID packet is stored. This address constitutes a destination address. This destination address is then reflected back to table 4520 as a link and to the database 4512. This is then put in database 4512 to indicate which ID packets have been filtered. This is via the acknowledgment function, which returns both the destination address and the underlying information. An example would be an entry such as a contact name. Note that, if system 4502 deleted an entry, the database 4512 would determine if there is a destination address linked to an External ID packet and, if so, indicate to database 4518 and propagate block 4522 that the item is deleted and then propagate the change. If it had not been linked in the filter operation, then the item would be deleted from the external side fo database 4512 (or disabled). Referring now to FIG. 46, there is illustrated a diagrammatic view of the operation wherein processes are created and propagated. In this view, there is provided on a system the process server 4602 which is operable to interface with a user interface 4604 to basically provide inputs to the process server, i.e., information necessary to define the process. The process server operates by assembling various logic blocks that are stored in the database 4606 to create Extents or processes or subprocesses which are utilized by the various nodes in the system. These are, after creation thereof, stored in a process database 4608. During generation of the processes, various ID packets are utilized, which are stored in an ID database 4610 and also routing information is stored in a routing database 4614. This routing information is information as to the various network addresses of all of the nodes in the system, such that the process can be effective. The process server interfaces with the local network 4616 which will basically interface with an ID server 4620 having associated therewith its ID database 4622, possibly an account server 4624 having associated therewith a process database 4626 for its associated portion of the processes distributed thereto and with a router 4628, having associated therewith process database 4630. It should be remembered, as described hereinabove, that all traffic on the system must go to the router first before being routed to the other process nodes. It can be seen that, once the process server has determined the processes and stored them in the process database 4608, it then determines where the processes need to be transmitted. Since the process defines a transaction from beginning to end, i.e., from transmission of certain information from an originating node to a destination node, there will be multiple processes that are carried out at one or more of the various nodes disposed in the transaction path. Each of these processes is created as a group and then distributed outwards. Referring now to FIG. 47, there is illustrated a diagrammatic view of the logical flow of creating a process. The logical instructions for the process were input at a block 4702, which have been input to a process generator 4704. The process generator requires access to various standard process blocks stored at a database 4706. The process generator will receive the logical instructions and then assemble a process of a transaction that will define a group of processes for each Extent and a group of Extents. This is illustrated as a plurality of sequentially performed processes of 4708. Each of the process blocks has input and output and requires information to be associated therewith. For example, there may be a process block that defines a destination route and requires information as to the originating node and the destination node for that particular process block. This will enable that process block to generate possibly a portion of the transaction packet, extract information from the database or reside on a conversion server for processing the transaction packet at the conversion server. By utilizing process blocks, the assembly of the overall Extent is facilitated in a much more expedient manner. Once all of the process blocks have been assembled, it being remembered that these are a sequence of instructions, the logical flow will be to a finish block 4710 to complete the process assembly and then generate the code with a code generator block 4712. This code generation constitutes the process which is then stored in the process database 4708 with a process number and a sequence number for a particular transaction. It should be remembered that the process server for a given transaction will associate a plurality of Extents together such that, once a channel ID is defined, each process in the channel ID will recognize a previous process and the data will flow through the system in accordance with this process sequence. Referring now to FIG. 48, there is illustrated a flow chart depicting the operation of propagating Extents, i.e., predetermined processes that are generated by the process generator 4702, to various nodes in the system. The program is initiated at a block 4802 and then proceeds to a block 4804 to determine the source and destination IDs in the process. The system will then flow to a function block 4806 to then &#x201c;ping&#x201d; all of the destination and source IDs required for this process. This is required to ensure that all of the destination and source IDs are actually on-line and working. The program will then flow to a decision block 4808 to determine if the ping operation has been successful. If not, the program will default to an Error block 4810 and, if all the tests came back successful, the program would flow to a function block 4812 to set a flag to that of a test mode. As will be described hereinbelow, each process must go through an evaluation step before it goes &#x201c;live&#x201d; in the overall transaction between two systems, nodes or customers. The program will then flow to a function block 4814 to distribute the various Extents that were created in the process to the respective ones of the nodes, it being remembered that the process is comprised of a group of subprocesses, the subprocesses distributed to various nodes. The program then flows to a function block 4816 to send some type of test trigger to test the system. When the system is actually created, it may be that there is a final destination node that is to receive information or an order. The order can be placed with some kind of notation that this is a test transaction, such that, when the trigger signal is received for the test operation, all of the resources in the transaction path are &#x201c;exercised&#x201d; to determine if the transaction has been completed in the manner which was contemplated by the original logical instructions that were input to the process generator. It may be that there are many addresses that are &#x201c;dummy&#x201d; in nature, such that the final destination of the process will end up at a dummy node with, for example, a dummy facsimile, a dummy order, or the such. The program then flows to a decision block 4818 to determine if the process has been approved. This could be a manual operation which evaluates the transaction flow to determine if it has been executed correctly, i.e., the correct order has been placed in the correct manner at the destination or that a particular process interfaces with another system in the correct manner. If it has not been approved, it may be that the process needs to be recreated, as indicated by block 4820. However, if it is approved, the program will flow to a function block 4822 wherein the process will be remapped to a live system, i.e., the flag may be set to a live mode or, in conjunction therewith, various addresses in the process are remapped to change some parameters thereof. The program then flows to a function block 4824 to set the flag to &#x201c;live&#x201d; and then to a function block 4826 to redistribute the subprocesses over to the associated nodes. It should be remembered that a copy of each of the subprocesses and the overall process are maintained in the process database 4608. The program then flows to a Done block 4828. Referring now to FIG. 49, there is illustrated a diagrammatic view of two systems that interface internal thereto with ID packets. There is provided a first system 4902 labeled SYS A and a second system 4904 labeled SYS B. The first system 4902 has associated therewith a host 4906, a network structure 4908, an ID server 4910 and a router 4912. The host 4906 has associated therewith a host database 4914. Similarly, the system 4904 has a host 4916, a network structure 4918, an ID server 4920 and a router 4922. The host 4916 has associated therewith a database 4924. The two systems 4902 and 4904 interface with each other through an interconnection 4926 between the routers 4912 and 4922. In some situations, there can exist two systems that have dissimilar databases, i.e., the software utilizes a significantly different operating system and database generation system resulting in a different database structure. When two distinctly different databases are utilized in two companies, it is difficult for the two companies to converse with each other without some type of adapter therebetween. This situation is exacerbated when the two companies are merged. For example, when two companies become a single entity and desire to have a single common database, it is necessary to convert both databases into a new database or to convert one database into the other database. This is not an uncommon situation. The problem exists when there are common aspects of two databases such as common products, common vendor IDs, etc. For example, there could be a common vendor between the two databases that was utilized for purchasing products from, or for shipping products thereto. Both databases would have information associated with this same vendor entered into their respective database structure in a significantly different manner, due to the dissimilarity of the database structures. However, even if the two database were the same, i.e., both Oracle&#xae; databases, they could have a different formats and the such for various fields, i.e., a different organization. The reason for this is that a great deal of latitude is provided to the system administrator when creating the database in defining the format of ID fields. It may be that one administrator for one database structure formats it with numbers and the other system administrator formats it with textual characters. This presents a problem in that comparison of IDs in a common field will not allow merging of records. As such, it is possible when merging into a new database that there could actually be two new vendor IDs generated in the new database structure for a single common vendor. As such, all links to the common vendor with two different vendor IDs would still be separate and distinct, as they were in the two different databases. In order for the two systems 4902 and 4904 to merge together into a single system, they would have to have a common database structure wherein the database 4914 and the database 4924 will merge into either a common separate database or one merged into the other. Referring now to FIG. 50, there is illustrated a table depicting the difference in the two systems and the way in which they might handle vendor IDs. In the example of FIG. 50, there is listed a vendor &#x201c;ABC&#x201d; that exists in both the database of SYS A and the database of SYS B. In SYS A, there is a unique vendor ID associated with vendor ABC, which vendor ID is &#x201c;123.&#x201d; Also, there is a unique ID packet associated therewith in SYS A identified as &#x201c;XXX.&#x201d; This ID packet XXX, of course, is representative of a unique number that has associated therewith the constituent parts as described hereinabove in the form of the core ID, the device ID and the item ID. This is for representative purposes only. In SYS B, the vendor ID is denoted as &#x201c;567&#x201d; and the ID packet is also provided as being a unique value &#x201c;YYY.&#x201d; The reason that the vendor IDs in SYS A and SYS B are different is due to the fact that the system administrator formatted them for different values. It could be also that they were assigned vendor IDs in a sequential manner and it was the time they were put in that defined what the vendor IDs would be. With respect to the ID packets, these are generated by each system's ID server and, therefore, would constitute a unique number. However, it could be that the classification portion of the ID packet, that embedded in the ID packet, could be the same. It would be the time stamp and sequence number that would create the unique difference. In any event, it can be seen that the vendor IDs for the two systems are different for the same vendor and, therefore, some conversion must be performed. Referring now to FIG. 51, there is illustrated a block diagram view of the merging operation of the two databases 4914 and 4924 into a single database 5102. Each of the records in the databases 4914 and 4924 are compared with a compare operation, illustrated as a block 5104, to determine if they are the same. The vendor IDs may be different, but the underlying information associated with that vendor ID would have similarities, if not being identical. For example, the name of the vendor would be the same, the address of the vendor would be the same, even the zip code of the vendor would be the same. By examining this information that underlies the ID packet and is associated with the vendor IDs in the respective databases, an evaluation can be made as to whether they are the same vendor. If so, then this will provide a TRUE output from the comparison block 5104. Each of the databases is processed through a separate conversion block&#x2014;5106 for SYS A and 5108 for SYS B. A multiplexing block 5110 is provided for selecting either the output of conversion block 5106 or the output of conversion block 5108. When the comparison is TRUE, this indicates that the data in both systems is identical and, as such, the conversion of that information to a format compatible with the database 5102 will be performed by both conversion blocks 5106 and 5108. For a TRUE operation, only one conversion operation needs to be selected and this, in the present example, would be that associated with block 5106. However, if it is FALSE, then the multiplex block 5110 would first select the output of conversion block 5106 for storage in database 5102 and then the output of conversion block 5108, such that both IDs were converted. As noted hereinabove, when an ID packet is converted, this would result in a new ID packet being generated and given a new vendor number for the converted information. However, each conversion operation during the merge could be different and different parameters and aspects thereof could be added or subtracted. Also, the new ID packet will have the underlying profile information associated therewith. In an alternate embodiment, illustrated in FIG. 52, information in one database is merged into another database and made compatible therewith. In this operation, the data in databases 4914 and 4924 are compared with a comparison block 5202 to determine if they are substantially identical, as was the case with respect to the comparison block 5104. Whenever a TRUE result occurs, this indicates that they are identical and, as such, there is no need to convert the data from database 4914. There need only be a selection of the data from the database 4924 which is provided by a selection block 5206. This would be stored in a merged database 5208 which is basically identical to the database 4924, albeit larger. Whenever there is a FALSE comparison, i.e., there is no match to a record in the database 4914 and the data in the database 4924, then this data will be converted through a selection block 5210 and a conversion block 5212. Although illustrated as being individually selected as records, typically all of the data in the database 4914 will be compared in a search operation to the data in database 4924 to determine if there is a match for that data in database 4924. If there is no match with the data in database 4924, then this data from database 4914 is converted and stored in database 5208. Database 5208 will be initialized with all of the data in database 4924 such that there effectively will not be an actual selection operation, although there could be such an operation. Referring now to FIG. 53, there is illustrated a diagrammatic view of the comparison operation. In this example, the data underlying the ID packet would be that associated with, for example, the name, the address, the zip code and the vendor ID code. This exists on an external database external to the database to which it is being merged, i.e., the internal database. This information is input to a compare block 5302 and this is compared with the name table in the internal database, the address table in the internal database and the zip code in the internal database. Many other parameters could be compared. This is a function of the compare operation wherein a compare operation &#x201c;pulls&#x201d; data from the other database for the purpose of evaluating its presence in the internal database. If it is determined that the profile data underlying the ID packet is identical, then a new ID packet need not be generated. However, if it is determined that this information is new, then a new ID packet would be generated with the profile information and possibly a new vendor ID code generated in the database. Referring now to FIG. 54, there is illustrated a flowchart depicting the comparison operation, which is initiated at a Start block 5402 and then proceeds to a block 5406 to pull the name from the external database and then compare it to the name table in the function block 5908. If a decision block 5910 determines that it is a TRUE comparison, then the address information will be pulled from the external database and compared to an address table, as indicated by function blocks 5912 and 5914. If the comparison is TRUE, as determined by decision block 5916, the flow will then pull the zip code from the external database and compare it to the zip code table, as indicated by function blocks 5918 and 5920. If this results in a TRUE comparison, as determined by a decision block 5922, the program will flow to a function block 5924 to use the existing ID packet. However, if any of the decision blocks 5910, 5916 or 5922 determine that it was not a true comparison, then the program will flow to a function block 5926 to create a new ID packet, as described hereinabove. The program will then return via a return block 5930. Referring now to FIG. 55, there is illustrated a diagrammatic view of the operation of transferring information from a system, SYS A, to a system, SYS B. This is a further explanation of the internal/external operation, as described hereinabove with respect to FIG. 45. When data is transferred between two systems, it can be transferred in the native form or it can be transferred in the form of ID packets, noting that the ID packets for two systems may be different, as they were created with two different ID servers. In the example illustrated in FIG. 55, SYS A has provided therein a database represented by Table 5502. This database is divided into, for example, two columns, one associated with a vendor number and one associated with a profile, such that each vendor number has associated therewith a profile. This vendor number could be an ID packet. However, it could merely be the native vendor number of SYS A. When the data or information regarding vendors is transferred to SYS B, it is transferred essentially intact, i.e., with the vendor numbers that exist in SYS A. (Note that the vendor number could be reflected as an ID packet.) This will result in a database or table 5504 being transmitted to SYS B as external data therein, referred to as a table EXT A. This EXT A database or table consists of all of the vendor numbers in the profile, as it existed in SYS A and in a database structure associated with SYS A. At SYS B, there is a mapping function performed, as indicated by an arrow 5506 that maps all or a portion of the information in the table 5504 to a new table 5508, which provides the vendor number in the database SYS B in compliance with all the rules associated therewith. As described hereinabove with respect to FIG. 45, this may merely require the generation of an ID packet that is generated utilizing the profile information of the table 5504. However, the table 5508 also provides a link back to table 5504 in a column 5510. The profile information in table 5504 contains, in addition to the substantive information relating to the vendor associated with a vendor number, various links and change flags. The operation of these will be discussed hereinbelow. Once the database has been created at a table 5510, which exists at the ID server for SYS B, this information is then propagated to the various account servers, as represented by tables 5512, there being three such tables. Each of these tables 5512 represent other nodes in SYS B that require information regarding the vendor numbers. These, in practice, could be other account servers that have their own ID servers associated therewith. They could, also, be such things as the conversion server, the router, etc. These are associated with nodes in the system that require information as to vendor numbers without requiring the node to constantly go back through the network to the main database at table 5508 to determine what the underlying information would be. Once the information in table 5508 at the ID server is propagated down to each of the nodes and the tables 5512, it is important that the ID server be apprised of the address for each location in each of the tables that the particular vendor number is linked to. This is stored in a column 5514. Therefore, the ID server 5508 has a link to both the data in the table 5504 and to all other databases that lie below the table 5514 in the hierarchal structure. Referring now to FIG. 56, there is illustrated a simplified schematic of how information is propagated through the network from one ID server, a source ID server 5602, down to a plurality of lower servers. In the illustrated embodiment of FIG. 56, there are illustrated three lower ID servers, an ID server 5604, an ID server 5606 and an ID server 5608, for three different systems, SYS B, SYS C and SYS D. The ID server 5604 has associated therewith two account servers 5610 and 5612 with ID server 5606 having a single account server 5614 associated therewith and ID server 5608 having two account servers 5616 and 5618 associated therewith. In operation, there will be a &#x201c;release&#x201d; operation that allows information to be transferred from one system to another. In the first operation, there will be a request made by one of the lower ID servers for information, which information will then be released to that ID server, i.e., this indicating that the data being released is a valid data. This is then transmitted down to the external side of one or more of the servers 5604-5608. At each of the servers 5604-5608, the data is converted to the database structure on the internal side thereof as internal data to that ID server. This is then propagated or released in a second operation to one or more of the account servers associated therewith, i.e., to a lower level. Thereafter, when information is changed, then a change or release is &#x201c;pushed&#x201d; from the higher level to the lower levels and this change then propagated downward. For example, if ID server 5602 for SYS A had propagated data such as a catalogue down to one or more of the servers 5604-5608, and then desires to create a change, it must change the information at every location that it is presently disposed at. Suppose that this information were disposed at two or more of the account servers 5610-5618. In order to facilitate this change, the ID server 5602 would merely have to push the change to each of the servers to which the original information had been released. Once the lower level ID servers 5604-5608 receive the change information, then they will make the corresponding change in all of the servers therebelow. The reason for this is that the ID server 5602 is aware of all locations to which data was originally released or pushed to and the ID servers 5604-5608 are aware of all the addresses of that particular information and can then propagate down the change to those servers. The system of FIG. 56 is illustrated in a simplistic form in FIG. 56a. In FIG. 56a, there is illustrated a database 5630 associated with SYS A ID server 5602. A particular data field or addressable location 5636 is illustrated. This is passed through a mapping function 5634 for storage in a database 5636 as an addressable information field 5638. This information in addressable field 5638 is then propagated down to each of three databases 5640, 5642 and 5644 at addressable locations 5646,5648 and 5650, respectively. When the change is required in the addressable location 5632 in the ID server 5602, this change merely needs to be &#x201c;pushed&#x201d; to the database 5636 into the addressable location 5638. This is facilitated, as described hereinabove, by pushing into the external side and then an Extent operating to reflect this change over to the SYS B database 5636. Once the change has been stored in addressable location 5638, by utilizing the links that were created in the database 5636, each of the addressable locations 5646-5650 can have a change pushed thereto. As such, there is a &#x201c;one source&#x201d; link to all of the information that exists within the network, this being that addressable location in the database 5638. By making the change at this one source, then all of the data in the system can be changed. Referring now to FIG. 57, there is illustrated a flow chart depicting the operation wherein data is transferred from one system to the external side of a second system. The program is initiated at a block 5702 and then proceeds to a function block 5704 wherein a transactional relationship is initiated. In this operation, a contact will be made from, for example, SYS B to SYS A requesting information. This information may be in the form of their vendor list, their product catalogue, etc. Also, the manner by which a transaction between the two companies will be effected is also determined. Once the relationship has been initiated, the program will flow to a decision block 5706 to determine if data in the form of vendor numbers, product catalogues, etc., is to be downloaded. If so, the program will flow along the &#x201c;Y&#x201d; path to a function block 5708 to transfer the SYS data to the external side of SYS B. At SYS B, this external data from SYS A is then mapped to the internal side of SYS B, as indicated by a function block 5910. The program then flows to decision block 5912 in order to determine if the SYS B address is to be sent to the SYS A system. This is optional. This is an operation wherein the actual location in SYS B on the internal side thereof can be transmitted to SYS A. This would allow, for example, SYS A to actually point to the location within SYS B at which the data will be populated, as described hereinabove. However, in the preferred embodiment of the disclosure, the addressing is typically maintained at SYS B and SYS A and is only allowed access to the external side of SYS B. If the option is selected wherein the internal address is to be sent back to SYS A, then the program will proceed to a function block 5914 to transmit this SYS B address back to the internal side of SYS A and then to a function block 5916 to store the SYS B address in the SYS A database. However, in the preferred embodiment, the program will flow along the &#x201c;N&#x201d; path to an End block 5924. Referring now to FIG. 58, there is illustrated a flow chart depicting the operation of propagating the data from the internal side of SYS B down to the internal components thereof, such as the conversion server, the router, etc. The program is initiated at a start block 5802 and then proceeds to a function block 5804 to create a filter database, i.e., to extract the desired information from the external side that was received from SYS A and map it to the internal side of SYS B, i.e., create data packets internal to SYS B. This was described hereinabove with reference to FIG. 45. The program then flows to a function block 5806 to propagate downward the information to the destination ones of the account servers, such as the conversion server, the router, etc. When this occurs, the destination device will return the address at the destination device at which the information is stored, this indicated by decision block 5810. When the destination address is received, the program will flow from decision block 5810 to a function block 5812 wherein a linkage created in the database associated with the internal side of the ID server of SYS B. The program will then flow to an End block 5814. It is noted that when the link addresses are created, this link provides a link between the ID server and SYS B and the destination device and also a link address is provided between the SYS B database at the ID server and the external side thereof, such that a change in the external side can be propagated through to the destination device, since the ID server on the internal side of SYS B has knowledge of where the information came from, i.e., a link to the external side, and knowledge of where the information that was mapped from the external side is stored. Referring now to FIG. 59, there is illustrated a flow chart depicting the operation of altering data in SYS A, which is initiated at the block 5902 and then proceeds to a function block 5904, wherein a transfer operation effected for a change in the SYS A database. The SYS A database on the internal side thereof has knowledge of the fact that information in this database resides in other locations on a network and remote locations on other networks. When a change is made to the database, these changes are noted and propagated to the external sides of systems at which the database was downloaded. This is indicated by a function block 5906. Once a flag or such is set on the external side of any one of the systems to which data from SYS A was downloaded, the internal side will recognize this flag as being set, i.e., recognize the change, and then a determination will be made as to whether this information was actually mapped to the internal side thereof. This is indicated by an operation in a decision block 5908. If the information is not in the filtered database, i.e., it was never mapped, then the program proceeds along the &#x201c;N&#x201d; path to an End Block 5910. If the data was mapped, then the program will flow along the &#x201c;Y&#x201d; path to map the new data over to the internal side of SYS B, i.e., make the change in the database at the ID server, and then &#x201c;push&#x201d; this change downward to the destination devices as indicated by a function block 5912. Once an acknowledgment is received, as indicated by a decision block 5914, the program proceeds to the End block 5910. Conversion Server Referring now to FIG. 60, there is illustrated a diagrammatic view of the Conversion Server operation. The Conversion Server, as described hereinabove, provides an operation wherein data and/or ID packets are transferred thereto for general processing in the intermediate or ID packet domain. However, when data is &#x201c;pulled&#x201d; from a host system or &#x201c;pushed&#x201d; to the host system, the host system will operate in its native database structure and language, i.e., there will be a predetermined format for that data. When two hosts must transfer data therebetween, some type of adaptor or conversion operation is required. The databases need not be totally different systems, but can merely be configured differently. For example, there may be an option on one database to select the format of a vendor code that is comprised of alpha characters as opposed to numeric characters, or it could be that the vendor is defined with a different value, albeit in the same format (this due to being defined independent of other systems). The vendor would be assigned a universal packet unique to that vendor such that, whenever information was being transferred from one system to the other, it would first be converted to the ID packet, this information transferred over to the next system and then converted from the ID packet over to the native format for the destination system. With further reference to FIG. 60, there are illustrated two hosts, a host A 6002 and a host B 6004. Host A has associated therewith a native database 6006 which is formatted and structured in accordance with the structure of the database. Similarly, the host B has associated therewith a native database 6008. Whenever information is transferred between the two databases, there must be some type of conversion operation. Consider the example where data is transferred from host A to host B. In this configuration, data must be pushed from the database 6006 over to the system (or it could be pulled, which will be described in more detail hereinbelow hereinbelow). A data manager 6010 is provided at host A which is operable to run a program that will fetch data from the native database 6006 and transfer it to the system. As described hereinabove, this data manager 6010 is an Extent that is a program that runs on or proximate to the host 6002. This Extent, when initiated, will fetch data in blocks and in a predetermined order. There are many operations that can be performed on the data in order to efficiently extract it and transfer it to the system. In addition, as also described hereinabove, every time data is fetched from the database 6006, there will be a proprietary address assigned to a proprietary column in the database 6006 in association with a time stamp. Of course, if a proprietary address had already been assigned, it will not be updated. This is for the purpose of keeping track of the data in the database 6006 without regard to any restructuring of the data therein. This is not a host A address. Once data is pulled from database 6006, it is routed in its native format to a data conversion block 6012. This is the Conversion Server. As described hereinabove, this operation requires first creating a transaction packet and transferring the data in accordance with various channel and feed ID packets. This will first, as described hereinabove, be sent to the router for redistribution to the Conversion Server. For simplicity, a direct path is illustrated to the conversion operation. In the conversion operation, the data is converted from the native format for database 6006 into an ID packet. These ID packets, as described hereinabove, were generated at the ID server and populated down to the Conversion Server. This Conversion Server will convert the data from the native data to an ID packet and store these in a database, represented by a table 6014. In this table, the data will be structured such that there is a relation between the ID packet, the address in the A database (this will be the address that is stamped into the data by the data manager 6010) and the information associated with the information from the database 6006 in addition to a pointer therefor. This pointer would be such a thing as a vendor ID. Once this has been created, the data can then be transferred to a processing block 6016 for processing the ID packet domain. This process typically takes place at the Conversion Server but can be transferred to another Conversion Server through the system or to another processing node. In any event, all processing can be effected with the ID packets in the ID packet domain. Such things as layering, consolidation and merging can be performed utilizing the ID packets. Since an ID packet is a defined length and each ID packet is unique, this facilitates the layering operation and also facilitates transfer of information (this due primarily to the finite length of a packet rather than the variable length packet). By substituting the ID packet as the index, then groups of data can be transferred with a single ID packet, i.e, a vendor ID, the address, etc., associated therewith, thus reducing data transmission. This advantage will be realized in EDI to a large extent. Once processed, the processed data in the form of the ID packets are sent to a second data Conversion Server 6018 for conversion from the ID packet to the native language or structure of the database 6008. This is reflected in a storage table 6020. This is then transferred to the database 6008 by first transferring it in the native language to a data manger 6022 associated with the host B. Although this transaction is illustrated as being pushed from host A to host B, it could be that host B would request information from host A and, &#x201c;pull&#x201d; the data therefrom. Further, there can be a process running in the process block 6016 that requires more data and can actually pull data from the respective host A or host B for the processing operation. This is merely a function of the type of Extent that runs on any single node in the system, noting that the operation of transferring between nodes is facilitated with Extents, i.e., the Extents comprise the input/output of each node. Referring now to FIG. 61, there is illustrated a flow chart depicting the operation of the conversion operation, which is initiated at a block 6102 and then proceeds to a function block 6104 wherein the data is pushed/pulled and transferred from host A, for example, to the Conversion Server. The program then flows to a function block 6106 to convert the data to an ID packet format and then to a function block 6106 to convert the data to the native language of host B. The program then flows to a function block 6108 to push/pull data between the Conversion Server and the host B and then the program flows to an End block 6110. Referring now to FIG. 62, there is illustrated a flow chart depicting the operation of the data manger, i.e., the Extent running at one of the hosts. This is initiated at a block 6102 and then proceeds to a decision block 6204 to determine if this is a push or pull operation. If it is a pull operation, the program will flow to a decision block 6206 to await an external request for data from another node in the system, at which time it will flow along a &#x201c;Y&#x201d; path to a function block 6208 to fetch data and then stamp the address and time if necessary. The program will then flow to a function block 6210 to create a transaction packet and then to a function block 6212 in order to transfer this data to a router, which operation will result in the data being transferred to the Conversion Server. The reason for this is that the Extent creates a transaction packet having associated therewith feed and channel ID packets that define the route to the Conversion Server and eventually to the destination server, i.e., host B in the example hereinabove. After transferring to the router, the program will flow back to the input of function block 6204. If the operation were a push operation, then the program would flow from decision block 6204 to a decision block 6214 in order to determine if the operation in the push mode has been completed. If not, the program flows to the function block 6208 in order to fetch the data, create the transaction packet and then transfer the transaction packet to the router. This will continue until all of the data necessary is transferred, at which time the program will flow along the &#x201c;Y&#x201d; path back to the input of decision block 6204 from decision block 6214. Referring now to FIG. 63, there is illustrated a diagrammatic view for one operation of the Conversion Server, i.e., that utilized for a consolidation operation. In a consolidation operation, there is provided some type of mapping function which takes information from two similar or dissimilar databases and consolidates them into a single format. One example of this is a company having a large number of sub-companies, each having a general ledger associated therewith. In this general ledger, there will be associated a plurality of charts of accounts (COA). There may be thousands of account definitions in each Chart of Account for each of the sub-companies. The problem is that a central office does not desire to have all of this detail, i.e., they wish to consolidate a large amount of information into a single account record. For example, it could be that one company would discriminate expenses associated with delivering documents into such things as First Class Mail, Express Mail, overnight couriers, hand delivery, etc. and keep track of each one of these expenses in a separate chart of account. It may be that a central system only desires to have a single account referred to as &#x201c;Postage&#x201d; wherein all of these accounts would be mapped thereto. Therefore, there must be provided some type of mapping function which allows this to occur. In the embodiment of FIG. 63, there is provided a first company having a COA 6302 wherein there are provided 350 separate accounts, and a second company having a COA 6304 with 250 accounts defined therein. It is desirable to map these into a single COA 6306 with only 150 accounts. Therefore, for each account in the separate COAs, there must be some type of mapping function. There is provided a mapping function 6310 for mapping from COA 6302 into 6306 and a mapping function 6312 for mapping from the COA 6304 into the COA 6306. Additionally, it may be that this mapping function must work between two dissimilar systems. With use of the Conversion Server described hereinabove, an ID packet is created which defines the final consolidated account in the COA 6306. This ID packet may be, for example, defined as &#x201c;postage expense.&#x201d; The ID packet would have an association in the ID server that would associate it with all of the other accounts associated with its company and in its company's native database. Therefore, whenever information for one or all of the accounts associated with the one ID packet were transferred to the associated mapping block, i.e., the Conversion Server, it would be recognized through a matching or searching operation that this particular account and the associated records would be associated with this ID packet and then they would be transferred to the COA 6306 in an ID packet format. At the destination, this ID packet would then be converted into the account associated therewith in its native database and then an update performed. Referring now to FIG. 64, there is illustrated a flow chart depicting this operation, which is initiated at a start block 6402 and then proceeds to a function block 6404 to fetch the record or the account in the above example. This is then forwarded to the Conversion Server, as indicated by a function block 6406 and then proceeds to a decision block 6408 to determine if there is an ID packet match. If the ID packet exists, then a simple conversion is performed and the program flows to a function block 6410 to select the ID packet and then transfer the information associated with this ID packet to the destination system utilizing the ID packet as a pointer, as indicated by a function block 6412, the program then flowing to a block 6414 to return the program. However, if the system determined that there were no ID packet match, i.e., that nobody had associated the account with a particular ID packet, this would result in an error and the program will flow along an &#x201c;N&#x201d; path from decision block 6408 to a function block 6416 in order to update and define the particular ID packet. Once an ID packet is defined, it would be defined at the ID server and then propagated down to the Conversion Server. The program would then flow back to the input of the decision block 6408 to again perform the matching operation, at which time it would proceed along the path to the destination. Referring now to FIG. 65, there is illustrated a flow chart depicting the operation of the update and definition process, which is initiated at a block 6502 and then proceeds to a function block 6504. Function block 6402 represents a system wherein the user will be prompted to make a selection, this operation typically performed at the ID server. The user will typically be provided a large number of accounts from which to select, these being the accounts at the destination, i.e., the point to which they are consolidated. The user will make a selection to, for example, associate a courier service with postage expense. This may be performed by the user at the sub-company or it may be performed at a central area. In any event, this ID packet must be generated at the ID server associated with the system that is transferring data. It could be, however, that the ID packet was generated at a central ID server and then propagated down to the local ID server. Once the user or system administrator has made a selection, as indicated by decision block 6506, the program will flow to a function block 6508 to update the ID server and the associated packet with this relational information. Once the ID packet has been updated, i.e., the same ID packet that existed before but with the additional relational links provided, the program will flow to a function block 6510 wherein this updated ID will be propagated down to the Conversion Server to replace the previous ID packet. The program will then flow to a return block 6512. Although the preferred embodiment has been described in detail, it should be understood that various changes, substitutions and alterations can be made therein without departing from the spirit and scope of the invention as defined by the appended claims.\",\n          \"DETAILED DESCRIPTION OF THE INVENTION The present invention is directed generally toward adenine nucleotide translocator (ANT) polypeptides, which as provided herein may refer to any ANT isoform; to expression constructs containing nucleic acids encoding ANT and to natural and synthetic small molecules that interact with ANT, including ANT binding ligands. The present invention relates in part to the unexpected findings that bacterial, insect, yeast or mammalian expression systems can be designed for reliable production of recombinant human ANT polypeptides in significant quantities. In certain aspects the invention provides compositions and methods for producing recombinant ANT polypeptides that employ regulated promoters, and in certain of these and other aspects the invention provides compositions and methods for producing recombinant ANT polypeptides that are ANT fusion proteins. In certain preferred embodiments, the design of such expression systems includes the use of a host cell that lacks endogenous ANT or in which endogenous ANT gene expression is substantially impaired, as provided herein. The present invention thus also pertains in part to methods for producing and isolating recombinant ANT polypeptides, including human ANT polypeptides and in preferred embodiments human ANT3 polypeptides, that may then be used in various binding assays and screening assays and the like. In view of the surprising observation that expression of recombinant human ANT polypeptides can be achieved at levels enabling such uses of these ANT polypeptide products, the present invention provides assays (including high throughput assays) for identifying agents that bind to recombinant human ANT. Accordingly, the present invention further relates in part to novel human ANT ligands, the synthesis, selection and characterization of which would heretofore have not been possible given the need for expressed recombinant ANT polypeptides to use in binding assays. The invention also pertains to agents that interact with ANT, including agents that enhance or impair any ANT functions known to the art, including but not limited to those described herein, and to incorporation of such agents into pharmaceutical compositions and their use in therapeutic methods. As dicussed above, the present invention relates in part to the unexpected finding that recombinant adenine nucleotide translocator (ANT) polypeptides, which includes full length ANT proteins and polypeptides, fragments and variants thereof, and further includes ANT fusion proteins as provided herein, can be produced in useful amounts by using a recombinant expression vector having a regulatory nucleic acid operably linked to a nucleic acid encoding ANT. In particular, the invention provides compositions and methods for producing recombinant ANT polypeptides through the use of a regulated promoter; the invention also provides recombinant ANT polypeptides that are ANT fusion proteins. The invention also pertains to compositions and methods to identify the presence of ANT polypeptides and to isolate recombinant ANT, and in addition to screening assays for compounds that interact with ANT. Accordingly, the invention provides certain advantages with regard to regulation of mitochondrial function, and in particular regulation of the mitochondrial permeability &#x201c;pore&#x201d;. By way of background, four of the five multisubunit protein complexes (Complexes I, III, IV and V) that mediate ETC activity are localized to the inner mitochondrial membrane, which is the most protein rich of biological membranes in cells (75% by weight); the remaining ETC complex (Complex II) is situated in the matrix. ANT represents the most abundant of the inner mitochondrial membrane proteins. In at least three distinct chemical reactions known to take place within the ETC, positively-charged protons are moved from the mitochondrial matrix, across the inner membrane, to the intermembrane space. This disequilibrium of charged species creates an electrochemical potential of approximately 220 mV referred to as the &#x201c;protonmotive force&#x201d; (PMF), which is often represented by the notation &#x394;&#x3c8; or &#x394;&#x3c8;m and represents the sum of the electric potential and the pH differential across the inner mitochondrial membrane (see, e.g., Ernster et al., 1981 J. Cell Biol. 91:227s and references cited therein). This membrane potential drives ANT-mediated stoichiometric exchange of adenosine triphosphate (ATP) and adenosine diphosphate (ADP) across the inner mitochondrial membrane, and provides the energy contributed to the phosphate bond created when ADP is phosphorylated to yield ATP by ETC Complex V, a process that is &#x201c;coupled&#x201d; stoichiometrically with transport of a proton into the matrix. Mitochondrial membrane potential, &#x394;&#x3c8;m, is also the driving force for the influx of cytosolic Ca2+ into the mitochondrion. Under normal metabolic conditions, the inner membrane is impermeable to proton movement from the intermembrane space into the matrix, leaving ETC Complex V as the sole means whereby protons can return to the matrix. When, however, the integrity of the inner mitochondrial membrane is compromised, as occurs during MPT that may accompany a disease associated with altered mitochondrial function, protons are able to bypass the conduit of Complex V without generating ATP, thereby &#x201c;uncoupling&#x201d; respiration because electron transfer and associated proton pumping yields no ATP. Thus, mitochondrial permeability transition involves the opening of a mitochondrial membrane &#x201c;pore&#x201d;, a process by which, inter alia, the ETC and ATP synthesis are uncoupled, &#x394;&#x3c8;m collapses and mitochondrial membranes lose the ability to selectively regulate permeability to solutes both small (e.g., ionic Ca2+, Na+, K+, H+) and large (e.g., proteins). Without wishing to be bound by theory, it is unresolved whether this pore is a physically discrete conduit that is formed in mitochondrial membranes, for example by assembly or aggregation of particular mitochondrial and/or cytosolic proteins and possibly other molecular species, or whether the opening of the &#x201c;pore&#x201d; may simply represent a general increase in the porosity of the mitochondrial membrane. MPT may also be induced by compounds that bind one or more mitochondrial molecular components. Such compounds include, but are not limited to, atractyloside and bongkrekic acid, which are known to bind to ANT. Methods of determining appropriate amounts of such compounds to induce MPT are known in the art (see, e.g., Beutner et al., 1998 Biochim. Biophys. Acta 1368:7; Obatomi and Bach, 1996 Toxicol. Lett. 89:155; Green and Reed, 1998 Science 281:1309; Kroemer et al., 1998 Annu. Rev. Physiol. 60:619; and references cited therein). Thus certain mitochondrial molecular components, such as ANT, may contribute to the MPT mechanism. As noted above, it is believed that adenine nucleotide translocator (ANT) mediates ATP/proton exchange across the inner mitochondrial membrane, and that ANT inhibitors such as atractyloside or bongkrekic acid induce MPT under certain conditions. Hence, it is desirable to obtain ANT in sufficient quantities for structural and functional assays that provide, for example, ANT ligands and other agents that interact with ANT, which will be useful for therapeutic management of mitochondrial pore activity. See also U.S. Ser. No. 09/161,172, entitled &#x201c;Compositions and Methods for Identifying Agents that Alter Mitochondrial Permeability Transition Pores&#x201d;, which is hereby incorporated by reference. The compositions and methods of the present invention can be adapted to any prokaryotic or eukaryotic ANT, including plant and animal ANTs, which may further include, for example, yeast, vertebrate, mammalian, rodent, primate and human ANTs, for which amino acid sequences and/or encoding nucleic acids will be known to those familiar with the art. Three human ANT isoforms have been described that differ in their tissue expression patterns. (Stepien et al., 1992 J. Biol. Chem. 267:14592; see also Wallace et al., 1998 in Mitochondria &#x26; Free Radicals in Neurodegenerative Diseases, Beal, Howell and Bodis-Wollner, Eds., Wiley-Liss, New York, pp. 283-307, and references cited therein.) Nucleic acid sequences for cDNAs encoding these three human ANT isoforms have been reported (FIG. 1; See Neckelmann et al., Proc. Nat'l. Acad. Sci. U.S.A. 84:7580-7584 (1987) for huANT1 cDNA [SEQ ID NO:1]; Battini et al., J. Biol. Chem. 262:4355-4359 (1987) for huANT2 cDNA [SEQ ID NO:2], and Cozens et al., J. Mol. Biol. 206:261-280 (1989) for huANT3 cDNA [SEQ ID NO:3]; see FIG. 2 for amino acid sequences of huANT1 [SEQ ID NO:31] huANT2 [SEQ ID NO:32] and huANT3 [SEQ ID NO:33].), and ANT gene sequences have been determined for a number of species (See, e.g., Li et al., 1989 J. Biol. Chem. 264:13998 for huANT1 genomic DNA; Liew et al. GenBank Acc. #N86710 for huANT2; Shinohara et al., 1993 Biochim. Biophys. Acta 1152:192 for rat ANT gene; for others see also, e.g., Ku et al., 1990 J. Biol. Chem. 265:16060; Adams et al., 1991 Science 252:1651; and WO 98/19714.). ANT sequences among mammalian species are highly conserved; for example, at the amino acid level murine ANT1 and ANT2 exhibit 98% sequence identity with human ANT2. Full length amino acid sequences of at least 29 ANT proteins have been reported to date from a variety of animal and plant species, with most of these deduced from nucleic acid sequences. (Fiore et al., 1998 Biochimie 80:137-150) The present invention further relates to nucleic acids which hybridize to ANT encoding polynucleotide sequences as provided herein, as incorporated by reference or as will be readily apparent to those familiar with the art, if there is at least 70%, preferably at least 90%, and more preferably at least 95% identity between the sequences. The present invention particularly relates to nucleic acids which hybridize under stringent conditions to the ANT encoding nucleic acids referred to herein. As used herein, the term &#x201c;stringent conditions&#x201d; means hybridization will occur only if there is at least 95% and preferably at least 97% identity between the sequences. The nucleic acids which hybridize to ANT encoding nucleic acids referred to herein, in preferred embodiments, encode polypeptides which either retain substantially the same biological function or activity as the ANT polypeptides encoded by the cDNAs of FIG. 1 [SEQ ID NOS:1, 2 and 3], or the deposited expression constructs. As used herein, to &#x201c;hybridize&#x201d; under conditions of a specified stringency is used to describe the stability of hybrids formed between two single-stranded nucleic acid molecules. Stringency of hybridization is typically expressed in conditions of ionic strength and temperature at which such hybrids are annealed and washed. Typically &#x201c;high&#x201d;, &#x201c;medium&#x201d; and &#x201c;low&#x201d; stringency encompass the following conditions or equivalent conditions thereto: high stringency: 0.1&#xd7;SSPE or SSC, 0.1% SDS, 65&#xb0; C.; medium stringency: 0.2&#xd7;SSPE or SSC, 0.1% SDS, 50&#xb0; C.; and low stringency: 1.0&#xd7;SSPE or SSC, 0.1% SDS, 50&#xb0; C. The deposits referred to herein will be maintained under the terms of the Budapest Treaty on the International Recognition of the Deposit of Micro-organisms for purposes of Patent Procedure. These deposits are provided merely as convenience to those of skill in the art and are not an admission that a deposit is required under 35 U.S.C. &#xa7; 112. The sequences of the nucleic acids contained in the deposited materials, as well as the amino acid sequences of the polypeptides encoded thereby, are incorporated herein by reference and are controlling in the event of any conflict with any description of sequences herein. A licensee may be required to make, use or sell the deposited materials, and no such license is hereby granted. Nucleic Acids The nucleic acids of the present invention may be in the form of RNA or in the form of DNA, which DNA includes cDNA, genomic DNA, and synthetic DNA. The DNA may be double-stranded or single-stranded, and if single stranded may be the coding strand or non-coding (anti-sense) strand. A coding sequence which encodes an ANT polypeptide for use according to the invention may be identical to the coding sequence known in the art for any given ANT, as described above and, for example, as shown for human ANT1 [SEQ ID NO:1], human ANT2 [SEQ ID NO:2] and human ANT3 [SEQ ID NO:3] in FIG. 1, or to that of any deposited clone, or may be a different coding sequence, which, as a result of the redundancy or degeneracy of the genetic code, encodes the same ANT polypeptide as, for example, the cDNAs of FIG. 1 or the deposited expression constructs. The nucleic acids which encode ANT polypeptides, for example the human ANT polypeptides having the amino acid sequences of FIG. 2 [SEQ ID NOS:31-33] or any other ANT polypeptides for use according to the invention, or for the ANT polypeptides encoded by the deposited constructs may include, but are not limited to: only the coding sequence for the ANT polypeptide; the coding sequence for the ANT polypeptide and additional coding sequence; the coding sequence for the ANT polypeptide (and optionally additional coding sequence) and non-coding sequence, such as introns or non-coding sequences 5&#x2032; and/or 3&#x2032; of the coding sequence for the ANT polypeptide, which for example may further include but need not be limited to one or more regulatory nucleic acid sequences that may be a regulated or regulatable promoter, enhancer, other transcription regulatory sequence, repressor binding sequence, translation regulatory sequence or any other regulatory nucleic acid sequence. Thus, the term &#x201c;nucleic acid encoding an ANT polypeptide&#x201d; encompasses a nucleic acid which includes only coding sequence for the polypeptide as well as a nucleic acid which includes additional coding and/or non-coding sequence(s). The present invention further relates to variants of the herein described nucleic acids which encode for fragments, analogs and derivatives of an ANT polypeptide, for example the human ANT1, ANT2 and ANT3 polypeptides having the deduced amino acid sequences of FIG. 2 [SEQ ID NOS:31-33] or any ANT polypeptide, including ANT polypeptides encoded by the cDNAs of the deposited expression constructs. The variants of the nucleic acids encoding ANTs may be naturally occurring allelic variants of the nucleic acids or non-naturally occurring variants. As is known in the art, an allelic variant is an alternate form of a nucleic acid sequence which may have at least one of a substitution, a deletion or an addition of one or more nucleotides, any of which does not substantially alter the function of the encoded ANT polypeptide. Thus, for example, the present invention includes nucleic acids encoding the same ANT polypeptides as shown in FIG. 2 [SEQ ID NOS:31-33], or the same ANT polypeptides encoded by the cDNAs of the deposited expression constructs, as well as variants of such nucleic acids, which variants encode a fragment, derivative or analog of any of the polypeptides of FIG. 2 (SEQ ID NO:2) or the polypeptides encoded by the cDNAs of the deposited expression constructs. Variants and derivatives of ANT may be obtained by mutations of nucleotide sequences encoding ANT polypeptides. Alterations of the native amino acid sequence may be accomplished by any of a number of conventional methods. Mutations can be introduced at particular loci by synthesizing oligonucleotides containing a mutant sequence, flanked by restriction sites enabling ligation to fragments of the native sequence. Following ligation, the resulting reconstructed sequence encodes an analog having the desired amino acid insertion, substitution, or deletion. Alternatively, oligonucleotide-directed site-specific mutagenesis procedures can be employed to provide an altered gene wherein predetermined codons can be altered by substitution, deletion or insertion. Exemplary methods of making such alterations are disclosed by Walder et al. (Gene 42:133, 1986); Bauer et al. (Gene 37:73, 1985); Craik (BioTechniques, January 1985, 12-19); Smith et al. (Genetic Engineering: Principles and Methods, Plenum Press, 1981); Kunkel (Proc. Natl. Acad. Sci. USA 82:488, 1985); Kunkel et al. (Methods in Enzymol. 154:367, 1987); and U.S. Pat. Nos. 4,518,584 and 4,737,462. Equivalent DNA constructs that encode various additions or substitutions of amino acid residues or sequences, or deletions of terminal or internal residues or sequences not needed for biological activity are also encompassed by the invention. For example, sequences encoding Cys residues that are not essential for biological activity can be altered to cause the Cys residues to be deleted or replaced with other amino acids, preventing formation of incorrect intramolecular disulfide bridges upon renaturation. Other equivalents can be prepared by modification of adjacent dibasic amino acid residues to enhance expression in yeast systems in which KEX2 protease activity is present. EP 212,914 discloses the use of site-specific mutagenesis to inactivate KEX2 protease processing sites in a protein. KEX2 protease processing sites are inactivated by deleting, adding or substituting residues to alter Arg-Arg, Arg-Lys, and Lys-Arg pairs to eliminate the occurrence of these adjacent basic residues. Lys-Lys pairings are considerably less susceptible to K-EX2 cleavage, and conversion of Arg-Lys or Lys-Arg to Lys-Lys represents a conservative and preferred approach to inactivating KEX2 sites. Polypeptides and Fusion Proteins The present invention further relates to ANT polypeptides, and in particular to methods for producing recombinant ANT polypeptides by culturing host cells containing ANT expression constructs, and to isolated recombinant human ANT polypeptides, including, for example, the human ANT1, ANT2 and ANT3 polypeptides which have the deduced amino acid sequence of FIG. 2 [SEQ ID NOS:31-33] or which have the amino acid sequence encoded by the deposited recombinant expression constructs, as well as fragments, analogs and derivatives of such polypeptides. The polypeptides and nucleic acids of the present invention are preferably provided in an isolated form, and in certain preferred embodiments are purified to homogeneity. The terms &#x201c;fragment,&#x201d; &#x201c;derivative&#x201d; and &#x201c;analog&#x201d; when referring to ANT polypeptides or fusion proteins, or to ANT polypeptides or fusion proteins encoded by the deposited recombinant expression constructs, refers to any ANT polypeptide or fusion protein that retains essentially the same biological function or activity as such polypeptide. Thus, an analog includes a proprotein which can be activated by cleavage of the proprotein portion to produce an active ANT polypeptide. The polypeptide of the present invention may be a recombinant polypeptide or a synthetic polypeptide, and is preferably a recombinant polypeptide. A fragment, derivative or analog of an ANT polypeptide or fusion protein, including ANT polypeptides or fusion proteins encoded by the cDNAs of the deposited constructs, may be (i) one in which one or more of the amino acid residues are substituted with a conserved or non-conserved amino acid residue (preferably a conserved amino acid residue) and such substituted amino acid residue may or may not be one encoded by the genetic code, or (ii) one in which one or more of the amino acid residues includes a substituent group, or (iii) one in which the ANT polypeptide is fused with another compound, such as a compound to increase the half-life of the polypeptide (for example, polyethylene glycol), or (iv) one in which additional amino acids are fused to the ANT polypeptide, including amino acids that are employed for purification of the ANT polypeptide or a proprotein sequence. Such fragments, derivatives and analogs are deemed to be within the scope of those skilled in the art from the teachings herein. The polypeptides of the present invention include ANT polypeptides and fusion proteins having amino acid sequences that are identical or similar to sequences known in the art. For example by way of illustration and not limitation, the human ANT (&#x201c;huANT&#x201d;) polypeptides of FIG. 2 [SEQ ID NOS:31-33] are contemplated for use according to the instant invention, as are polypeptides having at least 70% similarity (preferably a 70% identity) to the polypeptides of FIG. 2 [SEQ ID NOS:31-33] and more preferably 90% similarity (more preferably a 90% identity) to the polypeptides of FIG. 2 [SEQ ID NOS: 31-33] and still more preferably a 95% similarity (still more preferably a 95% identity) to the polypeptides of FIG. 2 [SEQ ID NOS:31-33] and to portions of such polypeptides, wherein such portions of an ANT polypeptide generally contain at least 30 amino acids and more preferably at least 50 amino acids. As known in the art &#x201c;similarity&#x201d; between two polypeptides is determined by comparing the amino acid sequence and conserved amino acid substitutes thereto of the polypeptide to the sequence of a second polypeptide. Fragments or portions of the polypeptides of the present invention may be employed for producing the corresponding full-length polypeptide by peptide synthesis; therefore, the fragments may be employed as intermediates for producing the full-length polypeptides. Fragments or portions of the nucleic acids of the present invention may be used to synthesize full-length nucleic acids of the present invention. The term &#x201c;isolated&#x201d; means that the material is removed from its original environment (e.g., the natural environment if it is naturally occurring). For example, a naturally occurring nucleic acid or polypeptide present in a living animal is not isolated, but the same nucleic acid or polypeptide, separated from some or all of the co-existing materials in the natural system, is isolated. Such nucleic acids could be part of a vector and/or such nucleic acids or polypeptides could be part of a composition, and still be isolated in that such vector or composition is not part of its natural environment. The term &#x201c;gene&#x201d; means the segment of DNA involved in producing a polypeptide chain; it includes regions preceding and following the coding region &#x201c;leader and trailer&#x201d; as well as intervening sequences (introns) between individual coding segments (exons). As described herein, the invention provides ANT fusion proteins encoded by nucleic acids that have the ANT coding sequence fused in frame to an additional coding sequence to provide for expression of an ANT polypeptide sequence fused to an additional functional or non-functional polypeptide sequence that permits, for example by way of illustration and not limitation, detection, isolation and/or purification of the ANT fusion protein. Such ANT fusion proteins may permit detection, isolation and/or purification of the ANT fusion protein by protein-protein affinity, metal affinity or charge affinity-based polypeptide purification, or by specific protease cleavage of a fusion protein containing a fusion sequence that is cleavable by a protease such that the ANT polypeptide is separable from the fusion protein. Thus ANT fusion proteins may comprise polypeptide sequences added to ANT to facilitate detection and isolation of ANT. Such peptides include, for example, poly-His or the antigenic identification peptides described in U.S. Pat. No. 5,011,912 and in Hopp et al., (1988 Bio/Technology 6:1204), or the XPRESS&#x2122; epitope tag (Invitrogen, Carlsbad, Calif.). The affinity sequence may be a hexa-histidine tag as supplied, for example, by a pBAD/His (Invitrogen) or a pQE-9 vector to provide for purification of the mature polypeptide fused to the marker in the case of a bacterial host, or, for example, the affinity sequence may be a hemagglutinin (HA) tag when a mammalian host, e.g., COS-7 cells, is used. The HA tag corresponds to an antibody defined epitope derived from the influenza hemagglutinin protein (Wilson et al., Cell 37:767 (1984)). ANT fusion proteins may further comprise immunoglobulin constant region polypeptides added to ANT to facilitate detection, isolation and/or localization of ANT. The immunoglobulin constant region polypeptide preferably is fused to the C-terminus of an ANT polypeptide. General preparation of fusion proteins comprising heterologous polypeptides fused to various portions of antibody-derived polypeptides (including the Fc domain) has been described, e.g., by Ashkenazi et al. (PNAS USA 88:10535, 1991) and Byrn et al. (Nature 344:677, 1990). A gene fusion encoding the ANT:Fc fusion protein is inserted into an appropriate expression vector. In certain embodiments of the invention, ANT:Fc fusion proteins may be allowed to assemble much like antibody molecules, whereupon interchain disulfide bonds form between Fc polypeptides, yielding dimeric ANT fusion proteins. ANT fusion proteins having specific binding affinities for pre-selected antigens by virtue of fusion polypeptides comprising immunoglobulin V-region domains encoded by DNA sequences linked in-frame to sequences encoding ANT are also within the scope of the invention, including variants and fragments thereof as provided herein. General strategies for the construction of fusion proteins having immunoglobulin V-region fusion polypeptides are disclosed, for example, in EP 0318554; U.S. Pat. No. 5,132,405; U.S. Pat. No. 5,091,513; and U.S. Pat. No. 5,476,786. The nucleic acid of the present invention may also encode a fusion protein comprising an ANT polypeptide fused to other polypeptides having desirable affinity properties, for example an enzyme such as glutathione-S-transferase. As another example, ANT fusion proteins may also comprise an ANT polypeptide fused to a Staphylococcus aureus protein A polypeptide; protein A encoding nucleic acids and their use in constructing fusion proteins having affinity for immunoglobulin constant regions are disclosed generally, for example, in U.S. Pat. No. 5,100,788. Other useful affinity polypetides for construction of ANT fusion proteins may include streptavidin fusion proteins, as disclosed, for example, in WO 89/03422; U.S. Pat. No. 5,489,528; U.S. Pat. No. 5,672,691; WO 93/24631; U.S. Pat. No. 5,168,049; U.S. Pat. No. 5,272,254 and elsewhere, and avidin fusion proteins (see, e.g., EP 511,747). As provided herein and in the cited references, ANT polypeptide seqences may be fused to fusion polypeptide sequences that may be full length fusion polypeptides and that may alternatively be variants or fragments thereof. The present invention also provides a method of targeting a polypeptide of interest to a membrane, and in particular embodiments to a cellular membrane, and in further embodiments to a mitochondrial membrane. This aspect of the invention is based on the unexpected observation that certain recombinant expression constructs as provided herein, which constructs include a nucleic acid encoding a first polypeptide that is an ANT polypeptide, and that is expressed as a fusion protein with a second polypeptide sequence, provide recombinant ANT fusion proteins capable of preferentially localizing to cell membranes. In certain embodiments the cell membrane is a prokaryotic cell membrane such as a bacterial cell membrane, for example an E. Coli membrane. In other embodiments the cell membrane is a eukaryotic cell membrane such as a yeast or a mammalian cell membrane, for example a membrane of any eukaryotic cell contemplated herein. A cell membrane as used herein may be any cellular membrane, and typically refers to membranes that are in contact with cytosolic components, including intracellular membrane bounded compartments such as mitochondrial inner and outer membranes as described above, and also intracellular vesicles, ER-Golgi constituents, other organelles and the like, as well as the plasma membrane. In preferred embodiments, an ANT fusion protein may be targeted to a mitochondrial membrane. In other preferred embodiments, for example, recombinant expression constructs according to the invention may encode ANT fusion proteins that contain polypeptide sequences that direct the fusion protein to be retained in the cytosol, to reside in the lumen of the endoplasmic reticulum (ER), to be secreted from a cell via the classical ER-Golgi secretory pathway, to be incorporated into the plasma membrane, to associate with a specific cytoplasmic component including the cytoplasmic domain of a transmembrane cell surface receptor or to be directed to a particular subcellular location by any of a variety of known intracellular protein sorting mechanisms with which those skilled in the art will be familiar. Accordingly, these and related embodiments are encompassed by the instant compositions and methods directed to targeting a polypeptide of interest to a predefined intracellular, membrane or extracellular localization. Vectors The present invention also relates to vectors and to constructs that include nucleic acids of the present invention, and in particular to &#x201c;recombinant expression constructs&#x201d; that include any nucleic acids encoding ANT polypeptides according to the invention as provided above; to host cells which are genetically engineered with vectors and/or constructs of the invention and to the production of ANT polypeptides and fusion proteins of the invention, or fragments or variants thereof, by recombinant techniques. ANT proteins can be expressed in mammalian cells, yeast, bacteria, or other cells under the control of appropriate promoters. Cell-free translation systems can also be employed to produce such proteins using RNAs derived from the DNA constructs of the present invention. Appropriate cloning and expression vectors for use with prokaryotic and eukaryotic hosts are described by Sambrook, et al., Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbor, N.Y., (1989). Generally, recombinant expression vectors will include origins of replication and selectable markers permitting transformation of the host cell, e.g., the ampicillin resistance gene of E. coli and S. cerevisiae TRP1 gene, and a promoter derived from a highly-expressed gene to direct transcription of a downstream structural sequence. Such promoters can be derived from operons encoding glycolytic enzymes such as 3-phosphoglycerate kinase (PGK), &#x3b1;-factor, acid phosphatase, or heat shock proteins, among others. The heterologous structural sequence is assembled in appropriate phase with translation initiation and termination sequences. Optionally, the heterologous sequence can encode a fusion protein including an N-terminal identification peptide imparting desired characteristics, e.g., stabilization or simplified purification of expressed recombinant product. Useful expression constructs for bacterial use are constructed by inserting into an expression vector a structural DNA sequence encoding a desired protein together with suitable translation initiation and termination signals in operable reading phase with a functional promoter. The construct may comprise one or more phenotypic selectable marker and an origin of replication to ensure maintenance of the vector construct and, if desirable, to provide amplification within the host. Suitable prokaryotic hosts for transformation include E. coli, Bacillus subtilis, Salmonella typhimurium and various species within the genera Pseudomonas, Streptomyces, and Staphylococcus, although others may also be employed as a matter of choice. Any other plasmid or vector may be used as long as they are replicable and viable in the host. As a representative but nonlimiting example, useful expression vectors for bacterial use can comprise a selectable marker and bacterial origin of replication derived from commercially available plasmids comprising genetic elements of the well known cloning vector pBR322 (ATCC 37017). Such commercial vectors include, for example, pKK223-3 (Pharmacia Fine Chemicals, Uppsala, Sweden) and GEM1 (Promega Biotec, Madison, Wis., USA). These pBR322 &#x201c;backbone&#x201d; sections are combined with an appropriate promoter and the structural sequence to be expressed. Following transformation of a suitable host strain and growth of the host strain to an appropriate cell density, the selected promoter, if it is a regulated promoter as provided herein, is induced by appropriate means (e.g., temperature shift or chemical induction) and cells are cultured for an additional period. Cells are typically harvested by centrifugation, disrupted by physical or chemical means, and the resulting crude extract retained for further purification. Microbial cells employed in expression of proteins can be disrupted by any convenient method, including freeze-thaw cycling, sonication, mechanical disruption, or use of cell lysing agents; such methods are well know to those skilled in the art. Thus, for example, the nucleic acids of the invention as provided herein may be included in any one of a variety of expression vector constructs as a recombinant expression construct for expressing an ANT polypeptide. Such vectors and constructs include chromosomal, nonchromosomal and synthetic DNA sequences, e.g., derivatives of SV40; bacterial plasmids; phage DNA; baculovirus; yeast plasmids; vectors derived from combinations of plasmids and phage DNA, viral DNA, such as vaccinia, adenovirus, fowl pox virus, and pseudorabies. However, any other vector may be used for preparation of a recombinant expression construct as long as it is replicable and viable in the host. The appropriate DNA sequence(s) may be inserted into the vector by a variety of procedures. In general, the DNA sequence is inserted into an appropriate restriction endonuclease site(s) by procedures known in the art. Standard techniques for cloning, DNA isolation, amplification and purification, for enzymatic reactions involving DNA ligase, DNA polymerase, restriction endonucleases and the like, and various separation techniques are those known and commonly employed by those skilled in the art. A number of standard techniques are described, for example, in Ausubel et al. (1993 Current Protocols in Molecular Biology, Greene Publ. Assoc. Inc. &#x26; John Wiley &#x26; Sons, Inc., Boston, Mass.); Sambrook et al. (1989 Molecular Cloning, Second Ed., Cold Spring Harbor Laboratory, Plainview, N.Y.); Maniatis et al. (1982 Molecular Cloning, Cold Spring Harbor Laboratory, Plainview, N.Y.); and elsewhere. The DNA sequence in the expression vector is operatively linked to at least one appropriate expression control sequences (e.g., a promoter or a regulated promoter) to direct mRNA synthesis. Representative examples of such expression control sequences include LTR or SV40 promoter, the E. coli lac or trp, the phage lambda PL promoter and other promoters known to control expression of genes in prokaryotic or eukaryotic cells or their viruses. Promoter regions can be selected from any desired gene using CAT (chloramphenicol transferase) vectors or other vectors with selectable markers. Two appropriate vectors are pKK232-8 and pCM7. Particular named bacterial promoters include lac, lacZ, T3, T7, gpt, lambda PR, PL and trp. Eukaryotic promoters include CMV immediate early, HSV thymidine kinase, early and late SV40, LTRs from retrovirus, and mouse metallothionein-I. Selection of the appropriate vector and promoter is well within the level of ordinary skill in the art, and preparation of certain particularly preferred recombinant expression constructs comprising at least one promoter or regulated promoter operably linked to a nucleic acid encoding an ANT polypeptide is described herein. In certain preferred embodiments the expression control sequence is a &#x201c;regulated promoter&#x201d;, which may be a promoter as provided herein and may also be a repressor binding site, an activator binding site or any other regulatory sequence that controls expression of a nucleic acid sequence as provided herein. In certain particularly preferred embodiments the regulated promoter is a tightly regulated promoter that is specifically inducible and that permits little or no transcription of nucleic acid sequences under its control in the absence of an induction signal, as is known to those familiar with the art and described, for example, in Guzman et al. (1995 J. Bacteriol. 177:4121), Carra et al. (1993 EMBO J. 12:35), Mayer (1995 Gene 163:41), Haldimann et al. (1998 J. Bacteriol. 180:1277), Lutz et al. (1997 Nuc. Ac. Res. 25:1203), Allgood et al. (1997 Curr. Opin. Biotechnol. 8:474) and Makrides (1996 Microbiol. Rev. 60:512), all of which are hereby incorporated by reference. In other preferred embodiments of the invention a regulated promoter is present that is inducible but that may not be tightly regulated. In certain other preferred embodiments a promoter is present in the recombinant expression construct of the invention that is not a regulated promoter; such a promoter may include, for example, a constitutive promoter such as an insect polyhedrin promoter as described in the Examples or a yeast phosphoglycerate kinase promoter (see, e.g., Giraud et al., 1998 J. Mol. Biol. 281:409). The expression construct also contains a ribosome binding site for translation initiation and a transcription terminator. The vector may also include appropriate sequences for amplifying expression. Transcription of the DNA encoding the polypeptides of the present invention by higher eukaryotes may be increased by inserting an enhancer sequence into the vector. Enhancers are cis-acting elements of DNA, usually about from 10 to 300 bp that act on a promoter to increase its transcription. Examples including the SV40 enhancer on the late side of the replication origin bp 100 to 270, a cytomegalovirus early promoter enhancer, the polyoma enhancer on the late side of the replication origin, and adenovirus enhancers. As noted above, in certain embodiments the vector may be a viral vector such as a retroviral vector. For example, retroviruses from which the retroviral plasmid vectors may be derived include, but are not limited to, Moloney Murine Leukemia Virus, spleen necrosis virus, retroviruses such as Rous Sarcoma Virus, Harvey Sarcoma virus, avian leukosis virus, gibbon ape leukemia virus, human immunodeficiency virus, adenovirus, Myeloproliferative Sarcoma Virus, and mammary tumor virus. The viral vector includes one or more promoters. Suitable promoters which may be employed include, but are not limited to, the retroviral LTR; the SV40 promoter; and the human cytomegalovirus (CMV) promoter described in Miller, et al., Biotechniques 7:980-990 (1989), or any other promoter (e.g., cellular promoters such as eukaryotic cellular promoters including, but not limited to, the histone, pol III, and &#x3b2;-actin promoters). Other viral promoters which may be employed include, but are not limited to, adenovirus promoters, thymidine kinase (TK) promoters, and B19 parvovirus promoters. The selection of a suitable promoter will be apparent to those skilled in the art from the teachings contained herein, and may be from among either regulated promoters or promoters as described above. The retroviral plasmid vector is employed to transduce packaging cell lines to form producer cell lines. Examples of packaging cells which may be transfected include, but are not limited to, the PE501, PA317, &#x3c8;-2, &#x3c8;-AM, PA12, T19-14X, VT-19-17-H2, &#x3c8;CRE, &#x3c8;CRIP, GP+E-86, GP+envAm12, and DAN cell lines as described in Miller, Human Gene Therapy, 1:5-14 (1990), which is incorporated herein by reference in its entirety. The vector may transduce the packaging cells through any means known in the art. Such means include, but are not limited to, electroporation, the use of liposomes, and CaPO4 precipitation. In one alternative, the retroviral plasmid vector may be encapsulated into a liposome, or coupled to a lipid, and then administered to a host. The producer cell line generates infectious retroviral vector particles which include the nucleic acid sequence(s) encoding the ANT polypeptides or fusion proteins. Such retroviral vector particles then may be employed, to transduce eukaryotic cells, either in vitro or in vivo. The transduced eukaryotic cells will express the nucleic acid sequence(s) encoding the ANT polypeptide or fusion protein. Eukaryotic cells which may be transduced include, but are not limited to, embryonic stem cells, embryonic carcinoma cells, as well as hematopoietic stem cells, hepatocytes, fibroblasts, myoblasts, keratinocytes, endothelial cells, and bronchial epithelial cells. As another example of an embodiment of the invention in which a viral vector is used to prepare the recombinant ANT expression construct, in one preferred embodiment, host cells transduced by a recombinant viral construct directing the expression of ANT polypeptides or fusion proteins may produce viral particles containing expressed ANT polypeptides or fusion proteins that are derived from portions of a host cell membrane incorporated by the viral particles during viral budding. In another preferred embodiment, ANT encoding nucleic acid sequences are cloned into a baculovirus shuttle vector, which is then recombined with a baculovirus to generate a recombinant baculovirus expression construct that is used to infect, for example, Sf9 or Trichoplusia ni (PharMingen, Inc., San Diego, Calif.) host cells, as described in Baculovirus Expression Protocols, Methods in Molecular Biology Vol. 39, Christopher D. Richardson, Editor, Human Press, Totowa, N.J., 1995; Piwnica-Worms, &#x201c;Expression of Proteins in Insect Cells Using Baculoviral Vectors,&#x201d; Section II in Chapter 16 in: Short Protocols in Molecular Biology, 2nd Ed., Ausubel et al., eds., John Wiley &#x26; Sons, New York, N.Y., 1992, pages 16-32 to 16-48. Host Cells In another aspect, the present invention relates to host cells containing the above described recombinant ANT expression constructs. Host cells are genetically engineered (transduced, transformed or transfected) with the vectors and/or expression constructs of this invention which may be, for example, a cloning vector, a shuttle vector or an expression construct. The vector or construct may be, for example, in the form of a plasmid, a viral particle, a phage, etc. The engineered host cells can be cultured in conventional nutrient media modified as appropriate for activating promoters, selecting transformants or amplifying particular genes such as genes encoding ANT polypeptides or ANT fusion proteins. The culture conditions for particular host cells selected for expression, such as temperature, pH and the like, will be readily apparent to the ordinarily skilled artisan. The host cell can be a higher eukaryotic cell, such as a mammalian cell, or a lower eukaryotic cell, such as a yeast cell, or the host cell can be a prokaryotic cell, such as a bacterial cell. Representative examples of appropriate host cells according to the present invention include, but need not be limited to, bacterial cells, such as E. coli, Streptomyces, Salmonella tvphimurium; fungal cells, such as yeast; insect cells, such as Drosophila S2, Trichoplusia ni (PharMingen, San Diego, Calif.) and Spodoptera SJ9; animal cells, such as CHO, COS or 293 cells; adenoviruses; plant cells, or any suitable cell already adapted to in vitro propagation or so established de novo. The selection of an appropriate host is deemed to be within the scope of those skilled in the art from the teachings herein. Various mammalian cell culture systems can also be employed to express recombinant protein. Examples of mammalian expression systems include the COS-7 lines of monkey kidney fibroblasts, described by Gluzman, Cell 23:175 (1981), and other cell lines capable of expressing a compatible vector, for example, the C127, 3T3, CHO, HeLa and BHK cell lines Mammalian expression vectors will comprise an origin of replication, a suitable promoter and enhancer, and also any necessary ribosome binding sites, polyadenylation site, splice donor and acceptor sites, transcriptional termination sequences, and 5&#x2032; flanking nontranscribed sequences, for example as described herein regarding the preparation of ANT expression constructs. DNA sequences derived from the SV40 splice, and polyadenylation sites may be used to provide the required nontranscribed genetic elements. Introduction of the construct into the host cell can be effected by a variety of methods with which those skilled in the art will be familiar, including but not limited to, for example, calcium phosphate transfection, DEAE-Dextran mediated transfection, or electroporation (Davis et al., 1986 Basic Methods in Molecular Biology). As will be aprreciated by those of ordinary skill in the art, in certain situations it may be desirable to prepare the compositions of the invention and to practice the methods of the invention under conditions where endogenous ANT expression by a host cell is compromised, in order to provide advantages associated with the expression of a desired ANT encoding construct. For example, detection of particular ANT encoding nucleic acid sequences or ANT polypeptides that are highly similar to those encoded by the host cell genome may be facilitated by inhibiting host cell ANT gene expression. As another example, where functional activity of an exogenously introduced recombinant ANT polypeptide is to be determined in a host cell or in a biological sample derived therefrom, it may also be advantageous to inhibit endogenous host cell ANT gene expression. Thus, in certain preferred embodiments of the invention, host cells may lack at least one isoform of an endogenous ANT, and in certain preferred embodiments the host cells may lack all endogenous ANT isoforms. For example, in the yeast system described by Giraud et al. (1998 J. Mol. Biol. 281:409) a S. cerevisiae triple null mutant is described that lacks all three yeast ANT isoforms and is unable to grow under anaerobic conditions. In other preferred embodiments, expression in host cells of at least one gene encoding an endogenous ANT isoform is substantially impaired. Substantial impairment of endogenous ANT isoform expression may be achieved by any of a variety of methods that are well known in the art for blocking specific gene expression, including site-specific or site-directed mutagenesis as described above, antisense inhibition of gene expression, ribozyme mediated inhibition of gene expression and generation of mitochondrial DNA depleted (&#x3c1;0) cells. Identification of oligonucleotides and ribozymes for use as antisense agents and DNA encoding genes for targeted delivery for genetic therapy involve methods well known in the art. For example, the desirable properties, lengths and other characteristics of such oligonucleotides are well known. Antisense oligonucleotides are typically designed to resist degradation by endogenous nucleolytic enzymes by using such linkages as: phosphorothioate, methylphosphonate, sulfone, sulfate, ketyl, phosphorodithioate, phosphoramidate, phosphate esters, and other such linkages (see, e.g., Agrwal et al., Tetrehedron Lett. 28:3539-3542 (1987); Miller et al., J. Am. Chem. Soc. 93:6657-6665 (1971); Stec et al., Tetrehedron Left. 26:2191-2194 (1985); Moody et al., Nucl. Acids Res. 12:4769-4782 (1989); Uznanski et al., Nucl. Acids Res. (1989); Letsinger et al., Tetrahedron 40:137-143 (1984); Eckstein, Annu. Rev. Biochem. 54:367-402 (1985); Eckstein, Trends Biol. Sci. 14:97-100 (1989); Stein In: Oligodeoxynucleotides. Antisense Inhibitors of Gene Expression, Cohen, Ed, Macmillan Press, London, pp. 97-117 (1989); Jager et al., Biochemistry 27:7237-7246 (1988)). Antisense nucleotides are oligonucleotides that bind in a sequence-specific manner to nucleic acids, such as mRNA or DNA. When bound to mRNA that has complementary sequences, antisense prevents translation of the mRNA (see, e.g., U.S. Pat. No. 5,168,053 to Altman et al.; U.S. Pat. No. 5,190,931 to Inouye, U.S. Pat. No. 5,135,917 to Burch; U.S. Pat. No. 5,087,617 to Smith and Clusel et al. (1993) Nucl. Acids Res. 21:3405-3411, which describes dumbbell antisense oligonucleotides). Triplex molecules refer to single DNA strands that bind duplex DNA forming a colinear triplex molecule, thereby preventing transcription (see, e.g., U.S. Pat. No. 5,176,996 to Hogan et al., which describes methods for making synthetic oligonucleotides that bind to target sites on duplex DNA). According to this embodiment of the invention, particularly useful antisense nucleotides and triplex molecules are molecules that are complementary to or bind the sense strand of DNA or mRNA that encodes an ANT polypeptide or a protein mediating any other process related to expression of endogenous ANT genes, such that inhibition of translation of mRNA encoding the ANT polypeptide is effected. A ribozyme is an RNA molecule that specifically cleaves RNA substrates, such as mRNA, resulting in specific inhibition or interference with cellular gene expression. There are at least five known classes of ribozymes involved in the cleavage and/or ligation of RNA chains. Ribozymes can be targeted to any RNA transcript and can catalytically cleave such transcripts (see, e.g., U.S. Pat. No. 5,272,262; U.S. Pat. No. 5,144,019; and U.S. Pat. Nos. 5,168,053, 5,180,818, 5,116,742 and 5,093,246 to Cech et al.). According to certain embodiments of the invention, any such ANT mRNA-specific ribozyme, or a nucleic acid encoding such a ribozyme, may be delivered to a host cell to effect inhibition of ANT gene expression. Ribozymes, and the like may therefore be delivered to the host cells by DNA encoding the ribozyme linked to a eukaryotic promoter, such as a eukaryotic viral promoter, such that upon introduction into the nucleus, the ribozyme will be directly transcribed. As used herein, expression of a gene encoding an endogenous adenine nucleotide translocator isoform is substantially impaired by any of the above methods for inhibiting when cells are substantially but not necessarily completely depleted of functional DNA or functional mRNA encoding the endogenous ANT isoform, or of the relevant ANT polypeptide. ANT isoform expression is substantially impaired when cells are preferably at least 50% depleted of DNA or mRNA encoding the endogenous ANT (as measured using high stringency hybridization as described above) or depleted of ANT polypeptide (as measured by Western immunoblot as described herein, see also, e.g., Giraud et al. 1998 J. Mol Biol. 281:409); and more preferably at least 75% depleted of endogenous ANT DNA, mRNA or polypeptide. Most preferably, ANT isoform expression is substantially impaired when cells are depleted of &#x3e;90% of their endogenous ANT DNA, mRNA, or polypeptide. Alternatively, expression of a gene encoding an endogenous adenine nucleotide translocator isoform may be substantially impaired through the use of mitochondrial DNA depleted &#x3c1;0 cells, which are incapable of mitochondrial replication and so may not continue to express functional ANT polypeptides. Methods for producing &#x3c1;0 cells are known and can be found, for example in PCT/US95/04063, which is hereby incorporated by reference. Protein Production The expressed recombinant ANT polypeptides or fusion proteins may be useful in intact host cells; in intact organelles such as mitochondria, cell membranes, intracellular vesicles other cellular organelles; or in disrupted cell preparations including but not limited to cell homogenates or lysates, submitochondrial particles, uni- and multilamellar membrane vesicles or other preparations. Alternatively, expressed recombinant ANT polypeptides or fusion proteins can be recovered and purified from recombinant cell cultures by methods including ammonium sulfate or ethanol precipitation, acid extraction, anion or cation exchange chromatography, phosphocellulose chromatography, hydrophobic interaction chromatography, affinity chromatography, hydroxylapatite chromatography and lectin chromatography. Protein refolding steps can be used, as necessary, in completing configuration of the mature protein. Finally, high performance liquid chromatography (HPLC) can be employed for final purification steps. The polypeptides of the present invention may be a naturally purified product, or a product of chemical synthetic procedures, or produced by recombinant techniques from a prokaryotic or eukaryotic host (for example, by bacterial, yeast, higher plant, insect and mammalian cells in culture). Depending upon the host employed in a recombinant production procedure, the polypeptides of the present invention may be glycosylated or may be non-glycosylated. Polypeptides of the invention may also include an initial methionine amino acid residue. Samples A &#x201c;biological sample containing mitochondria&#x201d; may comprise any tissue or cell preparation in which intact mitochondria capable of maintaining a membrane potential when supplied with one or more oxidizable substrates such as glucose, malate or galactose are or are thought to be present. Mitochondrial membrane potential may be determined according to methods with which those skilled in the art will be readily familiar, including but not limited to detection and/or measurement of detectable compounds such as fluorescent indicators, optical probes and/or sensitive pH and ion-selective electrodes (See, e.g., Ernster et al., 1981 J. Cell Biol. 91:227s and references cited therein; see also Haugland, 1996 Handbook of Fluorescent Probes and Research Chemicals&#x2014;Sixth Ed., Molecular Probes, Eugene, Oreg., pp. 266-274 and 589-594.). By &#x201c;capable of maintaining a potential&#x201d; it is meant that such mitochondria have a membrane potential that is sufficient to permit the accumulation of a detectable compound (e.g., DASPMI [2-,4-dimethylaminostyryl-N-methylpyridinium], TMRM [tetramethylrhodamine methyl ester], etc.) used in the particular instance. A biological sample containing mitochondria may, for example, be derived from a normal (i.e., healthy) individual or from an individual having a disease associated with altered mitochondrial function. Biological samples containing mitochondria may be provided by obtaining a blood sample, biopsy specimen, tissue explant, organ culture or any other tissue or cell preparation from a subject or a biological source. The subject or biological source may be a human or non-human animal, a primary cell culture or culture adapted cell line including but not limited to genetically engineered cell lines that may contain chromosomally integrated or episomal recombinant nucleic acid sequences, immortalized or immortalizable cell lines, somatic cell hybrid or cytoplasmic hybrid &#x201c;cybrid&#x201d; cell lines, differentiated or differentiatable cell lines, transformed cell lines and the like. A &#x201c;biological sample&#x201d; may comprise any tissue or cell preparation as just described for a biological sample containing mitochondria, but does not require the presence of intact mitochondria. Thus a &#x201c;biological sample&#x201d; may comprise any tissue or cell preparation and a &#x201c;biological sample containing at least one recombinant ANT polypeptide&#x201d; comprises any tissue or cell preparation in which an expressed recombinant ANT polypeptide or fusion protein as provided herein is thought to be present. A biological sample may, for example, be derived from a recombinant cell line or from a transgenic animal. Biological samples containing recombinant ANT may be provided by obtaining a blood sample, biopsy specimen, tissue explant, organ culture or any other tissue or cell preparation from a subject or a biological source. The subject or biological source may be a human or non-human animal, a primary cell culture or culture adapted cell line including but not limited to genetically engineered cell lines that may contain chromosomally integrated or episomal recombinant nucleic acid sequences, immortalized or immortalizable cell lines, somatic cell hybrid or cytoplasmic hybrid &#x201c;cybrid&#x201d; cell lines, differentiated or differentiatable cell lines, transformed cell lines and the like. Proteins As described herein, isolation of a mitochondrial pore component or a mitochondrial molecular species with which an agent identified according to the methods of the invention interacts refers to physical separation of such a complex from its biological source, and may be accomplished by any of a number of well known techniques including but not limited to those described herein, and in the cited references. Without wishing to be bound by theory, a compound that &#x201c;binds a mitochondrial component&#x201d; can be any discrete molecule, agent compound, composition of matter or the like that may, but need not, directly bind to a mitochondrial molecular component, and may in the alternative bind indirectly to a mitochondrial molecular component by interacting with one or more additional components that bind to a mitochondrial molecular component. These or other mechanisms by which a compound may bind to and/or associate with a mitochondrial molecular component are within the scope of the claimed methods, so long as isolating a mitochondrial pore component also results in isolation of the mitochondrial molecular species that directly or indirectly binds to the identified agent. Thus, for example, as provided herein, any ANT polypeptide including recombinant ANT polypeptides and fusion proteins may be a mitochondrial molecular component and/or a mitochondrial pore component, and any ANT ligand or agent that binds to an ANT polypeptide may be a compound that binds a mitochondrial component and/or an agent that affects mitochondrial pore activity. As described herein, the mitochondrial permeability transition pore may not be a discrete assembly or multisubunit complex, and the term thus refers instead to any mitochondrial molecular component (including, e.g. a mitochondrial membrane per se) that regulates the inner membrane selective permeability where such regulated function is impaired during IPT. As used herein, mitochondria are comprised of &#x201c;mitochondrial molecular components&#x201d;, which may be any protein, polypeptide, peptide, amino acid, or derivative thereof; any lipid, fatty acid or the like, or derivative thereof; any carbohydrate, saccharide or the like or derivative thereof, any nucleic acid, nucleotide, nucleoside, purine, pyrimidine or related molecule, or derivative thereof, or the like; or any other biological molecule that is a constituent of a mitochondrion. &#x201c;Mitochondrial molecular components&#x201d; includes but is not limited to &#x201c;mitochondrial pore components&#x201d;. A &#x201c;mitochondrial pore component&#x201d; is any mitochondrial molecular component that regulates the selective permeability characteristic of mitochondrial membranes as described above, including those responsible for establishing &#x394;&#x3a8;m and those that are functionally altered during MPT. Isolation and, optionally, identification and/or characterization of the mitochondrial pore component or components with which an agent that affects mitochondrial pore activity interacts may also be desirable and are within the scope of the invention. Once an agent is shown to alter MPT according to the methods provided herein and in U.S. Ser. No. 09/161,172, those having ordinary skill in the art will be familiar with a variety of approaches that may be routinely employed to isolate the molecular species specifically recognized by such an agent and involved in regulation of MPT, where to &#x201c;isolate&#x201d; as used herein refers to separation of such molecular species from the natural biological environment. Thus, for example, once an ANT ligand is prepared according to the methods provided herein, such approaches may be routinely employed to isolate the ANT polypeptide. Techniques for isolating a mitochondrial pore component such as an ANT polypeptide or fusion protein may include any biological and/or biochemical methods useful for separating the complex from its biological source, and subsequent characterization may be performed according to standard biochemical and molecular biology procedures. Those familiar with the art will be able to select an appropriate method depending on the biological starting material and other factors. Such methods may include, but need not be limited to, radiolabeling or otherwise detectably labeling cellular and mitochondrial components in a biological sample, cell fractionation, density sedimentation, differential extraction, salt precipitation, ultrafiltration, gel filtration, ion-exchange chromatography, partition chromatography, hydrophobic chromatography, electrophoresis, affinity techniques or any other suitable separation method that can be adapted for use with the agent with which the mitochondrial pore component interacts. Antibodies to partially purified components may be developed according to methods known in the art and may be used to detect and/or to isolate such components. Affinity techniques may be particularly useful in the context of the present invention, and may include any method that exploits a specific binding interaction between a mitochondrial pore component and an agent identified according to the invention that interacts with the pore component. For example, because ANT ligands as provided herein and other agents that influence MPT can be immobilized on solid phase matrices, an affinity binding technique for isolation of the pore component may be particularly useful. Alternatively, affinity labeling methods for biological molecules, in which a known MPT-active agent or a novel ANT ligand as provided herein may be modified with a reactive moiety, are well known and can be readily adapted to the interaction between the agent and a pore component, for purposes of introducing into the pore component a detectable and/or recoverable labeling moiety. (See, e.g., Pierce Catalog and Handbook, 1994 Pierce Chemical Company, Rockford, Ill.; Scopes, R. K., Protein Purification: Principles and Practice, 1987, Springer-Verlag, New York; and Hermanson, G. T. et al., Immobilized Affinity Ligand Techniques, 1992, Academic Press, Inc., California; for details regarding techniques for isolating and characterizing biological molecules, including affinity techniques. Characterization of mitochondrial pore component molecular species, isolated by MPT-active agent affinity techniques described above or by other biochemical methods, may be accomplished using physicochemical properties of the pore component such as spectrometric absorbance, molecular size and/or charge, solubility, peptide mapping, sequence analysis and the like. (See, e.g., Scopes, supra.) Additional separation steps for biomolecules may be optionally employed to further separate and identify molecular species that co-purify with mitochondrial pore components. These are well known in the art and may include any separation methodology for the isolation of proteins, lipids, nucleic acids or carbohydrates, typically based on physicochemical properties of the newly identified components of the complex. Examples of such methods include RP-HPLC, ion exchange chromatography, hydrophobic interaction chromatography, hydroxyapatite chromatography, native and/or denaturing one- and two-dimensional electrophoresis, ultrafiltration, capillary electrophoresis, substrate affinity chromatography, immunoaffinity chromatography, partition chromatography or any other useful separation method. Preferably extracts of cultured cells, and in particularly preferred embodiments extracts of biological tissues or organs may be sources of mitochondrial molecular components, including ANT polypeptides. Preferred sources may include blood, brain, fibroblasts, myoblasts, liver cells or other cell types. ANT Ligands As noted above, the binding of the adenine nucleotide translocator (ANT) is responsible for mediating transport of ADP and ATP across the mitochondrial inner membrane. ANT has also been implicated as the critical component of the mitochondrial permeability transition pore, a Ca2+ regulated inner membrane channel that plays an important modulating role in apoptotic processes. Additionally, ANT activity appears to be related to changes in ANT polypeptide conformation within the mitochondrial membrane, as evidenced by studies using agents that are capable of binding to ANT. (Block et al., 1986 Meths. Enzymol. 125:658) Accordingly, it is another aspect of the present invention to provide compositions and methods for producing and identifying agents that bind to ANT, which agents are also referred to herein as ANT ligands. Binding interactions between ANT and a variety of small molecules are known to those familiar with the art. For example, these interactions include binding to ANT by atractyloside, carboxyatractyloside, palmitoyl-CoA, bongkrekic acid, thyroxin, eosin Y and erythrosin B. (See, e.g., Stubbs, 1979 Pharm. Ther. 7:329; Klingenberg et al., 1978 Biochim. Biophys. Acta 503:193; Sterling, 1986 Endocrinol. 119:292; Majima et al., 1998 Biochem. 37:424; Block et al. 1986 Meths. Enzymol. 125:658; for erythrosin B and additional ANT inhibitors, see Beavis et al. 1993 J. Biol. Chem. 268:997; Powers et al. 1994 J. Biol. Chem. 269:10614.) The ANT ligands of the present invention represent novel atractyloside derivatives. Atractyloside (ATR) and its known derivatives, including carboxyatractyloside (CATR), naphthoyl-ATP, MANT-ATR and other ATR derivatives (see, e.g., Boulay et al., Analytical Biochemistry 128:323-330,1983; Roux et al., Analytical Biochemistry 234:31-37,1996; Lauquin et al., FEBS Letters 67:306-311,1976; and Gottikh et al., Tetrahedron 26:4419-4433, 1970; for other known ATR derivatives see, e.g. Block et al., 1986 Meths. Enzymol. 125:658) have proven invaluable in the elucidation of the structure and the mechanism of action of the adenine nucleotide translocator. According to the ANY ligands of the invention, the binding mode of ATR to ANT allows for modifications of the ATR 6&#x2032;-hydroxyl functionality without significantly altering ATR binding affinity for ANT. Thus, ANT ligands as provided herein may be ATR derivatives modified by chemical substitution at the 6&#x2032; hydroxyl position. In particular, the novel ANT ligands as provided herein further include long linker moieties at the 6&#x2032; position, which linkers may include a 6&#x2032;-amine linker, thereby permitting additional chemical modification to the ANT ligand as will be appreciated by those skilled in the art and as illustrated in the non-limiting Examples. Also, as shown in Examples 6-11, such linkers as provided herein may have carbon chain backbones of 2-20 carbon atoms, and in preferred embodiments 2-6 carbon atoms. The invention therefore provides ANT ligands that may be intermediates for conjugation to a variety of additional chemical moieties to yield further ATR derivatives that are ANT ligands within the scope of the invention. These include ANT ligands to which 125I may be covalently attached under mild reaction conditions; the invention also includes ANT ligands to which reactive amine groups may be covalently linked. ANT ligands which are such amine-containing ATR derivatives may then be reacted with a variety of fluorophores and haptens bearing, for example, reactive isothiocyanate, N-hydroxysuccinimide ester, anhydride and other useful functionalities to yield stable ATR derivatives including, for example, derivatives that have thiourea, amide or other linkages. Thus, ANT ligands as provided herein also include ATR derivatives that are detectable by virtue of substituents introduced at the 6&#x2032; position. Accordingly, detectable ATR derivatives as herein provided include ATR derivatives having a 6&#x2032; hydroxyl substitution that includes a radiolabeled substituent, for example 125I, 131I, 3H, 14C or 35S. Other ANT ligands that are detectable ATR derivatives may comprise fluorescent substituents, including those appropriately tagged with reporter molecules such as fluorophores and haptens having utility in high throughput screening assays for identifying agents that bind to ANT. More specifically, in preferred embodiments, an ANT ligand according to the present invention that includes fluorescent substituents has an extinction coefficient&#x2267;10,000 M&#x2212;1 (see Table 1); further, this property provides an advantage for using such ANT ligands according to the methods provided herein, and in particular for use in high throughput screening assays. Additionally, the ANT ligands of the invention exhibit high affinities for ANT, and in preferred embodiments have binding constants in the nanomolar range. In certain embodiments of the invention, ANT ligands may be ATR derivatives such as ATR-lanthanide chelating agents, which have utility in time-resolved fluorescence detection, for example detection of these compounds complexed to a lanthanide ion such as Eu3+, Tb3+, Sm3+ and Dy3+. In addition, ANT ligands may comprise ATR conjugated to readily detectable substituents such as highly fluorescent moieties, for example by way of illustration and not limitation, cyanine and coumarin derivatives. These and other highly fluorescent substituents permit the synthesis, according to the methods of the invention, of ANT ligands that are detectable with extremely high sensitivities. Those familiar with the art are aware of additional fluorescent substituents that may be used, for example, those disclosed in Haugland, 1996 Handbook of Fluorescent Probes and Research Chemicals&#x2014;Sixth Ed., Molecular Probes, Eugene, Oreg. In other embodiments, the invention provides detectable ANT ligands produced by coupling of biotin-NHS ester with the ATR derivatives as disclosed herein; these and other ANT ligands similarly generated according to the instant methods can be detected with commercially available enzyme-avidin conjugates using, for example, colorimetric, fluorescent or chemiluminescent techniques. In one embodiment, the ANT ligands of this invention have the following structured): including stereoisomers and pharmaceutically acceptable salts thereof, wherein R1 is hydroxyl, halogen, &#x2014;OC(&#x2550;O)R4 or &#x2014;NHR4; R2 is hydrogen or &#x2014;C(&#x2550;O)R5; R3 is &#x2014;CH3 or &#x2550;CH2; R4 is &#x2014;X-aryl, &#x2014;X-substituted aryl, &#x2014;X-arylalkyl, &#x2014;X-substituted arylalkyl, X-heteroaryl, or &#x2014;X-heteroarylalkyl, wherein X is an optional amido or alkylamido linker moiety; and R5 is alkyl. As used herein, the above terms have the meanings set forth below. &#x201c;Amido&#x201d; means &#x2014;NHC(&#x2550;O)&#x2014; or &#x2014;C(&#x2550;O)NH&#x2014;. &#x201c;Alkylamido&#x201d; means -(alkyl)-NHC(&#x2550;O)&#x2014; or -(alkyl)-C(&#x2550;O)NH&#x2014;, such as &#x2014;CH2NHC(&#x2550;O)&#x2014;, &#x2014;CH2CH2NHC(&#x2550;O)&#x2014;, &#x2014;CH2C(&#x2550;O)NH&#x2014;, &#x2014;CH2CH2C(&#x2550;O)NH&#x2014;, and the like. &#x201c;Alkyl&#x201d; means a straight chain or branched, noncyclic or cyclic, unsaturated or saturated aliphatic hydrocarbon containing from 1 to 8 carbon atoms. Representative saturated straight chain alkyls include methyl, ethyl, n-propyl, n-butyl, n-pentyl, n-hexyl, and the like; while saturated branched alkyls include isopropyl, sec-butyl, isobutyl, tert-butyl, isopentyl, and the like. Representative saturated cyclic alkyls include cyclopropyl, cyclobutyl, cyclopentyl, cyclohexyl, and the like; while unsaturated cyclic alkyls include cyclopentenyl and cyclohexenyl, and the like. Unsaturated alkyls contain at least one double or triple bond between adjacent carbon atoms (referred to as an &#x201c;alkenyl&#x201d; or &#x201c;alkynyl&#x201d;, respectively). Representative straight chain and branched alkenyls include ethylenyl, propylenyl, 1-butenyl, 2-butenyl, isobutylenyl, 1-pentenyl, 2-pentenyl, 3-methyl-1-butenyl, 2-methyl-2-butenyl, 2,3-dimethyl-2-butenyl, and the like; while representative straight chain and branched alkynyls include acetylenyl, propynyl, 1-butynyl, 2-butynyl, 1-pentynyl, 2-pentynyl, 3-methyl-1 butynyl, and the like. &#x201c;Aryl&#x201d; means an aromatic carbocyclic moiety such as phenyl or naphthyl (i.e., 1- or 2-naphthyl). &#x201c;Arylalkyl&#x201d; means an alkyl having at least one alkyl hydrogen atoms replaced with an aryl moiety, such as benzyl, &#x2014;(CH2)2phenyl, &#x2014;(CH2)3phenyl, and the like. &#x201c;Heteroaryl&#x201d; means an aromatic heterocycle ring of 5- to 10 members and having at least one heteroatom selected from nitrogen, oxygen and sulfur, and containing at least 1 carbon atom, including both mono- and bicyclic ring systems. Representative heteroaryls are pyridyl, furyl, benzofuranyl, thiophenyl, benzothiophenyl, quinolinyl, pyrrolyl, indolyl, oxazolyl, benzoxazolyl, imidazolyl, benzimidazolyl, thiazolyl, benzothiazolyl, isoxazolyl, pyrazolyl, isothiazolyl, pyridazinyl, pyrimidinyl, pyrazinyl, triazinyl, cinnolinyl, phthalazinyl, and quinazolinyl. &#x201c;Heteroarylalkyl&#x201d; means an alkyl having at least one alkyl hydrogen atom replaced with a heteroaryl moiety, such as &#x2014;CH2pyridinyl, &#x2014;CH2pyrimidinyl, and the like. &#x201c;Heterocycle&#x201d; means a 5- to 7-membered monocyclic, or 7- to 10-membered bicyclic, heterocyclic ring which is either saturated, unsaturated, or aromatic, and which contains from 1 to 4 heteroatoms independently selected from nitrogen, oxygen and sulfur, and wherein the nitrogen and sulfur heteroatoms may be optionally oxidized, and the nitrogen heteroatom may be optionally quaternized, including bicyclic rings in which any of the above heterocycles are fused to a benzene ring. The heterocycle may be attached via any heteroatom or carbon atom. Heterocycles include heteroaryls as defined above. Thus, in addition to the heteroaryls listed above, heterocycles also include morpholinyl, pyrrolidinonyl, pyrrolidinyl, piperidinyl, hydantoinyl, valerolactamyl, oxiranyl, oxetanyl, tetrahydrofuranyl, tetrahydropyranyl, tetrahydropyridinyl, tetrahydroprimidinyl, tetrahydrothiophenyl, tetrahydrothiopyranyl, tetrahydropyrimidinyl, tetrahydrothiophenyl, tetrahydrothiopyranyl, and the like. &#x201c;Heterocyclealkyl&#x201d; means an alkyl having at least one alkyl hydrogen atom replaced with a heterocycle, such as &#x2014;CH2morpholinyl, and the like. The term &#x201c;substituted&#x201d; as used herein means any of the above groups (i.e., alkyl, aryl, arylalkyl, heteroaryl, heteroarylalkyl, heterocycle and heterocyclealkyl) wherein at least one hydrogen atom is replaced with a substituent. In the case of a keto substituent (&#x201c;C(&#x2550;O)&#x201d;) two hydrogen atoms are replaced. Substituents include halogen, hydroxy, alkyl, haloalkyl, aryl, substituted aryl, arylalkyl, substituted arylalkyl, heterocycle, substitued heterocycle, heterocyclealkyl or substituted heterocyclealkyl. &#x201c;Halogen&#x201d; means fluoro, chloro, bromo and iodo. &#x201c;Haloalkyl&#x201d; means an alkyl having at least one hydrogen atom replaced with halogen, such as trifluoromethyl and the like. &#x201c;Alkoxy&#x201d; means an alkyl moiety attached through an oxygen bridge (i.e., &#x2014;O-alkyl) such as methoxy, ethoxy, and the like. In one embodiment, R2 is &#x2014;C(&#x2550;O)CH2CH(CH3)2, R3 is &#x2550;CH2, and the ANT ligand is an atractyloside derivative having the following structure (II): wherein R1 is as defined above. In another embodiment, R2 is &#x2014;C(&#x2550;O)CH2CH(CH3)2, R3 is &#x2014;CH3, and the ANT ligand is a dihydro-atractyloside derivative having the following structure (III): wherein R1 is as defined above. In still a further embodiment, R2 is &#x2014;OH, R3 is &#x2550;CH2, and the ANT ligand is an apoatractyloside derivative having the following structure (IV): wherein R1 is as defined above. In more specific embodiments of structures (II), (III) and (IV), R1 is &#x2014;OC(&#x2550;O)(aryl), &#x2014;OC(&#x2550;O)(substituted aryl), &#x2014;OC(&#x2550;O)(arylalkyl), &#x2014;OC(&#x2550;O)(substituted arylalkyl), &#x2014;NH(CH2)2NHC(&#x2550;O)(arylalkyl), &#x2014;NH(CH2)2NHC(&#x2550;O)(substituted arylalkyl). Representative R1 moieties in this regard include &#x2014;OC(&#x2550;O)(phenyl), &#x2014;OC(&#x2550;O)(1-naphthyl), &#x2014;OC(&#x2550;O)(substituted phenyl), &#x2014;OC(&#x2550;O)(substituted 1-naphthyl), &#x2014;OC(&#x2550;O)(CH2)1-3(phenyl), &#x2014;OC(&#x2550;O)(CH2)1-3(substituted phenyl), &#x2014;NH(CH2)2NHC(&#x2550;O)(CH2)1-3(phenyl), &#x2014;NH(CH2)2NHC(&#x2550;O)(CH2)1-3(substituted phenyl). In this context, representative substituted phenyl moieties include (but are not limited to) 4-hydroxyphenyl, 3-iodo-4-hydroxyphenyl, 3,5-iodo-4-hydroxyphenyl, 4-(4-hydroxyphenyl)phenyl, 4-(3-iodo-4-hyroxyphenyl)phenyl, 3-methyl-4-hyroxyphenyl, and 3-methyl-4-hydroxy-5-iodophenyl. The ANT ligands of structure (I) may readily be made by one skilled in the art of organic chemisty and, more particularly, by the techniques disclosed in Examples 6-11. The compounds of the present invention may generally be utilized as the free base. Alternatively, the compounds of this invention may be used in the form of acid addition salts. Acid addition salts of the free amino compounds of the present invention may be prepared by methods well known in the art, and may be formed from organic and inorganic acids. Suitable organic acids include maleic, fumaric, benzoic, ascorbic, succinic, methanesulfonic, acetic, oxalic, propionic, tartaric, salicylic, citric, gluconic, lactic, mandelic, cinnamic, aspartic, stearic, palmitic, glycolic, glutaric, and benzenesulfonic acids. Suitable inorganic acids include hydrochloric, hydrobromic, sulfuric, phosphoric, and nitric acids. Thus, the term &#x201c;pharmaceutically acceptable salt&#x201d; of structure (I) is intended to encompass any and all acceptable salt forms. With regard to stereoisomers, the compounds of structure (I) may have chiral centers and may occur as recemates, reacemic mixtures and as individual enantiomers or diastereomers. All such isomeric forms are included within the present invention, including mixtures thereof. Furthermore, some of the crystalline forms of the compounds of structure (I) may exist as polymorphs, which are included in the present invention. In addition, some of the compounds of structure (I) may also form solvates with water or other organic solvents. Such solvates are similarly included within the scope of this invention. Activities of ANT ligands are typically calculated from the IC50 as the concentration of a compound necessary to displace 50% of the detectable (i.e., detectably labeled, for example, radiolabeled) ligand from ANT molecules, which may be present as isolated or purified polypeptides or as components of preparations containing isolated mitochondria or submitochondrial particles (SMP) using established ligand binding assays or modifications thereof. For example, ANT ligands may be tested for their ability to compete with radiolabeled ATR, or with a radiolabeled ATR derivative such as compound 24 as provided herein, for binding to isolated ANT polypeptides or to ANT present in isolated mitochondria or SMP. As another example, the relative affinities for ANT of various ANT ligands as provided herein may be determined by a fluorescence assay that exploits the flourescent properties of compound 22 (Example 11), a naphthoyl-ATR derivative that is an ANT ligand having a fluorescence excitation peak at 312 nm and an emission peak at 400 nm. When compound 22 is bound to ANT, the fluorescence is quenched. When, however, compound 22 is displaced from ANT by a known concentration of ATR or an ATR derivative that is an ANT ligand, fluorescence dequenching that results from displacement of the fluorophore can be measured in real time. Briefly, a mitochondrial preparation (see, e.g., Example 13) is washed and resuspended in a suitable buffer in the presence of compound 22 (e.g., 10 mM Tris-120 mM KCl containing 3.6 nmoles of compound 22 per mg mitochondrial protein, 10 min at room temperature), washed to remove unbound fluorophore and placed in a fluorometer equipped with a light source and filter set appropriate for the fluorophore. Fluorescence intensity is monitored as a function of time, and a candidate ANT ligand is then added to determine its ability to compete with compound 22 for binding to ANT, as evidenced by a change in detectable relative fluorescence intensity units. After the fluorescence signal has stabilized, any additional compound 22 that remains bound to ANT is displaced by adding an excess (e.g., &#x3bc;M quantities) of ATR as a competitive inhibitor, to determine maximal signal intensity and therefrom calculate the proportion of compound 22 displaced by the candidate ANT ligand. Those having familiarity with the art will appreciate that variations and modifications may be made to ANT-binding assays such as those illustrated above and described in the Examples for determing IC50 values of candidate ANT ligands, and which are not intended to be limiting. Activity of each ANT ligand is reported as a &#x201c;Ki&#x201d; value calculated by the following equation: K i = IC 50 1 + L / K D where L=radioligand and KD=affinity of radioligand for receptor (Cheng and Prusoff, Biochem. Pharmacol. 22:3099, 1973). ANT ligands of this invention have a Ki of 100 &#x3bc;M or less. In a preferred embodiment of this invention, the ANT ligands have a Ki of less than 10 &#x3bc;M, and more preferably less than 1 &#x3bc;M. To this end, ANT ligands of this invention having a Ki of less than 100 &#x3bc;M include compound 5 (Example 7), compound 6 (Example 8), and compounds 22, 23, 24, 26, 29, 33, 35, 37, and 38 (Example 11). Preferred ANT ligands having a Ki of less than 10 &#x3bc;M include compounds 6, 22, 23, 24, 29, 33, 35 and 38, and more preferred ANT ligands having a Ki of less than 1 &#x3bc;M include compounds 6, 24, 33, and 38 as well as ATR. Assays It is another aspect of the invention to provide compositions and methods for the determination of the presence of ANT polypeptides and for the identification of agents that bind to, or that interact with, ANT polypeptides. Such compositions and methods will be useful for diagnostic and prognostic purposes, for example in the determination of the existence of altered mitochondrial function which, as described above, may accompany both normal and disease states. These compositions and methods will also be useful for the identification of agents that alter or regulate mitochondrial function based on ANT roles in mitochondrial activities, for example by way of illustration and not limitation, maintenance of mitochondrial membrane potential, ATP biosynthesis, induction of apoptosis, MPT and other mitochondrial function. In certain preferred embodiments these compositions and methods are useful as high throughput screening assays. In certain aspects the invention provides a method for determining the presence of an ANT polypeptide in a biological sample, comprising contacting a sample suspected of containing an ANT polypeptide with an ANT ligand under conditions and for a time sufficient to allow binding of the ANT ligand to an ANT polypeptide, and detecting such binding &#x201c;ANT ligands&#x201d; according to these aspects of the invention may include any novel ANT ligands as provided herein. The use of human ANT1, ANT2 and ANT3 according to these methods represent particularly preferred embodiments. Other preferred embodiments include the use of any ANT polypeptide or ANT fusion protein as provided herein. Accordingly, the instant method for determining the presence of ANT polypeptide in a sample will be useful for monitoring expression of ANT encoding constructs provided herein. In some preferred embodiments an ANT fusion protein is used that is a GST fusion protein, and in other preferred embodiments the ANT fusion protein is a His-tagged fusion protein. As provided herein, the biological sample may be a cell, a mitochondrion, submitochondrial particles, a cell membrane (including any cellular membrane as described herein), a cell extract, cell conditioned medium, a tissue homogenate or an isolated ANT. In other aspects, the invention provides a method for identifying an agent that binds to an ANT polypeptide, comprising contacting a candidate agent with a host cell expressing at least one recombinant ANT polypeptide under conditions and for a time sufficient to permit binding of the agent to the ANT polypeptide and detecting such binding. In various preferred embodiments the host cell may be a prokaryotic cell or a eukaryotic cell. In certain other preferred embodiments the host cell may lack at least one isoform of an endogenous ANT, for example, due to a mutation in one or more endogenous ANT encoding genes. In certain other embodiments host cell expression of at least one gene encoding an endogenous ANT isoform is substantially impaired, for example, through the use of ANT nucleic acid-specific ribozyme or antisense constructs as provided herein, or through the use of p&#xb0; cells, as also provided herein. According to other embodiments of this aspect of the invention, it may be preferred to use intact cells or, alternatively, to use permeabilized cells. Those having ordinary skill in the art are familiar with methods for permeabilizing cells, for example by way of illustration and not limitation, through the use of surfactants, detergents, phospholipids, phospholipid binding proteins, enzymes, viral membrane fusion proteins and the like; through the use of osmotically active agents; by using chemical crosslinking agents; by physicochemical methods including electroporation and the like, or by other permeabilizing methodologies. In other aspects, the invention provides a method for identifying an agent that binds to an ANT polypeptide comprising contacting a candidate agent with a biological sample containing at least one recombinant ANT polypeptide under conditions and for a time sufficient to permit binding of the agent to the ANT polypeptide, and detecting such binding. The use of human ANT1, ANT2 and ANT3 according to these methods represent particularly preferred embodiments. Other preferred embodiments include the use of any ANT polypeptide or ANT fusion protein as provided herein. In some preferred embodiments an ANT fusion protein is used that is a GST fusion protein, and in other preferred embodiments the ANT fusion protein is a His-tagged fusion protein. As provided herein, the biological sample may be a cell, a mitochondrion, submitochondrial particles, a cell membrane (including any cellular membrane as described herein), a cell extract, cell conditioned medium, a recombinant viral particle, a tissue homogenate or an isolated ANT. Detection of binding may be by any of a variety of methods and will depend on the nature of the candidate agent being screened. For example, certain candidate agents are inherently detectable as a consequence of their physicochemical properties, such as will be apparent to those skilled in the art and including spectrophotometric, colorimetric, fluorimetric, solubility, hydrophobic, hydrophilic, electrostatic charge, molecular mass or other physicochemical properties. As another example, certain candidate agents may be radioactively labeled with a readily detectable radionuclide, as is well known in the art. Certain candidate agents may also be directly or indirectly detectable by ANT protein affinity methodologies, for example by their ability to interfere with binding of an ANT-specific antibody to an ANT; or by their being removable from an assay solution using a protein affinity reagent that binds to a fusion polypeptide present as a portion of an ANT fusion protein. A candidate agent bound to an ANT polypeptide may be detected by any method known for the detection, identification or characterization of relevant molecules, including spectrophotometric, mass spectrometric, chromatographic, electrophoretic, calorimetric or any other suitable analytical technique. In another aspect the invention provides a method for identifying an agent that interacts with an ANT polypeptide comprising contacting a biological sample containing recombinant ANT with a detectable ANT ligand (or a known detectable molecule capable of binding to ANT) in the presence of a candidate agent, and comparing binding of the detectable ANT ligand (or known detectable ANT binding molecule) to recombinant ANT in the absence of the agent to binding of the detectable ANT ligand (or known detectable ANT binding molecule) to recombinant ANT in the presence of the agent, and therefrom identifying an agent that interacts with an ANT polypeptide. It will be appreciated that in certain preferred embodiments this aspect provides competitive binding assays wherein novel ANT ligands as provided hereinabove are useful. However, this aspect of the invention need not be so limited and may be modified to employ known detectable ANT binding molecules, in which case it should be pointed out that the selection of biological sample and/or of recombinant ANT as provided by the present invention offer unexpected advantages heretofore unknown in the art. Examples of known detectable ANT-binding molecules include suitably labeled ATP, ADP, ATR, CATR, palmitoyl-CoA, bongkrekic acid, thyroxin, eosin Y and erythrosin B or other ANT-binding molecules known in the art. (See, e.g., Block et al., 1986 Meths. Enzymol. 125:658.) The use of human ANT1, ANT2 and ANT3 according to these methods represent particularly preferred embodiments. Other preferred embodiments include the use of any ANT polypeptide or ANT fusion protein as provided herein. In some preferred embodiments an ANT fusion protein is used that is a GST fusion protein, and in other preferred embodiments the ANT fusion protein is a His-tagged fusion protein. As provided herein, the biological sample may be a cell, a mitochondrion, submitochondrial particles, a cell membrane (including any cellular membrane as described herein), a cell extract, cell conditioned medium, a recombinant viral particle, a tissue homogenate or an isolated ANT. The ANT ligands compounds are preferably part of a pharmaceutical composition when used in the methods of the present invention. The pharmaceutical composition will include at least one of a pharmaceutically acceptable carrier, diluent or excipient, in addition to one or more ANT ligands and, optionally, other components. &#x201c;Pharmaceutically acceptable carriers&#x201d; for therapeutic use are well known in the pharmaceutical art, and are described, for example, in Remingtons Pharmaceutical Sciences, Mack Publishing Co. (A. R. Gennaro edit. 1985). For example, sterile saline and phosphate-buffered saline at physiological pH may be used. Preservatives, stabilizers, dyes and even flavoring agents may be provided in the pharmaceutical composition. For example, sodium benzoate, sorbic acid and esters of p-hydroxybenzoic acid may be added as preservatives. Id. at 1449. In addition, antioxidants and suspending agents may be used. Id. &#x201c;Pharmaceutically acceptable salt&#x201d; refers to salts of the compounds of the present invention derived from the combination of such compounds and an organic or inorganic acid (acid addition salts) or an organic or inorganic base (base addition salts). The compounds of the present invention may be used in either the free base or salt forms, with both forms being considered as being within the scope of the present invention. The pharmaceutical compositions that contain one or more ANT substrates/ligands compounds may be in any form which allows for the composition to be administered to a patient. For example, the composition may be in the form of a solid, liquid or gas (aerosol). Typical routes of administration include, without limitation, oral, topical, parenteral (e.g., sublingually or buccally), sublingual, rectal, vaginal, and intranasal. The term parenteral as used herein includes subcutaneous injections, intravenous, intramuscular, intrasternal, intracavernous, intrameatal, intraurethral injection or infusion techniques. The pharmaceutical composition is formulated so as to allow the active ingredients contained therein to be bioavailable upon administration of the composition to a patient. Compositions that will be administered to a patient take the form of one or more dosage units, where for example, a tablet may be a single dosage unit, and a container of one or more compounds of the invention in aerosol form may hold a plurality of dosage units. For oral administration, an excipient and/or binder may be present. Examples are sucrose, kaolin, glycerin, starch dextrins, sodium alginate, carboxymethylcellulose and ethyl cellulose. Coloring and/or flavoring agents may be present. A coating shell may be employed. The composition may be in the form of a liquid, e.g., an elixir, syrup, solution, emulsion or suspension. The liquid may be for oral administration or for delivery by injection, as two examples. When intended for oral administration, preferred composition contain, in addition to one or more ANT substrates/ligands compounds, one or more of a sweetening agent, preservatives, dye/colorant and flavor enhancer. In a composition intended to be administered by injection, one or more of a surfactant, preservative, wetting agent, dispersing agent, suspending agent, buffer, stabilizer and isotonic agent may be included. A liquid pharmaceutical composition as used herein, whether in the form of a solution, suspension or other like form, may include one or more of the following adjuvants: sterile diluents such as water for injection, saline solution, preferably physiological saline, Ringer's solution, isotonic sodium chloride, fixed oils such as synthetic mono or digylcerides which may serve as the solvent or suspending medium, polyethylene glycols, glycerin, propylene glycol or other solvents; antibacterial agents such as benzyl alcohol or methyl paraben; antioxidants such as ascorbic acid or sodium bisulfite; chelating agents such as ethylenediaminetetraacetic acid; buffers such as acetates, citrates or phosphates and agents for the adjustment of tonicity such as sodium chloride or dextrose. The parenteral preparation can be enclosed in ampoules, disposable syringes or multiple dose vials made of glass or plastic. Physiological saline is a preferred adjuvant. An injectable pharmaceutical composition is preferably sterile. A liquid composition intended for either parenteral or oral administration should contain an amount of ANT substrates/ligands compound such that a suitable dosage will be obtained. Typically, this amount is at least 0.01 wt % of an ANT substrates/ligands compound in the composition. When intended for oral administration, this amount may be varied to be between 0.1 and about 70% of the weight of the composition. Preferred oral compositions contain between about 4% and about 50% of ANT substrates/ligands compound(s). Preferred compositions and preparations are prepared so that a parenteral dosage unit contains between 0.01 to 1% by weight of active compound. The pharmaceutical composition may be intended for topical administration, in which case the carrier may suitably comprise a solution, emulsion, ointment or gel base. The base, for example, may comprise one or more of the following: petrolatum, lanolin, polyethylene glycols, beeswax, mineral oil, diluents such as water and alcohol, and emulsifiers and stabilizers. Thickening agents may be present in a pharmaceutical composition for topical administration. If intended for transdermal administration, the composition may include a transdermal patch or iontophoresis device. Topical formulations may contain a concentration of the ANT substrates/ligands compound of from about 0.1 to about 10% w/v (weight per unit volume). The composition may be intended for rectal administration, in the form, e.g., of a suppository which will melt in the rectum and release the drug. The composition for rectal administration may contain an oleaginous base as a suitable nonirritating excipient. Such bases include, without limitation, lanolin, cocoa butter and polyethylene glycol. In the methods of the invention, the ANT substrates/ligands compound(s) may be administered through use of insert(s), bead(s), timed-release formulation(s), patch(es) or fast-release formulation(s). It will be evident to those of ordinary skill in the art that the optimal dosage of the ANT substrates/ligands compound(s) may depend on the weight and physical condition of the patient; on the severity and longevity of the physical condition being treated; on the particular form of the active ingredient, the manner of administration and the composition employed. It is to be understood that use of an ANT substrates/ligands compound in a chemotherapy can involve such a compound being bound to an agent, for example, a monoclonal or polyclonal antibody, a protein or a liposome, which assist the delivery of said compound. EXAMPLES The following Examples are offered by way of illustration and not by way of limitation. Example 1 Cloning and Expression of His-Tagged Human Ant Proteins in Bacteria A. PCR Amplification of ANT cDNAs Total cellular RNA prepared from whole human brain was obtained from a commercial source (Clontech, Palo Alto, Calif.). The RNA was purified by treatment with RNase-free DNase I (Roche Molecular Biochemicals, formerly Boehringer Mannheim Biochemicals, Indianapolis, Ind.) using 1 ul of DNase I (10 u/ul) in a buffer containing 40 mM Trsi-HCl, pH 7.0, 6 mM magnesium chloride and 2 mM calcium chloride for 30 minutes at 37&#xb0; C. This treatment was followed by two phenol/chloroform extractions, one chloroform extraction and an ethanol precipitation in the presence of sodium acetate. The RNA pellet was collected by centrifugation, washed with 70% ethanol, air dried, and resuspended in RNase-free sterile water. The RNA was reverse transcribed to generate cDNA using RNase H-deficient Reverse Transcriptase (SUPERSCRIPT&#x2122;; Life Technologies, Rockville, Md.). ANT cDNAs were amplified by polymerase chain reactions (PCR) in a thermal cycler using the following primers, AMPLITAQ&#x2122; DNA Polymerase (Perkin-Elmer), and reagents and buffers supplied in a GENEAMP&#x2122; PCR Reagent Kit (Perkin-Elmer), according to the manufacturer's instructions. In the following representations of the PCR primers, underlined nucleotides indicate sequences complementary to the 5&#x2032;-ends and 3&#x2032;-ends of the ANT cDNAs and double-underlined nucleotides indicate recognition sequences for the restriction enzymes XhoI (recognition sequence: 5&#x2032;-CTCGAG) and Asp718 (recognition sequence: 5&#x2032;-GGTACC). For human ANT1 (huANT1; SEQ ID NO:1), the following primers were used: Forward (sense): 5&#x2032;-TTATATCTCGAGTATGGGTGATCACGCTTGGAGCTT SEQ ID NO:4 CCTAAAG and Reverse (antisense): 5&#x2032;-TATATAGGTACCTTAGACATATTTTTTGATCTCATC SEQ ID NO:5 ATACAAC. For human ANT2 (huANT2; SEQ ID NO:2), the following primers were used: Forward (sense): 5&#x2032;-TTATATCTCGAGTATGACAGATGCCGCTGTGTCCTT SEQ ID NO:6 CGCCAAG and Reverse (antisense): 5&#x2032;-TATATAGGTACCTTATGTGTACTTCTTGATTTCATC SEQ ID NO:7 ATACAAG. For human ANT3 (huANT3; SEQ ID NO:3), the following primers were used: Forward (sense): 5&#x2032;-TTATATCTCGAGTATGACGGAACAGGCCATCTCCTT SEQ ID NO:8 CGCCAAA and Reverse (antisense): 5&#x2032;-TATATAGGTACCTTAGATCACCTTCTTGAGCTCGTC SEQ ID NO:9 GTACAGG. B. Generation of ANT Expression Constructs PCR products were digested with the restriction endonucleases XhoI and Asp718 (both enzymes from Roche Molecular Biochemicals) according to the manufacturer's recommendations using manufacturer-supplied reaction buffers. Restricted DNAs were purified by horizontal agarose gel electrophoresis and band extraction using the UltraClean GelSpin kit (Mo Bio Laboratories, Inc., Solana Beach, Calif.). The expression vector pBAD/His (&#x201c;B&#x201d; derivative; Invitrogen, Carlsbad, Calif.) was used. This vector contains the following elements operably linked in a 5&#x2032; to 3&#x2032; orientation: the inducible, but tightly regulatable, araBAD promoter; optimized E. coli translation initiation signals; an amino terminal polyhistidine(6xHis)-encoding sequence (also referred to as a &#x201c;His-Tag&#x201d;); an XPRESS&#x2122; epitope-encoding sequence; an enterokinase cleavage site which can be used to remove the preceding N-terminal amino acids following protein purification, if so desired; a multiple cloning site; and an in-frame termination codon. Plasmid pBAD/His DNA was prepared by digestion with the restriction endonucleases XhoI and Asp718 according to the manufacturer's instructions and subjected to horizontal agarose gel electrophoresis and band extraction using the UltraClean GelSpin kit (Mo Bio Laboratories). Restricted ANT cDNAs were ligated into the linearized plasmid with restricted expression vector DNA using T4 DNA ligase (New England Biolabs, Beverly, Mass.) using the manufacturer's reaction buffer and following the manufacturer's instructions. Competent recA1 hsdR endA1E. coli cells (strain TOP10F&#x2032;; Invitrogen, Catalog #C3030-03) were transformed with ligation mixtures containing the prokaryotic vector construct according to the manufacturer's instructions. Single colonies were selected and grown in 3-5 ml of LB broth (Sambrook, J., Fritsch, E. F., and Maniatis, T., Molecular Cloning.: A Laboratory Manual, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1989) containing 50 &#x3bc;g/ml ampicillin (Roche Molecular Biochemicals). Plasmid DNA was isolated from the bacterial cultures using the WIZARD&#x2122; Plus Series 9600 Miniprep Reagents System (Promega, Madison, Wis.). The recombinant huANT nucleotide sequences present in the expression constructs were determined and their authenticity confirmed relative to the published ANT sequences (FIG. 1; See Neckelmann et al., Proc. Nat'l. Acad. Sci. U.S.A. 84:7580-7584 (1987) for huANT1; Battini et al., J. Biol. Chem. 262:4355-4359 (1987) for huANT2, and Cozens et al., J. Mol. Biol. 206:261-280 (1989) for huANT3.) by DNA sequencing using the PRISM&#x2122; Ready BIG DYES Terminator Cycle Sequencing Kit (The Perkin-Elmer Corp., Norwalk, Conn.) and the following sequencing primers 5&#x2032;-TATGCCATAGCATTTTTATCC (SEQ ID NO: 10) and 5&#x2032;-CGCCAAAACAGCCAAGCT (SEQ ID NO:11). For each human ANT sequence, both primers are located inside the vector sequence adjacent to the DNA insertion. Sequence data was analyzed using the SEQUENCE NAVIGATORS analysis software package (Perkin-Elmer). This huANT3 expression construct was named pMK3A-huANT3. The expression plasmids encoding His-tagged human ANT1, ANT2 and ANT3 are referred to herein as follows: For human ANT1, &#x201c;pMK1 (His-tagged huANT1)&#x201d; or &#x201c;pMK1&#x201d;; for human ANT2, &#x201c;pMK2(His-tagged huANT2)&#x201d; or &#x201c;pMK2&#x201d;; for human ANT3 &#x201c;pMK3A (His-tagged hu ANT3&#x201d; or &#x201c;pMK3A&#x201d;; for human ANT3 from which extraneous linker N-terminal amino acids are deleted as detailed below, &#x201c;pMK3B (His-tagged hu ANT3, shortened epiotpe linker)&#x201d; or &#x201c;pMK3B&#x201d;. Plasmids pMK1, pMK2 and pMK3A have been deposited at the American Type Culture Collection (ATCC; Manassas, Va.) on Nov. 3, 1998, and given the accession numbers ATCC 98969, ATCC 98970 and ATCC 98971, respectively. The expression constructs comprising nucleotide sequences encoding human ANT1 (pMK1-huANT1) and human ANT2 (pMK2-huANT2) were restriction mapped to confirm their structures. The nucleotide sequences of plasmids pMK1-huANT1 and pMK2-huANT2 are determined using the methods and primers (SEQ ID NOS:10 and 11) described above. Treatment of the recombinant huANT3 protein expressed from pMK3A-huANT3 with enterokinase liberates the His-Tag/XPRESS&#x2122; epitope polypeptide from the huANT3 protein; however, the resultant huANT3 protein comprises several extraneous N-terminal amino acids (i.e., Pro-Ser-Ser-Ser-Met, where &#x201c;Met&#x201d; indicates the amino acid encoded by the translation initiation codon of huANT3). Although the extraneous amino acids probably have little or no effect on the recombinant huANT3 protein, a derivative expression construct in which the nucleotide sequence encoding the extraneous amino acids are deleted was prepared in the following manner. The QUIK-CHANGE&#x2122; Site-Directed Mutagenesis Kit (Stratagene, La Jolla, Calif.) essentially according to the manufacturer's instructions. Briefly, a reaction mixture comprising purified pMK3A-huANT3 DNA, the mutagenic oligonucleotide primers 5&#x2032;-GGAGATGGCCTGTTCCGTCATCTTATCGTCATCGTCGTACAGATC (SEQ ID NO:12; the underlined sequence is the reverse complement of the 5&#x2032; end of the huANT3 reading frame), and 5&#x2032;-GATCTGTACGACGATGACGATAAGATGACGGAACAGGCCATCTCC(SEQ ID NO:13; the underlined sequence corresponds to the 5&#x2032; end of the huANT3 reading frame), Pfu DNA polymerase and dNTPs in manufacturer-supplied reaction buffer was prepared. The mutagenic oligonucleotide primers were present in excess and cycles of DNA synthesis was carried out in a thermal cycler according to the manufacturer's protocol. The reaction products were treated with the restriction enzyme DpnI, which cleaves methylated and hemi-methylated DNAs but leaves unmethylated DNA (i.e., annealed products of the reaction) intact, and used to transform EPICUREAN COLI&#x2122; XL-1-Blue E. coli cells (Stratagene). Plasmid DNA was prepared from twelve randomly selected transformants and the nucleotide sequence of the region containing the multiple cloning site cassette was determined according to the methods described above. Of the twelve plasmids, only one retained the original sequence found in pMK-huANT3, and three contained undesired point mutations. One of the eight &#x201c;correct&#x201d; plasmids was chosen and named pMK3B-huANT3. C. Expression of His-Tagged huANT3 Cultures of E. coli cells containing pMK3A-huANT3 were grown in LB media containing 50 ug/ml ampicillin to mid-log phase (OD600&#x2dc;0.5) and induced for 3-4 hours with increasing doses of arabinose (i.e., 0.00002%, 0.0002%, 0.002%, 0.02%, and 0.2%). One ml of each culture was centrifuged at 5,000&#xd7;g for 10 minutes at 4&#xb0; C. to pellet the cells. Cell pellets were resuspended, and the cells were lysed, by adding 100 ul of Phosphate Buffered Saline (PBS; pH 7.4) containing 1% cholate, 1% n-dodecyl maltoside, and 0.1% 2-mercaptoethanol (in the preceding text, and throughout the specification, unless specified otherwise, all chemicals are from Sigma, St. Louis, Mo.). Total protein content in the lysates was determined using the BCA (bicinchoninic acid; Smith et al., 1985, Anal. Biochem. 150:76-85) Protein Assay kit (Pierce Chemical Co., Rockford, Ill.). Ten &#x3bc;g of total protein were loaded per lane onto an SDS polyacrylamide gel, electrophoresed and transferred to a nitrocellulose membrane (HYBOND&#x2122; ECL Nitrocellulose Membrane, Amersham Pharmacia Biotech, formerely Amersham Life Sciences, Piscataway, N.J.). Human ANT3 fusion proteins were detected in a western blot using ANTI-XPRESS&#x2122; Antibody (Invitrogen) and horseradish peroxidase-conjugated anti-mouse secondary antibody (Amersham Pharmacia Biotech) according to the manufacturers' instructions. The results are shown in FIG. 3. From left to right in the figure, the following samples are shown: lanes &#x201c;M&#x201d;, molecular weight markers; lane &#x201c;0&#x201d;, untransformed E. coli cells; lane &#x201c;o/n&#x201d;, E. coli comprising pMK3A-huANT3 grown overnight without induction; lane &#x201c;1&#x201d;-&#x201c;5&#x201d;, E. coli comprising pMK3A-huANT3 grown induced with increasing doses of arabinose (0.00002%, 0.0002%, 0.002%, 0.02% and 0.2%, respectively). As expected, untransformed (lane 0) and uninduced (lane o/n) E coli showed no XPRESS&#x2122;-huANT3 material. However, expression of recombinant ANT3 fusion protein with a molecular weight of 36.6 kD was observed in lanes 3 and 4 (0.002% and 0.02% arabinose, respectively). No XPRESS&#x2122;-huANT3 material was detected in lanes 1 and 2 (0.00002% and 0.0002% arabinose, respectively) indicating that the degree of induction was insufficient under these conditions. Cells that were grown in the presence of the highest concentration of arabinose (0.2%, lane 5) began to lyse and died before the time of harvest; consequently, no recombinant protein was detected. This indicated that very high expression of recombinant huANT in E. coli caused cell death, as is sometimes the case during overexpression of heterologous proteins in bacteria. D. Recombinant huANT3 Localizes to the Bacterial Membrane In order to locate the expressed human ANT 3 within E. coli cells, cells were grown in culture and induced with arabinose as described above, and then fractionated into different compartments (e.g., membranes, inclusion bodies and cytosol). Bacteria were pelleted by centrifugation at 5,000&#xd7;g for 10 minutes at 4&#xb0; C. The cell pellets were resuspended in {fraction (1/10)}volume of cell buffer A (50 mM Tris-HCl, pH 8.0, 2 mM EDTA, 100 ug/ml lysozyme, and 0.1% Triton X-100) and incubated for 15 minutes at 30&#xb0; C. in an orbital shaker. The cell mixture was sonicated for 2 minutes and membranes were pelleted by centrifugation at 12,000&#xd7;g for 15 minutes at 4&#xb0; C. The supernatant, representing the cytosol, was removed for analysis (FIG. 4, lane 4), as was a portion of the pellet containing membranes and inclusion bodies (FIG. 4, lane 3). The remaining portion of the pellet was washed twice with cell buffer B (10 mM Tris-HCl, pH 7.0, 0.1 mM EDTA, and 1 mM DTT) and centrifuged at 12,000&#xd7;g for 15 minutes at 4&#xb0; C. The pellet was resuspended in cell buffer C (20 mM Tris-HCl, pH 8.0, 100 mM sodium chloride, and 6 M guanidinium hyrochloride) and incubated for 1 hour at room temperature. The solution was then centrifuged at 12,000&#xd7;g for 15 minutes at 4&#xb0; C. The supernatant (containing solubilized inclusion bodies; lane 1, FIG. 4) and the pellet (containing insoluble inclusion bodies; lane 2, FIG. 4) were analyzed by Western blotting as described above. The results are shown in FIG. 4. Recombinant huANT 3 (molecular weight 36.6 kD) was detected in lanes 2, 3, and 4, as well as the positive control lane (+) (total cell lysate previously tested for presence of ANT3 protein by Western immunoblot analysis, as described above). The greatest amount of recombinant huATN3 was detected in lane 3, which represents the membrane fraction. This indicates that the majority of the huANT3 fusion protein integrated into the E. coli cellular membrane. Smaller protein signals were visible in lanes 2 and 4, representing the insoluble inclusion body fraction which might have contained some membranes with integrated ANT 3, and the cytosolic fraction where protein synthesis takes place. No protein was detectable in the soluble inclusion body fraction in lane 1, indicating that controlled expression of ANT3 in the bacteria did not result in the formation of inclusion bodies, which is an undesirable consequence of over-expression of some heterologous proteins in bacteria. E. Purification of ANT Proteins ANT proteins, and ANT fusion proteins, produced by the expression systems described herein have been purified using a variety of methods. The purification of ANT proteins, particularly human ANT proteins, is described in this Example. Regardless of which of following protein purification methods is used, or others that can be derived from the present disclosure, it is important to add sufficient amounts of DNase and RNase to eliminate the viscosity associated with some bacterial lysates (typically 10 &#x3bc;g/mL of each enzyme; both from Roche Biochemicals) when the bacterial cells are lysed (or immediately thereafter). An alternative or additional means by which viscosity has been minimized and ANT solubility has been optimized is vigorous sonication, as opposed to standard sonication, of the lysates. The term &#x201c;vigorous sonication&#x201d; refers to, for example, sonication with a Branson Sonifier (Model 450) 2&#xd7;(30 seconds each time) at 50% duty cycle and 80% output using a tapered, flat-tipped probe (as opposed to sonication with a cup and horn apparatus). Although either type of sonication will suffice, better yields have typically been observed when vigorous sonication has been used. Furthermore, in various ANT purification methods that have been used, it was often desirable to make the lysate at least 1% Triton-X, in order to solubilize the maximum possible amount of ANT protein, after which insoluble material is removed by a high-speed (i.e., about 100,000 g) spin. Typically, protease inhibitors such as, for example, pepstatin, leupeptin, phenylmethylsulfonyl fluoride (PMSF) and/or aprotinin (all from Sigma) have been present at effective levels (typically 10 &#x3bc;g/mL) during the preparation. Depending on the particular ANT protein or ANT fusion protein being isolated, all four protease inhibitors or any effective combination thereof are used. For example, in preparations of GST-huANT3 fusion proteins, best results were seen when all four protease inhibitors were used, although acceptable results have been obtained when only leupeptin and pepstatin were used. One method incorporates novel methods with several techniques previously used only for purifying ANT proteins from non-human mammals, i.e., bovine cardiac tissue and rats (Aquila et al., 1982, Hoppe-Seyler's Z. Physiol. Chem. 363:345-349; and Sterling, 1986, Endocrinology 119:292-295). In brief, bacterial cells expressing a GST-ANT3 fusion protein were lysed by lysozyme treatment, and 14C-palmityl-CoA (Sigma) was added at a concentration of 50 nmol per gram of E. coli. Because it associates with ANT proteins, 14C-palmityl-CoA acts as a radiolabeled tracer that can be used to follow the ANT protein in subsequent purification steps The lysates were then sonicated and made 6% Triton X-100 (Sigma) and incubated at 4&#xb0; C. for 1 hr to solubilize material. A high-speed spin was used to remove insoluble material, and the resulting solute was applied either (1) for small scale preparations, to hydroxyapatite beads (Bio-Rad Laboratories, Hercules, Calif.), or (2) in the case of larger preparations (i.e., &#x2267;1 liter of bacterial culture), to a hydroxyapatite column (Bio-Rad) essentially according to the manufacturer's instructions. Unlike other intramembrane mitochondrial proteins, ANT has a low affinity for hydroxyapatite (Klingenberg et al., 1978, Biochim. et Biophys. Acta 503:193-210). The hydroxyapatite column was eluted with Column Buffer A (10 mM MOPS, pH 7.2, 100 Mm NaCl, 9.5% Triton X100) and washed with Column Buffer B (10 mM MOPS, pH 7.2, 100 mM NaCl, 400 mM sodium phosphate). Non-recombinant ANT proteins from non-human species are eluted in the void volume with Column Buffer A, and the GST-huANT3 fusion protein was expected to be present in the void volume as well; Column Buffer B was used to wash the column in the event that GST-huANT3 fusion protein behaves differently. Samples were collected in such a manner as to have a final concentration of 30 of mM octyl glucoside (Calbiochem), a nonionic detergent that helps solubulize ANT proteins with minimal effect on activity (Sterling, 1986, Endrocrinol. 119: 292-295). The bead-extracted supernatant or column eluent was collected, and Triton X-100 was removed therefrom using the EXTRACTI-GEL&#x2122; affinity matrix (Pierce) essentially according to the manufacturer's instructions (see also Berman et al., 1985, Biochemistry 24:7140-7147). Varying amounts of GST-huANT3 prepared in the above manner were subject to PAGE and the gel was stained using a colloidal blue protein stain (Novex, San Diego, Calif.). The stained gel displayed a single band having a molecular weight corresponding to that predicted for the fusion protein. Based on the intensity of bands from samples of varying volumes, and the known volume of the preparation and minimal sensitivity of the stain, the yield from 100 mL of bacterial culture was estimated to be about 50 ug. In one of the lanes of the gel, approximately 500 ng of protein was loaded, and no contaminating bands were detected; this indicates that the GST-huANT3 protein was from at least about 90% pure to at least about 95% pure. GST-huANT3 fusion proteins (see preceding Examples) have been purified by this method, and other ANT fusion proteins, including His-tagged huANT3 and other His-tagged ANT proteins, are purified in like fashion. Purified huANT fusion proteins are used to produce purified human ANT proteins as follows. GST-huANT fusion proteins are further purified via glutathione-agarose beads (Sigma) essentially according to the manufacturer's instructions. In brief, a solution comprising GST-huANT fusion proteins is contacted with glutathione-agarose beads, and the beads are washed to release undesirable contaminants. Next, the [bead:GST-huANT] complexes are treated with an appropriate enzyme, i.e., one that separates the huANT polypeptide from the remainder of the fusion protein. In the case of the GST-huANT3 fusion protein described herein (i.e., that encoded by pMK3C), thrombin (Sigma) cleaves the fusion protein in such a manner so as to produce two polypeptides: a first polypeptide corresponding to the GST moiety, and a second polypeptide which corresponds to human ANT3 with an additional six amino acids (i.e., Gly-Ser-Pro-Gly-Ile-Leu) present at its N-terminus. His-tagged huANT fusion proteins are further purified via Nickel-coated resins (such as, e.g., PROBOND&#x2122; Ni2+ charged agarose resin; Invitrogen) essentially according to the manufacturer's instructions. In brief, a solution comprising His-tagged huANT fusion proteins is contacted with the Nickel-coated resin, and the resin is washed to release undesirable contaminants. Next, the [resin:His-tagged huANT] complexes are treated with an appropriate enzyme, i.e., one that separates the huANT polypeptide from the remainder of the fusion protein. In the case of the His-tagged huANT3 fusion proteins described herein, enterokinase (Sigma, or EKMAX&#x2122; from Invitrogen may be used) cleaves the fusion protein in such a manner so as to produce two polypeptides: a first polypeptide comprising the His-tag and XPRESS&#x2122; epitope moieties, and a second polypeptide which corresponds to human ANT3. If the expression construct used is pMK3A, the resultant purified human ANT3 protein has an additional four amino acids (i.e., Pro-Ser-Ser-Ser) at its N-terminus. If pMK3B is the expression construct present in the cells from which His-tagged huANT3 is isolated, the resultant purified human ANT3 protein has the sequence of native huANT3, i.e., SEQ ID NO:3. In both of the preceding purification steps, an ANT fusion protein bound to a solid support is treated with an enzyme (i.e., thrombin or enterokinase) that liberates an ANT protein from the remainder of the fusion protein, which remains bound to the solid support. ANT protein is released into the liquid phase which is then collected to generate a solution comprising the ANT protein and some amount of the liberating enzyme. The amount of liberating enzyme needed is minimal because the treatment is catalytic in nature; nevertheless, some enzyme remains in the preparation. If desired, enzyme molecules may be removed from the preparation using any of a variety of means known in the art. For example, an enzyme may be removed from a solution by contacting the solution with a resin conjugated to a ligand having a high affinity for the enzyme. In the case of enterokinases, one such resin is the EK-AWAY&#x2122; resin (Invitrogen) which comprises the soybean trypsin inhibitor having a high affinity for enterokinases. Methods of treating GST fusion proteins with thrombin and purifying the desired recombinant protein have been described previously (see, for example, Smith and Corcoran, Unit 16.7 in Chapter 16 in Short Protocols in Molecular Biology 2nd Ed., Ausubel et al., eds, John Wiley &#x26; Sons, New York, N.Y., 1992, pages 16-28 to 16-31. In general, however, any suitable means for separating the liberating enzyme from any given ANT protein may be used. F. Growth Inhibition As noted in the above discussion of the results presented in FIG. 3, very high expression of recombinant huANT3 in E. coli caused cell death. Such a result is sometimes observed during over-expression of heterologous proteins in bacteria. Although not wishing to be bound by any particular theory, because the recombinant huANT3 protein localized to the bacterial membrane, and because ANT3 functions as an ATP/ADP exchanger in the inner mitochondrial membrane and under appropriate conditions may exhibit pore properties suggestive of a role in membrane permeability, one possible explanation for the observed cell death would be an inappropriate enhancement of the permeability of the bacterial membrane. If this in fact the case, inhibitors of mitochondrial ANT might prevent the death of E. coli overexpressing huANT3. As noted above, under certain conditions atractyloside or bongkrekic acid may exhibit inhibition of ANT activity, such that either of these inhibitors, other known ANT-active agents and potentially other ANT ligands as provided herein may be employed in the instant Example described using bongkrekic acid. In order to test this hypothesis, the following experiments are carried out. E. coli harboring pMK3A-huANT3 are grown with no arabinose or with 0.2% or more arabinose, the latter concentration having been previously shown to induce toxic levels of huANT3, and various concentrations (0, 5, 20, 50 and 200 A) of bongkrekic acid (Biomol Research Laboratories, Inc., Plymouth Meeting, Mass.), an inhibitor of ANT (Henderson and Lardy, 1970, J. Biol. Chem. 245:1319-1326) that binds to ANT (see, e.g., Vignais et al., 1976, Biochim. Biophys. 440:688-696). The ability of bongkrekic acid to prevent the lysis of E. coli overexpressing huANT3, or any other ANT protein for that matter, indicates that the toxic effect of such overexpression is due to an activity associated with normally functioning ANT. ANT proteins produced by this expression system, and others described herein, are also purified using known methods for purifying ANT proteins from humans and other mammals. See for example, Klingenberg et al., 1978 Biochim. Biophys. Acta 503:193-210; Aquila et al., 1982 Hoppe-Seyler's Z. Physiol. Chem. 363:345-349; and Sterling, 1986 Endocrinol. 119:292-295. The bacterial toxicity of extreme overexpression of ANT in this system can be used to screen and identify novel inhibitors of ANT, as such compounds will be expected to also prevent lysis of E. coli overexpressing ANT proteins. In order to achieve a greater degree of specificity for the ANT protein produced from an expression vector, the yeast expression system for ANT proteins (see Example 4, infra) is used in a mutant yeast strain that is resistant to bongkrekic acid (Lauquin et al., 1975 FEBS Letters 35:198-200). Example 2 Expression of GST-huANT3 Fusion Proteins A. Generation of GST-huANT3 Expression Constructs Human ANT3 cDNA was amplified from pMK3A-huANT3 by PCR as in Example 1 but using the following primers. In the following representations of PCR primers, underlined nucleotides indicate sequences complementary to the 5&#x2032;-ends and 3&#x2032;-ends of the ANT cDNAs and double-underlined nucleotides indicate recognition sequences for the restriction enzymes XhoI (recognition sequence: 5&#x2032;-CTCGAG) or EcoR1 (recognition sequence: 5&#x2032;-GAATTC). The primers used for PCR amplification were: Forward (sense): 5&#x2032;-CCCGGGGAATTCTGATGACGGAACAGGCCATCTC SEQ ID NO:14 C and Reverse (antisense): 5&#x2032;-CCCGGGCTCGAGTTAGAGTCACCTTCTTGAGCTC SEQ ID NO:15 The expression vector pGEX-4T-2 (Amersham Pharmacia Biotech) was used to generate huANT3 fusion proteins comprising an enzymatic polypeptide and an ANT polypeptide. This vector comprises a lacIq (repressor) gene a tac promoter operably linked to a glutathione S-transferase (GST) gene from Schistosoma japonicum. (Smith et al., 1988, Gene 67:31-40), the coding sequence of which has been modified to comprise a thrombin cleavage site-encoding nucleotide sequence immediately 5&#x2032; from a multiple cloning site. GST fusion proteins can be detected by Western blots with anti-GST or by using a calorimetric assay; the latter assay utilizes glutathione and 1-chloro-2-4-dinitrobenzene (CDNB) as substrates for GST and yields a yellow product detectable at 340 nm (Habig et al., 1974, J. Biol. Chem. 249:7130-7139). GST fusion proteins produced from expression constructs derived from this expression vector can be purified by, e.g., glutathione affinity chromatography, and the desired polypeptide released from the fusion product by thrombin. Thus, this expression vector provides for the rapid purification of fusion proteins, and release of proteins with relatively few extraneous N-terminal amino acids, although the resulting recombinantly produced protein contains two additional amino acids at the amino terminus (Gly-Ser). The tac promoter may be induced by the addition to cultured cells of, e.g., 1-5 mM isopropyl-beta-D-thiogalactopyranoside (IPTG; Fluka, Milwaukee, Wis.) and provides for high-level expression. Plasmid pGEX-4T-2 was prepared by digestion with the restriction endonucleases EcoR1 and Asp718 according to the manufacturer's instructions and subjected to horizontal agarose gel electrophoresis and band extraction using the UltraClean GelSpin kit (Mo Bio Laboratories). Restricted ANT cDNAs were ligated with the restricted expression vector DNA as described in the preceding Example. Single colonies were selected for grown in 3-5 ml of LB broth containing 50 ug/ml ampicillin (Roche Molecular Biochemicals), and plasmid DNA was isolated from the bacterial cultures using the WIZARD&#x2122; Plus Series 9600 Miniprep Reagents System (Promega). To confirm their authenticity, the recombinant huANT nucleotide sequences present in the pGEX deriavtive plasmid were determined as described in the preceding Example using the previously described oligonucleotide primers and 5&#x2032; and 3&#x2032; PGEX Sequencing Primers (Amersham Pharmacia Biotech). The resultant GST-huANT3 expression construct was named pMK3C-GST-huANT3 (also referred to herein as pMK3C). Plasmid pMK3C has been deposited at the American Type Culture Collection (ATCC; Manassas, Va.) on Nov. 3, 1998, and given the accession number ATCC 98973. Thrombin treated recombinant huANT3 protein produced from the pMK3C-GST-huANT3 expression construct includes several extraneous N-terminal amino acids, i.e., Gly-Ser-Pro-Gly-Ile-Leu-Met, where &#x201c;Met&#x201d; indicates the amino acid encoded by the translation initiation codon of huANT3. There is, however, no evidence that the extraneous six amino terminal amino acids have any effect on the resultant recombinant huANT3 protein. In order to confirm expression of the GST-huANT3 fusion protein, the following experiments were carried out. Eight independently isolated pMK3C-GST-huANT3 transformants and one control (vector-transformed) isolate were grown overnight in LB-ampicillin and then diluted 1:20 in 2 ml of fresh media. After 3 hours of growth at 37&#xb0; C., IPTG was added to a final concentration of 0.1 mM. Cell growth was continued for 2 hours, after which 1.5 of cells were tranferred to microfuge tubes, pelleted, resuspended in 300 uL of cold PBS containing 1% Triton X-100, and sonicated twice for 8 seconds. The sonicates were spun for 5 min. at 4&#xb0; C., the supernatant was transferred to fresh microfuge tubes and 50 uL of glutathione-agarose beads (Sigma) were added to produce a 50% slurry. After a 5 min. incubation at ambient temperature, the beads were spun and washed with 1 ml of PBS three times. The washed pellet was resuspended in SDS spl buffer (62.5 mM Tris, pH 6.8, 2% SDS, 10% glycerol, 5% beta-mercaptoethanol and sufficient bromophenol blue to provide visible coloration), and 30 uL of each preparation (equivalent to 15 uL of culture) was subjected to SDS-PAGE. The gel was stained using a Colloidal Coomassie (G-250) Staining Kit (Novex, San Diego, Calif.). A band of the predicted molecular weight of the GST-huANT3 fusion protein was readily apparent, with the same intensity, in each of the 8 preparations from pMK3C-GST-huANT3 transformants; this band was absent in the control preparation. B. Western Blot Analysis of Expression of huANT3 Fusion Proteins E. coli transformed with either (1) pMK3A-huANT3 (the pBAD/His-huANT3 expression construct) or (2) pMK3C-GST-huANT3 (the pGEX/GST-huANT3 expression construct) were lysed by the addition of lysozyme (100 &#x3bc;g/&#x3bc;l; Sigma) for 20 min at room temperature, followed by one freeze/thaw cycle. The negative control for the former transformant was a parallel culture of the transformed cells that had not undergone arabinose induction. The control for the latter transformant was a parallel culture of E. coli that had been transformed with the pGEX-4T-2 vector only. Total protein concentrations of each lysate were determined using the BCA Protein Assay kit (Pierce Chemical Co.), and equivalent amounts of total protein from each lysate preparation were mixed with equivalent volumes of 2&#xd7; Laemmli electrophoresis buffer and subjected to SDS-PAGE. The proteins were electrophoretically transferred to nitrocellulose, which was then contacted with antibodies against the appropriate epitope included in each vector (i.e., ANTI-XPRESS&#x2122; from Invitrogen for pMK3A-huANT3 and polyclonal goat anti-GST from Amersham Pharmacia Biotech, formerly Nycomed Amersham plc and Pharmacia &#x26; UpJohn Inc. for pMK3C-GST-huANT3). In a separate experiment, the bacterial lysate from the pMK3C-GST-huANT3 transformants was incubated with agarose-glutathione beads (Sigma) according to the manufacturer's instructions (see the preceding section and Smith et al., Expression and Purification of Glutathione S-Transferase Fusion Proteins, Unit 16.7 of Chapter 16 in: Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, N.Y., 1992, pages 16-28 to 16-31). The beads were suspended in Laemmli sample buffer and subjected to SDS-PAGE and Western analysis as described above. Although the yield of GST-huANT3 was low, perhaps because the fusion protein is inserted into the bacterial membrane, a sufficient amount of material was recovered for the experiment. The results (FIG. 5) show that a specific band of the predicted molecular weight (His-Tag+enterokinase site+antigenic site+huANT3=38 kDa) was observed in the arabinose induced E. coli that were transformed with the pBAD/his-huANT3 vector, but was absent in the non-induced control culture. Similarly, a band corresponding to GST-huANT3 was observed in the pMK3C-GST-huANT3-transformed E. coli, while only the unaltered GST band was observed in control E. coli transformed with the expression vector. Purification of the GST-huANT3 fusion protein using agarose-GSH beads produced a band of equivalent size to that observed in the crude lysate of pMK3C-GST-huANT-transformed bacteria. Example 3 Expression of ANT3 in Insect Cells A. Generation of Baculovirus Expression Constructs DNA comprising nucleotide sequences encoding huANT3 was amplified by PCR from a whole human brain cDNA library (Clontech) using the following primers. In the following representations of PCR primers, underlined nucleotides indicate sequences complementary to the 5&#x2032;-ends and 3&#x2032;-ends of the ANT cDNAs and double-underlined nucleotides indicate recognition sequences for the restriction enzymes BamHI (recognition sequence: 5&#x2032;-GGATCC) or EcoR1 (recognition sequence: 5&#x2032;-GAATTC). The PCR primers used were: Forward (sense): 5&#x2032;-TTATAGGATCCATGACGGAACAGGCCATCTCCTT SEQ ID NO:16 CGCCAAA and Reverse (antisense): 5&#x2032;-TTAAAGAATTCTTAGATCACCTTCTTGAGCTCGT SEQ ID NO:17 CGTACAG. PCR products were digested with the restriction endonucleases Bam-HI (New England Biolabs) and EcoRI (New England Biolabs) according to the manufacturer's recommendations. Subsequent purification was carried out by horizontal agarose gel electrophoresis and band extraction using the UltraClean GelSpin kit (Mo Bio Laboratories, Inc.). The Baculovirus transfer vector pBlueBacHis2 (B version, Invitrogen) comprises, in 5&#x2032; to 3&#x2032; orientation, a constitutive polyhedrin promotor operably linked to nucleotide sequences encoding (1) a translation initiation sequence, (2) an N-terminal polyhistidine sequence, (3) an XPRESS&#x2122; epitope tag for detection and purification of the recombinant protein and (4) an enterokinas cleavage site, followed by a multiple cloning site wherein cDNAs can be inserted. The transfer vector pBlueBacHis2 was prepared by digestion with the restriction endonucleases BamHI and EcoRI according to the manufacturer's recommendation, and the restricted DNA was subject to horizontal agarose gel electrophoresis and band extraction using the UltraClean GelSpin kit (Mo Bio Laboratories, Inc.). The restricted PCR products were ligated with the restricted expression vector DNA as in the preceding Examples. Competent E. coli TOP10F&#x2032; cells (Invitrogen) were transformed with the ligation recation following the manufacturer's instructions. Single colonies were selected for growth in 3-5 ml of LB broth containing 50 ug/ml ampicillin. Plasmid DNA was isolated from the bacterial cultures using the WIZARD&#x2122;Plus Series 9600 Miniprep Reagents System (Promega). The recombinant ANT gene sequences were determined and their authenticities confirmed (SEQ ID NOS:1, 2 and 3 correspond to human ANTs 1, 2 and 3, respectively) by DNA sequencing using the Prism Ready Dye Terminator Cycle Sequencing Kit (Perkin-Elmer, Catalog #402080) and the following primers: Polyhedrin Forward Sequencing Priming Site, 5&#x2032;-AAATGATAACCATCTCGC (SEQ ID NO:18); Baculovirus Reverse Sequencing Priming Site, 5&#x2032;-ACTTCAAGGAGAATTTCC(SEQ ID NO:19); primers internal to the ANT 3 coding sequence (sense strand), 5&#x2032;-ACTTCGCCTTCACGGATA (SEQ BD NO:20); and 5&#x2032;-TACGGCCAAGGGCATTCT (SEQ ID NO:21); primers internal to the ANT 3 coding sequence (antisense strand), 5&#x2032;-TGAAGCGGAAGTTCCTAT (SEQ D NO:22); and 5&#x2032;-ATGCCGGTTCCCGTACGA (SEQ ID NO:23). Sequence data were analysed using the SEQUENCE NAVIGATOR&#x2122; analysis software package (Perkin-Elmer). An isolated plasmid having the correct sequence was named pMK4A-huANT3. Although pMK4A-huANT3 contains authentic huANT3-encoding sequences, the ANT3 reading frame is not synchronous with the reading frame of the His-Tag/XPRESS&#x2122; epitope of the expression vector. Accordingly, pMK4A-huANT3 is not expected to produce recombinant ANT protein, although cells harboring it may be used as controls. In order to generate an in-frame derivative of pMK4A-huANT3, the plasmid was mutagenized using the QUIK-CHANGE&#x2122; Site-Directed Mutagenesis Kit (Stratagene) as in Example 1, except that the mutagenic oligonucleotide primers used were 5&#x2032;-GGCCTGTTCCGTCATCTTATCGTCATCGTCG (SEQ ID NO:24; the underlined sequence is the reverse complement of the 5&#x2032; end of the huANT3 reading frame), and 5&#x2032;-CGACGATGACGATAAGATGACGGAACAGGCC (SEQ ID NO:25; the underlined sequence corresponds to the 5&#x2032; end of the huANT3 reading frame). Several transformants were isolated, and plasmid DNA purified therefrom. The nucleotide sequences of the plasmid DNAs were determined and one having the &#x201c;correct&#x201d; sequence was identified and named pMK4B-huANT3. The baculovirus expression plasmids encoding human ANT3 are referred to as &#x201c;pMK4A (baculovirus shuttle, out-of-frame hu ANT3) or &#x201c;pMK4A&#x201d;; and &#x201c;pMK4B (baculovirus shuttle, in-frame hu ANT3)&#x201d; or &#x201c;pMK4B&#x201d;. Plasmid pM4B has been deposited at the American Type Culture Collection (ATCC; Manassas, Va.) on Nov. 3, 1998, and given the accession number ATCC 98972. In order to insert sequences encoding the huANT3 protein (and assoicated regulatory sequences) into the baculovirus genome, insect cells (MAXBAC&#x2122; Spodoptera frugiperda Sf9 cells, Invitrogen, Carlsbad, Calif.; or Trichoplusia ni cells, PharMingen, San Diego, Calif.) were co-transfected with the baculoviral transfer construct pMK4B-huANT3 and linear baculoviral (Autographa californica nuclear polyhedrosis virus, AcMNPV) DNA engineered to contain a promoterless 3&#x2032; fragment of the lacZ gene (BAC-N-BLUE&#x2122;, Invitrogen) using the BAC-N-BLUE&#x2122; Transfection Kit (Invitrogen) following the manufacturer's instructions. Recombinant baculovirus plaques express functional beta-galactosidase and were identified as blue plaques in the presence of X-gal (5-bromo-4-chloro-3-indoyl-beta-D-glactosidase). These recombinant viruses are expression constructs that express human ANT3 polypeptide in insect cells, as shown by the following experiments. B. Western Blot Analysis of Baculovirus Expression Systems High titer viral stock was produced, and recombinant protein was expressed in infected Sf9 (Invitrogen, Carlsbad, Calif.) or T ni (PharMingen, San Diego, Calif.) cells according to the manufacturer's instructions (see also Piwnica-Worms, Expression of Proteins in Insect Cells Using Baculovirus Vectors, Section II of Chapter 16 in: Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, N.Y., 1992, pages 16-32 to 16-48; Kitts, Chapter 7 in: Baculovirus Expression Protocols, Methods in Molecular Biology, Vol. 39, C. R. Richardson, Ed., Humana Press, Totawa, N.J., 1995, pages 129-142). Transfected Sf9 cells were pelleted by centrifugation and lysed by adding 100 &#x3bc;l of MSB buffer (210 mM mannitol (Sigma), 70 mM sucrose (Fluka), 50 mM Tris-HCl, pH 7.4, 10 mM EDTA) and performing three freeze-thaw cycles. A total cellular fraction, a cytosolic fraction, a submitochondrial partical fraction, a mitochondrial fraction and a plasma membrane fraction were prepared as follows. The cell lysate was centrifuged at 600 g for 10 minutes at 4&#xb0; C. to prepare a plasma membrane pellet. The supernatant was removed and set aside. The plasma membrane pellet was washed with 100 ul of MSB, centrifuged at 600 g for 10 minutes at 4&#xb0; C., and used for the analysis. The supernatant was removed, combined with the first supernatant and mixed. Half of the supernatant was used to prepare a mitochondrial fraction and a cytosolic fraction by centrifugation at 14,000 g for 15 minutes at 4&#xb0; C.; the pellet represents the mitochondrial fraction and the supernatant represents the cytosol. The other half of the supernatant was centrifuged at 14,000 g for 15 minutes at 4&#xb0; C. to produce a mitochondria-containing pellet that was resuspended in MSB, incubated with 0.25 mg/ml digitonin (Roche Molecular Biochemicals, formerly Boehringer Mannheim, Indianapolis, Ind.) for 2 min and sonicated for 3 min at 50% duty cycle in a cup-horn sonicator to produce submitochondrial particles (SMPs). (See Example 13 for details regarding mitochondrial preparation from transfected T. ni cells.) The protein content for each fraction was determined using the BCA Protein Assay kit (Pierce Chemical Co.), and 8 ug of total protein were loaded per lane onto an SDS polyacrylamide gel, electrophoresed and transferred to a HYBOND&#x2122; ECL Nitrocellulose Membrane (Amersham Life Science). Fusion proteins were detected in a western blot using ANTI-XPRESS&#x2122; Antibody (Invitrogen, Catalog #R910-25) and horseradish peroxidase-conjugated anti-mouse secondary antibody (Amersham Life Science) following the manufacturers' instructions. The results of the Western analysis are shown in FIG. 6. Recombinant GST-huANT3 fusion protein (molecular weight 36.6 kD) was detected in total cells, mitochondria, submitochondrial particles and the plasma membrane. The signal was most intense in mitochondria and submitochondrial particles, whereas no band was detectable in the cytosolic fraction. These data suggest that the human recombinant huANT3 fusion protein integrated into the mitochondrial membranes much more efficiently than into the plasma membranes. Furthermore, all of the recombinant protein integrated into membranes since no signal was detected in the cytosolic fraction. The final lane of the autoradiogram shows His-tagged huANT3 isolated from cell lysates using magnetic agarose beads coupled to Ni according to the manufacturers instructions (Qiagen; Hilden, Germany). Thus, as in E. coli, huANT3 is expressed in the baculovirus/Sf9 system. Furthermore, recombinantly produced 6xHis- and epitope-tagged huANT3 fusion protein is appropriately localized to the mitochondria in Sf9 cells despite the presence of over 35 extraneous N-terminal amino acids, and can be isolated from cellular fractions by means that take advantage of the His-Tag moiety's affinity for metals such as, e.g., nickel. Example 4 Expression of ANT3 in Yeast A. Expression Constructs and Host Cells Human ANT3 cDNA was amplified by PCR as in Example 1 but using the following primers. In the following representations of PCR primers, underlined nucleotides indicate sequences complementary to the 5&#x2032;-ends and 3&#x2032;-ends of the ANT cDNAs and double-underlined nucleotides indicate recognition sequences for the restriction enzymes XhoI (recognition sequence: 5&#x2032;-CTCGAG) or Asp718 (recognition sequence: 5&#x2032;-GGTACC). The primers used for PCR amplification were: Forward (sense; SEQ ID NO:28): 5&#x2032;-TTAATGGGTACCATGACGGAACAGGCCATCTCCTTCGCCAAA, and Reverse (antisense; SEQ ID NO:29): 5&#x2032;-TTATACTCGAGTTAGATCACCTTCTTGAGCTCGTCGTACAGG. PCR products, and expression vector DNAs, were digested with the restriction endonucleases XhoI and Asp718 (both enzymes from Roche Molecular Biochemicals) according to the manufacturer's recommendations using manufacturer-supplied reaction buffers. The expression vector pYES2 (Invitrogen) was used. This vector contains a multiple cloning site located immediately downstream from an inducible GAL1 promoter, as well as the 2u origin of replication and the S. cerevisiae URA3 gene for high-copy maintenance and selection in ura3 yeast cells, respectively. The restricted DNAs were purified by horizontal agarose gel electrophoresis and band extraction using the UltraClean GelSpin kit (Mo Bio Laboratories), ligated to each other, and used to transform E. coli cells, as in the preceding Examples. Plasmid DNA was isolated from several transformants, and the nucleotide sequence of the insert DNA was determined and confirmed to be that of huANT3. One confirmed plasmid was chosen to be used for further study and was designated pMK5A (huANT3). A second yeast huANT3 expression vector, pMK5B, was constructed as follows. Plasmids pMK5A and pYESTrp2 (Invitrogen) were digested with restriction enzymes BglI and PvuII (both from New England Biolabs) and gel purified, ligated and used to transform E. coli as above. The expression vector pYES2Trp is similar to pYES2 but comprises a TRP1 selectable marker. Plasmid DNA was isolated from several transformants and restriction mapped to confirm the structure of the expected expression construct. One confirmed plasmid was chosen to be used for further study and was designated pMK5B (huANT3). A third yeast huANT3 expression vector, pMK5C, was constructed using the expression vector pYPGE2, which comprises a TRP1 selectable marker and the strong PGK promoter upstream from a multiple cloning site (Brunelli and Pall, 1993 Yeast 9:1299-1308). Plasmid pYPGE2 DNA was digested with nol and Asp718, gel-purified and ligated with the XhoI- and Asp718-restricted huANT3 PCR product of Example 1. The ligation mixture was used to transform E. coli, and plasmid DNA was isolated from several transformants and restriction mapped to confirm the structure of the expected expression construct. One confirmed plasmid was chosen to be used for further study and was designated pMK5C (huANT3). In order to generate yeast expression systems, the S. cerevisiae strain INVSc1 (MAT&#x3b1;, his3&#x394;1, leu2, trp1-289, ura3-52) was transformed with purified pMK5A, pMK5B and pMK5C DNAs using the S.c. EASYCOMP&#x2122; Transformation Kit (Invitrogen). A second S. cerevisiae strain, J&#x394;1&#x394;3 (MAT&#x3b1;, ade2-I, leu2-3, leu2-112, his3-11, his3-15, trp1-1, ura3-1, can1-00, AAC1::LEU2, AAC2::HIS3, AAC3::URA3) was also transformed with the expression constructs. The AAC genes encode the three isoforms of the mitochondrial ADP/ATP translocator in S. cerevisiae and are interrupted in strain J&#x394;1&#x394;3 (Giraud et al., J. Mol. Biol. 281:409-418 (1998)). It is thus expected that transformants of J&#x394;1&#x394;3, which are incapable of expressing endogenous ANT (AAC) proteins, will only express the human ANT protein encoded by the expression construct with which they have been transformed. B. Northern Blot Analyses of Yeast Expression Systems In order to examine levels of huANT3 mRNA production in strain J&#x394;1&#x394;3, Northern analyses of cells transformed with pMK5B and pMK5C were performed according to methods known in the art. In brief, transformed cells and control (untransformed) cells grown to mid-log phase, harvested and lysed. RNA was extracted from the lysates, electrophoresed and transferred to a nitrocellulose filter (see Treco, Preparation of Yeast RNA, Unit 13.12 of Chapter 13 in Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, N.Y. (1992), 13:44-46 and Seldon, Analysis of RNA by Northern Hybridization, Unit 4.9 of Chapter 4, Id, 4:23-25). The XhoI- and Asp718-restricted huANT3 PCR product of Example 1 was radiolabelled and used as a probe, and an RNA preparation from human spleen tissue was used as a positive control. The results (FIG. 10) demonstrate the appropriately-sized ANT3-specific RNA is produced in human spleen and in yeast cells transformed with either expression vector, but not in untransformed yeast cells. The pYPGE2-derived expression construct pMK5C, which directs ANT3 expression from the PGK promoter, clearly results in more ANT3 RNA than the pYES2Trp-derived construct pMK5B, in which ANT3 expression is driven by the GAL1 promoter. In either case, however, significant levels of huANT3-specific RNA were produced in a yeast background that lacks any endogenous adenosine nucleotide translocator proteins. C. Western Analyses of Yeast Expression Systems 1. Production of antibody to huANT3 As the huANT3 produced from the yeast expression constructs lacks an epitope tag, it was necessary to produce an antibody to huANT3 in order to evaluate recombinant production of the protein. A Nonspecific (antipeptide) antibody specific to huANT3 was prepared as follows. A synthetic polypeptide corresponding to a portion of huANT3 located near the carboxy terminus and predicted to have high antigenicity according to the Jameson-Wolf Index (Wolf et al., Comput. Appl. Biosci. 4:187-191 (1988)) was synthesized using known means by Alpha Diagnostic International (San Antonio, Tex.) and determined to be at least about 70% pure, preferably at least about 90% pure, by HPLC and MS analyses. The sequence of the synthetic polypeptide (SEQ ID NO:30) is: Cys-Trp-Arg-Lys-Ile-Phe-Arg-Asp-Glu-Gly-Gly-Lys- Ala-Phe-Phe The synthetic polypeptide was conjugated to a carrier molecule, keyhole limpet hemocyanin (KLH), using MSB (m-maleimidobenzoyl-N-hydroxysuccinimide ester; Pierce Chemical Co., Rockford, Ill.), and the conjugated material was used to immunize several rabbits, according to known means (Collawn and Paterson, Units 11.14 and 11.15 in Chapter 11 in: Short Protocols in Molecular Biolog, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, N.Y. (1992) 11:37-41. The rabbits were or are bled at 0 (preimmune, 2 mL), 7, 9, 11, 13 (15 mL for each bleed) or 15 weeks (50 mL) post-inoculation. Sodium azide (0.1%) was or is added to the bleeds as preservative. 2. Western Analyses Western analyses of yeast expression systems are performed essentially as described in the preceding Examples, except that different methods are used to prepare protein preparations from yeast cells as opposed to bacterial or insect cells. Such methods of isolating proteins from yeast are known in the art (see, for example, Dunn and Wobbe, Preparation of Protein Extracts from Yeast, Unit 13.13 of Chapter 13 in Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds. John Wiley &#x26; Sons, New York, N.Y. (1992), 13:46-50). The intracellular distribution of huANT3 in, e.g., membrane or mitochondrial fractions, is determined as in the preceding Examples. Example 5 Expression of ANT3 in Mammalian Cells The preceding Examples describe a variety of means by which ANT and ANT fusion proteins can be recombinantly produced in various systems. Although such ANT proteins can be used in a variety of assays (see infra), it may be desirable to isolate large amounts of the native ANT protein from mammalian cells. In particular, as described in this Example, it may be desirable to produce recombinant viral particles in which ANT proteins are displayed in the viral envelope. Such ANT-displaying viral particles are expected to be very stable and useful in a variety of assays including, for example, those in which compounds binding to ANT proteins are screened and identified. Another useful outcome of mammalian expression systems is the generation and isolation of human mitochondria in which a particular ANT isoform is over-represented in order to determine the specific biological role(s) of such isoforms. For example, ANT3 is apparently ubiquitously expressed in human tissues, whereas ANT1 is primarily expressed in heart and skeletal muscle (Stepien et al., 1992, J. Biol. Chem. 267:14592-14597). Directed overexpression of huANT1 in cultured heart or muscle cells is expected to result in mitochondria that contain mostly the ANT1 isoform. Such &#x201c;ANT isoform-enriched&#x201d; mitochondria can be isolated and tested for various mitochondrial functions. Constructs for expressing ANT proteins in mammalian cells are prepared in a stepwise process. First, expression cassettes that comprise a promoter (and associated regulatory sequences) operably linked to nucleotide sequences encoding an ANT protein are constructed in bacterial plasmid-based systems; these expression cassette-comprising constructs are evaluated and optimized for their ANT-producing ability in mammalian cells that are transiently transfected therewith. Second, the ANT expression cassettes are transferred to viral systems that produce recombinant proteins during lytic growth of the virus (e.g., SV40, BPV, EBV, adenovirus; see below) or from a virus that can stably integrate into and transduce a mammalian cellular genome (e.g., a retroviral expression construct). A. Transient expression With regards to the first step, commercially available &#x201c;shuttle&#x201d; (i.e., capable of replicaton in both E. coli and mammalian cells) vectors that comprise promoters that function in mammalian cells and can be operably linked to an ANT-encoding sequence include, but are not limited to, SV40 late promoter expression vectors (e.g., pSVL, Pharmacia), glucocorticoid-inducible promoter expression vectors (e.g., pMSG, Pharmacia), Rous sarcoma enhancer-promoter expression vectors (e.g., pRc/RSV, Invitrogen) and CMV early promoter expression vectors, including deriavtives thereof having selectable markers to agents such as Neomycin, Hygromycin or ZEOCIN&#x2122; (e.g., pRc/CMV2, pCDM8, pcDNA1.1, pcDNA1.1/Amp, pcDNA3.1, pcDNA3.1/Zeo and pcDNA3.1/Hygro, Invitrogen) In general, preferred shuttle vectors for ANT genes are those having selectable markers (for ease of isolation and maintenance of transformed cells) and inducible, and thus regulatable, promoters (as overexpression of ANT genes may have toxic effects). Methods for transfecting mamallian cells are known in the art (see, Kingston et al., &#x201c;Transfection of DNA into Eukaryotic Cells,&#x201d; Section I of Chapter 9 in: Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, N.Y., 1992, pages 9-3 to 9-16). A control plasmid, such as pCH110 (Pharmacia), may be cotransfected with the ANT expression construct being examined so that levels of ANT can be normalized to a gene product expressed from the control plasmid. Western analyses of mammalian expression systems are performed essentially as described in the preceding Examples, except that different methods are used to prepare protein preparations from mamallian cells as opposed to bacterial, insect or yeast cells. Such methods of isolating proteins from yeast are known in the art (see, for example, Kingston and Sheen, Unit 9.6A and Brasier, Unit 9.6B of Chapter 9 in: Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, N.Y., 1992, pages 9-17 to 9-23). Preferred expression cassettes, consisting essentially of a promoter and associated regulatory sequences operably linked to an ANT gene of interest, are identified by the ability of cells transiently transformed with a vector comprising a given ANT expression cassette to express high levels of ANT protein when induced to do so; these expression cassettes are incorporated into viral expression vectors. B. Viral Expression Nucleic acids, preferably DNA, comprising preferred expression cassettes are isolated from the transient expression constructs in which they were prepared, characterized and optimized (see preceding section). A preferred method of isolating such expression cassettes is by amplification by PCR, although other methods (e.g., digestion with appropriate restriction enzymes) can be used. Preferred expression cassettes are introduced into viral expression vectors, preferably retroviral expression vectors, in the following manner. A DNA molecule comprising a preferred expression cassette is introduced into a retroviral transfer vector by ligation (see preceding Examples). Two types of retroviral transfer vectors are known in the art: replication-incompetent and replication-competent. Replication-incompetent vectors lack viral genes necessary to produce infectious particles but retain cis-acting viral sequences necessary for viral transmission. Such cis-acting sequences include the &#x3a8; packaging sequence, signals for reverse transcription and integration, and viral promoter, enhancer, polyadenylation and other regulatory sequences. Replication-competent vectors retain all these elements as well as genes encoding virion structural proteins (typically, those encoded by genes designated gag, pol and env) and can thus form infectious particles in a variety of cell lines. In contrast, these functions are supplied in trans to replication-incompetent vectors in a packaging cell line, i.e, a cell line that produces mRNAs encoding gag, pol and env genes but lacking the &#x3a8; packaging sequence. See, generally, Cepko, Unit 9.10 of Chapter 9 in: Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, N.Y., 1992, pages 9-30 to 9-35. A retroviral construct comprising an ANT expression cassette produces RNA molecules comprising the cassette sequences and the &#x3a8; packaging sequence. These RNA molecules correspond to viral genomes that are encapsidated by viral structural proteins in an appropriate cell line (by &#x201c;appropriate&#x201d; it is meant that, for example, a packaging cell line must be used for constructs based on replication-incompetent retroviral vectors). Infectious viral particles are then produced, and released into the culture supernatant, by budding from the cellular membrane. The infectious particles, which comprise a viral RNA genome that includes the ANT expression cassette, are prepared and concentrated according to known methods. It may be desirable to monitor undesirable helper virus, i.e., viral particles which do not comprise an ANT expression cassette. See, generally, Cepko, Units 9.11, 9.12 and 9.13 of Chapter 9 in: Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, N.Y., 1992, pages 9-36 to 945. Viral particles comprising an ANT expression cassette are used to infect in vitro (e.g., cultured cells) or in vivo (e.g., cells of a rodent, or of an avian species, which are part of a whole animal). Tissue explants or cultured embryos may also be infected according to methods known in the art. See, generally, Cepko, Unit 9.14 of Chapter 9 in: Short Protocols in Molecular Biology, 2nd Ed., Asubel et al., eds., John Wiley &#x26; Sons, New York, N.Y., 1992, pages 9-45 to 9-48. Regardless of the type of cell used, production of ANT protein is directed by the recombinant viral genome. In a preferred embodiment, recombinantly produced ANT proteins are inserted into the cell membrane of cultured cells. Because the retroviral expression construct produces viral particles by budding of the cell membrane, the resultant viral particles delivered to the culture supernatant have ANT protein incorporated into their capsules, preferably on the surface of the particles. Such ANT-displaying viral particles are expected to provide a stable format for ANT proteins and to thus be useful in assays using ANT proteins, either directly or as a source material from which ANT can be further purified. If it is desired to minimize the amount of ANT protein inserted into mitochondrial membranes, &#x3c1;0 cells, which have been treated in such a manner as to be nearly or completely devoid of mitochondria, are used as host cells. C. ANT Antisense Constructs Antisense versions of the preceding transient and viral ANT expression constructs are prepared by exchanging the antisense (non-encoding) strand for a sense (ANT protein encoding) strand in a construct. Such ANT antisense constructs are useful as research reagents, i.e., to reduce levels of expression of one or more isoforms in a cell transformed or infected with such a construct in order to determine the effects of such treatment on cellular physiology. ANT antisense constructs are also useful as gene therapy agents that interfere with the translation of one or more isoforms of ANT. Example 6 Synthesis and Properties of Representative ATR Derivatives A number of atractyloside (ATR) derivatives were prepared for use as ligands for adenine nucleotide translocators (ANTs) in the context of high-throughput screening assays. These compounds bind with high affinity (i.e., in the nM range) to ANT and are thus useful for screening libraries of chemical compounds for molecules having high specificity for ANT (regardless of isoform) The structure of ATR is set forth below as compound (1). Compounds (3) and (4) represent novel fluorescent derivatives of ATR, while compound (2) is an ATR derivative which permits introduction of the 125I under mild conditions. Purification Compounds 2, 3 and 4 were purified by silica gel chromatography using CH2Cl2/MeOH/AcOH (75:25:1) as the eluting solution. Detection was achieved by staining with a 0.5% solution of vanillin in H3PO4/H2O (1/1). Further purification was accomplished by reversed-phase HPLC using a Microsorb C8 column (250&#xd7;10 mm). The column was eluted at a flow rate of 2.0 mL/min with a linear gradient of methanol/acetic acid/i M ammonium acetate 98:1:1 (&#x201c;Solvent B&#x201d;) and H2O/acetic acid/1 M ammonium acetate aqueous solution 98:1:1 (&#x201c;Solvent A&#x201d;). The effluent was monitored for absorbance at 254 nm. Compound-containing fractions were pooled, evaporated, and repeatedly co-evaporated with added methanol (3&#xd7;5 mL), Synthesis of Compound 2 Atractyloside 1 (0.10 mmol) was dried by repeated evaporation of added pyridine (3&#xd7;5 mL) and the resulting gummy residue dissolved in pyridine (5 mL). To the resulting solution, 0.20 mmol of toluenesulfonyl chloride was added. The reaction mixture was stirred at ambient temperature for 1.5 h. Then, another portion of toluenesulfonyl chloride (0.20 mmol) was added and the reaction left stirring an additional 1.5 h. 1 mL of methanol was added to the reaction mixture which was then stirred for 0.5 h, after which solvents were removed by evaporation. Residual pyridine was removed by evaporation of additional methanol (5&#xd7;10 mL). Silica gel chromatography followed by reversed-phase HPLC using a linear gradient of 50-80% of solvent B in solvent A for 30 min, resulted in the compound 2 eluting at 68% solvent B. Yield: 4.3 mg, 4.9%. ESI-MS (M&#x2014;H) found: 879, calc.: 879. Synthesis of Compounds 3 and 4 7-Diethylamino-2-oxo-2H-chromene-3-carboxylic acid or 0.20 mmol of 1-pyrenebutyric acid and 0.60 mmol of 1,1&#x2032;-carbonyldiimidazole in 1 mL of dimethylformamide were allowed to react for 15 min. To the activated carboxylic acid was added a solution of atractyloside 1 in H2O (4 mL) and the resulting reaction mixture was stirred at ambient temperature for 16 h. Evaporation left a gummy residue which was purified by silica gel chromatography followed by reversed-phase HPLC. Using a linear gradient of 10-80% of solvent B in solvent A for 50 min (for compound 3) or 50-100% of solvent B in solvent A for 50 min (for compound 4) resulted in compound 3 eluting at 75% B and compound 4 eluting at 82% B. Yields: compound 3, 3.1 mg, 8.0%; compound 4, 1.3 mg, 3.6%. ESI-MS (M&#x2212;H) compound 3 found: 968, calc.: 968; compound 4 found: 995, calc.: 995. Properties of Representative ATR Derivatives As summarized in Table 1 below, compounds 3 and 4 were found to be more advantageous in terms of fluorescence characteristics and sensitivity compared to the existing ATR derivatives Naphthoyl-ATR and MANT-ATR as reported by Boulay et al., Analytical Biochemistry 128:323-330,1983; Roux et al., Analytical Biochemistry 234:31-37,1996; and Lauquin et al., FEBS Letters 67:306-311,1976. TABLE 1 Excitation Emission Extinction Coefficient (M&#x2212;1) ATR Derivative (nm) (nm) (Predicted) Naphthoyl-ATR 300 405 6,200 MANT-ATR 350 460 5,800 Compound 4 341 391 17,420 Compound 3 417 470 46,400 Example 7 Synthesis of Representative ATR Derivative The further representative ATR derivative, compound 5, was prepared by the procedure set forth below. Synthesis of Compound 5 Dipotassium atractylate (0.10 mmol) was dissolved in 50% aq. ethanol (5 mL) and palladium on charcoal (10%, 17 mg) was added to the reaction mixture. After flushing the system with hydrogen, the reaction mixture was stirred under an atmosphere of hydrogen gas for 3 h. Removal of catalyst by filtration through Celite, washing with 50% aq. ethanol (101 mL), and evaporation of solvents afforded a white solid. Yield after thorough drying under high vacuum; 78.3 mg (97.3%). ESI-MS (N-2H+K) found: 765, calc.: 765. 1H-NMR analysis confirmed the absence of alkenic protons: DMSO-d6) &#x3b4; 0.88(d, 3H), 0.89(d, 3H), 1.02(d, 3H). Example 8 Synthesis of Representative Iodinated ATR Derivative Compound 2 of Example 6 may be used as intermediate for conjugation of variety of chemical moieties to yield further ATR derivatives. In this example, compound 2 is employed to introduce 125I under mild conditions to yield the following compound 6. Synthesis of Compound 6 Five &#x3bc;l of 0.2 M sodium phosphate (pH 5) was combined with 21 ul of Na125I (9.25 mCi) in its shipping container (specific activity, 2100 Ci/mmol; Amersham, Piscataway, N.J.). Ten ul (200 ug, 212 nmol) of compound 2 of Example 1 was added to the mixture. The pH was checked with litmus paper to confirm that it did not rise above pH 5. The mixture was allowed to stand at ambient temperature overnight (17.5 hours) to yield radiolabelled compound 6. (Non-radioactive iodinated ATR derivative, for use as a &#x201c;cold&#x201d; competitor in binding studies, may be prepared in the same manner using unlabeled iodine). The iodinated derivative was purified over a C18 analytical column (4&#xd7;6&#xd7;250 mm) (Phenomenex, Torrance, Calif.) using a 25%-55% acetonitrile gradient in running buffer (1% triethylammonium acetate, pH 4.5). A flow rate of 1 ml/min was used to run the gradient over 30 min. The desired product eluted at 25 min. ESI-MS: 835 (M&#x2212;H), 707 (m-2H&#x2014;I). Example 9 Synthesis of Representative ATR Derivatives Activation of carboxylic acids with carbonyl diimidazole and their reaction with ATR has been the method of choice for synthesis of various 6&#x2032;-O-acyl derivatives. The relatively low reactivity of the 6&#x2032;-hydroxyl of ATR and the presence of an allylic secondary hydroxyl in the aglycon as well as the sulfated glucose moiety, are all factors that have a negative impact on the efficiencies of these acylation reactions. Hence, yields are generally poor and the approach requires a large excess of acylating reagents. Two strategies for introduction of an amine functionality in the ATR system are described below that permit synthesis of a broader range of ATR derivatives. In the first strategy, as depicted by Scheme 1, displacement of the primary tosylate from compound 2 (Example 1) with azide followed by reduction yields the corresponding 6&#x2032;-amine (compound 7). Alternatively, the amine group can be introduced as part of a spacer, which permits introduction of more sterically demanding functional moieties. Thus, reacting the 6&#x2032;-O-succinoyl derivative (compound 8; see Brandolin et al., 1974 FEBS Lett. 46:149.) with a monoprotected diamine followed by deprotection affords compound 9 as illustrated by Reaction Scheme 2. The amine-containing ATR derivatives 7 and 9 may then be reacted with a variety of fluorophors and haptens bearing reactive isothiocyanate, N-hydroxysuccinimide ester and anhydride functionalities to yield stable ATR-derivatives having thiourea and amide linkages. Representative ATR derivatives that were prepared include ATR-lanthanide chelating agents (compounds 10, 11, 12, 13, 20 and 21) that have utility for time-resolved fluorescence detection of these compounds complexed to Eu3+. In addition, ATR was conjugated to cyanine (compounds 14 and 15) and fluorescein analogues (compounds 16 and 17) that are detectable by fluorescence with extremely high sensitivities. Coupling of biotin-NHS ester with the ATR derivatives of compounds 7 and 9 provided ATR-biotin conjugates (compounds 18 and 19) that can be detected with commercially available enzyme-avidin conjugates using colorimetric, fluorescent or chemiluminescent techniques. More specifically, a solution of compound 2 in DMF was treated with azide ion for 8 hours at 80&#xb0; C. to give the 6&#x2032;-azido-ATR, that was purified by silica gel chromatography using a CH2Cl2/CH3OH solvent system supplemented with 1% acetic acid. Staudinger-reduction using 1.5 equivalents of triphenylphosphine in a THT/H2O mixture for 4 hours at RT afforded the amine of compound 7, that was isolated after silica gel chromatographic purification. To accommodate more sterically demanding functional moieties, 6&#x2032;-O-succinoyl-ATR may be condensed with commercially available monoprotected diamines (Calbiochem-Novabiochem Corp, San Diego, Calif.) to produce ATR-mono-protected amine derivatives. Thus, EDC-mediated coupling of 6&#x2032;-O-succinoyl-ATR in DMF with 1.1 equivalents of mono-protected FMOC diamines yield the amide that was deprotected using piperidine or 1,8-diazabicyclo[5.4.0]undec-7-ene (DBU) in acetonitrile to furnish the ATR derivative of compound 9. The amines were purified by silca gel chromatography as described above. The ATR-amine derivatives of compounds 7 and 9 were coupled to a variety of fluorophors, chelates and haptens that contained amine-reactive functionalities, such as isothiocyanates, anhydrides and NHS esters in aqueous DMF to generate the ATR derivatives of compounds 10 through 19: These compounds were purified by a combination of silica gel chromatography and preparative reverse phase chromatography on a C-8 column using CH3OH(H2O gradient containing 0.1-1% acetic acid. Example 10 Synthesis of Representative ATR Derivatives and Intermediates Therefor General Procedure for Coupling Atractyloside or Dihydroatractyloside to Organic Acids Carboxylic acid (200 &#x3bc;mol) and 1,1&#x2032;-carbonyldiimidazole (700 mmol) were dissolved in DMF (2 mL) and stirred for 15 min. To the activated acid, a solution of atractyloside (ATR) (100 &#x3bc;mol, 85 mg) or dihydroatractyloside (100 &#x3bc;mol) in DMF:H2O (1:2, 6 mL) was added (in 1 mL portions over ca 30 sec). The reaction mixture was stirred at room temperature for 60 min, after which solvents were removed by rotary evaporation on a water bath in which the temperature was kept below 40&#xb0; C. The residue was stripped of traces of DMF by repeated evaporation of added EtOH:H2O (1:1, 3&#xd7;20 mL). The residue was then taken up in MeOH: H2O (1:1, 10 mL), sonicated if necessary, and filtered through a 0.45 &#x3bc;m filter. Evaporation, re-dissolution in &#x2dc;1.3 mL buffer, and purification by reverse phase HPLC furnished the desired 6&#x2032;-O-acyl-ATR in yields ranging from 5-15%. Reverse Phase HPLC Conditions for Purification of Atractyloside Derivatives Purification by reverse phase HPLC (RP-HPLC) was performed in a 10&#xd7;250 mm C-8 column, using a gradient of MeOH:AcOH:1 M NH4OAc (buffer B, 98:1:1) in H2O:AcOH: 1 M NH4OAc (buffer A, 98:1:1). Typically, a gradient of B in A from 50-80% over 30 min was employed. For more lipophilic derivatives, the gradient was from 60-90% or 70-100% B over the same time period. Synthesis of Dihydroatractyloside Atractyloside, dipotassium salt (254 mg, 0.3 mmole) in 15 ml of EtOH/H2O (1:1) was hydrogenated under atmospheric pressure for 3 hours using 51 mg of activated palladium/carbon (10%) as catalyst. The catalyst was removed by filtration through celite and the celite bed was washed with 15 ml of EtOH/H2O. The filtrates were concentrated by rotary evaporation, followed by drying under high vacuum overnight to provide the product as white solid (236 mg), that was pure by NMR. 6&#x2032;-Tosyl-Atractyloside Atractyloside.3H2O (255 mg. 0.3 mmole) was dried by co-evaporation with dry pyridine (3&#xd7;5 ml) and then was kept under high vacuum for 16 hrs. The dried atractyloside was dissolved in 15 ml of dry pyridine and 114 mg of tosyl chloride (0.6 mmole) was added. The reaction was stirred for 2 hrs at 23&#xb0; C. and then an additional 114 mg of tosyl chloride was added and the reaction was allowed to continue for an additional 1.5 hrs. Methanol (1 ml) was added to the reaction mixture to scavenge excess tosyl chloride and the mixture was stirred for several minutes. The mixture was evaporated to dryness, and residual pyridine was removed by evaporation of added methanol (2&#xd7;30 ml). The crude product was partially purified by silica gel flash chromatography (CH2Cl2/MeOH, 3:1 with 1% AcOH). The product containing fractions were dissolved in 3 ml of H2O:MeOH:1M NH4OAc:HOAc (49:49:1:1) and purified by RP-HPLC using a 50%-80% gradient of MeOH:AcOH:1 M NH4OAc (buffer B, 98:1:1) in H2O:AcOH:1 M N OAc (buffer A, 98:1:1). Product containing fractions were pooled, evaporated, and subjected to additional co-evaporation of 3&#xd7;10 ml of MeOH. Tosyl-atractyloside was obtained as a glassy material weighing 60 mg. General Procedure for Lactoperoxidase-Catalyzed Iodination of 4-Hydroxyphenyl Derivatives To a solution of 4&#x2032;-hydroxy-biphenyl-4-carboxylic acid atractylosid-6&#x2032;-yl ester (1.0 mg, 1.0 mmol) in H2O (120 &#x3bc;L) were added aqueous solutions of lactoperoxidase (50 &#x3bc;L, 200 IU/mL), NaI (10 &#x3bc;L, 100 mM) and H2O2 (20 &#x3bc;L, 100 mM). The reaction was left at room temperature for 1 h after which it was frozen at &#x2212;20&#xb0; C. The next day (&#x2dc;20 hours later) the reaction mixture was thawed and subjected to RP-HPLC. Three major peaks were eluted and electrospray ionization-mass spectrometic (ESI-MS) analysis confirmed their identity as unreacted starting material, and monoiodinated, and diiodinated atractyloside derivatives. These conditions can be modified to drive the reaction completely to the mono and di-iodinated forms with additional aliquots of NaI and/or enzyme. General for Synthesis of Iodophenols The iodination procedure described in Acta Chem, Scand. 12, 188 (1958) was used for mono-iodination of 3-(4-hydroxyphenyl)propionic acid, 4&#x2032;-hydroxy-4-biphenylcarboxylic acid and 4-hydroxybenzoic acid. This method is also applicable for the mono-iodination of 3-(4-methoxyphenyl)-propionic acid and di-iodination of 3-(3-iodo-4-hydroxyphenyl)propionic acid. Thus a solution of KI (1.99 gm, 12 mmole) and iodine (1.22 gm, 4.8 mmole) in 20 ml of H2O was added in a dropwise fashion to a solution of 3-(4-hydroxyphenyl)propionic acid (0.83 gm, 5 mmole) in 100 ml of concentrated aqueous ammonia solution over 20 min. The reaction mixture was stirred for an additional 40 min and then subjected to vacuum to remove the ammonia. The mixture was dried further by rotary evaporation to afford an oily residue. The crude material was partitioned between 2M HCl (50 ml) and ether (2&#xd7;50 ml) and the ether layers were combined and concentrated to give a yellowish solid residue. Flash silica gel chromatography using 95:5 CH2Cl2/MeOH as eluant, concentration of product containing fractions and recrystallization in 1:1 benzene hexane afforded 790 mg of 3-iodo-4-hydroxyphenylpropionic acid. 3-(3,5-Diiodo-4-hydroxyphenyl)propionic acid was prepared in similar fashion using 5.2 equivalents of KI and 2.1 equivalents of 12. Following crystallization from toluene, the di-iodo derivative was obtained in 77% yield. 5-iodo-6-hydroxy-2-naphthoic Acid Mono-iodination of 6-hydroxy-2-naphthoic acid and 4-hydroxybenzoic acid was carried out following the procedure of Edgar and Falling (J. Org. Chem. 55, 5287, 1990). Thus, 0.75 gm of (4.34 mmol) was dissolved in 19 ml of MeOH and 1.04 gm of NaI (and 0.27 gm of NaOH was added. The solution was cooled to 0&#x25a1;C. and aqueous NaOCl (4% solution, 12.9 ml) was added dropwise over 75 min. The resulting mixture was stirred for 1 hr at 0&#xb0; C. and then treated with 7 ml of 10% aqueous sodium metabisulfite. The mixture was adjusted to pH 7 using 5% HCl and extracted with 40 ml of ether. The organic layer was washed with brine and dried over MgSO4. The solution was concentrated to an off-white solid, that was recrystallized from toluene/CH3OH to provide 0.42 gm of 5-iodo-6-hydroxy-2-naphthoic acid. Reaction of 6&#x2032;-Tostylatractyloside with 1,2-ethylenediamine 6&#x2032;-Tosyl-Atractyloside (25 mg) was dissolved in 2 ml of 1,2-ethylenediamine and the mixture was stirred at 23&#xb0; C. overnight. The 1,2-ethylenediamine was removed in vacuo, the residue was dissolved in MeOH/H2O (2:1) and 10.6 mg of the product was isolated in by RP-HPLC using the conditions described above. Proton nmr and mass spectra indicate the loss of the isovaleryl group. Reaction of N-(6-Deoxy-apo-atractylosyl)-ethanediamine with Bolton Hunter Reagent N-(6-Deoxy-apo-atractylosyl)-ethanediamine (10.6 mg) in 2.75 ml of DMF/DMSO (8:3) was reacted with 60 mg of 4-hydroxyphenylpropionyl-N-hydroxysuccinimidyl ester (Bolton Hunter reagent) at 23&#xb0; C. for 16 hrs. The solution was diluted with water and purified by RP-HPLC to afford 9.1 mg of the desired compound. Lactoperoxidase catalyzed iodination of the compound leads to a quantitative conversion to the di-iodo-apo-atractyloside derivative. Reaction of Atractyloside with Succinic Anhydride Atractyloside.3H2O (255 mg. 0.3 mmole) was dried by co-evaporation with dry pyridine (3&#xd7;5 ml) and then was kept under high vacuum for 16 hrs. The dried atractyloside was dissolved in 6 ml of dry pyridine and 60 mg of succinic anhydride (0.6 mmole) was added. The mixture was kept at 80&#xb0; C. for 30 min, another 60 mg (0.6 mmole) of succinic anhydride was added and the reaction mixture was stirred for an additional 3 hrs. The pyridine was removed in vacuo, and the residue was triturated with 10 ml of MeOH. The 6&#x2032;-O-succinyl-ATR derivative was collected by filtration as a white solid, washed with MeOH and dried overnight over P2O5. Reaction of 6&#x2032;-O-Succinylatractyloside with Tyramine and Iodination 6&#x2032;-O-Succinylatractyloside (85 mg) was dissolved in 2 ml of DMF and 28 mg of tyramine and 100 mg of PyBOP was added. The mixture was stirred at 23&#xb0; C. for 16 hrs. The crude mixture was subjected to RP-HPLC and the desired amide was isolated in 19.7 mg. Lactoperoxidase catalyzed iodination of the product using the standard conditions described above provides the mono and the di-iodinated products. Synthesis of 4-(4-Hydroxy-3-Methyl)-Butyric Acid To 4 gm (30 mmol) of AlCl3 in 50 ml of 1,2-dichloroethane at 0&#xb0; C. was added 2.7 gm (27 mmol) of succinic anhydride and the mixture was stirred for 20 min. 2-Methylanisole (3.1 ml, 25 mmol) was added, and the reaction mixture was warmed to 23&#xb0; C. and stirred for 12 hrs. The mixture was poured into 300 ml of ice-cold water and the precipitate was filtered off. The precipitate was washed with 2&#xd7;300 ml of water to afford a white solid. The solid material was dried under vacuum to afford 3.51 gm of product that was used in the next reaction. 3-(4-Methoxy-3-methylbenzoyl)propionic acid (4.4 gm, 20 mmol) and 4.49 gm of KOH pellets (80 mmol) were dissolved in 30 ml of ethylene glycol and 3.88 ml of hydrazine hydrate (80 mmol) was added to the stirred solution in four portions. The resulting reaction mixture was heated at 155&#xb0; C. for 24 hrs in an oil bath. After cooling, the reaction mixture was taken up in 100 ml of benzene and washed with 10% aqueous citric acid. The organic layer was washed with another portion of citric acid, dried over anhydrous sodium sulfate, and concentrated in vacuo to afford an oily residue that crystallized upon standing. The material was triturated with hot hexane and the solvent was evaporated off to afford 3.55 gm of a crystalline solid that was homogeneous by silica gel tic using hexane/ethyl acetate (8:2) as eluting solvent. 4-(4-Methoxy-3-methylphenyl)butyric acid (3.12 gm) was heated in 120 ml of a 1:1 mixture of 48% aqueous HBr/acetic acid at 155&#xb0; C. for 24 hrs. The reaction mixture was cooled to room temperature and was extracted with 200 ml of benzene/ether (1:1). The organic layer was dried over anhydrous magnesium sulfate and concentrated to afford a light brown solid residue. The reaction products were separated by silica gel flash chromatography using hexane/ethyl acetate (3:1) as eluting solvent to provide 0.86 mg of 4-(4-hydroxy-3-methyl)-butyric acid as a light yellow solid. Representative Synthesis of Atractyloside Derivatives To a solution of 3-(4-hydroxyphenyl)propionic acid (HPP) (0.498 g, 3.0 mmole) in 10 ml of anhydrous DMF was added carbonyldiimidazole (0.486 g, 3.0 mmole). The mixture was stirred at room temperature for 30 minutes and added in portions (2 ml/hour) to a solution of atractyloside (ATR) (0.086 g, 0.1 mmole) in 1 ml of anhydrous DMF The reaction mixture was stirred at room temperature overnight and quenched with 1 ml of water. The solvent was evaporated under vacuum and the residue was dissolved in ethyl acetate (75 ml) and water (50 ml). The aqueous layer is separated, extracted with ethyl acetate (3&#xd7;75 ml) and concentrated under vacuum. The residue was dissolved in 1.5 ml of methanol/water (1/1), filtered through a 0.2 mm filter and purified using HPLC with a preparative C-8 column (microsorb, 10&#xd7;250 mm) using a linear gradient elution of 30%-60% solvent B with a flow rate of 2.0 ml/min (solvent A: H2O/HOAc/NH4Oac (1.0M, aq.): {1000/1/1}; solvent B: CH3OH/HOAc/NH4OAc(1.0 M, aq.): {1000/1/1}). The title compound (compound 36 of Example 11 below) was obtained as a white film (6.2 mg). Example 11 Further Representative ATR Derivatives Following the procedures set forth in Example 10, the following ATR derivatives were prepared. Atractyloside Derivatives Cpd R MS NMR 22 &#x2002;879.3 (M &#x2212; H)&#x2212;&#x2002;799.3 (M &#x2212; SO3&#x2014;H)&#x2212; 1H NMR (CD3OD) &#x3b4;0.63(s,3H),0.95(d,3H), 0.96(d,3H),4.94(s,1H), 5.09(s,1H),7.53(m,2H), 7.60(m,1H),7.93(d,1H), 8.07(d,1H), 8.26(d,1H), 8.88(d,1H) 23 1083 (M &#x2212; H) 1003 (M &#x2212; SO3&#x2014;H)&#x2212; 1H NMR (CDOD) &#x3b4;0.65(s,3H),0.95(d,3H), 0.96(d,3H),5.02(s,1H), 5.13(s,1H),6.61(ddd,2H), 6.69(d,2H),6.86(dd,2H), 7.34(d,1H),8.33(dd,1H), 8.67(s,1H) 24 &#x2002;999 (M &#x2212; H)&#x2212; 1H NMR (CD3OD) &#x3b4;0.94(s,3H),0.95(d,3H), 0.96(d,3H),5.05(s,1H), 5.15(s,1H),6.75(d,1H), 7.04(dd,1H),7.52(d,1H) 25 &#x2002;921 (M &#x2212; H)&#x2212;&#x2002;841 (M &#x2212; SO3&#x2014;H)&#x2212; 1NMR (CD3OD) &#x3b4;0.75(s,3H),0.95(d,3H), 0.96(d,3H),4.97(s,1H), 5.08(s,1H),6.89(d,2H), 7.52(d,2H),7.64(d,2H), 8.07(d.2H) 26 1047 (M &#x2212; H)&#x2212;&#x2002;967 (M &#x2212; SO3&#x2014;H)&#x2212; 1H NMR (CD3OD) &#x3b4;0.73(s,3H),0.95(d,3H), 0.96(d,3H),4.98(s,1H), 5.09(s,1H),6.93(d,1H), 7.52(dd,1H),7.98(d.1H), 8.08(d,2H) 27 &#x2014; 1H NMR (CD3OD) &#x3b4;0.81(s,3H),0.95(d,3H), 0.96(d,3H),5.04(s,1H), 5.15(s,1H),6.80(d,2H), 7.90(d,2H),7.64(d,2H) 28 1097.1 (M &#x2212; H)&#x2212;1017.0 (M &#x2212; SO3&#x2014;H)&#x2212; 1H NMR (CD3OD) &#x3b4;0.83(s,3H),0.95(d,3H), 0.96(d,3H),5.03(s,1H), 5.15(s,1H),6.84(d,1H), 7.88(dd,1H),8.33(d,1H) 29 1043.2 (M &#x2212; 2H + Na)&#x2212;1021.2 (M &#x2212; H)&#x2212; 1NMR (CD3OD) &#x3b4;0.66(s,3H),0.95(d,3H), 0.96(d,3H),4.93(s,1H), 5.05(s,1H),7.22(d,1H), 7.87(d,1H),8.04(dd,1H), 8.08(d,1H),8.50(d,1H) 30 &#x2002;944 (M &#x2212; H)&#x2212;&#x2002;864 (M &#x2212; SO3&#x2014;H)&#x2212; 1H NMR (CD3OD) &#x3b4;0.97(s,3H),0.98(d,6H), 5.06(s,1H),5.17(s,1H), 5.70(d,2H),7.03(d,2H). 31 1070 (M &#x2212; H)&#x2212;&#x2002;990 (M &#x2212; SO3&#x2014;H)&#x2212; &#x2014; 32 1116 (M &#x2212; SO3&#x2014;H)&#x2212; &#x2014; 33 1125 (M &#x2212; H)&#x2212;1147 (M &#x2212; 2H + N)&#x2212; 1H NMR (CD3OD) &#x3b4;0.95(s,3H),0.96(s,6H), 5.05(s,1H),5.15(s,1H), 7.58(s,2H) 34 &#x2002;923.2 (M &#x2212; 2H + Na)&#x2212;&#x2002;901.3 (M &#x2212; H)&#x2212; &#x2014; 35 1049.3 (M &#x2212; 2H + Na)&#x2212; &#x2014; 36 &#x2002;873 (M &#x2212; H)&#x2212;&#x2002;895 (M &#x2212; 2H + Na)&#x2212; 1 H NMR (CD3OD) &#x3b4;0.94(s,3H),0.96(d,6H), 5.05(s,1H),5.16(s,1H), Dihydroactractyloside Derivatives Cpd R MS NMR 37 &#x2002;897 (M &#x2212; 2H + Na)&#x2212;&#x2002;875 (M &#x2212; H)&#x2212; 1H NMR (CD3OD) &#x3b4;0.93(s,3H),0.95(d,3H), 0.96(d,3H),1.08(d,3H), 6.69(d,2H),7.02(d,2H), resonances from alkenic protons absent 38 1105 (M &#x2212; 2H + Na)&#x2212;1083 (M &#x2212; H) 1H NMR (CD3OD) &#x3b4;0.93(s,3H),0.95(d,3H), 0.96(d,3H),1.08(d,3H), 6.74(d,1H),7.05(dd,1H), 7.53(d,1H), resonances from alkenic protons absent Apoactractyloside Derivatives Cpd R MS NMR 39 &#x2002;831(M &#x2212; H)&#x2212; 1H NMR (CD3OD) &#x3b4;1.00(s,3H),5.07(s,1H), 5.17(s,1H),6.69(d,2H), 7.03(d,2H) 40 1083 (M &#x2212; H)&#x2212;1105 (M &#x2212; 2H + Na)&#x2212; &#x2014; Example 12 Binding Assays Using Recombinant huANT3 A. GST-huANT3 Recombinantly Produced in E. coli: Following arabinose induction, transformed or sham transformed (vector only) E. coli were collected by centrifugation at 2000 g for 10 min. The bacterial pellets were resuspended in MSB, to which lysozyme (100 &#x3bc;g/ml) was added. After 20 min at room temperature, the lysates were subjected to one freeze-thaw cycle followed by sonication as described above The resultant membrane preparation was used for binding assays. To estimate maximal binding and the extent of overexpression of the huANT3, 25 &#x3bc;g of membrane protein was incubated with varying amounts of [32P]ATP (1-500 &#x3bc;M) in binding buffer (120 mM KCl, 10 mM Tris, 1 mM EDTA, pH 7.4) for 2 hr at room temperature. The membranes with bound ATP were sedimented by centrifugation at 5000 g for 5 min, and washed once with binding buffer. Membrane pellets were then mixed with 5 ml scintillation cocktail and counted. The results are presented in Table 2. TABLE 2 Saturation Binding of [32P]ATP to E. coli Membranes cpm bound by: Sham huANT3- [ATP], uM Transformed Cells Producing Cells 0.1 109 191 0.5 95 49 1.0 147 325 5.0 123 N.D. 10 214 263 50 549 2,727 100 718 5,772 500 2,140 9,715 N.D., not determined The data presented in Table 2 indicate that the affinity of the ATP binding was &#x2dc;6 &#x3bc;M. ATP binding was completed abolished by the addition of atractyloside (10 &#x3bc;M) to the assay. These results support the contention that the measured ATP binding was predominantly to recombinantly produced ANT3 protein. Agarose-glutathione beads were incubated with solubilized (using Dnase, Rnase and 0.1% Triton X-100; see Example 1, section D) bacterial lysate (see Example 2), and substituted for the E. coli membranes in binding assays. Best results (i.e., more specific binding) were seen when the beads were preincubated with bovine serum albumin (BSA, 0.1%) Specific ATP binding (1,070 cpm) was also observed in this experiment (compare to nonspecific binding of 279 cpm in the presence of 10 mM non-radiolabeled ATP). B. huANT3 from a Sf9/Baculovirus Expression System Sf9 cells expressing huANT3 were grown in spinner flasks. The cells were harvested by centrifugation at 2,000 g for 5 min. The cell pellet was resuspended in MSB and subjected to 3 freeze-thaw cycles. Cell membranes and debris were removed by centrifugation at 600 g for 5 min; mitochondria were collected by centrifuging the supernatant at 20,000 g for 15 min. The mitochondrial pellets were suspended in MSB, and used for binding assays as described above. Homologous competition of [32P]ATP binding was performed using 25 &#x3bc;g mitochondrial protein per assay. As illustrated in FIG. 7, ATP bound to the mitochondria with Kd=13 &#x3bc;M, a value consistent with ATP binding to ANT. Furthermore, the ATP binding was displaced by low concentrations of atractyloside (FIG. 8). Homologous competition binding assays using [125I]atractyloside revealed specific binding with Kd=12 nM (FIG. 9). These findings are consistent with the presence of functional huANT3 in the mitochondrial preparations. His-tagged huANT3 protein was purified from baculovirus-infected Sf9 solubilized cell lysates using Ni-agarose magnetic beads; Sf9 cells that had not been infected were used as negative controls. The beads were incubated with [32P]ATP (1 or 100 &#x3bc;M) for 2 hr. The beads were washed and then counted to determine the amount of bound ATP. As shown in Table 3, the [32P]ATP binding was significantly higher in material recovered from the infected cells than in the controls. Binding saturation had essentially been achieved with 1 &#x3bc;M ATP. TABLE 3 Binding of [32P]ATP to Purified His-Tagged huANT3 cpm bound by: Control huANT3- [ATP], uM (Uninfected) Cells Producing Cells 1.0 43 149 100 30 160 Example 13 Competitive Binding Assays Atractyloside analogs (Example 6; Table 1; see also Examples 7-9) were used in pseudo-homologous competition binding assays using Sf9/huANT3 mitochondria. Mitochondria (see Example 12, 25 &#x3bc;g/tube) were incubated with 0.5 nM [125I]atractyloside and varying concentrations of non-radiolabeled atractyloside or fluorescent atractyloside derivatives as described above (FIGS. 7-9). The results (Table 4) show that three of the atractyloside derivatives (ANT-, Pyrene- and Coumarin-atractyloside) had relative binding affinities similar to that of authentic atractyloside (IC50&#x3c;500 mM relative to atractyloside). Each of these derivatives is fluorescent, and may therefore be useful as detectable ligands for binding assays. TABLE 4 Competitive Binding Assays Using [125I]Atractyloside cpm [125I]Atractyloside bound in the presence of: [ATR Derivative], nM ATR COU-ATR PYR-ATR MANT-ATR 0.0 227 437 437 437 1.0 224 391 350 &#x2014; 50 146 &#x2014; &#x2014; &#x2014; 10 &#x2002;42 371 349 229 50 &#x2002;26 &#x2014; &#x2014; &#x2014; 100 &#x2014; 277 362 195 1,000 &#x2002;36 174 238 &#x2014; 5,000 &#x2002;45 &#x2014; &#x2014; &#x2014; 10,000 &#x2003;0 100 &#x2003;0 &#x2014; Abbreviations and Symbols: ATR, atractlyoside. COU-ATR, Coumarin-atractyloside, (Table 1, compound 3). PYR-ATR, Pyrene-atractyloside, (Table 1, compound 4). MANT-ATR, MANT-atractyloside, (Table 1, Roux et al. 1996 Anal. Bioch. 234:31) - - - , not determined. Atractyloside analogs (Example 6; Table 1; see also Examples 7-9) were also used in pseudo-homologous competition binding assays using T. ni/huANT3 mitochondria or bovine mitochondria. Mitochondria from noninfected T. ni cells, or T. ni cells infected with a baculovirus expressing huANT3 (see Example 3) were prepared as follows: T. ni cells were prepared by a subcontractor (PharMingen, San Diego, Calif.) as portions of about 250 mg of cells per tube. Each portion was suspended in 1 ml of MSB with protease inhibitors (leupeptin, final concentration 10 ug/ml; pepstatin, final concentration 10 ug/ml; aprotinin, final concentration, 2 ug/ml; phenylmethylsulfonyl fluoride, [PMSF], final concentration, 100 &#x3bc;M; all from Sigma Chemical Co., St. Louis, Mo.). The resuspended cell suspensions were frozen and thawed twice, then homogenized using a rotating teflon-coated probe and a close-fitting glass container (10 passes). The cellular homogenate was centrifuged (3,700 rpm, approximately 1,500&#xd7;g) at 4&#xb0; C. for 5 minutes; this supernatant from the first spin was saved. The pellet was washed with about 500 &#x3bc;l of MSB with protease inhibitors, centrifuged (3,800 rpm, approximately 1,600&#xd7; g) at 4&#xb0; C. for 5 minutes, and supernatant from this spin was combined with the supernatant from the first spin. The combined supernatant was centrifuged (14,000 rpm, approximately 20,800&#xd7;g) at 4&#xb0; C. for 15 minutes, and the pellet was resuspended in 300 &#x3bc;l of a 1:1 solution of (a) 20 mM MOPS and (b) MSB, wherein both (a) and (b) contain the previously described protease inhibitors. The resultant suspension was frozen and thawed three times. Bovine mitochondria were prepared as follows: Essentially all of the fat and cholesterol in clogged arteries was removed from two bovine hearts which were then cut into 1-inch cubes. The cubes were ground in a meat grinder using the fine setting. Three hundred (300) gm portions of the ground heart were weighed out and, to each was added 400 ml of Isolation Buffer (IB; 250 mM sucrose, 1 mM sodium succinate, 0.2 mM K+ EDTA, 10 mM Tris-base, pH 7.8). (All buffers were filter sterilized, and column buffers were degassed, and, unless otherwise noted, all steps were carried out at 0 to 4&#xb0; C. on ice or in pre-cooled rotors and centrifuges.) The preparations were mixed in a blender two times for 15 seconds on high setting and, in between and after blends, the pH was adjusted to 7.8 with 2M Tris-base. The homogenate was centrifuged for 20 minutes at 1,200&#xd7;g, and the supernatant was poured through two layers of cheese cloth and adjusted to pH 7.8 with 2M Tris-base. The supernatant was then centrifuged for 30 minutes at 11,000&#xd7;g. The supernatant was decanted, and the buff-colored outer pellet was dislodged with about 10 ml of IB and discarded. The brown inner pellet (heavy mitochondria) was resuspended in IB (about 10 ml per pellet). The pellets were homogenized in a glass-teflon homogenizer (2 passes at high drill speed). Samples were combined and centrifuged for 30 minutes at 11,000&#xd7;g. The supernatant was decanted, and the pellets were resuspended in 60 ml of IB per 900 gm of ground heart. This centrifugation step was repeated and the pellets were finally resuspended in IB (60 ml per 900 gm of ground heart). One kilogram of beef heart typically yields about one (1) gram of mitochondria. The mitochondrial preparations were divided into aliquots (typically, 50 &#x3bc;l for T. ni mitochondria or 20 &#x3bc;l for bovine mitochondria) and then either used directly in assays or flash frozen and stored at &#x2212;80&#xb0; C. The total protein content in the mitochondrial preparations was determined using the enhanced protocol (30 minutes at 60&#xb0; C.; see http://www.ruf rice.edu/&#x2dc;bioslabs/methods/protein/BCA.html) of the bicinchoninic acid (BCA) assay (available in kit form from Pierce, Rockford, Ill.). In the &#x201c;Tube Assay,&#x201d; mitochondria (from about 1 to 10 &#x3bc;g of total protein) were resuspended in 100 &#x3bc;l of Tris-KCl buffer with 0.1% BSA, pH 7.4. 125I-labeled compound 24 (Example 11) was added to a final concentration of 0.5 nM. When used, competitors were added at these concentration ranges: unlabeled atractyloside or compound 24 final concentration from 5 nM to 10 &#x3bc;M; unlabeled ADP (a lower affinity competitor) was added at a final concentration of 500 nM to 1 mM. The reaction mixes were incubated on ice for 60 minutes and then pelleted by centrifugation (approximately 16,000&#xd7;g) for 11 minutes at 4&#xb0; C. Unbound 125I-compound 24 was removed by aspiration. The pellets were contact-washed with Tris-KCl buffer, pH 7.4, and recentrifuged. The resultant pellets were aspirated and the radioactivity (dpm) in each was determined by gamma counting. Representative results are shown in FIGS. 11-18. The data presented in FIGS. 11 and 12 show that mitochondria (5 log of protein/tube) from both beef heart (FIG. 11) and T. ni cells expressing huANT3 (FIG. 12) specifically bind 125I-compound 24 in a manner that is inhibited by increasing concentrations of unlabeled compound 24, but, as expected, little or no binding is seen when mitochondria are excluded from the reaction mixes. FIGS. 13 and 14 show competitive inhibition of 1-compound 24 binding to mitochondria (1 &#x3bc;g of protein/tube) from beef heart (FIG. 13) and T. ni cells expressing huANT3 (FIG. 14) by compound 24 that is not detectably labeled, unlabeled atractyloside (ATR), and unlabeled adenosine diphosphate (ADP). In both instances, ATR and compound 24 yield comparable competition curves, although ATR appears to have a slightly higher affinity than compound 24. However, both ATR and compound 24 bind with much higher (about 1,000 fold) affinity than the low affinity ANT ligand ADP. FIG. 15 shows the competitive inhibition, by unlabeled ATR, of binding of 125I-compound 24 to mitochondria (1 &#x3bc;g of protein/tube) from T. ni cells expressing huANT3 and control T. ni cells (i.e., non-infected T. ni cells). As shown in FIG. 15, there was only slight inhibition of 125I-compound 24 binding to control (nontransformed) mitochondria by higher concentrations of ATR. In contrast, binding of 125I-compound 24 to mitochondria from T. ni cells expressing huANT3 was increasingly inhibited by higher concentrations of ATR. FIG. 16 shows competitive inhibition, by unlabeled compound 24 and by bongkrekic acid (BKA), of 125I-compound 24 binding to mitochondria (1 &#x3bc;g of protein/tube) from beef heart. BKA effectively displaced labeled compound 24, albeit with a slightly lower affinity than unlabeled compound 24. FIG. 17 shows competitive inhibition of 125I-compound 24 binding to beef heart mitochondria by either of the ATR derivatives, compound 23 (see Example 11) and compound 28 (see Example 11). As shown in FIG. 17, ATR exhibited an IC50 of approximately 44 nM, compound 23 an IC50 of approximately 105 nM, and compound 28 an IC50 of approximately 695 nM. In FIG. 18, data are presented depicting competitive inhibition of 125-compound 24 binding to beef heart mitochondria by the ATR derivative compound 5 (see Example 7). As shown in FIG. 18, the IC50 for compound 5 was approximately 3.3 &#x3bc;M. Competitive binding assays were also performed using recombinant His-tagged huANT3 (see Example 3) immobilized on Ni beads (FIG. 19) instead of mitochondria. To prepare the bead-immobilzed huANT3, mitochondria from T. ni cells infected with a baculovirus expressing huANT3 (see Example 3) were solubilized with 0.5% O-glucopyranoside in the presence of 0.5 nM 125I-compound 24, Ni-agarose beads (Qiagen, Hilden, Germany), and various concentrations of ATR or BKA as unlabeled competitors. After 1 hour at 4&#xb0; C., the beads were washed and radioactivity that remained associated with the beads was counted. (Background binding of [125I] compound 24 to Ni-agarose beads (Qiagen) in the absence of His-tagged huANT3 was approximately 700-800 cpm and was not subtracted from the radioactivity shown in FIG. 19.) The results (FIG. 19) show that both ATR and BKA effectively compete with compound 24 in a manner similar to that observed in assays using intact mitochondria (cf. FIGS. 14 and 15). Example 14 High Throughput Screening Assay for Compounds Targeted to ANT Proteins and Polypeptides The recombinantly produced ANT proteins, ANT fusion proteins and detectably labeled ANT ligands described herein are incorporated into automated assay systems. Such automated systems are useful for high throughput screening (HTS) of candidate ANT-binding compounds or chemical libraries comprising such compounds. Such compounds may be further characterized and developed as drug candidates and drugs useful for preventing, treating or curing diseases or disorders resulting from the overexpression or dysfunction of one or more ANT proteins or from the overexpression or dysfunction of a factor that positively regulates or stimulates ANT proteins. A preferred element of many automated assay systems is the incorporation of a target molecule (in the present instance, an ANT protein) into a 96-well plate. This format is readily adaptable for use in a variety of automated label detection systems. For HTS assays, robotic label detection systems are preferred. As one example of an HTS comprising the elements describes herein, the GST-huANT3 fusion protein of Example 2 is contacted with REACTI-BIND&#x2122; glutathione-coated 96-well plates (Pierce). Glutathione coated strip-well plates are preferably used for assays comprising radiolabeled ANT ligands (e.g., iodinated atractyloside derivates; see Example 7), whereas black opaque glutathione coated 96-well plates are preferred for assays comprising fluorescent ANT ligands (such as are described in, e.g., Examples 6-9); both types of glutathione coated plates are commercially available (Pierce). In a typical assay, 1 to 50 ug of GST-huANT3 protein (i.e., total solubilized protein prepared as in Example 2) is added per gluthathione-coated well to each well of a 96-well plate. Iodinated atractyloside derivate (125I-ATR) is added to the wells (0.5 nmol/well). In a control experiment, unlabeled atractyloside (ATR; Sigma) is used as a &#x2018;mock&#x2019; drug at a concentration of from about 1 to about 10,000 &#x3bc;M. That is, unlabeled ATR is used to displace a labeled atractyloside derivative (e.g., 125I-ATR). Unlabeled ATR thus acts as a positive control for an HTS in which various compounds are screened for their ability to displace a labeled ANT ligand. As an example of the automated label detection systems used in the HTS assays of the Example, when the detectably labeled ANT ligand of the assay is 125I-ATR, an automatic gamma counter is used. Alternatively, 125I-ATR can be used in scintillation proximity assays (SPA). For example, a GST-huANT fusion protein is contacted with ScintiStrip 96-well plates coated with glutathione (EG&#x26;G Wallac). The polystyrene of these plates contains a scintillating agent that emits beta radiation when excited by a gamma emitter in close proximity thereto. The beta radiation is then detected by any appropriate automatic beta counter. When fluorescent ANT ligands are used in the HTS assay, an automatic fluorescence counter is used and may be, for example, a FLUOROCOUNT&#x2122; Counter (Packard Instrument Company, Meriden, Conn.).\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"claims\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 5,\n        \"samples\": [\n          \"1-41. (Cancelled) 42. An isolated recombinant human adenine nucleotide translocator 2 (ANT2) polypeptide comprising the amino acid sequence set forth in SEQ ID NO:32, produced by a method comprising culturing a host cell comprising a recombinant expression construct comprising at least one regulated promoter operably linked to a nucleic acid encoding the ANT2 polypeptide. 43-46. (Cancelled) 47. An isolated adenine nucleotide translocator 2 (ANT2) fusion protein comprising an ANT2 polypeptide fused to at least one additional polypeptide sequence, wherein said ANT2 polypeptide has at least 95% amino acid sequence identity to the amino acid sequence of a human ANT2 polypeptide as set forth in SEQ ID NO:32, and wherein the ANT2 fusion protein is capable of binding an ANT ligand. 48. The fusion protein of either claim 47 or claim 114 wherein said one additional polypeptide sequence is an enzyme sequence or a variant or fragment thereof. 49. The fusion protein of either claim 47 or claim 114 wherein said fusion protein localizes to membranes. 50. The fusion protein of claim 49 wherein said membranes are mitochondrial membranes. 51. An isolated adenine nucleotide translocator 2 (ANT2) fusion protein comprising an ANT2 polypeptide fused to at least one additional polypeptide sequence cleavable by a protease, said ANT2 polypeptide being separable from the fusion protein by cleavage with the protease, wherein said ANT2 polypeptide has at least 95% amino acid sequence identity to the sequence of a human ANT2 polypeptide set forth in SEQ ID NO:32, and wherein the ANT2 fusion protein is capable of binding an ANT ligand. 52-56. (Cancelled) 57. The fusion protein of either claim 47 or claim 115 wherein the additional polypeptide sequence is a polypeptide having affinity for a ligand. 58-113. (Cancelled) 114. An isolated recombinant human adenine nucleotide translocator fusion protein comprising an adenine nucleotide translocator (ANT) polypeptide fused to at least one additional polypeptide sequence, wherein the ANT polypeptide comprises an amino acid sequence of a human ANT2 polypeptide as set forth in SEQ ID NO:32. 115. An isolated human adenine nucleotide translocator fusion protein comprising an adenine nucleotide translocator (ANT) polypeptide fused to at least one additional polypeptide sequence cleavable by a protease that separates the ANT polypeptide from the remainder of the fusion protein, wherein the ANT polypeptide comprises the amino acid sequence of a human ANT2 sequence as set forth in SEQ ID NO:32. 116. The isolated polypeptide of claim 42 in which one amino acid is substituted.\",\n          \"1. A method for transferring information between first and second systems with dissimilar first and second database structures, respectively, comprising the steps of: extracting data from the first system; routing the extracted data from the first system to a first conversion server; at the first conversion server, converting the data from a format compatible with the first database structure to an intermediate format; routing the data in the intermediate format to a second conversion server; at the second conversion server, converting the data from the intermediate format to a format compatible with the second database structure; and storing the data converted at the second conversion server in the second system. 2. The method of claim 1, wherein the step of extracting is initiated in response to an external request. 3. The method of claim 1, wherein the step of extracting is performed in response to a trigger operation at the first system which initiates the step of extracting and routing to the first conversion server. 4. The method of claim 1, wherein the first and second conversion servers are disposed on a network and communicate with the first and second systems and with each other through the network and/or in the step of routing data comprises routing data over the network. 5. The method of claim 1, wherein the data is comprised of a plurality of records, with each record including at least one index and the step of converting in the first conversion server operable to convert the index compatible with the first database structure [AB] to an intermediate index that is relationally associated with the index in the first database structure; and the step of converting from the intermediate format to the format compatible with the second database structure comprises converting the intermediate index to an index compatible with the second database structure. 6. The method of claim 5, wherein the intermediate index has a finite length, such that all data transactions in between the first and second conversion servers are operable to transmit finite length index values. 7. The method of claim 6, and further comprising an intermediate processing step wherein at least a portion of the converted data that is transmitted from the first conversion server to the second conversion server is processed in the intermediate format.\",\n          \"1-41. (Cancelled) 42. An isolated recombinant human adenine nucleotide translocator 1 (ANT1) polypeptide comprising the amino acid sequence set forth in SEQ ID NO:31, wherein the recombinant human ANT1 polypeptide is capable of binding an ANT ligand and is produced by a method comprising culturing a host cell comprising a recombinant expression construct comprising at least one regulated promoter operably linked to a nucleic acid encoding the ANT1 polypeptide. 43-46. (Cancelled) 47. An isolated adenine nucleotide translocator 1 (ANT1) fusion protein comprising an ANT1 polypeptide fused to at least one additional polypeptide sequence, wherein said ANT1 polypeptide has at least 95% amino acid sequence identity to the sequence of a human ANT1 polypeptide as set forth in SEQ ID NO:31, and wherein the ANT1 fusion protein is capable of binding an ANT ligand. 48. The fusion protein of either claim 47 or claim 114 wherein said one additional polypeptide sequence is an enzyme sequence or a variant or fragment thereof. 49. The fusion protein of either claim 47 or claim 114 wherein said fusion protein localizes to membranes. 50. The fusion protein of claim 49 wherein said membranes are mitochondrial membranes. 51. An isolated adenine nucleotide translocator 1 (ANT1) fusion protein comprising an ANT1 polypeptide fused to at least one additional polypeptide sequence cleavable by a protease, said ANT1 polypeptide being separable from the fusion protein by cleavage with the protease, wherein said ANT1 polypeptide has at least 95% amino acid sequence identity to the sequence of a human ANT1 polypeptide set forth in SEQ ID NO:31 and wherein the ANT1 fusion protein is capable of binding an ANT ligand. 52-56. (Cancelled) 57. The fusion protein of either claim 47 or claim 115 wherein the additional polypeptide sequence is a polypeptide having affinity for a ligand. 58-113. (Cancelled) 114. An isolated recombinant human adenine nucleotide translocator fusion protein comprising an adenine nucleotide-translocator (ANT) polypeptide fused to at least one additional polypeptide sequence, wherein the ANT polypeptide comprises an amino acid sequence of a human ANT1 polypeptide as set forth in SEQ ID NO:31. 115. An isolated human adenine nucleotide translocator fusion protein comprising an adenine nucleotide translocator (ANT) polypeptide fused to at least one additional polypeptide sequence cleavable by a protease that separates the ANT polypeptide from the remainder of the fusion protein, wherein the ANT polypeptide comprises the amino acid sequence of a human ANT1 sequence as set forth in SEQ ID NO:31. 116. An isolated polypeptide according to claim 42 in which one amino acid residue is substituted.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"status\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 1,\n        \"samples\": [\n          \"ISS\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"}},"metadata":{},"execution_count":5}]},{"cell_type":"code","source":["import numpy as np\n","\n","np.shape(train_df)"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"mYlr2OZX06Q0","executionInfo":{"status":"ok","timestamp":1711408820476,"user_tz":420,"elapsed":9,"user":{"displayName":"Rishikesh Sivakumar","userId":"10957495338496577769"}},"outputId":"612a2759-2ecf-46c8-b422-4c7370079893"},"execution_count":null,"outputs":[{"output_type":"execute_result","data":{"text/plain":["(5, 4)"]},"metadata":{},"execution_count":6}]},{"cell_type":"code","source":["if torch.cuda.is_available():\n","   dev = torch.device(\"cuda:0\")\n","   print(\"Running on the GPU\")\n","else:\n","   dev = torch.device(\"cpu\")\n","   print(\"Running on the CPU\")"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"lm-jn7mrtfZY","executionInfo":{"status":"ok","timestamp":1711408820476,"user_tz":420,"elapsed":5,"user":{"displayName":"Rishikesh Sivakumar","userId":"10957495338496577769"}},"outputId":"ff52de0c-cfed-40e3-8a05-18241604fdf6"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["Running on the GPU\n"]}]},{"cell_type":"code","source":["tokenizer = T5Tokenizer.from_pretrained('t5-base')\n","model = T5ForConditionalGeneration.from_pretrained('t5-base',\n","                                             return_dict=True)\n","#moving the model to GPU\n","model.to(dev)"],"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":1000,"referenced_widgets":["73eb22912e4449298bd418b3b7e823da","73ecc175f4b04408aea3786b5c9d0b2f","c7988b785d3346b9a03133d441080b87","dade3047b1944ef0a22e031eee7ef07e","3384cac3e2e8406d9ea8bbfd65c87136","7f770f22ae0e4c14926f8bf81049d4c2","617cb6ee91f14a11b8f024413280460d","2d3fb1e8c53347c9966f6669cbe8f93f","9e5ce0b8408b494bb583513bf0c1e4f8","6cda654b41574bfcb9bf4a8ef8c0aa78","8470bd11052b455e8d10bf7b6eb5cb0d","7691709fd81c4a34927860dc8c91751c","88bc50ab935b49a99292aa907990453d","d8266e99c1fd4dc2a3e62bd0ab4524e0","44b8fcdac57440fc8b72495697e6eae4","97d5902c8d5b434dbd149d9dc98884e4","a4e20156bf5842b39315fe025791c534","80628e51d1e245e59a8057db248d35a4","70ce768bdce441daab87f41ff24d6eb7","d893d3d5f92443f7bbead9749667f4b3","98119439c548494ca454c4750c79c65d","d128daf8ef9844bb885dc74223c81ef2","4ce328f50ec846a3a33a577bb3af4774","ca9186809d5b44718f3b34b60c4f7015","d1872dac52fe42729838fc218cf48a07","9a0732edb9ce423f87290430fa158f63","de28447cbce44dc2a51e54fd5204815d","48ec762c8ea34121853ab2ad6bc3e4e7","da835df94c1342e1a11c4309be6d7e10","08c901f3359044daaedce3384c412865","f34d969eefa249dabf1db793e37822cf","0794da8ad58c48679ada18842604fe7e","f3214b0068894e0eaad6de46da4ddc70","12d2a3d58c8a4a618dbdc330c906def3","3bd08447fff44b3aa14bd9cf5a9de3e1","862b994d613d458193a2f1122620d9a3","811ad5d09a7e4702a2d2c42e248649d3","10455389e55b42259b5015fb6931c7e0","c5bd1b7764004ef9b7e93c7118b97c1f","d0b4db582654429e92780b96b5c439ce","a5ba4c4f8d36425cb42f095570ee913f","837d7790b24d49dabea13905d5350c50","4ef460b335b7444590c430a4b8bc82fd","8f8903b08bb543a99d0ac0c8c1916295","ec1828f9d3bb4d40a1cb02c7a17bac33","dcb5d24e65c74fe89aecb5b14139346d","5d4680477e004459879f20cf530d32f4","aa60ba0a44bd4b5eb96cd90a11898201","f7bddc60aa0240ee82491739df915edb","675c5f258a8d408484ce97f0806a2ad3","a6dc881ed8fd47c68dacc2d8163b25c7","fd008b9ee15d49d3a75a3d34469161a0","f85341a4a9364047808d49e1e3fbdb67","a0429ab624b44019b67c383cfcd79cc7","5fc67793c7734fe282d2cd47b13358f7"]},"id":"6pPnqWRBt0f7","executionInfo":{"status":"ok","timestamp":1711408830907,"user_tz":420,"elapsed":10434,"user":{"displayName":"Rishikesh Sivakumar","userId":"10957495338496577769"}},"outputId":"7f6d4d56-5884-4730-d705-312a2726a770"},"execution_count":null,"outputs":[{"output_type":"stream","name":"stderr","text":["/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_token.py:88: UserWarning: \n","The secret `HF_TOKEN` does not exist in your Colab secrets.\n","To authenticate with the Hugging Face Hub, create a token in your settings tab (https://huggingface.co/settings/tokens), set it as secret in your Google Colab and restart your session.\n","You will be able to reuse this secret in all of your notebooks.\n","Please note that authentication is recommended but still optional to access public models or datasets.\n","  warnings.warn(\n"]},{"output_type":"display_data","data":{"text/plain":["spiece.model:   0%|          | 0.00/792k [00:00<?, ?B/s]"],"application/vnd.jupyter.widget-view+json":{"version_major":2,"version_minor":0,"model_id":"73eb22912e4449298bd418b3b7e823da"}},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["tokenizer.json:   0%|          | 0.00/1.39M [00:00<?, ?B/s]"],"application/vnd.jupyter.widget-view+json":{"version_major":2,"version_minor":0,"model_id":"7691709fd81c4a34927860dc8c91751c"}},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["config.json:   0%|          | 0.00/1.21k [00:00<?, ?B/s]"],"application/vnd.jupyter.widget-view+json":{"version_major":2,"version_minor":0,"model_id":"4ce328f50ec846a3a33a577bb3af4774"}},"metadata":{}},{"output_type":"stream","name":"stderr","text":["You are using the default legacy behaviour of the <class 'transformers.models.t5.tokenization_t5.T5Tokenizer'>. This is expected, and simply means that the `legacy` (previous) behavior will be used so nothing changes for you. If you want to use the new behaviour, set `legacy=False`. This should only be set if you understand what it means, and thoroughly read the reason why this was added as explained in https://github.com/huggingface/transformers/pull/24565\n","Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.\n"]},{"output_type":"display_data","data":{"text/plain":["model.safetensors:   0%|          | 0.00/892M [00:00<?, ?B/s]"],"application/vnd.jupyter.widget-view+json":{"version_major":2,"version_minor":0,"model_id":"12d2a3d58c8a4a618dbdc330c906def3"}},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["generation_config.json:   0%|          | 0.00/147 [00:00<?, ?B/s]"],"application/vnd.jupyter.widget-view+json":{"version_major":2,"version_minor":0,"model_id":"ec1828f9d3bb4d40a1cb02c7a17bac33"}},"metadata":{}},{"output_type":"execute_result","data":{"text/plain":["T5ForConditionalGeneration(\n","  (shared): Embedding(32128, 768)\n","  (encoder): T5Stack(\n","    (embed_tokens): Embedding(32128, 768)\n","    (block): ModuleList(\n","      (0): T5Block(\n","        (layer): ModuleList(\n","          (0): T5LayerSelfAttention(\n","            (SelfAttention): T5Attention(\n","              (q): Linear(in_features=768, out_features=768, bias=False)\n","              (k): Linear(in_features=768, out_features=768, bias=False)\n","              (v): Linear(in_features=768, out_features=768, bias=False)\n","              (o): Linear(in_features=768, out_features=768, bias=False)\n","              (relative_attention_bias): Embedding(32, 12)\n","            )\n","            (layer_norm): T5LayerNorm()\n","            (dropout): Dropout(p=0.1, inplace=False)\n","          )\n","          (1): T5LayerFF(\n","            (DenseReluDense): T5DenseActDense(\n","              (wi): Linear(in_features=768, out_features=3072, bias=False)\n","              (wo): Linear(in_features=3072, out_features=768, bias=False)\n","              (dropout): Dropout(p=0.1, inplace=False)\n","              (act): ReLU()\n","            )\n","            (layer_norm): T5LayerNorm()\n","            (dropout): Dropout(p=0.1, inplace=False)\n","          )\n","        )\n","      )\n","      (1-11): 11 x T5Block(\n","        (layer): ModuleList(\n","          (0): T5LayerSelfAttention(\n","            (SelfAttention): T5Attention(\n","              (q): Linear(in_features=768, out_features=768, bias=False)\n","              (k): Linear(in_features=768, out_features=768, bias=False)\n","              (v): Linear(in_features=768, out_features=768, bias=False)\n","              (o): Linear(in_features=768, out_features=768, bias=False)\n","            )\n","            (layer_norm): T5LayerNorm()\n","            (dropout): Dropout(p=0.1, inplace=False)\n","          )\n","          (1): T5LayerFF(\n","            (DenseReluDense): T5DenseActDense(\n","              (wi): Linear(in_features=768, out_features=3072, bias=False)\n","              (wo): Linear(in_features=3072, out_features=768, bias=False)\n","              (dropout): Dropout(p=0.1, inplace=False)\n","              (act): ReLU()\n","            )\n","            (layer_norm): T5LayerNorm()\n","            (dropout): Dropout(p=0.1, inplace=False)\n","          )\n","        )\n","      )\n","    )\n","    (final_layer_norm): T5LayerNorm()\n","    (dropout): Dropout(p=0.1, inplace=False)\n","  )\n","  (decoder): T5Stack(\n","    (embed_tokens): Embedding(32128, 768)\n","    (block): ModuleList(\n","      (0): T5Block(\n","        (layer): ModuleList(\n","          (0): T5LayerSelfAttention(\n","            (SelfAttention): T5Attention(\n","              (q): Linear(in_features=768, out_features=768, bias=False)\n","              (k): Linear(in_features=768, out_features=768, bias=False)\n","              (v): Linear(in_features=768, out_features=768, bias=False)\n","              (o): Linear(in_features=768, out_features=768, bias=False)\n","              (relative_attention_bias): Embedding(32, 12)\n","            )\n","            (layer_norm): T5LayerNorm()\n","            (dropout): Dropout(p=0.1, inplace=False)\n","          )\n","          (1): T5LayerCrossAttention(\n","            (EncDecAttention): T5Attention(\n","              (q): Linear(in_features=768, out_features=768, bias=False)\n","              (k): Linear(in_features=768, out_features=768, bias=False)\n","              (v): Linear(in_features=768, out_features=768, bias=False)\n","              (o): Linear(in_features=768, out_features=768, bias=False)\n","            )\n","            (layer_norm): T5LayerNorm()\n","            (dropout): Dropout(p=0.1, inplace=False)\n","          )\n","          (2): T5LayerFF(\n","            (DenseReluDense): T5DenseActDense(\n","              (wi): Linear(in_features=768, out_features=3072, bias=False)\n","              (wo): Linear(in_features=3072, out_features=768, bias=False)\n","              (dropout): Dropout(p=0.1, inplace=False)\n","              (act): ReLU()\n","            )\n","            (layer_norm): T5LayerNorm()\n","            (dropout): Dropout(p=0.1, inplace=False)\n","          )\n","        )\n","      )\n","      (1-11): 11 x T5Block(\n","        (layer): ModuleList(\n","          (0): T5LayerSelfAttention(\n","            (SelfAttention): T5Attention(\n","              (q): Linear(in_features=768, out_features=768, bias=False)\n","              (k): Linear(in_features=768, out_features=768, bias=False)\n","              (v): Linear(in_features=768, out_features=768, bias=False)\n","              (o): Linear(in_features=768, out_features=768, bias=False)\n","            )\n","            (layer_norm): T5LayerNorm()\n","            (dropout): Dropout(p=0.1, inplace=False)\n","          )\n","          (1): T5LayerCrossAttention(\n","            (EncDecAttention): T5Attention(\n","              (q): Linear(in_features=768, out_features=768, bias=False)\n","              (k): Linear(in_features=768, out_features=768, bias=False)\n","              (v): Linear(in_features=768, out_features=768, bias=False)\n","              (o): Linear(in_features=768, out_features=768, bias=False)\n","            )\n","            (layer_norm): T5LayerNorm()\n","            (dropout): Dropout(p=0.1, inplace=False)\n","          )\n","          (2): T5LayerFF(\n","            (DenseReluDense): T5DenseActDense(\n","              (wi): Linear(in_features=768, out_features=3072, bias=False)\n","              (wo): Linear(in_features=3072, out_features=768, bias=False)\n","              (dropout): Dropout(p=0.1, inplace=False)\n","              (act): ReLU()\n","            )\n","            (layer_norm): T5LayerNorm()\n","            (dropout): Dropout(p=0.1, inplace=False)\n","          )\n","        )\n","      )\n","    )\n","    (final_layer_norm): T5LayerNorm()\n","    (dropout): Dropout(p=0.1, inplace=False)\n","  )\n","  (lm_head): Linear(in_features=768, out_features=32128, bias=False)\n",")"]},"metadata":{},"execution_count":8}]},{"cell_type":"code","source":["optimizer = Adafactor(model.parameters(),lr=1e-3,\n","                      eps=(1e-30, 1e-3),\n","                      clip_threshold=1.0,\n","                      decay_rate=-0.8,\n","                      beta1=None,\n","                      weight_decay=0.0,\n","                      relative_step=False,\n","                      scale_parameter=False,\n","                      warmup_init=False)"],"metadata":{"id":"3bxQTX9yt2xi"},"execution_count":null,"outputs":[]},{"cell_type":"code","source":["from IPython.display import HTML, display\n","def progress(loss,value, max=100):\n"," return HTML(\"\"\" Batch loss :{loss}\n","      <progress\n","value='{value}'max='{max}',style='width: 100%'>{value}\n","      </progress>\n","             \"\"\".format(loss=loss,value=value, max=max))"],"metadata":{"id":"wO141c3juHay"},"execution_count":null,"outputs":[]},{"cell_type":"code","source":["import pandas as pd\n","from torch.utils.data import Dataset\n","\n","class TextToTextDataset(Dataset):\n","    def __init__(self, tokenizer, filepath, input_column, target_column, max_token_len=512):\n","        self.tokenizer = tokenizer\n","        self.data = pd.read_csv(filepath)\n","        self.input_texts = self.data[input_column]\n","        self.target_texts = self.data[target_column]\n","        self.max_token_len = max_token_len\n","\n","    def __len__(self):\n","        return len(self.data)\n","\n","    def __getitem__(self, idx):\n","        input_text = self.input_texts[idx]\n","        target_text = self.target_texts[idx]\n","\n","        # Tokenize input text\n","        input_tokens = self.tokenizer.encode_plus(input_text, max_length=self.max_token_len,\n","                                                   truncation=True, padding=\"max_length\",\n","                                                   return_tensors=\"pt\")\n","\n","        # Tokenize target text\n","        target_tokens = self.tokenizer.encode_plus(target_text, max_length=self.max_token_len,\n","                                                    truncation=True, padding=\"max_length\",\n","                                                    return_tensors=\"pt\")\n","\n","        # Input IDs and attention masks for the model\n","        inputs = {\n","            \"input_ids\": input_tokens[\"input_ids\"].flatten(),\n","            \"attention_mask\": input_tokens[\"attention_mask\"].flatten()\n","        }\n","\n","        # Target IDs, ignoring the first token (usually <s> token)\n","        # We do not need attention mask for targets as they are used as labels\n","        targets = target_tokens[\"input_ids\"].flatten()\n","\n","        return inputs, targets"],"metadata":{"id":"hW4znAse4bPI"},"execution_count":null,"outputs":[]},{"cell_type":"code","source":["from torch.utils.data import Dataset\n","\n","class TextDataset(Dataset):\n","    def __init__(self, csv_file, tokenizer, max_length=512): # adjust max length depending on tokenizer\n","        self.data_frame = pd.read_csv(csv_file)\n","        self.tokenizer = tokenizer\n","        self.max_length = max_length\n","        # self.label_map = label_map\n","        # redefine self.data_frame for integer valued labels\n","        # self.data_frame['status'] = self.data_frame['status'].map(self.label_map)\n","\n","    def __len__(self):\n","        return self.data_frame.shape[0]\n","\n","    def __getitem__(self, idx):\n","        text = self.data_frame.iloc[idx]['detail']\n","        label = self.data_frame.iloc[idx]['claims']\n","\n","        # Encoding the text using the BERT tokenizer\n","        text_encoding = self.tokenizer.encode_plus(\n","            text,\n","            add_special_tokens=True,\n","            max_length=self.max_length,\n","            return_token_type_ids=False,\n","            padding='max_length',\n","            truncation=True,\n","            return_attention_mask=True,\n","            return_tensors='pt',\n","        )\n","\n","        label_encoding = self.tokenizer.encode_plus(\n","            label,\n","            add_special_tokens=True,\n","            max_length=self.max_length,\n","            return_token_type_ids=False,\n","            padding='max_length',\n","            truncation=True,\n","            return_attention_mask=True,\n","            return_tensors='pt',\n","        )\n","        source_ids = text_encoding[\"input_ids\"].squeeze()\n","        source_mask = text_encoding[\"attention_mask\"].squeeze()\n","        target_ids = label_encoding[\"input_ids\"].squeeze()\n","        target_mask = label_encoding[\"attention_mask\"].squeeze()\n","\n","        return {\n","            \"source_ids\": source_ids.to(dtype=torch.long),\n","            \"source_mask\": source_mask.to(dtype=torch.long),\n","            \"target_ids\": target_ids.to(dtype=torch.long),\n","            \"target_ids_y\": target_ids.to(dtype=torch.long),\n","        }"],"metadata":{"id":"PF1ES7a24k1w"},"execution_count":null,"outputs":[]},{"cell_type":"code","source":["train_data = TextDataset(csv_path, tokenizer)\n","# val_data = TextDataset(val_csv, tokenizer)"],"metadata":{"id":"7JXW4d686ckI"},"execution_count":null,"outputs":[]},{"cell_type":"code","source":["train_data[0]"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"2igN4RFI7Ky9","executionInfo":{"status":"ok","timestamp":1711410729416,"user_tz":420,"elapsed":7,"user":{"displayName":"Rishikesh Sivakumar","userId":"10957495338496577769"}},"outputId":"3fc18e2b-234c-412f-a788-efb144bc045a"},"execution_count":null,"outputs":[{"output_type":"execute_result","data":{"text/plain":["{'source_ids': tensor([  309, 25067, 26017,  9562,  3347,   262,     4, 15837,  3765,   262,\n","          4633,  7039,   196, 11810,   134,   389,  4648,    19,  2569,   270,\n","            77,    24,  4951, 21286,  1459,     6,  3487, 17127,     6,    11,\n","          2491,  9328,    21,  2323,    13,  2547, 12311,  2699,    16,  4896,\n","             6,  2839,  8258,     5,    71,     3, 24079,     6,  8308,  1229,\n","            13, 15741,   372,  2491, 15293,    11,  3487,  6002, 11573,    19,\n","           990,    11,  3030,    16,   490,    97,     5,     3, 22917,     6,\n","          2323,    33, 10431,    13,  3173, 10164,    53,  3487,    11,    87,\n","           127,  1055,  2491,  9328,  4089,     5, 19148,     3, 17282,  2097,\n","            33,  8152,    12,   482, 20218,  1413,    18, 11650,  2416,    13,\n","         27804,   859,  4145,    11,  1438,     5,   100,  1147,   795,     3,\n","             9,  4732,    21,  1288,    18, 22955,  1357,   492,    11,  2491,\n","          1459,    57,   936,    18,   348, 11438,    42,  9738,    29,  7578,\n","         20893,  1002,     5,   368,  1146,   593, 22996,  1014,  2323,  7931,\n","            38,    46, 13591,    29,    17, 15037,    13,   175,  2842,     5,\n","            37,     3,     7,  1489,  2020,    11,  8308,  1405,    13,     8,\n","          4648,  1250,     8,  1229,    12,    36,  4896,  1427,  8473,    38,\n","          4145,    33, 10380,     6,  1766,  4108,     6,    11,   126,  3262,\n","          1124,    33, 15110,     5,    37,  4648,    65,     3,     9,  1148,\n","           620,    13,  1564,    16,     8,   844,    13, 15226,     6,  4136,\n","          3732,     6,   433,   610,     6, 21286,  8308, 20407,     7,     6,\n","          1456,     7,     6,    42,   136,   616,    16,    84,  1438,    33,\n","          2546,    11, 16647,    16,     8,  1288,    18, 22955,  9328,    13,\n","          4145,     5,    71,   248,   381,    13,   490,   296,  1564,  7789,\n","             8,  2874,    13,  1317,    41,     7,    15,    51,    23,    18,\n","            61, 21286,  3173,    42,  2323,     6,  5908,    16,  4145,  6640,\n","          2957,     8, 10970,    13,  1017,  1766,     6,  2178,     3,     9,\n","          1643,  1899,    13,  1832,  1438,     6,    11,  2699,    16,     3,\n","             9,  4896,     6,  2839,  1164,     5, 28107,    19,     3,     9,\n","          3427,   677,    13,    80,   224,   917,     6,    68,  4062,    92,\n","           560,   433,   610,    11,  4136,  3732,    16,   681,     6, 12794,\n","            11,   463,    16,  1456,     7,     6,    11, 21286,   610,    11,\n","          8308, 20407,     7,    16,  3867,   585,     5,   486,     8,  2030,\n","           593,     6,   175,   982,    33,  2389, 16384,    57,   609,     3,\n","           390,    30,   936, 26207,    11,   351,     5,   886, 16783,    11,\n","         19291,    40,  2097,  3223,    12,   199,  4602,  1364,   593,  3379,\n","            13,     8,  4038,   982,     5,    94,    19,  5327,    12,   915,\n","             3,     9,  2268,  4648,    24,     6,   116,  6960,     6,   133,\n","          1716,    38,     3,     9,  2021,  3052,    21,   936,  1357, 13730,\n","            42,  1716,    38,     8,  2069, 10045,   358,    13,     3,     9,\n","          1551, 21286,  1232,    13,  9738,    29,  7578,  1357, 13730,     6,\n","          2040,  3114,  1162, 31663,  1014, 16651,    41,  9156,   137,    71,\n","           743,     6,  3236,  4648,    21, 21286,  3487, 14302,    16,  4896,\n","          8258,   225,    43,     8,   826,  2605,    10, 13380,   179,  4648,\n","            11,  4136, 12628,   356,  5376,   302,    17, 13954,   190,  1745,\n","            18,  4563, 14302,  2977,    18,   715,     3,    60, 17939,   257,\n","            16,   495,    13,  4896,  1419,  1112,    86, 13398,    12,  1453,\n","            13,  5030,     6,   415, 29007,   389,     3,  9156,  4648,    19,\n","          2569,   270,    77,    24,   795,    21,   175,  6803,    16,     3,\n","             9,  3236,     6, 30238,  3107,     5,    37,     3,  9156,  4648,\n","          5806,     3,     9,   775,   356,    13,  2399,   585,   844,  3334,\n","            12,     1]),\n"," 'source_mask': tensor([1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,\n","         1, 1, 1, 1, 1, 1, 1, 1]),\n"," 'target_ids': tensor([ 1300,    71,  8308,   331,  3026,   358,    21, 14498, 17127,    13,\n","          1438,    11, 14498,  2491,  9328,     6,   243,   358,  8315,    53,\n","            10,     3,     9,  5030,  1229,    21,  9153,  1222,  4175,   117,\n","             3,     9, 29101,    63,    13,  7827,  2895,    12,   243,  5030,\n","          1229,    21,     3, 26685,  1356,     6,   243, 29101,    63,    13,\n","          7827,   379,     3,     9,  2069,  5015,  1218,     6,     3,     9,\n","           166, 21286,  3102,  1218,     6,    11,     3,     9,   511, 21286,\n","          3102,  1218,   117,   243,  2069,  5015,  1218,     3, 11600,     3,\n","             9,   166,  8373,  1968,    28,   243,   166,  3102,  1218,    11,\n","             3,     9,   511,  8373,  1968,    28,   243,   511,  3102,  1218,\n","             6,   243,   166,  8373,  9085,  1438,    11,  1912,   179,  4145,\n","            13,   243,   166,  3102,  1218,     6,   243,   511,  8373,  9085,\n","          1438,    11,  1912,   179,  4145,    13,   243,   511,  3102,  1218,\n","             6,   243,   166,  3102,  1218,  5936,    53,   243,   166,  8373,\n","            12,  2082,  1438,  1316,    21,  5505,  4145,   831,    57,   243,\n","           166,  3102,  1218,     6,   243,   511,  3102,  1218,  5936,    53,\n","           243,   511,  8373,    12,  2082,  1438,  1316,    21,  5505,  4145,\n","           831,    57,   243,   511,  3102,  1218,     6,   243,   166,  3102,\n","          1218,     3, 22469,    28,   243,   511,  3102,  1218,    21,     3,\n","         13134,  1438,  1316,    21,     3, 26685,  4145,   831,    57,   243,\n","           166,  3102,  1218,     6,   243,   511,  3102,  1218,     3, 22469,\n","            28,   243,   166,  3102,  1218,    21,     3, 13134,  1438,  1316,\n","            21,     3, 26685,  4145,   831,    57,   243,   511,  3102,  1218,\n","             6,   243,  2069,  5015,  1218,  4281,  2253,  7233,  3028,    16,\n","          1353,    13, 11157,  1968,    28,  2491,   772,    11,  1358,  3028,\n","            16,  1353,    13,  5962,    13,  1857,  1438,     5,  1682,    71,\n","           358,    38,   356,  7444,    16,  1988,   209,   213,    77,   243,\n","          2069,  5015,  1218,  3806,     7,   243,   166,    11,   511,  8373,\n","             7,    45,   331,  9085,  2250,    13,  4145,     3, 28400,    12,\n","          2491,  1308,     5,  1877,    71,   358,    38,   356,  7444,    16,\n","          1988,   209,   213,    77,   243,  2069,  5015,  1218,  3806,     7,\n","           243,   166,    11,   511,  8373,     7,    45,   331,  9085,   381,\n","            11,   686,    13,  1438,     5,  2853,    71,   358,    38,   356,\n","          7444,    16,  1988,   209,   213,    77,   243,  2069,  5015,  1218,\n","          3806,     7,   243,   166,    11,   511,  8373,     7,    45,   331,\n","          9085,   381,    11,   686,    13,  4145,     5,  3594,    71,   358,\n","            38,   356,  7444,    16,  1988,   314,   213,    77,   243,   331,\n","           963,  2250,    13,  4145,     3, 28400,    12,  2491,  1308,    11,\n","           381,    11,   686,    13,  1438,     5,  4357,    71,   358,    38,\n","           356,  7444,    16,  1988,   209,   213,    77,   243,  2069,  5015,\n","          1218,   911,     7,   331,  9085,  2253, 17765,     6,   243,   331,\n","           379, 13080,    16,    15, 11433,  2197,    16,  1353,    13,  3487,\n","            11,  2491,   772,     5,  4306,    71,   358,    38,   356,  7444,\n","            16,  1988,   431,   213,    77,   243,  2069,  5015,  1218,  3806,\n","             7,     3,     9, 14640,    63,   610,  1809,    21,     3,    35,\n","          1161,    75,    53, 27354,    17,  6002, 11573,     3,   390,    30,\n","           331,  9085,  2253, 17765,    11,   243,   166,    11,   511,  8373,\n","             7,     5,  4848,    41,   254,   663,  1361,    61,  5835,    41,\n","           254,   663,  1361,    61,  5477,    71,   358,    38,   356,  7444,\n","            16,  1988,   489,   213,    77,   243,   331,   856,  5475,  2250,\n","            13,  4145,     3, 28400,    12,  2491,  1308,     6,   381,    11,\n","           686,     1]),\n"," 'target_ids_y': tensor([ 1300,    71,  8308,   331,  3026,   358,    21, 14498, 17127,    13,\n","          1438,    11, 14498,  2491,  9328,     6,   243,   358,  8315,    53,\n","            10,     3,     9,  5030,  1229,    21,  9153,  1222,  4175,   117,\n","             3,     9, 29101,    63,    13,  7827,  2895,    12,   243,  5030,\n","          1229,    21,     3, 26685,  1356,     6,   243, 29101,    63,    13,\n","          7827,   379,     3,     9,  2069,  5015,  1218,     6,     3,     9,\n","           166, 21286,  3102,  1218,     6,    11,     3,     9,   511, 21286,\n","          3102,  1218,   117,   243,  2069,  5015,  1218,     3, 11600,     3,\n","             9,   166,  8373,  1968,    28,   243,   166,  3102,  1218,    11,\n","             3,     9,   511,  8373,  1968,    28,   243,   511,  3102,  1218,\n","             6,   243,   166,  8373,  9085,  1438,    11,  1912,   179,  4145,\n","            13,   243,   166,  3102,  1218,     6,   243,   511,  8373,  9085,\n","          1438,    11,  1912,   179,  4145,    13,   243,   511,  3102,  1218,\n","             6,   243,   166,  3102,  1218,  5936,    53,   243,   166,  8373,\n","            12,  2082,  1438,  1316,    21,  5505,  4145,   831,    57,   243,\n","           166,  3102,  1218,     6,   243,   511,  3102,  1218,  5936,    53,\n","           243,   511,  8373,    12,  2082,  1438,  1316,    21,  5505,  4145,\n","           831,    57,   243,   511,  3102,  1218,     6,   243,   166,  3102,\n","          1218,     3, 22469,    28,   243,   511,  3102,  1218,    21,     3,\n","         13134,  1438,  1316,    21,     3, 26685,  4145,   831,    57,   243,\n","           166,  3102,  1218,     6,   243,   511,  3102,  1218,     3, 22469,\n","            28,   243,   166,  3102,  1218,    21,     3, 13134,  1438,  1316,\n","            21,     3, 26685,  4145,   831,    57,   243,   511,  3102,  1218,\n","             6,   243,  2069,  5015,  1218,  4281,  2253,  7233,  3028,    16,\n","          1353,    13, 11157,  1968,    28,  2491,   772,    11,  1358,  3028,\n","            16,  1353,    13,  5962,    13,  1857,  1438,     5,  1682,    71,\n","           358,    38,   356,  7444,    16,  1988,   209,   213,    77,   243,\n","          2069,  5015,  1218,  3806,     7,   243,   166,    11,   511,  8373,\n","             7,    45,   331,  9085,  2250,    13,  4145,     3, 28400,    12,\n","          2491,  1308,     5,  1877,    71,   358,    38,   356,  7444,    16,\n","          1988,   209,   213,    77,   243,  2069,  5015,  1218,  3806,     7,\n","           243,   166,    11,   511,  8373,     7,    45,   331,  9085,   381,\n","            11,   686,    13,  1438,     5,  2853,    71,   358,    38,   356,\n","          7444,    16,  1988,   209,   213,    77,   243,  2069,  5015,  1218,\n","          3806,     7,   243,   166,    11,   511,  8373,     7,    45,   331,\n","          9085,   381,    11,   686,    13,  4145,     5,  3594,    71,   358,\n","            38,   356,  7444,    16,  1988,   314,   213,    77,   243,   331,\n","           963,  2250,    13,  4145,     3, 28400,    12,  2491,  1308,    11,\n","           381,    11,   686,    13,  1438,     5,  4357,    71,   358,    38,\n","           356,  7444,    16,  1988,   209,   213,    77,   243,  2069,  5015,\n","          1218,   911,     7,   331,  9085,  2253, 17765,     6,   243,   331,\n","           379, 13080,    16,    15, 11433,  2197,    16,  1353,    13,  3487,\n","            11,  2491,   772,     5,  4306,    71,   358,    38,   356,  7444,\n","            16,  1988,   431,   213,    77,   243,  2069,  5015,  1218,  3806,\n","             7,     3,     9, 14640,    63,   610,  1809,    21,     3,    35,\n","          1161,    75,    53, 27354,    17,  6002, 11573,     3,   390,    30,\n","           331,  9085,  2253, 17765,    11,   243,   166,    11,   511,  8373,\n","             7,     5,  4848,    41,   254,   663,  1361,    61,  5835,    41,\n","           254,   663,  1361,    61,  5477,    71,   358,    38,   356,  7444,\n","            16,  1988,   489,   213,    77,   243,   331,   856,  5475,  2250,\n","            13,  4145,     3, 28400,    12,  2491,  1308,     6,   381,    11,\n","           686,     1])}"]},"metadata":{},"execution_count":37}]},{"cell_type":"code","source":["def train(epoch, tokenizer, model, device, loader, optimizer):\n","\n","    \"\"\"\n","    Function to be called for training with the parameters passed from main function\n","\n","    \"\"\"\n","\n","    model.train()\n","    for _, data in enumerate(loader, 0):\n","        y = data[\"target_ids\"].to(device, dtype=torch.long)\n","        y_ids = y[:, :-1].contiguous()\n","        lm_labels = y[:, 1:].clone().detach()\n","        lm_labels[y[:, 1:] == tokenizer.pad_token_id] = -100\n","        ids = data[\"source_ids\"].to(device, dtype=torch.long)\n","        mask = data[\"source_mask\"].to(device, dtype=torch.long)\n","\n","        outputs = model(\n","            input_ids=ids,\n","            attention_mask=mask,\n","            decoder_input_ids=y_ids,\n","            labels=lm_labels,\n","        )\n","        loss = outputs[0]\n","\n","        if _ % 10 == 0:\n","            training_logger.add_row(str(epoch), str(_), str(loss))\n","            console.print(training_logger)\n","\n","        optimizer.zero_grad()\n","        loss.backward()\n","        optimizer.step()\n"],"metadata":{"id":"i05y6wxzBB8M"},"execution_count":null,"outputs":[]},{"cell_type":"code","source":["import torch\n","from torch.utils.data import DataLoader\n","#from torch.optim import Adafactor\n","\n","train_dataloader = DataLoader(train_data, batch_size=1, shuffle=True)\n","criterion = torch.nn.CrossEntropyLoss()\n","'''optimizer = Adafactor(model.parameters(),lr=1e-3,\n","                      eps=(1e-30, 1e-3),\n","                      clip_threshold=1.0,\n","                      decay_rate=-0.8,\n","                      beta1=None,\n","                      weight_decay=0.0,\n","                      relative_step=False,\n","                      scale_parameter=False,\n","                      warmup_init=False)'''\n","optimizer = torch.optim.Adam(model.parameters(), lr=0.001)\n"],"metadata":{"id":"WNA4KQRv9QZ7"},"execution_count":null,"outputs":[]},{"cell_type":"code","source":["epoch = 1\n","train(epoch, tokenizer, model, dev, train_dataloader, optimizer)"],"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":275},"id":"Gj6YVamR86yH","executionInfo":{"status":"error","timestamp":1711410730854,"user_tz":420,"elapsed":1440,"user":{"displayName":"Rishikesh Sivakumar","userId":"10957495338496577769"}},"outputId":"78be32d0-938e-4672-d752-8c9f0cad11d7"},"execution_count":null,"outputs":[{"output_type":"error","ename":"NameError","evalue":"name 'training_logger' is not defined","traceback":["\u001b[0;31m---------------------------------------------------------------------------\u001b[0m","\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)","\u001b[0;32m<ipython-input-40-4631bea69767>\u001b[0m in \u001b[0;36m<cell line: 2>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0mepoch\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;36m1\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0mtrain\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mepoch\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtokenizer\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmodel\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdev\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtrain_dataloader\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0moptimizer\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m","\u001b[0;32m<ipython-input-38-ebc22a9b197a>\u001b[0m in \u001b[0;36mtrain\u001b[0;34m(epoch, tokenizer, model, device, loader, optimizer)\u001b[0m\n\u001b[1;32m     24\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     25\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0m_\u001b[0m \u001b[0;34m%\u001b[0m \u001b[0;36m10\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 26\u001b[0;31m             \u001b[0mtraining_logger\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0madd_row\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mepoch\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0m_\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mloss\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     27\u001b[0m             \u001b[0mconsole\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mtraining_logger\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     28\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;31mNameError\u001b[0m: name 'training_logger' is not defined"]}]},{"cell_type":"code","source":[],"metadata":{"id":"QN2QNEux9OQn"},"execution_count":null,"outputs":[]}]}